## Proteomics Clinical Applications

<!-- image -->

www.clinical proteomics-journal.com

WILEY

1-2/2017

<!-- image -->

<!-- image -->

## High performance mass spectrometry based proteomics reveals enzyme and signaling pathway regulation in neutrophils during the early stage of surgical trauma

Samina Arshid 1,3 ‚àó , Muhammad Tahir 1,2 ‚àó , Belchor Fontes 3 , Edna Frasson de Souza Montero 3 , Mariana S. Castro 1 , Simone Sidoli 2 ‚àó‚àó , Peter Roepstorff 2 and Wagner Fontes 1

- 1 Laboratory of Biochemistry and Protein Chemistry, Department of Cell Biology, Institute of Biology, University of Brasilia, Bras' ƒ±lia, Brazil
- 2 Department of Biochemistry and Molecular Biology, University of Southern Denmark, Odense, Denmark
- 3 Laboratory of Surgical Physiopathology (LIM-62), Faculty of Medicine, University of SÀú ao Paulo, Brazil

Purpose: In clinical conditions trauma is associated with high mortality and morbidity. Neutrophils play a key role in the development of multiple organ failure after trauma

Experimental design: To have a detailed understanding of the neutrophil activation at primary stages after trauma, neutrophils are isolated from control and surgical trauma rats in this study. Extracted proteins are analyzed using nano liquid chromatography coupled with tandem mass spectrometry.

Results: A total of 2924 rat neutrophil proteins are identified in our analysis, of which 393 are found differentially regulated between control and trauma groups. By using functional pathways analysis of the 190 proteins up-regulated in surgical trauma, we found proteins related to transcription initiation and protein biosynthesis. On the other hand, among the 203 proteins down-regulated in surgical trauma we found enrichment for proteins of the immune response, proteasome degradation and actin cytoskeleton. Overall, enzyme prediction analysis revealed that regulated enzymes are directly involved in neutrophil apoptosis, directional migration and chemotaxis. Our observations are then confirmed by in silico protein-protein interaction analysis.

Conclusions and clinical relevance: Collectively, our results reveal that neutrophils drastically regulate their biochemical pathways after the early stages of surgical trauma, showing lower activity. This implies higher susceptibility of the trauma patients to infection and bystander tissues damage.

## Keywords:

Inflammation / Neutrophil / Proteome / Surgery / Trauma

<!-- image -->

/pc

Additional supporting information may be found in the online version of this article at the publisher's web-site

## 1 Introduction

Surgical trauma (ST) is a stressful experience that alters the physical and immune function in humans and animals [1,2]. Surgical stress has been found associated with overactivation of the innate immune response with subsequent immunosuppression in the early post-operative period that has been linked with an increase in neutrophil count and decrease in

Correspondence : Prof. Wagner Fontes, Laboratory of Biochemistry and Protein Chemistry, Department of Cell Biology, Institute of Biology, University of Brasilia, Bras' ƒ±lia-DF , 70910-900, Brazil E-mail : wagnerf@unb.br

percentage of neutrophils involved in phagocytic activity after surgical stress [3]. Short surgical procedures are very common medical practices [4] and the purpose of this study is to reveal and evaluate the initial changes in neutrophil activation at the proteomic level resulting from surgery, providing solid basis and better understanding for the underlying mechanisms.

Received: January 7, 2016 Revised: June 28, 2016 Accepted: September 23, 2016

## Clinical Relevance

Short surgical procedures are very common in everyday clinical practices that start the initial traumatic events resulting in the activation of polymorphonuclear neutrophils (PMNs), a primary line of defense against invading foreign particles, causing damage to the bystander tissues and, in severe cases, multiple organ dysfunctioning (MOD). To understand the underlying molecular events of the PMN activation in response to surgical trauma, it is of outmost importance to study the proteome of neutrophils at early stages of trauma. Our findings are the first comparative proteome analysis of PMNs from surgically traumatized rats and healthy controls by using nLC-

MS/MS proteomic approach. Interestingly our analysis revealed the enzymes that are involved in neutrophil apoptosis, ROS production, directional migration, and a down regulation in proteins related to immune response, proteasome degradation and actin cytoskeleton. Our findings open a new window to better understand the PMNs activation in the early stages of surgical trauma and provide a detailed database for further profound studies. Analysis of the functional roles of the reported enzymes and the proteins encountered in the pathway analyses can be helpful and considered as future therapeutic targets in neutrophil biology at primary stages of trauma.

Polymorphonuclear neutrophils (PMN), the most abundant white blood cells, are involved in the first line of defense. Their number and life span increases in tissue and in circulation during inflammation due to the presence of cytokines or other inflammatory agents. These agents lead to a delay in apoptotic pathways in neutrophils that after entering into tissues release cytotoxic molecules such as reactive oxygen species (ROS) and proteases [5-7]. Systemic inflammatory response syndrome (SIRS) and multiple organ failure (MOF) have been observed after trauma associated with excessive systemic PMN activation [8]. Neutrophil infiltration and accumulation in the tissues has been found in trauma and septic patients due to delayed apoptosis that can produce more cytotoxic products and promote tissue damage [9]. Increase in PMN priming has been noticed after RBC transfusions in injured patients [10, 11] due to generation and storage of platelet-activating factor (PAF) and PAFlike phospholipids, including lyso-phosphatidyl (lyso-PCs) in RBCsthat also delay PMN apoptosis [12]. Studies have shown a decrease in the chemotactic response of neutrophils in trauma patients, due to changes in potent chemoattractants [13, 14].

It is clear that blocking or depleting neutrophils in experimental models results in a reduction in post-operative complications [15]. Therefore, in order to control their response it is fundamental to have deeper understanding of their biochemical mechanisms, such as which enzymes and signaling pathways are activated. In this study we performed liquid chromatography coupled to tandem mass spectrometry (LC-MS/MS) based proteomics using iTRAQ-based relative quantification in the neutrophils of rats subjected to laparotomy. The proteins found to be differentially regulated when compared with control rats were found to be mostly involved in oxidoreductase, transferase and hydrolase activities and their functional association analysis highlighted their role in protein biogenesis, metabolism, immune response and motility.

## 2 Materials and methods

## 2.1 Materials and reagents

DTT and SDS were from GE Healthcare Bio-Sciences (Uppsala, Sweden). Protease Inhibitor Mix and Quant-iT TM Protein Assay Kit were from GE Healthcare Biosciences (Pittsburgh, USA). Iodoacetamide was from GE Healthcare (Amersham, UK). TEAB was from Sigma Aldrich Switzerland. PhosSTOP was from Roche - Germany. Trypsin was from Promega (Madison, WI). Vivacon 30KDa filters were from Sartorius Stedim Biotech (Goettingen, Germany). iTRAQ reagents were purchased from Applied Biosystems (Foster City, CA). TSKGel Amide-80 2 mm, 3 ùõç m particle size was from Tosoh Bioscience (Stuttgart, Germany). Poros Oligo R2 and R3 reversed-phase material were obtained from PerSeptive Biosystems (Framingham, MA). TFA was from Fluka (St. Louis, MO). Empore C8 extraction disk was obtained from 3 M Bioanalytical Technologies (St. Paul, MN). ReproSil-Pur C18 AQ 3 ùõç m material was from Dr. Maisch, (Ammerbuch-Entringen, Germany). All other solvents and chemicals used in this work were HPLC grade or higher and low-binding polypropylene microtubes from Sorenson Bioscience (Salt Lake City, UT) were used for sample preparation procedures.

## 2.2 Experimental subjects and surgical procedure

Male Wistar rats weighing 250-350 g with no inflammatory diseases were collected from the animal house of Faculty of Medicine, University of SÀúo Paulo (FMUSP), SÀú ao Paulo a state, Brazil. To use rats as experimental subjects the project was first approved by the ethical committee of FMUSP with protocol no. 8186. Twenty rats were divided randomly into two groups, the control group and surgical group and each

group had 10 rats. Absence of inflammatory processes was confirmed by hemocytometric analysis before the rats were included in the experiment. Rats from both the groups were anesthetized according to Tahir et al. [16] and rats from the surgical group were subjected to the abdominal surgery. After midline abdominal surgical incision, the abdominal cavity was kept open for 45 min and the intestinal tract was placed between gauze pads soaked with warm 0.9% NaCl solution to simulate intra-abdominal manipulation. Then the intestines were placed back into the cavity, the abdominal wall was sutured and the animals observed for 120 min. The control group was anesthetized and kept under observation for the same period.

## 2.3 Sample collection and neutrophil isolation

After 165 min of surgical procedures (or observation), about 10-12 mL of blood was collected from the heart (right ventricle catheter). Rat neutrophils were isolated by using Ficoll gradient protocol according to Russo-Carbolante et al. [17] with minor modifications. Briefly, about 10 mL of blood were layered carefully upon equal amount of Ficoll in a Falcon tube and centrifuged at 400 rcf for 45 min. After centrifugation, the upper layer was discarded and the bottom layer (rich in erythrocytes and neutrophils) was mixed with 6% dextran solution in 0.15 M of NaCl. The final volume was increased to 14 mL by adding phosphate buffer saline (PBS) solution. The samples were homogenized and mixed well and incubated at 37 C for 20 min. The transparent supernatant layer was col-¬∞ lected in a fresh falcon tube and centrifuged for 10 min at 270 rcf. The pellet was washed by adding 5 mL of milli-Q water for 15-20 s to lyse residual erythrocytes and then 5 mL of 2x PBS solution, mixed and centrifuged for 10 min at 500 rcf. After centrifugation, the neutrophil pellet was resuspended in 1 mL of 1x PBS solution to count under the microscope.

## 2.4 Neutrophil lysis and protein digestion

After counting, 3 √ó 10 6 neutrophils were lysed in 200 ùõç L of lysis buffer (2% SDS, 20 mM TEAB, 100 mM DTT, Phosphatase and Protease Inhibitors Mix) by using a tip-sonicator for 10 cycles with 15 s of each cycle and 1 min of interval on ice. After sonication the samples were heated at 80 C for 10 min ¬∞ and then centrifuged and quantified for protein concentration by using Quant-iT TM Protein Assay Kit (Cat. No. Q33210). SDSwasremoved from the samples according to Wisniewski et al. [18] by using 30 KDa Vivacon spin filters. The DTT reduced samples were alkylated with 400 ùõç L of iodoacetamide (IAA) buffer (50 mM IAA in 1% SDC and 20 mM TEAB solution) in the dark for 20 min at room temperature. Samples were washed with 400 ùõç L of 1% SDC and 20 mM TEAB solution. Proteins were digested on the filters at 37 C ¬∞ overnight by using Promega Trypsin in 1:50 trypsin to sample ratio in 400 ùõç L of 1% SDC and 20 mM TEAB solution.

After digestion, the samples were acidified to a final concentration of 0.1% by adding TFA and filter units were shifted to new collection tubes, centrifuged at 14000 rcf for 20 min and filtrates having peptides were saved. Peptides were desalted by homemade microcolumns of Poros Oligo R2/R3 packed resins (1 cm long) in p200 tips [19]. Peptide quantification was carried out by using Biochrome 30 amino acid composition analyzer (Cambridge, UK) as described in Laursen et al. [20].

## 2.5 iTRAQ labeling and HILIC-fractionation

Peptides from two rats of the same group were pooled together to make one biological replicate resulting in five biological replicates for each of the two conditions. Biological replicates from each condition were iTRAQ labeled according to the manufacturer's instructions. Briefly, 100 ùõç g of lyophilized, desalted digest from each replicate were resuspended individually in 30 ùõç L of 300 mM TEAB and added to a vial with iTRAQ label resuspended in 70 ùõç L of ethanol. The vials were mixed and incubated at room temperature for 2 h. The labeled peptides were combined in 1:1 proportion based on the amino acid analysis quantification. Equal proportion combination and iTRAQ labeling was confirmed by MALDI-MS/MS analysis (Bruker Daltonics, Billerica, CA, USA). All replicates from the control group were labeled with iTRAQ-114 and surgical trauma replicates were labeled with iTRAQ-115. Prior to LC-MS/MS analysis, 20 ùõç g of multiplexed iTRAQ-labeled peptides of each replicate were separated into seven fractions by using hydrophilic interaction chromatography (HILIC) on an Agilent 1200 HPLC system [21]. For that, lyophilized samples were resuspended in solvent B (90% acetonitrile, 0.1% TFA) and loaded into a TSK-gel Amide 80 HILIC HPLC column (length = 17 cm, inner diameter = 250 ùõç m,particle size = 3 ùõç m). Peptides were eluted at 6 ùõç L/min by a decreasing gradient of 90% ACN/0.1% TFA to 60% ACN/0.1% TFA over 35 min. Fractions were collected automatically in a microwell plate at 1 min of intervals after UV detection at 210 nm. Collected fractions were combined in seven fractions, based on the uniform distribution of the UV absorbance pattern, and lyophilized.

## 2.6 LC-MS/MS and data analysis

Dried fractions were resuspended in buffer A (0.1% formic acid) and loaded onto a Proxeon Easy-nLC system (Thermo Fisher Scientific, Odense, Denmark), coupled online with LTQ-Orbitrap Velos mass spectrometer (Thermo Fisher Scientific, Bremen, Germany). Peptides were loaded into a picofrit 18 cm long fused silica capillary column (75 ùõç minner diameter) packed in-house with reversed-phase Repro-Sil Pur C18-AQ3 ùõç mresin. A gradient of 180 min was set for peptide elution from 0-34% buffer-B (95% ACN/0.1% formic acid) at a flow rate of 250 nL/min. The MS method was set up in

a data-dependent acquisition (DDA) mode. For full MS scan, the mass range of 400-1200 m/z was analyzed in the Orbitrap Velos at 30 000 FWHM (400 m/z ) resolution and 1 √ó 10 6 AGC target value. Each MS scan in the Orbitrap analyzer was followed by MS/MS of the seven most intense ions ( /2265 2 + charge state). Fragment ions were acquired in the Orbitrap using a resolution of 7500 after high-energy collision dissociation (HCD) with energy set at 36 [22]. Raw data were viewed in Xcalibur v.2.1 (Thermo Scientific).

For the resulting raw files Proteome Discoverer software (v.1.4.0.288, Thermo Fisher Scientific) was used to perform database searching and relative quantification. MS/MS spectra were searched against UniProt Rodents database (updated November 2013) by using an in-house MASCOT (v2.3, Matrix Science, London, UK) server. The searches were performed with the following parameters: oxidation (M) and deamidation (NQ) were set as variable modifications, while carbamidomethylation (C) was set as fixed modification. Precursor mass tolerance was 10 ppm; MS/MS tolerance was 0.05 Da. Trypsin was selected as digestion enzyme and up to two missed cleavages were allowed. Using Percolator as validator [23], results were filtered for 1% false discovery rate (FDR) and minimal of two peptides were accepted for protein identification using Proteome Discoverer. Protein iTRAQ ratios were log2 transformed and normalized by the average value to compensate for possible imprecisions during equal mixing of the samples. Proteins were considered for quantitative analysis only if present in at least two replicates. One-tail Student's t -test ( p -value &lt; 0.05) was used to assess statistical validation for significantly regulated proteins between conditions. Protein Center (Thermo Scientific) was used to interpret annotation and statistical GO classification of the identified proteins. More detailed annotation and enzyme activity prediction and classification (Enzyme Commission numbers) were retrieved from the online UniProt database service. Functional analysis of the trauma relevant proteins clusters was carried out by Wikipathways using WEB-based GEne SeT AnaLysis Toolkit (WebGestalt) [24], whereas protein-protein interaction networks analysis was acquired from STRING with highest confidence score (0.900) [25].

Data Availability-Mass spectrometer output files (Raw data) have been deposited in a public repository, the Peptide Atlas database (https://db.systemsbiology.net/sbeams/ cgi/PeptideAtlas/PASS\_View?identifier = PASS00790), under the identifier PASS00790.

## 3 Results

## 3.1 Protein identification and relative protein abundance analysis

In this study, we performed a comprehensive comparative proteomic analysis of neutrophils from surgically trauma-

tized rats to evaluate the early effect of surgical trauma on the rat neutrophil proteome. The neutrophils were isolated from both groups control and surgical trauma from blood through Ficoll density gradient method. After isolation, proteins were extracted, quantified and pooled followed by trypsin digestion on Vivacon filters. All experiments were performed in five biological replicates. The resulting peptides were identified and quantified using nLC-MS/MS with iTRAQ labeling. Finally, nLC-MS/MS was preceded by HILIC fractionation to improve the depth of the analysis (Fig. 1).

We identified a total of 2924 rat neutrophil proteins, 393 (13.4% of the total identified proteins) were significantly different in terms of relative abundance ( p &lt; 0.05) in the surgical trauma group as compared with the control. Out of 393 proteins, 190 proteins ( /2248 6.5%) showed significant up-regulation in surgical trauma whereas 203 proteins (6.94%) presented down-regulation as compared to the control group, being 86% of the regulated proteins found in at least four out of five biological replicates and 52% presented a fold change larger than 1.3 (Supporting Information Table 1).

## 3.2 Gene ontology analysis of differentially regulated protein

The gene ontology (GO) classification of the differentially regulated proteins according to their expected subcellular localization revealed that most of the proteins are localized in cytoplasm, membrane, nucleus, cytoskeleton, organelle lumen and cytosol (Supporting Information Fig. 1A-C). The cytoskeleton, membrane, nucleus and vacuole showed higher numberofdown-regulated proteins and cytosol, endoplasmic reticulum, mitochondrion had higher count of up-regulated proteins, whereas ribosomal proteins were only up-regulated. The most regulated biological processes involved were cell communication, cell organization and biogenesis, metabolic processes, response to stimuli and transport. The molecular functions assigned to these regulated proteins were catalytic activity and binding ability for nucleotides, RNA, proteins and metal ions.

## 3.3 Predicted enzyme activity

Enzyme activity prediction for the ST responsive proteins (Fig. 2) showed that oxidoreductases and transferases are predominant among the down-regulated proteins, whereas ligases were found more frequently among the proteins with significantly up-regulation from neutrophils after abdominal surgery. Table 1 shows all the differentially regulated proteins with their predicted enzyme activity.

Figure 1. Experimental workflow for the quantitative proteomic analysis of rat neutrophil subjected to surgical trauma.

<!-- image -->

## 3.3.1 Oxidoreductases

All the three interesting lipoxygenases 12-LO, 15-LO and 5LO, Q02759/ P48999 (EC:1.13.11.31/ 1.13.11.33 /1.13.11.34) were found down-regulated in ST among the oxidoreductases in our analysis.

## 3.3.2 Transferases

Among the significantly up-regulated transferases the most repeatedly annotated transferases were Dolichylphosphate ùõÉ -D-mannosyltransferase (EC: 2.4.1.83/Q9WU83) and Dolichyl-diphosphooligosaccharide protein glycotransferase (EC 2.4.99.18/ O54734, P46978, P61806, Q3TDQ1). Ten transferases significantly down-regulated in ST rats are annotated with EC:2.7.11.1 and classified as non-specific serine/threonine protein kinases (including STE20-like and PAK 2, among others). One of the serine/threonine protein kinases with down-regulation is PAK2. Protein kinase C delta (P09215/ 2.7.11.13) was identified in our analysis with downregulation.

Among the hydrolases , 6 of them are annotated as DNA helicases (EC 3.6.4.12) and 3 as RNA helicases (EC 3.6.4.13) (Table 2). An important up-regulated enzyme after ST in this category is (P43527/EC 3.4.22.36) Caspase-1, which is a multifactorial enzyme. Our results from the significantly up-regulated hydrolases included myeloblastin/proteinase 3 (Q61096/ EC 3.4.21.76), cathepsin H (P00786/EC 3.4.22.16) and cathepsin D (P24268/ EC 3.4.23.5) while cathepsin C (Q64560/ 3.4.14.1) was present among down-regulated hydrolases after ST. Three other interesting hydrolases found with down-regulation are (P24135/3.1.4.11) phosphoinositide phospholipase C (PLC), inositol-phosphate phosphatase (P97697/ 3.1.3.25) and diphosphoinositolpolyphosphate diphosphatase (Q8R2U6/ 3.6.1.52). Caspase 3 (P55213/3.4.22.56) was also found significantly downregulated.

## 3.3.3 Isomerases

Four isomerases were found with up-regulation in neutrophils after surgical trauma, interestingly three of these are annotated as EC: 5.2.1.8 (P17742/ P29117/Q62446) prolyl

Figure 2. Predicted enzyme activities of proteins with differential regulation level in ST as compare to control.

<!-- image -->

Table 1. Predicted enzyme activities of the quantified proteins

| Enzyme description                                                                      | EC number     | Accession   | GENE symbol      | Sig. abundance   |
|-----------------------------------------------------------------------------------------|---------------|-------------|------------------|------------------|
| Oxidoreductases                                                                         |               |             |                  |                  |
| malate dehydrogenase, mitochondrial precursor                                           | EC 1.1.1.37   | P04636      | Mdh2             | High             |
| Thioredoxin-dependent peroxide reductase, mitochondrial                                 | EC 1.11.1.15  | Q9Z0V6      | Prdx3            | High             |
| fatty acyl-CoA reductase 1 isoform 1                                                    | EC 1.2.1.n2   | Q922J9      | Far1             | High             |
| biliverdin reductase A precursor                                                        | EC 1.3.1.24   | P46844      | Blvra            | High             |
| coproporphyrinogen-III oxidase, mitochondrial precursor                                 | EC 1.3.3.3    | P36552      | Cpox             | High             |
| C-1-tetrahydrofolate synthase, cytoplasmic                                              | EC 1.5.1.5    | P27653      | Mthfd1           | High             |
| 3-hydroxyacyl-CoA dehydrogenase type-2                                                  | EC 1.1.1.178  | O70351      | Hsd17b10         | Low              |
| 3-hydroxyacyl-CoA dehydrogenase                                                         | EC 1.1.1.51   | O70351      | Hsd17b10         | Low              |
| Arachidonate 15-lipoxygenase                                                            | EC 1.13.11.31 | Q02759      | Alox15           | Low              |
| L-lactate dehydrogenase C chain                                                         | EC 1.1.1.27   | P19629      | Ldhc             | Low              |
| 3-hydroxyacyl-CoA dehydrogenase                                                         | EC 1.1.1.35   | O70351      | Hsd17b10         | Low              |
| type-2                                                                                  | EC 1.1.1.44   | P85968      | LOC100360180,Pgd | Low              |
| 6-phosphogluconate dehydrogenase, decarboxylating eosinophil peroxidase precursor       | EC 1.11.1.7   | P49290      | Epx              | Low              |
| Arachidonate 15-lipoxygenase                                                            | EC 1.13.11.33 | Q02759      | Alox15           | Low              |
| arachidonate 5-lipoxygenase                                                             | EC 1.13.11.34 | P48999      | Alox5            | Low              |
| protein-methionine sulfoxide oxidase MICAL1                                             | EC 1.14.13.-  | D3ZBP4      | Mical1           | Low              |
| leukotriene-B(4) omega-hydroxylase 2                                                    | EC 1.14.13.3  | Q3MID2      | Cyp4f18          | Low              |
| metalloreductase STEAP4                                                                 | EC 1.16.1.-   | Q4V8K1      | Steap4           | Low              |
| peroxisomal acyl-coenzyme A oxidase 1                                                   | EC 1.3.3.6    | P07872      | Acox1            | Low              |
| Transferases                                                                            |               |             |                  |                  |
| lysophospholipid acyltransferase 5                                                      | EC 2.3.1.23   | Q5FVN0      | Lpcat3           | High             |
| nucleoside diphosphate kinase B                                                         | EC 2.7.13.3   | P19804      | Nme2             | High             |
| lysophospholipid acyltransferase 5                                                      | EC 2.3.1.-    | Q5FVN0      | Lpcat3           | High             |
| citrate synthase, mitochondrial precursor                                               | EC 2.3.3.1    | Q8VHF5      | Cs               | High             |
| dolichol-phosphate mannosyltransferase subunit 1                                        | EC 2.4.1.83   | Q9WU83      | Dpm1             | High             |
| dolichyl-diphosphooligosaccharide- protein glycosyltransferase 48 kDa subunit precursor | EC 2.4.99.18  | O54734      | Ddost            | High             |
| dolichyl-diphosphooligosaccharide- protein glycosyltransferase subunit STT3A            | EC 2.4.99.18  | P46978      | Stt3a            | High             |
| dolichyl-diphosphooligosaccharide- protein glycosyltransferase subunit                  | EC 2.4.99.18  | P61806      | Dad1             | High             |
| DAD1 Dolichyl-diphosphooligosaccharide- protein glycosyltransferase subunit             | EC 2.4.99.18  | Q3TDQ1      | Stt3b            | High             |
| STT3B glycosaminoglycan xylosylkinase                                                   | EC 2.7.1.-    |             |                  |                  |
|                                                                                         |               | Q8VCS3      | Fam20b           | High             |
| galactokinase                                                                           | EC 2.7.1.6    | Q9R0N0      | Galk1            | High             |
| nucleoside diphosphate kinase B                                                         | EC 2.7.4.6    | P19804      | Nme2             | High             |
| succinyl-CoA:3-ketoacid coenzyme A                                                      | EC 2.8.3.5    | B2GV06      | Oxct1            | High             |
| transferase 1, mitochondrial precursor                                                  | EC 2.3.1.51   | Q8BYI6      | Lpcat2           | Low              |
| lysophosphatidylcholine acyltransferase 2                                               |               | Q8BYI6      |                  | Low              |
| lysophosphatidylcholine acyltransferase 2                                               | EC 2.3.1.67   |             | Lpcat2           |                  |

Table 1. Continued

| Enzyme description                                               | EC number    | Accession     | GENE symbol       | Sig. abundance   |
|------------------------------------------------------------------|--------------|---------------|-------------------|------------------|
| interferon-induced, double-stranded RNA-activated protein kinase | EC 2.7.1.2   | Q63184        | Eif2ak2           | Low              |
| 5'-AMP-activated protein kinase catalytic subunit alpha-1        | EC 2.7.11.26 | P54645        | Prkaa1            | Low              |
| 5'-AMP-activated protein kinase catalytic subunit alpha-1        | EC 2.7.11.27 | P54645        | Prkaa1            | Low              |
| 5'-AMP-activated protein kinase catalytic subunit alpha-1        | EC 2.7.11.31 | P54645        | Prkaa1            | Low              |
| UMP-CMP kinase                                                   | EC 2.7.4.6   | Q4KM73        | Cmpk1             | Low              |
| Transketolase                                                    | EC 2.2.1.1   | P50137        | Tkt               | Low              |
| transaldolase                                                    | EC 2.2.1.2   | Q9EQS0        | Taldo1            | Low              |
| lysophosphatidylcholine acyltransferase 2                        | EC 2.3.1.23  | Q8BYI6        | Lpcat2            | Low              |
| ATP-citrate synthase isoform 2                                   | EC 2.3.3.8   | Q91V92        | Acly              | Low              |
| nicotinamide phosphoribosyltransferase                           | EC 2.4.2.12  | Q80Z29        | Nampt             | Low              |
| glutathione S-transferase P                                      | EC 2.5.1.18  | P04906        | Gstp1             | Low              |
| phosphatidylinositol 5-phosphate 4-kinase type-2 alpha           | EC 2.7.1.149 | Q9R0I8        | Pip4k2a           | Low              |
| diacylglycerol kinase zeta                                       | EC 2.7.1.17  | O08560        | Dgkz              | Low              |
| tyrosine-protein kinase CSK                                      | EC 2.7.1.2   | P32577        | Csk               | Low              |
| tyrosine-protein kinase BTK                                      | EC 2.7.1.2   | P35991        | Btk               | Low              |
| protein-tyrosine kinase 2-beta                                   | EC 2.7.1.2   | P70600        | Ptk2b             | Low              |
| NAD kinase 2, mitochondrial                                      | EC 2.7.1.23  | Q1HCL7        | Nadk2,Nadkd1      | Low              |
| STE20-like serine/threonine-protein kinase                       | EC 2.7.11.1  | O08815        | Slk               | Low              |
| Phosphoinositide 3-kinase regulatory subunit 4                   | EC 2.7.11.1  | P0C0R5        | Pik3r4            | Low              |
| 5'-AMP-activated protein kinase catalytic subunit alpha-1        | EC 2.7.11.1  | P54645        | Prkaa1            | Low              |
| serine/threonine-protein kinase TAO3                             | EC 2.7.11.1  | Q53UA7        | Taok3             | Low              |
| interferon-induced, double-stranded RNA-activated protein kinase | EC 2.7.11.1  | Q63184        | Eif2ak2           | Low              |
| Serine/threonine-protein kinase PAK 2                            | EC 2.7.11.1  | Q64303        | Pak2,LOC100910732 | Low              |
| serine/threonine-protein kinase 17B                              | EC 2.7.11.1  | Q91XS8        | Stk17b            | Low              |
| serine/threonine-protein kinase MST4                             | EC 2.7.11.1  | Q99JT2        | 2610018G03Rik     | Low              |
| serine/threonine-protein kinase 4                                | EC 2.7.11.1  | Q9JI11        | Stk4              | Low              |
| serine/threonine-protein kinase WNK1                             | EC 2.7.11.1  |               | Wnk1              | Low              |
| isoform 3 cAMP-dependent protein kinase                          | EC 2.7.11.11 | Q9JIH7 P68180 |                   | Low              |
| protein kinase C delta type                                      | EC 2.7.11.13 | P09215        | Prkcd             | Low              |
| calcium/calmodulin-dependent protein kinase type 1               | EC 2.7.11.17 | Q63450        | Camk1             | Low              |
| UMP-CMP kinase                                                   | EC 2.7.4.14  | Q4KM73        | Cmpk1             | Low              |
| Hydrolases                                                       |              |               |                   |                  |
| C-1-tetrahydrofolate synthase, cytoplasmic                       | EC 3.5.4.9   | P27653        | Mthfd1            | High             |
| ras GTPase-activating protein-binding protein 1                  | EC 3.6.4.13  | P97855        | G3bp1             | High             |
| isoamyl acetate-hydrolyzing esterase 1 homolog                   | EC 3.1.-.-   | Q711G3        | Iah1              | High             |
| peptidyl-tRNA hydrolase 2, mitochondrial isoform b               | EC 3.1.1.29  | Q8R2Y8        | Ptrh2             | High             |
| palmitoyl-protein thioesterase 1                                 | EC 3.1.2.22  | P45479        | Ppt1              | High             |
| precursor myotubularin-related protein 6                         | EC 3.1.3.-   | Q8VE11        | Mtmr6             | High             |
| 1-phosphatidylinositol                                           | EC 3.1.4.11  | Q99JE6        | Plcb3             | High             |

Table 1. Continued

| Enzyme description                                                         | EC number               | Accession     | GENE symbol        | Sig. abundance   |
|----------------------------------------------------------------------------|-------------------------|---------------|--------------------|------------------|
| N-acetylglucosamine-6-sulfatase precursor                                  | EC 3.1.6.14             | Q8BFR4        | Gns                | High             |
| Tissue alpha-L-fucosidase                                                  | EC 3.2.1.51             | P17164        | Fuca1              | High             |
| xaa-Pro aminopeptidase 1                                                   | EC 3.4.11.9             | O54975        | Xpnpep1            | High             |
| eukaryotic translation initiation factor 3 subunit F                       | EC 3.4.19.12            | Q9DCH4        | Eif3f              | High             |
| Myeloblastin                                                               | EC 3.4.21.76            | Q61096        | Prtn3              | High             |
| pro-cathepsin H precursor                                                  | EC 3.4.22.16            | P00786        | Ctsh               | High             |
| Caspase-1                                                                  | EC 3.4.22.36            | P43527        | Casp1              | High             |
| Cathepsin D                                                                | EC 3.4.23.5             | P24268        | Ctsd               | High             |
| inorganic pyrophosphatase                                                  | EC 3.6.1.1              | Q9D819        | Ppa1               | High             |
| sarcoplasmic/endoplasmic reticulum calcium ATPase 2 isoform b              | EC 3.6.3.8              | P11507        | Atp2a2             | High             |
| SWI/SNF-related matrix-associated                                          | EC 3.6.4.-              | Q91ZW3        | Smarca5            | High             |
| actin-dependent regulator of chromatin subfamily A member 5                |                         | P49717        | Mcm4               | High             |
| DNA replication licensing factor MCM4                                      | EC 3.6.4.12             | P49718        | Mcm5               | High             |
| DNA replication licensing factor MCM5 DNA replication licensing factor     | EC 3.6.4.12 EC 3.6.4.12 | P97310        | Mcm2               | High             |
| DNA replication licensing factor MCM6                                      | EC 3.6.4.12             | P97311        | Mcm6               | High             |
| ras GTPase-activating protein-binding protein 1                            | EC 3.6.4.12             | P97855        | G3bp1              | High             |
| chromodomain-helicase-DNA-binding protein 4                                | EC 3.6.4.12             | Q6PDQ2        | Chd4               | High             |
| putative pre-mRNA-splicing factor ATP-dependent RNA helicase               | EC 3.6.4.13             | O35286        | Dhx15              | High             |
| DHX15 isoform 2 eukaryotic initiation factor 4A-I                          |                         |               |                    |                  |
| isoform 1                                                                  | EC 3.6.4.13             | P60843        | Eif4a1             | High             |
| ATP-dependent RNA helicase DDX3X transitional endoplasmic reticulum ATPase | EC 3.6.4.13 EC 3.6.4.6  | Q62167 P46462 | Ddx3x Vcp          | High High        |
| inositol monophosphatase 1                                                 | EC 3.1.3.94             | P97697        | Impa1              | Low              |
| diphosphoinositol polyphosphate phosphohydrolase 2                         | EC 3.6.1.-              | Q8R2U6        | Nudt4              | Low              |
| neutral cholesterol ester hydrolase 1                                      | EC 3.1.1.-              | B2GV54        | Nceh1              | Low              |
| platelet-activating factor acetylhydrolase 2, cytoplasmic                  | EC 3.1.1.47             | P83006        | Pafah2             | Low              |
| Sialate O-acetylesterase                                                   | EC 3.1.1.53             | P82450        | Siae               | Low              |
| acyl-protein thioesterase 1                                                | EC 3.1.2.-              | P70470        | Lypla1             | Low              |
| eosinophil cationic protein precursor                                      | EC 3.1.27.-             | P70709        | LOC100361866,Ear11 | Low              |
| ubiquitin-like domain-containing CTD phosphatase 1                         | EC 3.1.3.16             | Q5FWT7        | Ublcp1             | Low              |
| serine/threonine-protein phosphatase 6 catalytic subunit                   | EC 3.1.3.16             | Q9CQR6        | Ppp6c              | Low              |
| protein phosphatase 1F                                                     | EC 3.1.3.16             | Q9WVR7        | Ppm1f              | Low              |
| inositol monophosphatase 1                                                 | EC 3.1.3.25             | P97697        | Impa1              | Low              |
| phosphoserine phosphatase                                                  | EC 3.1.3.3              | Q5M819        | Psph               | Low              |
| receptor-type tyrosine-protein phosphatase C isoform 4 precursor           | EC 3.1.3.48             | P04157        | Ptprc              | Low              |
| 1-phosphatidylinositol 4,5-bisphosphate                                    |                         | P24135        |                    | Low              |
| phosphodiesterase gamma-2                                                  | EC 3.1.4.11             |               | Plcg2              |                  |
| beta-mannosidase precursor Neutral alpha-glucosidase AB                    | EC 3.2.1.25 EC 3.2.1.84 | Q4FZV0 Q8BHN3 | Manba Ganab        | Low Low          |
|                                                                            | EC 3.4.11.2             |               |                    | Low              |
| aminopeptidase N precursor                                                 |                         | P15684        | Anpep              |                  |

( Continued )

Table 1. Continued

| Enzyme description                                                            | EC number             | Accession     | GENE symbol   | Sig. abundance   |
|-------------------------------------------------------------------------------|-----------------------|---------------|---------------|------------------|
| tripeptidyl-peptidase 2                                                       | EC 3.4.14.1           | Q64560        | Tpp2          | Low              |
| Signal peptidase complex catalytic subunit SEC11A                             | EC 3.4.21.89          | P42667        | Sec11a        | Low              |
| caspase-3                                                                     | EC 3.4.22.56          | P55213        | Casp3         | Low              |
| proteasome subunit alpha type-2                                               | EC 3.4.25.1           | P49722        | Psma2         | Low              |
| adenosine deaminase                                                           | EC 3.5.4.4            | Q920P6        | Ada           | Low              |
| protein prune homolog                                                         | EC 3.6.1.1            | Q6AYG3        | Prune         | Low              |
| ectonucleoside triphosphate diphosphohydrolase 1                              | EC 3.6.1.5            | P97687        | Entpd1        | Low              |
| diphosphoinositol polyphosphate phosphohydrolase 2                            | EC 3.6.1.52           | Q8R2U6        | Nudt4         | Low              |
| m7GpppX diphosphatase                                                         | EC 3.6.1.59           | Q9DAR7        | Dcps          | Low              |
| E1A-binding protein p400                                                      | EC 3.6.4.-            | Q8CHI8        | Ep400         | Low              |
| regulator of nonsense transcripts 1 isoform a                                 | EC 3.6.4.-            | Q9EPU0        | Upf1          | Low              |
| fumarylacetoacetase                                                           | EC 3.7.1.2            | P25093        | Fah           | Low              |
| Lyases                                                                        |                       |               |               |                  |
| multifunctional protein ADE2                                                  | EC 4.1.1.21           | P51583        | Paics         | High             |
| DNA-(apurinic or apyrimidinic site) lyase                                     | EC 4.2.99.18          | P43138        | Apex1         | Low              |
| Isomerases                                                                    |                       |               |               |                  |
| peptidyl-prolyl cis-trans isomerase A                                         | EC 5.2.1.8            | P17742        | Ppia          | High             |
| peptidyl-prolyl cis-trans isomerase F , mitochondrial precursor               | EC 5.2.1.8            | P29117        | Ppif          | High             |
| peptidyl-prolyl cis-trans isomerase FKBP3                                     | EC 5.2.1.8            | Q62446        | Fkbp3         | High             |
| thromboxane-A synthase                                                        | EC 5.3.99.5           | P49430        | Tbxas1        | High             |
| ribose-5-phosphate isomerase                                                  | EC 5.3.1.6            | P47968        | Rpia          | Low              |
| glucose-6-phosphate isomerase                                                 | EC 5.3.1.9            | Q6P6V0        | Gpi           | Low              |
| DNA topoisomerase 2-beta                                                      | EC 5.99.1.3           | Q64399        |               | Low              |
| Ligases                                                                       |                       |               |               |                  |
| Glycine-tRNA ligase                                                           | EC 6.1.1.14           | Q5I0G4        | Gars          | High             |
| succinyl-CoA ligase [ADP-forming] subunit beta, mitochondrial precursor       | EC 6.2.1.5            | Q9Z2I9        | Sucla2        | High             |
| E3 ubiquitin/ISG15 ligase TRIM25                                              | EC 6.3.2.19           | Q61510        | Trim25        | High             |
| glutathione synthetase                                                        | EC 6.3.2.3            | P46413        | Gss           | High             |
| multifunctional protein ADE2                                                  | EC 6.3.2.6            | P51583        | Paics         | High             |
| E3 ubiquitin/ISG15 ligase TRIM25                                              |                       | Q61510        | Trim25        | High             |
|                                                                               | EC 6.3.2.n3           |               |               |                  |
| C-1-tetrahydrofolate synthase, cytoplasmic threonine-tRNA ligase, cytoplasmic | EC 6.3.4.3 EC 6.1.1.3 | P27653 Q9D0R2 | Mthfd1 Tars   | High Low         |
| Lysine-tRNA ligase                                                            | EC 6.1.1.6            | P37879        | LOC100766627  | Low              |
| lysine-tRNA ligase isoform 2                                                  | EC 6.1.1.6            | Q99MN1        | Kars          | Low              |

isomerase (Pin1), a unique cis-trans-prolyl isomerase. Another isomerase found with up-regulation is thromboxane-A synthase (P49430/ EC: 5.3.99.5)

## 3.3.4 Ligases

Most of the identified ligases with differential regulation include tRNA ligases and Glutathione synthase (P46413/ EC: 6.3.2.3).

## 3.4 Functional pathways and in silico protein-protein interactions analysis of ST affected proteins

The functional pathways analysis of the trauma related proteins was carried out by Wikipathways using WEB-based GEne SeT AnaLysis Toolkit (WebGestalt) [24]. Table 2 lists the top ten pathways enriched for significantly up and downregulated proteins in ST compared with control. The determined pathways for proteins regulated in ST rats, include protein biosynthesis and several other pathways related

Table 2. Predicted Wiki pathways analysis for the proteins with significantly differential regulation

| Pathway name                         | Overlap genes   | Reference genes   | Ratio   | Raw P -value   | Adjusted P -value   |
|--------------------------------------|-----------------|-------------------|---------|----------------|---------------------|
| Pathways for up regulated proteins   |                 |                   |         |                |                     |
| Cytoplasmic Ribosomal Proteins       | 12              | 86                | 36.06   | 9.15E-16       | 2.20E-14            |
| Translation Factors                  | 8               | 47                | 43.99   | 1.18E-11       | 1.42E-10            |
| Electron Transport Chain             | 8               | 100               | 20.67   | 5.88E-09       | 4.70E-08            |
| mRNA processing                      | 8               | 130               | 15.9    | 4.65E-08       | 2.79E-07            |
| TNF-alpha NF-kB Signaling Pathway    | 9               | 203               | 11.46   | 1.12E-07       | 5.38E-07            |
| DNA Replication                      | 4               | 40                | 25.84   | 1.78E-05       | 7.12E-05            |
| Oxidative phosphorylation            | 4               | 58                | 17.82   | 7.81E-05       | 0.0003              |
| TCA Cycle                            | 3               | 28                | 27.69   | 0.0002         | 0.0005              |
| G1 to S cell cycle control           | 4               | 73                | 14.16   | 0.0002         | 0.0005              |
| Cell cycle                           | 4               | 95                | 10.88   | 0.0005         | 0.0012              |
| Pathways for down regulated proteins |                 |                   |         |                |                     |
| B Cell Receptor Signaling Pathway    | 12              | 199               | 14.66   | 4.83E-11       | 1.69E-09            |
| IL-5 Signaling Pathway               | 8               | 80                | 24.31   | 1.59E-09       | 2.78E-08            |
| Pentose Phosphate Pathway            | 4               | 8                 | 121.53  | 1.92E-08       | 2.24E-07            |
| Proteasome Degradation               | 6               | 59                | 24.72   | 1.68E-07       | 1.47E-06            |
| Regulation of Actin Cytoskeleton     | 8               | 156               | 12.46   | 3.04E-07       | 1.77E-06            |
| EGFR1 Signaling Pathway              | 9               | 213               | 10.27   | 2.83E-07       | 1.77E-06            |
| Kit Receptor Signaling Pathway       | 6               | 70                | 20.83   | 4.71E-07       | 2.21E-06            |
| IL-6 Signaling Pathway               | 7               | 114               | 14.92   | 5.06E-07       | 2.21E-06            |
| Type II interferon signaling (IFNG)  | 6               | 76                | 19.19   | 7.70E-07       | 2.99E-06            |
| T Cell Receptor Signaling Pathway    | 7               | 139               | 12.24   | 1.92E-06       | 6.72E-06            |

to the immune system, oxidative phosphorylation and energy metabolism like TCA and Pentose phosphate pathway, proteasome degradation and regulation of actin cytoskeleton. Furthermore, protein-protein interaction analysis using STRING (Fig. 3) facilitated the visualization of pathways where regulated proteins were enriched.

## 4 Discussion

The over activation of the innate immune response followed by immunosuppression in the early post-operative period has beenlinked with an increase in neutrophil count and decrease in percentage of neutrophils involved in phagocytic activity after surgical trauma [3].

After activation of PMNs an interaction between the neutrophil and endothelium occurs followed by rolling, adhesion and diapedesis [26]. The changes in selectins [27], integrins [28] and adhesion molecules [29] on both the endothelium and PMNs during ST could be a final step towards the EC and neutrophil dysfunction. Proteomic analysis using advanced techniques helped us to trace the changes caused by signaling from the site of surgery on the circulating neutrophil proteome that lead to alteration in PMNs activity [30].

We found 393 significantly regulated proteins after ST compared with control in rats. Our analysis showed the initial change in transcription initiation and protein biosynthesis with up regulation while immune response, proteasome degradation and actin cytoskeleton were downregulated in PMNs after surgical trauma.

Our results show down-regulation of most of the proteins having a role in the cytoskeleton. It is clear from the literature that neutrophils are highly motile defense cells and the cytoskeleton plays an important role in their motility [31], chemotaxis, trans-endothelial migration and phagocytosis [32]. Down-regulation of cytoskeleton proteins found in our dataset suggests decreased neutrophil motility and low chemotaxis after surgical trauma.

Different enzymes have been linked to the neutrophil activation and function under different conditions. In this study, the finding of a large number of proteins predicted to have enzyme activity further highlights the effect of surgical trauma on the neutrophil proteome and its function.

The enzyme prediction analysis of significantly regulated proteins suggested the importance of oxidoreductases in impaired chemotaxis and phagocytosis with increase in life span. Increase in systemic neutrophil life span has been known as a major cause of tissue damage after trauma [6]. Oxidoreductases are generally involved in conversion of molecular oxygen to oxygen free radicals, superoxide, hydroperoxide, singlet oxygen and hydrogen peroxide. Down regulation of such proteins is potentially harmful, as these molecules then modify lipids, proteins, DNA and carbohydrates, increasing oxidative stress. In fact, oxidoreductases constitute the most important free radical scavenger systems [33]. Lipoxygenases (LO) are important oxidoreductases, identified in our study, that introduce oxygen into unsaturated fatty acids [34]. After inflammatory stimuli, arachidonic acid is released from the cell membrane as a result of the action of cytosolic phospholipase A2 on phospholipids [35, 36]. The 12/15-LO further metabolizes arachidonic acid and liberates short-lived,

Figure 3. (A) STRING proteinprotein interaction analysis of the significantly up-regulated proteins in ST rats revealing protein networks involved in mRNA processing, TCA cycle, translation factors, and cytoplasmic ribosomal proteins. (B) STRING protein-protein interaction analysis of the significantly down-regulated proteins in ST rats revealing protein networks involved in proteasome degradation, regulation of actin cytoskeleton, immune response, and apoptosis. Color lines represent modes of interaction as detailed in the legend.

<!-- image -->

<!-- image -->

peroxidized products, which are reduced, or enzymatically converted to 12-hydroxyeicosatetraenoic acid (12-HETE) as well as hepoxilins, lipoxins and others [37]. Besides 12/15-LO, the lipoxygenase 5-LO also metabolizes arachidonic acid and produces leukotrienes (LTA4 /LTB4) in inflammatory cells [38].

A study blocking the 12/15-LO in mice showed disturbance in neutrophil recruitment, possibly by alterations in the

CXCL1/CXCR2 chemokine axis [39]. Similarly, both CXCR2 and LTB4 receptors were found down-regulated after exposure to IL8 after trauma injury [13]. Shureiqi et al. showed that the inhibition of arachidonate 15-lipoxygenase causes cell survival and decreases apoptosis in cancer cells suggesting its role as a pro-apoptotic molecule [40], whereas another study by Tang et al. also reported that arachidonate lipoxygenases play an important role in cell survival and

apoptosis [41]. This significant down-regulation of all the three identified lipoxygenases encountered in our analysis after surgical trauma may suggest their role in down regulation of CXCR2 and LTB4 receptors, leading to impaired chemotaxis and phagocytosis in neutrophils with a long life span.

It has been found in the literature that two transferases, Dolichyl-phosphate ùõÉ -D-mannosyltransferase and Dolichyldiphosphooligosaccharide protein glycotransferase, identified as up-regulated in our analysis, participate in n-glycan biosynthesis which is an essential process for maintenance of the receptor surface exposed in neutrophils [42]. Although not directly involved in signaling, glycosylation appears to be important in the maintenance of neutrophil responsiveness to CXC-chemokines during inflammation [43].

Most of the identified down-regulated transferases in this study are not well characterized with respect to their role in neutrophils after ST, opening space for new hypotheses. On the other hand, some down-regulated serine/threonine protein kinases like PAK2 are of great importance. It has been found that in neutrophils, p21-activated kinase (PAK2) can be stimulated by a number of chemokines [44, 45]. Some other studies showed that PAKs are involved in a variety of cellular events such as rapid cytoskeletal responses, transcriptional events and the development of malignancy [46].

Protein kinase C delta, another important transferase, was also found down-regulated after surgery, which has also been reported in mediating spontaneous PMN apoptosis [47].

Among the hydrolases an important enzyme caspase1 was found up-regulated. It participates in the initial inflammatory response after hemorrhagic shock and peripheral tissue trauma. It is involved in the maturation of cytokines (IL-1 ùõÉ and IL-18), in cell death pathways specifically in pyroptosis during infection, in the regulation of both glucose and lipid metabolism, and cell survival [48, 49]. Alongwithcaspase1,neutrophilderivedserineproteases, e.g. cathepsin G (CG), neutrophil elastase (NE), and proteinase 3 (PR3), are also involved in IL-1 ùõÉ maturation at distinct sites [50]. Our results show that up-regulated transferases include myeloblastin/proteinase 3, cathepsin H and cathepsin D, while cathepsin C was identified as down-regulated. Following abdominal surgery, up-regulation of these enzymes in neutrophils may lead to a higher systematic production of IL-1, a cytokine involved in chronic inflammatory disease. Most of the down-regulated hydrolases are involved in the metabolism of different biomolecules, including galactose, lipids, amino acids and purines.

Another hydrolase, phosphoinositide phospholipase C (PLC), was found down-regulated. It produces IP3 and diacylglycerol (DAG) from PI-4, 5 bisphosphate (PI(4,5)P2). IP3 further mediates calcium release and DAG activates several PKC isoforms. It has been found that inhibition of PLC resulted into impaired phagocytosis in macrophages and PLC has been found accumulated at the phagocytic cup [51]. Ins(1,4,5)P3 is further converted by the actions of sev-

eral distinct kinases and phosphatases to a variety of inositol phosphates, that negatively regulate PtdIns(3,4,5)P3 signaling pathway [52]. Therefore, the down regulation of PLC reinforces the hypothesis of neutrophils with impaired phagocytosis after ST discussed above. In this context two more hydrolases, inositol-phosphate phosphatase and diphosphoinositol-polyphosphate diphosphatase, were found down-regulated that along with PLC regulate PtdIns(3,4,5)P3 signaling pathway, consequently playing a critical role in controlling neutrophil function [52]. Another important hydrolase, Caspase-3, was found down-regulated. Its activation leads to apoptotic signals in neutrophils whereas decrease in procaspases-3 was found to be associated with increased lifespan of transmigrated PMN [9, 53]. Increase in systemic neutrophil life span has been known as a major cause of tissue damage after trauma [6].

Threeinteresting up-regulated isomerases were annotated as EC: 5.2.1.8 that acts as regulator of TNFùõÇ induced NADPH oxidase hyper activation in human neutrophils and thus can be an essential player in TNFùõÇ induced inflammatory diseases [54]. Another isomerase, thromboxane-A synthase, was also found up-regulated in this study, which is involved in the production of thromboxane that further induces production of superoxide in neutrophils [55].

Glutathione synthase is one of the ligases that plays an important role in glutathione synthesis, was found up-regulated as compared to control. It has anti-inflammatory activity and inhibits the production and action of several inflammatory cytokines and chemokines [56].

All the presented enzyme families have essential roles in the immune response and biochemical mechanisms of the neutrophil. Our data suggest that neutrophils in rats affected by surgical trauma have critical regulation of these essential enzymes, potentially causing the high life span with high ROS production and tissue damage.

It is clear from Table 2 and STRING analysis (Fig. 3A) that most of the up-regulated proteins have their role in mRNA processing, translation initiation and other translational activities. This up-regulation of translational proteins suggests higher protein expression, which is in accordance with the high neutrophil activation followed by protein secretion [57-59]. Some of the up-regulated proteins in ST as compared to control (Supporting Information Table 1) were clustered in TCA cycle and oxidative phosphorylation pathways, which are crucial for energy conservation in the form of ATP. It was also found in our study that four significantly down-regulated proteins were assigned to the pentose phosphate pathways (PPP) and also glucose6-phosphate dehydrogenase (G6PD). PPP associated proteins with down-regulation may suggest a metabolic shift from PPP in neutrophil activation after surgical trauma suggesting a low NETs formation at the primary stages of trauma [60].

Although neutrophils do not divide, in our study some of the proteins having their role in DNA replication and cell cycle were also identified with significantly up-regulation in

ST rats. This needs attention and further studies to find out their exact role in activated neutrophils after surgical trauma.

The proteasome is an important multi-subunit enzyme complex having a central role in regulating various biological processes like apoptosis, cell cycle progression, immune response and stress related cellular responses [61]. Six proteins from the proteasome degradation pathway were found significantly down-regulated in ST rats in our study. This downregulation of proteasomal proteins was in accordance to the previous study on neonatal and adult neutrophils [62,63] but in contrast to the recently published work where they analyzed the human neutrophil at proteomic and RNA level from trauma patients [59]. As trauma often results in MOF by activating neutrophils [8] and the participation of the proteasome in the pathogenesis of lung injury has also been investigated [64], these and our experimental findings together may suggest a time window to consider in blocking activated neutrophil proteasomal activity to avoid final injuries and later organ dysfunction after trauma.

Functional and protein-protein interaction network analysis using STRING [25] highlighted the down-regulation of proteins (Fig. 3B) involved in the immune response suggesting the low immune response and high susceptibility of the trauma patients to infection as previously described [65,66]. A pro-apoptotic protein Prkcd and an important apoptotic cascade molecule Caspase-3 were found down-regulated during our analysis that supports an increased lifespan of the trauma activated neutrophils [67-70].

A group of proteins involved in regulation of actin cytoskeleton and neutrophil motility was found downregulated, suggesting reduced directionality and migration. Butler et al. also proved that neutrophils from the thermally injured patients show less chemokinesis in the absence of bacterial infection [71]. Another study by Kurihara et al. found defects in the directionality of the neutrophils from thermally injured rats as compared to the sham controls towards fMLP [72]. Hence, this decrease in the trauma activated neutrophil motility will not only reduce the neutrophil recruitment to the site of infection but also will increase damage to the bystander tissues.

Studies on abdominal surgery showed marked elevations of cytokines and chemokines like GRO/KC, MCP-1, IL-18, TNFùõÇ , IL-12 and RANTES which play a key role during the inflammation [73]. Our results showed nine regulated proteins from the TNF-alpha and NF-kB signaling pathways, suggesting an important role of TNF in neutrophil survival after trauma however NFùõã B was found associated with both pro- and anti-inflammatory roles [74]. Whereas down regulation of IL6 signaling after surgical trauma requires further investigation as the deficiency of IL6 results into protection from injury by IL-6-survival signals in neutrophils [75]. Similarly, down regulation of anti-inflammatory cytokine IL-5 signaling pathway was observed. It seemed the TNF alpha pathway was modified by neutrophils after trauma with the additional role of PKC ùõÖ , PLC, and caspase-3 activation. The

changes found associated in the neutrophil proteome after trauma could be related to higher susceptibility of the trauma patients to infection leading to further tissues damage.

In conclusion, our comparative high throughput proteomic approach revealed the changes at regulation level in the rat neutrophil proteome at the early stages after surgical trauma. The enzyme prediction and functional pathways analysis of regulated proteins confirm that surgical trauma results in higher rates of mRNA processing and protein synthesis increased life span and decreased neutrophil recruitment, directional motility and immune response. These protein regulations suggest that neutrophils in ST have lower activity and this causes higher susceptibility of the trauma patients to infection and bystander tissues damage, resulting in multiple organs dysfunctioning. Based on these findings it might be possible to develop a therapeutic approach where the neutrophil response could be better modulated to prevent damage by the systemic inflammatory response syndrome.

This study opens a new window to the better understanding of the neutrophil proteome and provides a database for further profound studies. Validation of the functional roles of the reported enzymes and the proteins encountered in the pathway analyses can be helpful and considered as future therapeutic targets in neutrophil biology at primary stages of trauma.

The authors acknowledge TWAS, CNPq, CAPES, FAPESP, FAP-DF and FUB-UnB for financial support.

Theauthors declare that there is no conflict of interest regarding the publication of this manuscript.

## 5 References

- [1] Plytycz, B., Seljelid, R., Stress and immunity: minireview. Folia. Biol. (Krakow) 2002, 50 , 181-189.
- [2] Teles, L. M., Aquino, E. N., Neves, A. C., Garcia, C. H. et al., Comparison of the neutrophil proteome in trauma patients and normal controls. Protein Pept. Lett. 2012, 19 , 663-672.
- [3] Savitha, D., Rao, K. R., Girish, S. P ., Effect of surgical stress on neutrophil function. Indian J. Physiol. Pharmacol. 2008, 52 , 302-306.
- [4] Fontes, B., Fontes, W., Utiyama, E. M., Birolini, D., The efficacy of loop colostomy for complete fecal diversion. Dis. Colon. Rectum. 1988, 31 , 298-302.
- [5] Edwards, S. W., Hallett, M. B., Seeing the wood for the trees: the forgotten role of neutrophils in rheumatoid arthritis. Immunol. Today 1997, 18 , 320-324.
- [6] Savill, J. S., Wyllie, A. H., Henson, J. E., Walport, M. J. et al., Macrophage phagocytosis of aging neutrophils in inflammation. Programmed cell death in the neutrophil leads to its recognition by macrophages. J. Clin. Invest. 1989, 83 , 865875.
- [7] Liberio, M. S., Joanitti, G. A., Azevedo, R. B., Cilli, E. M. et al., Anti-proliferative and cytotoxic activity of pentadactylin isolated from Leptodactylus labyrinthicus on melanoma cells. Amino. Acids. 2011, 40 , 51-59.

- [8] Botha, A. J., Moore, F . A., Moore, E. E., Kim, F . J. et al., Postinjury neutrophil priming and activation: an early vulnerable window. Surgery 1995, 118 , 358-364; discussion 364-355.
- [9] Power, C. P ., Wang, J. H., Manning, B., Kell, M. R. et al., Bacterial lipoprotein delays apoptosis in human neutrophils through inhibition of caspase-3 activity: regulatory roles for CD14 and TLR-2. J. Immunol. 2004, 173 , 52295237.
- [10] Johnson, J. L., Moore, E. E., Offner, P . J., Partrick, D. A. et al., Resuscitation with a blood substitute abrogates pathologic postinjury neutrophil cytotoxic function. J. Trauma 2001, 50 , 449-455; discussion 456.
- [11] Aquino, E. N., Neves, A. C., Santos, K. C., Uribe, C. E. et al., Proteomic analysis of neutrophil priming by PAF. Protein Pept. Lett. 2016, 23 , 142-151.
- [12] Silliman, C. C., Clay, K. L., Thurman, G. W., Johnson, C. A. et al., Partial characterization of lipids that develop during the routine storage of blood and prime the neutrophil NADPH oxidase. J. Lab. Clin. Med. 1994, 124 , 684-694.
- [13] Adams, J. M., Hauser, C. J., Livingston, D. H., Lavery, R. F . et al., Early trauma polymorphonuclear neutrophil responses to chemokines are associated with development of sepsis, pneumonia, and organ failure. J. Trauma 2001, 51 , 452-456; discussion 456-457.
- [14] Carvalho, L. O., Aquino, E. N., Neves, A. C., Fontes, W., The neutrophil nucleus and its role in neutrophilic function. J. Cell. Biochem. 2015, 116 , 1831-1836.
- [15] Fabian, T . C., Croce, M. A., Stewart, R. M., Dockter, M. E. et al., Neutrophil CD18 expression and blockade after traumatic shock and endotoxin challenge. Ann. Surg. 1994, 220 , 552561; discussion 561-553.
- [16] Tahir, M., Arshid, S., Heimbecker, A. M., Castro, M. S. et al., Evaluation of the effects of ischemic preconditioning on the hematological parameters of rats subjected to intestinal ischemia and reperfusion. Clinics 2015, 70 , 61-68.
- [17] Russo-Carbolante, E. M. S., Azzolini, A. E. C. S., Polizello, A. C. M., Lucisano-Valim, Y . M., Comparative Study of Four Isolation Procedures to Obtain Rat Neutrophils. Comp Clin Path 2002, 11 , 71-76.
- [18] Wisniewski, J. R., Zougman, A., Nagaraj, N., Mann, M., Universal sample preparation method for proteome analysis. Nat. Methods 2009, 6 , 359-362.
- [19] Gobom, J., Nordhoff, E., Mirgorodskaya, E., Ekman, R. et al., Sample purification and preparation technique based on nano-scale reversed-phase columns for the sensitive analysis of complex peptide mixtures by matrix-assisted laser desorption/ionization mass spectrometry. J. Mass Spectrum. 1999, 34 , 105-116.
- [20] Laursen, I., Hojrup, P ., Houen, G., Christiansen, M., Characterisation of the 1st SSI purified MBL standard. Clin. Chim. Acta 2008, 395 , 159-161.
- [21] Palmisano, G., Lendal, S. E., Engholm-Keller, K., Leth-Larsen, R. et al., Selective enrichment of sialic acid-containing glycopeptides using titanium dioxide chromatography with analysis by HILIC and mass spectrometry. Nat. Protoc. 2010, 5 , 1974-1982.

- [22] Fontes, W., Cunha, R. B., Sousa, M. V., Morhy, L., Improving the recovery of lysine in automated protein sequencing. Anal. Biochem. 1998, 258 , 259-267.
- [23] Brosch, M., Yu, L., Hubbard, T ., Choudhary, J., Accurate and sensitive peptide identification with Mascot Percolator. J. Proteome Res. 2009, 8 , 3176-3181.
- [24] Wang, J., Duncan, D., Shi, Z., Zhang, B., WEB-based GEne SeT AnaLysis Toolkit (WebGestalt): update 2013. Nucleic Acids Res. 2013, 41 , W77-83.
- [25] Szklarczyk, D., Franceschini, A., Wyder, S., Forslund, K. et al., STRING v10: protein-protein interaction networks, integrated over the tree of life. Nucleic Acids Res. 2015, 43 , D447-D452.
- [26] Brown,G.E., Silver, G. M., Reiff, J., Allen, R. C. et al., Polymorphonuclear neutrophil chemiluminescence in whole blood from blunt trauma patients with multiple injuries. J. Trauma 1999, 46 , 297-305.
- [27] Kubes, P ., Jutila, M., Payne, D., Therapeutic potential of inhibiting leukocyte rolling in ischemia/reperfusion. J. Clin. Invest. 1995, 95 , 2510-2519.
- [28] Granger, D. N., Korthuis, R. J., Physiologic mechanisms of postischemic tissue injury. Annu. Rev. Physiol. 1995, 57 , 311332.
- [29] Hierholzer, C., Kalff, J. C., Audolfsson, G., Billiar, T . R. et al., Molecular and functional contractile sequelae of rat intestinal ischemia/reperfusion injury. Transplantation 1999, 68 , 1244-1254.
- [30] Chouker, A., Martignoni, A., Schauer, R., Dugas, M. et al., Beneficial effects of ischemic preconditioning in patients undergoing hepatectomy: the role of neutrophils. Arch. Surg. 2005, 140 , 129-136.
- [31] Anderson, S. I., Behrendt, B., Machesky, L. M., Insall, R. H. et al., Linked regulation of motility and integrin function in activated migrating neutrophils revealed by interference in remodelling of the cytoskeleton. Cell. Motil. Cytoskeleton. 2003, 54 , 135-146.
- [32] Cicchetti, G., Allen, P . G., Glogauer, M., Chemotactic signaling pathways in neutrophils: from receptor to actin assembly. Crit. Rev. Oral Biol. Med. 2002, 13 , 220-228.
- [33] Bellisola, G., Casaril, M., Gabrielli, G. B., Caraffi, M. et al., Catalase activity in human hepatocellular carcinoma (HCC). Clin. Biochem. 1987, 20 , 415-417.
- [34] Kuhn, H., O'Donnell, V. B., Inflammation and immune regulation by 12/15-lipoxygenases. Prog. Lipid Res. 2006, 45 , 334-356.
- [35] Khanapure, S. P ., Garvey, D. S., Janero, D. R., Letts, L. G., Eicosanoids in inflammation: biosynthesis, pharmacology, and therapeutic frontiers. Curr. Top. Med. Chem. 2007, 7 , 311-340.
- [36] Nascimento, A., Chapeaurouge, A., Perales, J., Sebben, A. et al., Purification, characterization and homology analysis of ocellatin 4, a cytolytic peptide from the skin secretion of the frog Leptodactylus ocellatus. Toxicon 2007, 50 , 1095-1104.
- [37] Chiang, N., Serhan, C. N., Dahlen, S. E., Drazen, J. M. et al., The lipoxin receptor ALX: potent ligand-specific and stereoselective actions in vivo. Pharmacol. Rev. 2006, 58 , 463-487.

- [38] Samuelsson, B., Dahlen, S. E., Lindgren, J. A., Rouzer, C. A. et al., Leukotrienes and lipoxins: structures, biosynthesis, and biological effects. Science (New York, N.Y.) 1987, 237 , 1171-1176.
- [39] Rossaint, J., Nadler, J. L., Ley, K., Zarbock, A., Eliminating or blocking 12/15-lipoxygenase reduces neutrophil recruitment in mouse models of acute lung injury. Crit. Care 2012, 16 , R166.
- [40] Shureiqi, I., Xu, X., Chen, D., Lotan, R. et al., Nonsteroidal anti-inflammatory drugs induce apoptosis in esophageal cancer cells by restoring 15-lipoxygenase-1 expression. Cancer. Res. 2001, 61 , 4879-4884.
- [41] Tang, D. G., Chen, Y . Q., Honn, K. V., Arachidonate lipoxygenases as essential regulators of cell survival and apoptosis. Proc. Natl. Acad. Sci. USA 1996, 93 , 5241-5246.
- [42] Rollins, B. J., Chemokines. Blood 1997, 90 , 909-928.
- [43] Ludwig, A., Ehlert, J. E., Flad, H. D., Brandt, E., Identification of distinct surface-expressed and intracellular CXC-chemokine receptor 2 glycoforms in neutrophils: Nglycosylation is essential for maintenance of receptor surface expression. J. Immunol. 2000, 165 , 1044-1052.
- [44] Ding, J., Knaus, U. G., Lian, J. P ., Bokoch, G. M. et al., The renaturable 69and 63-kDa protein kinases that undergo rapid activation in chemoattractant-stimulated guinea pig neutrophils are p21-activated kinases. J. Biol. Chem. 1996, 271 , 24869-24873.
- [45] Huang, R., Lian, J. P ., Robinson, D., Badwey, J. A., Neutrophils stimulated with a variety of chemoattractants exhibit rapid activation of p21-activated kinases (Paks): separate signals are required for activation and inactivation of paks. Mol. Cell Biol. 1998, 18 , 7130-7138.
- [46] Kumar, R., Vadlamudi, R. K., Emerging functions of p21activated kinases in human cancer cells. J. Cell. Physiol. 2002, 193 , 133-144.
- [47] Khwaja, A., Tatton, L., Caspase-mediated proteolysis and activation of protein kinase Cdelta plays a central role in neutrophil apoptosis. Blood 1999, 94 , 291-301.
- [48] Gurcel, L., Abrami, L., Girardin, S., Tschopp, J. et al., Caspase-1 activation of lipid metabolic pathways in response to bacterial pore-forming toxins promotes cell survival. Cell 2006, 126 , 1135-1145.
- [49] Shao, W., Yeretssian, G., Doiron, K., Hussain, S. N. et al., The caspase-1 digestome identifies the glycolysis pathway as a target during infection and septic shock. J. Biol. Chem. 2007, 282 , 36321-36329.
- [50] Hazuda, D. J., Strickler, J., Kueppers, F ., Simon, P . L. et al., Processing of precursor interleukin 1 beta and inflammatory disease. J. Biol. Chem. 1990, 265 , 6318-6322.
- [51] Botelho, R. J., Teruel, M., Dierckman, R., Anderson, R. et al., Localized biphasic changes in phosphatidylinositol4,5-bisphosphate at sites of phagocytosis. J. Cell Biol. 2000, 151 , 1353-1368.
- [52] Luo, H. R., Mondal, S., Molecular control of PtdIns(3,4,5)P3 signaling in neutrophils. EMBO Rep. 2015, 16 , 149-163.
- [53] Le'Negrate, G., Rostagno, P ., Auberger, P ., Rossi, B. et al., Downregulation of caspases and Fas ligand expression, and

- increased lifespan of neutrophils after transmigration across intestinal epithelium. Cell. Death. Differ. 2003, 10 , 153-162.
- [54] Boussetta, T ., Gougerot-Pocidalo, M. A., Hayem, G., Ciappelloni, S. et al., The prolyl isomerase Pin1 acts as a novel molecular switch for TNF-alpha-induced priming of the NADPH oxidase in human neutrophils. Blood 2010, 116 , 5795-5802.
- [55] Vaughan, J. E., Walsh, S. W., Ford, G. D., Thromboxane mediates neutrophil superoxide production in pregnancy. Am. J. Obstet. Gynecol. 2006, 195 , 1415-1420.
- [56] Villa, P ., Saccani, A., Sica, A., Ghezzi, P ., Glutathione protects mice from lethal sepsis by limiting inflammation and potentiating host defense. J. Infect. Dis. 2002, 185 , 1115-1120.
- [57] Humphreys, J. M., Hughes, V., Edwards, S. W., Stimulation of protein synthesis in human neutrophils by gammainterferon. Biochem. Pharmacol. 1989, 38 , 1241-1246.
- [58] Hughes, V., Humphreys, J. M., Edwards, S. W., Protein synthesis is activated in primed neutrophils: a possible role in inflammation. Biosci. Rep. 1987, 7 , 881-890.
- [59] Zhou, J. Y., Krovvidi, R. K., Gao, Y ., Gao, H. et al., Traumaassociated human neutrophil alterations revealed by comparative proteomics profiling. Proteomics. Clin. Appl. 2013, 7 , 571-583.
- [60] Azevedo, E. P ., Rochael, N. C., Guimaraes-Costa, A. B., de Souza-Vieira, T . S. et al., A metabolic shift towards pentose phosphate pathway is necessary for amyloid fibril- and phorbol 12-myristate 13-acetate -induced neutrophil extracellular trap (NET) formation. J. Biol. Chem. 2015, 290 , 22174-22183.
- [61] Almond, J. B., Cohen, G. M., The proteasome: a novel target for cancer chemotherapy. Leukemia 2002, 16 , 433-443.
- [62] Molloy, E. J., O'Neill, A. J., Grantham, J. J., SheridanPereira, M. et al., Granulocyte colony-stimulating factor and granulocyte-macrophage colony-stimulating factor have differential effects on neonatal and adult neutrophil survival and function. Pediatr. Res. 2005, 57 , 806-812.
- [63] Zhu, J., Zhang, H., Guo, T ., Li, W. et al., Quantitative proteomics reveals differential biological processes in healthy neonatal cord neutrophils and adult neutrophils. Proteomics 2014, 14 , 1688-1697.
- [64] Tian, X. F ., Zhang, X. S., Li, Y . H., Wang, Z. Z. et al., Proteasome inhibition attenuates lung injury induced by intestinal ischemia reperfusion in rats. Life Sci. 2006, 79 , 20692076.
- [65] Pillay, J., Kamp, V. M., van Hoffen, E., Visser, T . et al., A subset of neutrophils in human systemic inflammation inhibits T cell responses through Mac-1. J. Clin. Invest. 2012, 122 , 327336.
- [66] Hietbrink, F ., Koenderman, L., Althuizen, M., Leenen, L. P ., Modulation of the innate immune response after trauma visualised by a change in functional PMN phenotype. Injury 2009, 40 , 851-855.
- [67] Junger, W. G., Rhind, S. G., Rizoli, S. B., Cuschieri, J. et al., Prehospital hypertonic saline resuscitation attenuates the activation and promotes apoptosis of neutrophils in patients with severe traumatic brain injury. Shock 2013, 40 , 366374.

- [68] Chitnis, D., Dickerson, C., Munster, A. M., Winchurch, R. A., Inhibition of apoptosis in polymorphonuclear neutrophils from burn patients. J. Leukoc. Biol. 1996, 59 , 835-839.
- [69] Paunel-Gorgulu, A., Kirichevska, T ., Logters, T ., Windolf, J. et al., Molecular mechanisms underlying delayed apoptosis in neutrophils from multiple trauma patients with and without sepsis. Mol. Med. 2012, 18 , 325-335.
- [70] Hsing, C. H., Chen, C. L., Lin, W. C., Lin, C. F ., Propofol treatment inhibits constitutive apoptosis in human primary neutrophils and granulocyte-differentiated human HL60 cells. PLoS One 2015, 10 , e0129693.
- [71] Butler, K. L., Ambravaneswaran, V., Agrawal, N., Bilodeau, M. et al., Burn injury reduces neutrophil directional migration speed in microfluidic devices. PLoSOne 2010, 5 , e11921.

- [72] Kurihara, T., Jones, C. N., Yu, Y. M., Fischman, A. J. et al., Resolvin D2 restores neutrophil directionality and improves survival after burns. FASEB J. 2013, 27 , 22702281.
- [73] Buttenschoen, K., Fathimani, K., Buttenschoen, D. C., Effect of major abdominal surgery on the host immune response to infection. Curr. Opin. Infect. Dis. 2010, 23 , 259267.
- [74] Lawrence, T., The nuclear factor NFùõã B pathway in inflammation. Cold Spring Harb. Perspect. Biol. 2009, 1 , a001651.
- [75] Dienz, O., Rud, J. G., Eaton, S. M., Lanthier, P . A. et al., Essential role of IL-6 in protection against H1N1 influenza virus by promoting neutrophil survival in the lung. Mucosal Immunol. 2012, 5 , 258-266.