<!-- image -->

Published in final edited form as:

J Leukoc Biol. 2019 December ; 106(6): 1221-1232. doi:10.1002/JLB.2RI0919-302R.

## Decoding communication patterns of the innate immune system by quantitative proteomics

A. Sukumaran 1 , J.M. Coish 2 , J. Yeung 1 , B. Muselius 1 , M. Gadjeva 3 , A.J. MacNeil 2 , J. Geddes-McAlister 1,*

1 Molecular and Cellular Biology Department, University of Guelph, Guelph, ON, Canada, N1G 2W1;

2 Department of Health Sciences, Brock University, St. Catharines, ON, L2S 3A1;

3 Division of Infectious Diseases, Department of Medicine, Brigham and Women's Hospital, Harvard Medical School, Boston, MA, USA 02115.

## Abstract

The innate immune system is a collective network of cell types involved in cell recruitment and activation using a robust and refined communication system. Engagement of receptor-mediated intracellular signaling initiates communication cascades by conveying information about the host cell status to surrounding cells for surveillance and protection. Comprehensive profiling of innate immune cells is challenging due to low cell numbers, high dynamic range of the cellular proteome, low abundance of secreted proteins, and the release of degradative enzymes (e.g., proteases). However, recent advances in mass spectrometry-based proteomics provides capability to overcome these limitations through profiling the dynamics of cellular processes, signaling cascades, posttranslational modifications, and interaction networks. Moreover, integration of technologies and molecular datasets provide a holistic view of a complex and intricate network of communications underscoring host defense and tissue homeostasis mechanisms. In this Review, we explore the diverse applications of mass spectrometry-based proteomics in innate immunity to define communication patterns of the innate immune cells during health and disease. We also provide a technical overview of mass spectrometry-based proteomic workflows, with a focus on bottom-up approaches, and we present the emerging role of proteomics in immune-based drug discovery while providing a perspective on new applications in the future.

## Keywords

innate immunity; mass spectrometry-based proteomics; leukocytes; phagocytic cells; signaling cascades; cellular mapping; activation; drug discovery

Author Contributions

## 1. Introduction

## Innate immune system

Innate immunity is the product of a collective network of cell types that depend on a robust and refined communication system. Tasked with surveilling diverse tissue sites, leukocytes coordinate cell recruitment and activation through secretion of a diverse array of proteins (e.g., cytokines, chemokines, proteases). Leukocytes engaged in an ongoing pattern recognition event generate a distinct receptor-mediated intracellular signal (e.g., Toll-like receptors (TLRs)). This engagement contributes to the eventual elimination of the perceived threat via effector functions that include secretion-mediated communication with cognate cell types. The classic illustration of this communication in action is macrophage TLR4mediated recognition of Gram-negative bacteria, promoting signaling adaptor-dependent (e.g., MyD88 and TRIF) secretion factors that act as 'beacons' and mobilize supporting leukocyte traffic to the site, including among others, considerable numbers of circulating neutrophils. This example is refined to meet various unique challenges (i.e. microbial species) through calibration of the particular panel of mediators secreted, dictating the cell types recruited and activated. Evidence from rare genetic mutations that compromise aspects of innate immunity suggest that it is a remarkably effective system 1 .

Leukocytes arise from hematopoietic stem cells in the bone marrow and follow either myeloid or lymphoid differentiation lineages. On the myeloid side, a common progenitor gives rise to monocytes, macrophages, and dendritic cells (DC) 2 . Monocytes circulate in blood and mature upon entering tissues where they receive local tissue signals that shape their phenotype 3-5 . Macrophages are tissue-resident leukocytes that exhibit an array of pattern recognition receptors (PRRs), which recognize a diverse collection of foreign invaders through sensing of non-self-molecular motifs called pathogen-associated molecular patterns (PAMPs) 6-8 . Upon activation of PRRs, communicational proteins (e.g., cytokines) are manufactured by the cell, cueing and mobilizing a repertoire of effector cells to the compromised locale (Figure 1) 9 . Macrophages are phagocytically adept, but also continuously scan and shape the local tissue environment and, along with several other cell types, engage in setting nucleic acid-based traps (e.g., neutrophil extracellular traps (NETs)) via NETosis 10 . Beyond their role in host defense, macrophage position and functional plasticity make it no surprise that these cells play key roles in many pathologies from atherosclerosis and rheumatoid arthritis to inflammatory bowel disease and cancer to name a few 11,12 . DCs are also tissue-resident, but mobile, and make contributions that overlap with macrophages through subtype-specific (conventional, plasmacytoid, monocyte-derived) expression of PRRs. Such expression not only condition the adaptive response through shuttling antigen to secondary lymphoid tissues and presenting it to T cells, but also represent an important link to decisions around engagement of the adaptive arm of the immune system through interactions with natural killer (NK) cells 13,14 . Granulocytes are also derived from the myeloid lineage and include neutrophils (the most abundant circulating leukocyte), as well as eosinophils and basophils (also found in circulation), and long-lived tissue-resident mast cells (famous for their role in allergic inflammation, with still emerging roles in microbial defense and tissue homeostasis) 15,16 . These cells collectively shape the innate effector response and cytokine milieu during engagement with various

microbial species via varied PRR expression and secretome functional outcomes from degranulation to phagocytosis, creating a networked response to microbial detection 17,18 . Myeloid-derived suppressor cells (MDSCs) have also emerged in the past decade as a heterogenous group of cells with at least two types having phenotypic and morphological similarity to either neutrophils or monocytes. In addition, MDSCs contribute significantly to T cell inhibition with considerable implications in chronic infection and the tumor microenvironment, again through a network of interactions that implicate several other innate immune cells 19 . The lymphoid lineage, while heavily featuring the adaptive arm in T and B cells, also gives rise to important contributors to innate immunity. The innate lymphoid cell (ILCs) lineage features the classic innate lymphocyte - NKs - and blurs the lines between adaptive and innate immunity 20 . NK cells are established as critical monitors for virally-infected and tumor cells, but also play regulatory roles in innate immunity, as mentioned above. ILCs within each of the three currently defined classes (i.e., ILC1-3), have broadly analogous functions to the T helper cell lineage based on the key transcription factors (e.g., interferon regulatory factors and NFÎº B) and cytokines (e.g., interleukins and interferons). These define each class and contribute to tissue homeostasis, local remodeling and repair, microbial defense, and in cancer immunosurveillance 21 . Clearly evident from the evolving identification and classification of this heterogeneous innate immune cell spectrum is that sophisticated multi-parameter analysis platforms, such as those accessible at the interface of fluorescence-activated cell sorting (FACS) and mass spectrometry-based proteomics, are essential to achieving a holistic view of the complex and intricate network of communications underlying this host defense and tissue homeostasis mechanism.

## Mass spectrometry-based quantitative proteomics

The field of immunology has benefited substantially over the past two decades from technological advances in mass spectrometry-based proteomics 22 . The application of highresolution mass spectrometry for proteomics provides a powerful and robust tool for profiling and quantifying proteins within tissues, organs, and cells 23,24 . Comprehensive information on the dynamics of cellular processes, signaling cascades, post-translational modifications, and interaction networks and complexes is achieved. The applications, techniques, advantages, disadvantages of each is presented in this Review (Table 1). Bottomup or shotgun proteomics commonly accompanies discovery-driven experiments, which enzymatically digests proteins into peptides prior to identification by a mass spectrometer. Alternative applications include top-down proteomics, which analyzes intact proteins and enables the accurate and reliable identification of proteoforms (different proteins derived from the same gene by alternative splicing, alternative start site, or a combination) 25,26 . In addition, targeted proteomics, which focuses on a limited set of predefined peptides in a complex mixture, is routinely used for biomarker discovery and development 27 . In this Review, we focus on the application of bottom-up proteomics to define communication signals of innate immune cells. This approach is unbiased, comprehensive, and promotes identification of total proteomes, secretomes, modifications, interactions, and cellular networks in a single platform.

Considerations for high-quality proteomic profiling of immunological samples, as well as comparison of methodologies for profiling cellular secretomes were recently presented 28,29 .

Here, we provide a brief overview of the guiding principles of mass spectrometry-based proteomics - beginning with extraction of proteins from a variety of sample types. This may include chemical disruption by solubilization and denaturation of proteins in the presence of a chaotropic agent (e.g., urea) or boiling in a detergent (e.g., sodium dodecyl sulfate) or involve extensive mechanical disruption (e.g., probe sonication). Depending on the experimental design, sequence-specific proteases (e.g., trypsin or LysC) are employed to digest proteins into peptides. The peptides are then purified on C18 resin with the use of STop And Go Extraction (STAGE) tips prior to separation along a solvent gradient (commonly acetonitrile), ionization, and identification by mass spectrometry 30 . Sample processing options include methods for the absolute and relative quantification of proteins or peptides using metabolic (e.g., labeling with stable amino acids), chemical (e.g., addition of mass tags), or label-free approaches 31-35 . Moreover, fractionation of samples based on size, mass, or charge reduces sample complexity and promotes deeper proteome coverage 36 .

Measurement of samples by mass spectrometry begins with an MS1 scan to record masses present at a given time. This is followed by an MS2 (MS/MS) scan to detect and fragment peptides or ions based on fragment masses for identification. Traditionally, mass spectrometry processing was performed using data-dependent acquisition (DDA). In DDA, the top N most abundant ions are selected for fragmentation and aligned with a pre-defined database 37 . However, recent technological and computational advances have developed dataindependent acquisition (DIA). In DIA, all ions within a defined mass-to-charge (m/z) window are fragmented together and the window is rapidly moved over the entire m/z range. DIA represents the next-generation of mass spectrometry-based proteomics and in the future a hybrid of DDA and DIA for improved library completeness would significantly enhance comprehension of biological systems.

## Bioinformatics and data integration

A common bottleneck in mass spectrometry-based proteomics is the generation of large datasets, which require sophisticated bioinformatic workflows and platforms, including both publicly- and commercially-available products. The development and support of these platforms, including MaxQuant, Perseus, R, OpenSWATH, OpenMS, and DIA-Umpire, for analysis, visualization, and interpretation has expanded substantially in the past few years, making processing of MS datasets more accessible and reproducible 38-44 . Beyond mass spectrometry-based proteomics for investigation of biological systems, the movement towards multi-OMICs data integration strategies is becoming more prevalent. For example the combination of proteomics, transcriptomics, genomics, and metabolomics datasets in a single experiment, is rapidly becoming an informative and advanced approach for comprehensive profiling. In addition, the generation of new connections among OMICs platforms encourages the development of machine learning tools (e.g., Big Data Analytics) and deposition of data into public repositories (e.g., PRIDE and PASSEL). A general workflow for quantitative proteomic profiling of innate immune cells is provided in Figure 2.

In this Review, we explore the diverse applications of mass spectrometry-based proteomics in innate immunity. Specifically, we emphasize the role of quantitative proteomics within the

past five years to define communication patterns of innate immune cells during health and disease. We also provide a technical overview of mass spectrometry-based proteomic workflows, with a focus on bottom-up approaches. Finally, we present the emerging role of proteomics in immune-based drug discovery, providing a perspective on new applications in the future.

## 2. Quantitative proteomic profiling of innate immune cells

The application of mass spectrometry-based proteomics in innate immunity enables comprehensive and quantitative profiling of cellular processes, secretion, modifications, and interactions. Proteomics uncovers novel functions of well-characterized biological systems, by defining cellular maps of immune cells and provides a resource for the scientific community. Through exploration of immune cell activation and stimulation, proteomics characterizes communication signals of cells, reporting new regulation networks of the innate immune system. Moreover, detection and characterization of cellular proteome signatures associated with disease suggest new targets for therapeutic intervention.

## Global cellular mapping

Proteomic mapping of immune cells can profile distinctions between cell types, specific compartments or organelles, as well as differentiation and developmental stages. For example, the combination of FACS collections into microreactors (specially designed tips for washing and storage of cells until lysis) and proteomics, enabled quantitative profiling of 12 freshly acquired primary immune cell types using only 2 Âµg of protein 45 . This study describes a simple-to-implement sample preparation protocol for TMT (tandem mass tags)based protein analysis of FAC-sorted cells with minimal sample handling steps and processing time. Here, over 7,000 protein groups were quantified from the 12 cell types. A principal component analysis (PCA) and gene enrichment analysis demonstrated separation by pathways (e.g., B-cell antigen receptor and interleukin (IL)-12 pathways in B cells, T-cell receptor signaling for T cells) and clustering of similar cell types together (e.g., T and B cells). Moreover, comparison of proteome and transcriptome datasets showed a dampening of correlation between protein abundance and mRNA expression levels. These results suggest an influence on translational control and possibly protein stability. Here, the combination of newly constructed collection microreactors, on-column coupled TMTlabeling and desalting, and optimized small-scale fractionation of peptides enabled deep proteome profiling of multiple cells types in a single experiment.

Another example profiled multiple neutrophil populations (i.e., neutrophil granules, exocytoseable storage cells, and plasma membranes) using a four-layer Percoll density gradient (combines sedimentation with flotation to separate the desired subsets) to explore the 'targeting-by-timing' hypothesis 46-48 . This hypothesis suggests that granules are filled with proteins synthesized at the time when granules are formed. The authors provide a resource of protein markers within each neutrophil subcellular fraction and investigate possible correlation between proteome and transcriptome profiling. In total, 1,292 proteins were identified, including a newly-discovered granule subset (Ficolin-granule) rich in ficolin-1 (oligomeric lectins) and poor in gelatinase (matrix metalloproteinase 9). Of the

total proteins identified, 126 of the most prominent proteins were selected for data integration with mRNA expression profiles to evaluate the 'targeting-by-timing' hypothesis. This analysis showed that a positive correlation between mRNA array data and proteomic data was observed for the majority of the 126 selected proteins; however, significant discrepancies were also observed.

Macrophages, monocytes, and DCs share a common bone marrow-derived precursor known as the monocyte-macrophage DC progenitor (MDP). In DC development, a common DC progenitor has been identified, but a comparable monocyte-macrophage-restricted progenitor cell has not been defined. Here, a common monocyte progenitor (cMoP) was differentiated in vitro and confirmed in vivo, followed by comparison of cMoP, MDP, and monocyte proteomes 49 . This analysis showed that although cMoP was more closely related to MDP than to monocytes, the cMoP was further committed to monocyte development than MDP, confirming its intermediate status during differentiation. Moreover, gene ontology analysis of cMoP and monocyte cells demonstrated an enrichment in monocytes associated with cell adhesion and migration, phagocytosis, and innate immune responses. Conversely, cMoP enriched categories included proteins responsible for cell cycle, division, and DNA- and RNA-metabolic processes. Overall, this study emphasizes the power of quantitative proteomics for characterization of immune-related cells and the opportunity to provide novel mechanistic insights.

In another example, exploration of the relationship between siglec-E and TLR4 during macrophage differentiation was performed by quantitative proteomics 50 . Siglec-E is a murine CD33-related siglec (sialic acid-binding Ig-like lectins), which functions as an inhibitory receptor and is primarily expressed on neutrophils, tissue macrophages, and splenic DCs, but its role as a negative regulator of TLR4 signaling is controversial 51,52 . Proteomic profiling showed that Siglec-E does not suppress inflammatory TLR4 signaling. Moreover, a siglec-E-dependent alteration (e.g., constitutive tyrosine phosphorylation) in macrophage protein composition was identified, suggesting a functional role in host defense. The results of this study may appear 'negative' in their findings as the data do not support a role of siglec-E in regulation of TLR4 signaling functions. However, a causative role between siglec-E phenotype and macrophage differentiation was observed, illustrating the power of proteomics to define biological changes.

Profiling of developmental stages improves our understanding of immune cell transitions and provides foundational information for further protein and pathway characterization. For example, quantitative proteomics characterized distinct developmental stages (i.e., fully active CD56 bright , transitional CD56 dim CD57 - , and terminally differentiated CD56 dim CD57 + ) of primary human NK cells 53,54 . Isobaric tags for relative and absolute quantification (iTRAQ) were used as a chemical labeling technique, which adds isobaric tags to peptides for mass shift detections in the mass spectrometer. In this study, combining strong cation exchange chromatography with LC-MS/MS, 3,412 proteins were identified, including a core proteome associated with CD56 +  NK cells. In addition, distinct clustering between CD56 bright  and CD56 dim  cells, as well as high similarity among CD57 -  and CD57 + cells was observed. This analysis also defined 11 novel NK-associated proteins and identified 13 proteins displaying large variance between donors. These highly variable

proteins suggest a potential for patient stratification through the development of clinical markers. The results of this study support the current model of NK cell differentiation and provide several novel candidates for follow-up functional analyses.

## Defining activation signals

Immune cell activation results in the regulation of many cellular processes and the release of signaling molecules for inflammatory initiation responses. For example, secretome profiling of lipopolysaccharide (LPS)-stimulated macrophages from primary cells derived from WT, MyD88 knockout, TRIF knockout, and double knockout mice identified 775 proteins. Over the 16 h time course, 52 cytokines were consistently released across the conditions 55 . This study showed that secreted protein abundance was dependent on the presence of at least MyD88 or TRIF and redundancy between these adaptor proteins was evident through proinflammatory secretory results. However, a synergistic role of the adaptor proteins was also observed among anti-inflammatory mediators (Timp1, IL-19, and IL-10). These results support the requirement for adaptor coupling in regulatory cellular homeostasis. This foundational study used mass spectrometry-based proteomics to comprehensively profile the impact of immune cell activation on the extracellular environment and defines novel communication patterns regulating the innate immune system.

More recently, profiling of communication signals of multiple human innate immune cells defined novel communication patterns and uncovered a complex social architecture 56 . Signals between cells are primarily communicated through the production of proteins by 'sending cells' that bind to receptors of 'receiving cells'. Given the limitations of low cell numbers, high dynamic range of the cellular proteome, and low secreted protein abundance, profiling of immune cell communication has been limited. In this study, FAC sorting of 28 hematopoietic cell types from human donors, combined with quantitative, high-resolution mass spectrometry and advanced bioinformatics of steady and activated states, enabled the comprehensive profiling of interactions among immune cells at the total proteome and secretome levels. This work generated a plethora of data (&gt;80% coverage of immunesystem-annotated proteins) defining differences between inter- and intra-cellular interactions, including the diversity of the signals. In addition, a PCA demonstrated a clear distinction between lymphoid and myeloid cell functions, including differences in the number of communication connections among the immune cells. Furthermore, to facilitate the interpretation of the complex datasets and to support proteomic profiling of innate immune cells by other researchers, the authors developed an interactive online database (MaxQB). The development of such a resource substantially extends the impact of this research and supports our improved understanding of immune system communication.

Communication signals also induce enzyme production for activation or degradation of proteins in signalling cascades. In mast cells, the production of proteases controls their biological influence. For example, the proteolytic degradation of tumor necrosis factor (TNF) by mast cells prevent hyperinflammation in severe sepsis 57 . Proteomic profiling of mast cells defined the secretome or releasates generated by bone marrow derived-cultured mast cells vs. peritoneal cell-derived mast cells following IgE-mediated activation 58 . The results determined that bone marrow (390 unique proteins) vs. progenitor (91 unique

proteins) mast cells have distinct releasate profiles. These distinctions include the increased abundance of the transglutaminase coagulation factor XIIIA (involved in fibrin cross-linking for clot stabilization) in the IgE-mediated bone marrow releasate, compared to an absence of this protein in the progenitor cells. Further characterization detected proteolytic degradation of the coagulation factor by mast cell chymase for reduced activity during homeostatic and septic states. Earlier work supported this proteomic investigation of mast cell secretomes by profiling the SLP76 (cytosolic adapter protein that nucleates signaling complexes from immunoreceptors) interactome in resting and activated primary mouse mast cells 59,60 . Here, affinity purification of SLP76 binding partners detected a transmembrane adapter LAT2 and a serine/threonine kinase as the most recruited molecules during mast cell activation. This work developed novel tools (e.g., generation of knock-in mice with a key signaling molecule with a C-terminal one-strep-tag) for signalosome investigation in murine models and uncovered novel signaling molecules in mast cells through quantitative proteomics.

Stimulation of immune cells and, in particular, phagocytic cells is influenced by the microenvironment where biological (e.g., LPS) or physiological (e.g., oxygen levels) conditions impact cellular responses. For example, the connection between neutrophil activation and degranulation in response to strain-specific Streptococcus pyogenes secreted factors (e.g., superantigens and cytotoxins) was explored through profiling of the neutrophil secretome 61 . In total, 42 secreted proteins were identified in the presence of S. pyogenes strain 8003 and 18 were identified for strain 5448, with 14 common proteins. Among the proteins unique to either strain, a cysteine proteinase (SpeB) and a phosphoglycerate kinase (PGK) were identified. SpeB was associated with neutrophil activation and PGK regulated the release of resistin (an azurophilic or primary granule marker), demonstrating its role as a stimulatory factor in neutrophil degranulation. Taken together, the characterization of a secreted factors provides insight into S. pyogenes strain-specific activation of neutrophils and the connection between invasiveness.

In consideration of physiological changes in the microenvironment, profiling of macrophages under changing O 2  conditions was explored. Phenotype plasticity of macrophages is associated with a broad spectrum of activation states or polarizations, focusing on pro- and anti-inflammatory states and classical (M1) vs. alternative activation (M2a) 62 . In this study, quantitative proteomics measured protein abundance profiles of 5,102 proteins from macrophages at different polarization states mediated by high (18.6%) and low (3%) oxygen conditions 63 . The results showed polarization-specific markers (e.g., CD14, CD40, CD74, CD163, CD206, and CD274) and suggested an impact of oxygen on rates of phagocytosis of apoptotic cells. In particular, the impact of oxygen was linked to changes in protein abundance of an arachidonate 15-lipoxygenase (ALOX15). This protein was associated with an increase in phagocytosis in IL-4/IL-13-polarized cells under low oxygen conditions. Here, proteomics uncovered the connection between macrophage polarization and the surrounding physical microenvironment, supporting the accurate functional characterization of proteins.

Another recent study explored the effects of low-dose ethanol (reported to activate or inhibit TLR4 64 ) by semi-quantitative proteomics following treatment of a macrophage cell line (RAW 264.7) 65 . The results identified 1,206 proteins across all treatments (untreated,

ethanol, LPS, or ethanol and LPS combination) with 504 proteins meeting a stringent filtering criterion of â¥5 peptide spectral matches. In the presence of ethanol, complex metabolic changes occurred, including increased abundance of proteins associated with carbohydrate metabolism (e.g., glycolysis, lactic acid production, metabolism, transmission, and catabolism of carbohydrate) and proteins related to liver disease (e.g., necrosis, inflammation, cell damage). Moreover, ethanol was identified as a regulatory hub for 15 proteins, including three with previous roles in LPS signal transduction (e.g., CD14 antigen, prohibitin, and heat shock proteins). This work highlights a connection between co-exposure to ethanol and LPS in macrophages. Furthermore, the findings support a role for ethanol in inflammatory diseases and the ability of proteomics to comprehensively profile the environmental influences on cellular health.

## Innate immune cells in disease

Innate immune cells also play important roles in a variety of diseases. For example, the recently discovered MDSCs are present in the majority of cancer patients and experimental cancer systems where the production of exosomes in a tumor microenvironment induces the production of immune suppressive MDSC 66-68 . Recently, isolation and proteomic profiling of MDSC-derived exosomes resulted in the identification of 412 proteins from exosomes in conventional MDSC and inflammatory MDSC 69 . Annexins, 26S proteasome-associated, histone variants, metabolic enzymes (e.g., the pentose phosphate pathway and glycolysis), and immune system-related proteins were identified. Statistical analyses defined 63 proteins with a significant change in abundance between the two MDSC populations, including a decrease in abundance of proteins involved in the innate immune response (e.g., chitinase-3like protein, complement C3, CD5 antigen-like) in the conventional samples. Conversely, 30 proteins increased in abundance in inflammatory MDSC, including GTP and ATP binding proteins (e.g., ATP-citrate synthase, ADP-ribosylation factor 1), biosynthetic proteins (e.g., serine tRNA ligase, valine tRNA ligase), and a vacuolar sorting protein. Overall, proteomic profiling of MDSC-associated exosomes supports the immunosuppressive activity of MDSC and suggests a cellular mode of communication mediating anti-tumor immunity within the host.

Other studies focusing on MDSCs have investigated the role of ubiquitination, important for protein trafficking, in exosomes 70,71 . For example, enrichment of ubiquitinated proteins using immunoprecipitation, and immunoprecipitation of peptides containing glycinylglycine-modified lysine residues followed by LC-MS/MS, identified 50 ubiquitinated proteins carried by MDSC-derived exosomes. Five of the ubiquitinated proteins are associated with formation of endosomes and exosomes, as well as detection of ubiquitinated histones. These include HMG B1 (a proinflammatory mediator and driver of MDSC accumulation and suppressive potency), providing a connection among MDSC exosome profiles, ubiquitination, and the immunosuppressive roles of MDSC 72 . Other examples of proteomic profiling of MDSCs include an integrative study investigating the protein, mRNA, and miRNA (microRNAs) contents of conventional and inflammatory MDSC and MDSC-derived exosomes 73 . Here, RNA cargo also mediates MDSC immunosuppressive activity, suggesting a potential mechanistic redundancy between functional proteins and RNAs. MDSC-associated exosomes contained proteins involved in

exosome biogenesis and protein loading (e.g., CD9 and Vps4B) and loading of particular miRNAs into exosomes (e.g., nuclear ribonucleoprotein). Again, this study supports the intentional loading of specific proteins (and miRNAs) into exosomes, contributing to their role in anti-tumor activity.

Immunosuppressive cells also enhance tumor progression/metastasis and counteract classical anti-neoplastic treatments. In a recent study, 3,609 proteins were identified among immature myeloid DCs, MDSCs modeling tumor-infiltrating subsets, and/or modeling non-cancerous (NC)-MDSCs by quantitative proteomics 74 . In neoplastic MDSC, a core of kinases, controlling lineage-specific (phosphoinositide and tyrosine kinases) and cancer-induced (extracellular signal-induced kinase and protein kinase C), were differentially abundant. The kinases distinguished the neoplastic MDSCs from the myeloid DC lineage. The data show that AKT and ERK drove MDSC differentiation from myeloid precursors. Taken together, a distinct kinase signature and well-defined interactomes support an influence on the antitumor activity of MDSCs.

Proteomic exploration into the connection between innate immune cells and monogenic diseases, cardiovascular disease (CVD), and asthma has also been analyzed. For example, the function of neutrophil granulocytes is affected by monogenic diseases and current diagnostic tests are limited in their robustness and reproducibility when diagnosing patients. A recent analysis using DIA on neutrophils identified 4,154 proteins across 84 samples 75 . The experimental set up included neutrophil isolation by filter-aided sample preparation (FASP), a method for complete solubilization of the proteome by sodium dodecyl sulfate, followed by urea exchange on a standard filtration device for single-run analyses of organelles and improved proteome coverage 76 . Notably, integration of the proteome and transcriptome datasets showed no correlation, suggesting naÃ¯ve neutrophil transcriptome profiling was not reflective of function and therefore, the study focused on the proteome. Overall, comparisons of healthy donor proteomes vs. patients with severe congenital neutropenia (a deficiency of neutrophils), chronic granulomatous disease (an inherited primary immunodeficiency disease), and leukocyte adhesion deficiency (immune system malfunction leading to immunodeficiency) was performed. The analysis demonstrated clustering of disease vs. healthy profiles. In addition, a common decrease in abundance of granule proteins and minimal overlap among diseases based on the most abundant proteins was observed. Overall, this study enabled genetic diagnosis of diseases based on proteome signatures and demonstrated the value in functional characterization of proteomic profiling compared to transcriptome analysis.

In CVD, pro-inflammatory macrophage activation plays a prominent role. Given the functional heterogeneity of macrophages, characterization of the interplay between pro- and anti-inflammatory macrophage polarization is challenging 77,78 . Recently, quantitative interactome mapping of human macrophages in the presence of pro-inflammatory signal interferon-gamma (IFN Î³ ) or anti-inflammatory/pro-resolving IL-4 was performed 79 . This investigation combined the interactome data with a network proximity-based prediction method to identify key drivers of macrophage activation relevant to CVD. The top-ranked potential regulators of macrophage activation, including Guanylate binding protein 1 (GBP1; family of GTPases) and tryptophanyl-tRNA synthetase (enzyme catalyzing aminoacylation

of tRNA) showed significant enrichment with immune system and CVD-related signatures based on integration of network topology, gene expression, and protein abundance. Functional characterization of these candidate proteins through loss-of-function experiments demonstrated regulation of a pro-inflammatory cytokine (CCL2), secretion, and increased phosphorylation of STAT1 (connects coronary heart disease and inflammatory response) 80 . Taken together, interactome profiling of heterogenous macrophage populations uncovered novel disease-specific regulators of pro-inflammatory activation, suggesting potential CVD drug targets.

Lastly, a comprehensive analysis of eosinophils cellular proteome and phosphoproteome connected acute activation of IL-5 (a cytokine produced by type-2 T helper cells and mast cells) with allergic rhinitis or allergic asthma 81 . A total proteome analysis identified 7,086 proteins and a phosphoproteome analysis identified 4,802 site-specific phosphorylation events. The authors report a global reorganization of eosinophils upon IL-5 activation with minimal changes in the proteome (five significantly different proteins). However, dynamic changes among 220 significantly different phosphoisoforms, including MAPK and CAMKII pathways were measured. This study also defines the impact of often-contaminating platelets in eosinophil studies and provides a recourse for future studies.

## 3. Uncovering novel therapeutic avenues with proteomics Drug discovery

Proteomic profiling of innate immune cells at baseline or during infection may uncover novel mechanisms employed by cells to combat infection with potential as new therapeutic agents. Recently, the production of Î± -mannosidase by neutrophils suggested a novel strategy to treat bacterial keratitis (cornea infection) 82 . In Pseudomonas aeruginosa infections, neutrophils surround biofilms but are unable to breach the bacterial structures, resulting in limited host response and prolonged infection 83 . The presence of an exopolysaccharide matrix covering P. aeruginosa biofilms is proposed to inhibit complement activation and neutrophil phagocytosis through entrapment of enzymes (e.g., ecotin) with biofilmdegradation activity 84,85 . Here, a mechanism of biofilm decomposition by activated neutrophils derived from Swiss Webster mice (naturally resistant to P. aeruginosa infection of the eye) defined the increased production of polysaccharide degrading enzymes. These include Î± -mannosidases, which was higher in the resistant mouse strain at baseline and during bacterial invasion compared to a susceptible mouse strain (C57BL6/N). This study reports the deepest proteome profiling of circulating neutrophils to date (&gt;4,000 proteins) and demonstrates clear clustering among neutrophils derived from each mouse strain and under each condition 22 . Moreover, a global overview of neutrophil activation patterns in the presence or absence of P. aeruginosa is provided. Furthermore, in vivo experimentation reveals the ability of Î± -mannosidase to solubilize biofilms at the ocular surface and suggests a novel treatment strategy with anti-bacterial protection.

## Mechanisms of action and drug optimization

The application of proteomics to identify targets or interacting partners of unique compounds helps define the mechanism of action of a drug and supports new therapeutic avenues. For example, stimulator of interferon genes (STING) is a receptor in the

endoplasmic reticulum, which propagates innate immune sensing of cytosolic pathogenderived and self-DNA. Recently, a high-throughput screening approach identified amidobenzimidazole (ABZIs) small molecules that modulate STING function (i.e., compound 1) 86 . A newly developed linking strategy combined two ABZI-based compounds to increase binding affinity, as determined by immobilization of a compound derivative (i.e., compound 2) followed by LC-MS/MS in a competitive binding assay. For functional characterization of binding, phosphorylation analysis identified dose-dependent phosphorylation, dose-dependent secretion of IFN Î² , as well as increased production of IFN Î³ -induced protein 10, IL-6, and TNF. Moreover, hydrogen-deuterium mass spectrometry (uses isotope labeling to probe the rate that protein backbone amide hydrogens undergo exchange) defined the conformation of STING in the presence of compound 2, resulting in the generation of compound 3 87 . The improved potency of compound 3 was defined through induction of STING-dependent activation of type-1 IFN and pro-inflammatory cytokines in a murine model leading to an adaptive immune response and anti-tumor activity. Overall, proteomics played a supporting role in this study, but the diversity and adaptability of mass spectrometry applications was highlighted (e.g., binding partner identification and definition of structural conformations).

Another study screened a library of 182,710 small molecule NETosis inhibitors and identified LDC7559 as a blocker of NET formation 88 . Here, a competitive binding assay employing an inhibitor-derivative, combined with affinity chromatography and LC-MS/MS, identified Gasdermin D (GSDMD) as a target of the compound. Subsequent functional and mechanistic characterization defined the interaction between the compound and GSDMD for inhibition of both pyroptosis and NETosis. Used as a tool for discovery, proteomics played a crucial role in defining a direct target of LDC7559 (i.e., GSDMD) leading to the discovery of an inhibitor with the potential to modulate GSDMD activity in a variety of inflammatory diseases (e.g., cancer, autoimmune, vascular diseases).

## Drug delivery and bioavailability

Drug delivery and bioavailability can also be explored with proteomics. For example, encapsulation of therapeutics improves bioavailability and the release kinetics of therapeutics 89 . The recent introduction of leukolike vectors (LLV), consisting of purified leukocyte membranes coated onto nanoporous silicon particles, provides an enhanced biomimetic drug delivery system 90 . Here, DIA-based proteomic profiling of membranes eluted from LLV identified 334 proteins from the LLV surface (50% associated with the plasma membrane) with roles in transport, signaling, and immunity. Functional bioinformatic analysis of the LLV surface confirmed enrichment of proteins linked to canonical leukocyte pathways. Mapping of proteins to KEGG pathways supported significant enrichment of the leukocyte transendothelial migration pathway (involved in interaction and adherence of leukocytes to the inflamed endothelium) 91 . Overall, proteomics confirmed the presence of a leukocyte membrane-enriched coating on LLVs with transfer of &gt;150 proteins associated with leukocyte membranes.

## 4. Discussion and conclusion

This Review highlights examples of the diverse applications of mass spectrometry-based proteomics to define cellular maps, activation states, and associations with diseases related to innate immune cells. Moreover, we describe applications of proteomics in drug discovery for the development of novel therapeutics. Recent technological advances in mass spectrometry-based proteomics, including adoption of DIA in diverse research areas, improved instrument sensitivity, speed, and robustness, as well as multidimensional applications to study proteome structure and function enhance our abilities to study biological mechanisms. However, given the low abundance of some cell types and the extensive production of degradative proteases, improvements to proteomic workflows are required for unbiased, comprehensive proteome profiling. For example, the combination of FAC sorting of multiple cell types from fresh cells (non-frozen) using an in vivo murine model or the combination of FAC sorting of multiple hematopoietic cell types from human donors demonstrates the power and possibilities of quantitative proteomics. In addition, consideration of cell types (e.g., immortalized cell lines, primary cell lines, or in vivo) profiled within an experiment is crucial for accurate biological relevance in an observed system.

A rapidly-advancing area of research and technology development is the movement towards single cell proteomics platforms. This approach has been well demonstrated in genomic and transcriptomic profiling, but technological and instrumentation limitations restrict its application in proteomics. The ability to profile single cell changes at the protein level is appealing in the study of innate immune cell communication given the presence of cell types with low population numbers, which are frequently ignored during analyses because of limits in detection. Currently, advances towards single cell proteomics involve technological developments (e.g., nanopore sensors and mass cytometry) and demonstrate promise in the field 92-94 .

Beyond technological advances, bioinformatic improvements provide open-source, publiclyavailable software platforms and data repositories. These platforms provide an opportunity for multiple research groups to benefit from the large amount of data generated during mass spectrometry analyses. For example, it is common for many proteins to remain uncharacterized or unexplored given the large number of proteins identified in a single proteomics experiment. The limitation is also associated with experimental limitations of biological follow-up required for each additional factor. Therefore, the deposition of proteomic datasets into publicly-available repositories (e.g., PRIDE, PASSEL) is strongly supported to allow these unexplored biological insights to be prioritized and further investigated by other researchers 98 .

These repositories also enable data integration strategies to investigate biological problems from a systems perspective. However, consideration of discrepancies among information gathered at different molecular levels (e.g., RNA and protein) must be done to tease apart assumptions about correlation and causation. For example, recent discussions have focused on the limitations of correlating transcript levels with protein abundance 95-97 . Specifically, correlation between reduced transcript levels and protein abundance is linked to several

factors, including the intracellular stability of a protein (e.g., protein turnover rates) and post-translational regulation. Such limitations need to be carefully considered when attempting to integrate proteome and transcriptome datasets.

Despite the demonstrated multi-dimensionality, broad applicability, and impressive biological insights provided by mass spectrometry-based proteomics, its adoption into mainstream research labs is still cost and expertise (i.e., data analysis and interpretation) prohibitive. To overcome such limitations, robust sample preparation and data analysis pipelines are necessary to promote the comprehensive and reliable profiling of innate immune cellular maps, communication signals, and biologically-relevant interactions and networks of cells. In summary, mass spectrometry-based proteomics has contributed substantially to our understanding of innate immunity and promises to enhance our comprehension of innate immune cells communication in the years to come.

## Acknowledgments

The authors thank members of the Geddes-McAlister lab for their critical review of the manuscript and helpful suggestions, as well as Jan Rieckmann for the R-script used to obtain data in Figure 2.

## Abbreviations:

| TLR    | Toll-like receptor                     |
|--------|----------------------------------------|
| DC     | Dendritic cells                        |
| PRR    | Pattern recognition receptors          |
| PAMPs  | Pathogen associated molecular patterns |
| NK     | Natural killer                         |
| MDSCs  | Myeloid-derived suppressor cells       |
| ILCs   | Innate lymphoid cells                  |
| FACS   | Fluorescence-activated cell sorting    |
| STAGE  | STop And Go Extraction                 |
| DDA    | Data dependent acquisition             |
| DIA    | Data independent acquisition           |
| PRIDE  | PRoteomics IDEntifications database    |
| PASSEL | PeptideAtlas SRM Experiment Library    |
| TMT    | Tandem mass tags                       |
| PCA    | Principal Component Analysis           |
| IL     | Interleukin                            |

| MDP      | Monocyte-macrophage DC progenitor                         |
|----------|-----------------------------------------------------------|
| cMoP     | Common monocyte progenitor                                |
| Siglec   | Sialic acid-binding Ig-like lectins                       |
| iTRAQ    | Isobaric tags for relative and absolute quantification    |
| LPS      | Lipopolysaccharide                                        |
| LC-MS/MS | Liquid chromatography-mass spectrometry/mass spectrometry |
| TNF      | Tumor necrosis factor                                     |
| NETs     | Neutrophil extracellular traps                            |
| PGK      | Phosphoglycerate kinase                                   |
| ALOX15   | Arachidonate 15-lipoxygenase                              |
| NC       | Non-cancerous                                             |
| CVD      | Cardiovascular disease                                    |
| FASP     | Filter-aided sample preparation                           |
| IFN      | Interferon                                                |
| STING    | Stimulator of interferon genes                            |
| ABZI     | Amidobenzimidazole                                        |
| GSDMD    | Gasdermin D                                               |
| LLV      | Leukolike vectors                                         |
| KEGG     | Kyoto Encyclopedia of Genes and Genomes                   |
| CLRs     | C-type lectins                                            |
| RIG      | Retinoic acid inducible gene                              |
| RLR      | Retinoic acid inducible gene-like receptors               |
| NOD      | Nucleotide oligomerization domain                         |
| NLR      | Nucleotide oligomerization domain-like receptors          |
| IRFs     | Interferon regulatory factors                             |

## References

- 1. Mortaz E et al. Pattern recognitions receptors in immunodeficiency disorders. Eur. J. Pharmacol (2017). doi:10.1016/j.ejphar.2017.01.014
- 2. YÃ¡Ã±ez A et al. Granulocyte-Monocyte Progenitors and Monocyte-Dendritic Cell Progenitors Independently Produce Functionally Distinct Monocytes. Immunity (2017). doi:10.1016/ j.immuni.2017.10.021

J Leukoc Biol. Author manuscript; available in PMC 2020 December 01.

- 3. Xue J et al. Transcriptome-Based Network Analysis Reveals a Spectrum Model of Human Macrophage Activation. Immunity (2014). doi:10.1016/j.immuni.2014.01.006
- 4. van de Laar L et al. Yolk Sac Macrophages, Fetal Liver, and Adult Monocytes Can Colonize an Empty Niche and Develop into Functional Tissue-Resident Macrophages. Immunity (2016). doi:10.1016/j.immuni.2016.02.017
- 5. Netea MG et al. Trained immunity: A program of innate immune memory in health and disease. Science (2016). doi:10.1126/science.aaf1098
- 6. Wang P, Han X, Mo B, Huang G &amp; Wang C LPS enhances TLR4 expression and IFNÎ³ production via the TLR4/IRAK/NFÎº B signaling pathway in rat pulmonary arterial smooth muscle cells. Mol. Med. Rep (2017). doi:10.3892/mmr.2017.6983
- 7. Sprokholt JK et al. RIG-I-like receptor activation by dengue virus drives follicular T helper cell formation and antibody production. PLoS Pathog. (2017). doi:10.1371/journal.ppat.1006738
- 8. Mahla RS, Reddy MC, Vijaya Raghava Prasad D &amp; Kumar H Sweeten PAMPs: Role of sugar complexed PAMPs in innate immunity and vaccine biology. Frontiers in Immunology (2013). doi:10.3389/fimmu.2013.00248
- 9. Hayashi F, Means TK &amp; Luster AD Toll-like receptors stimulate human neutrophil function. Blood (2003). doi:10.1182/blood-2003-04-1078
- 10. Brinkmann V et al. Neutrophil Extracellular Traps Kill Bacteria. Science (80-.) (2004). doi:10.1126/science.1092385
- 11. Murray PJ &amp; Wynn TA Obstacles and opportunities for understanding macrophage polarization. J. Leukoc. Biol (2011). doi:10.1189/jlb.0710409
- 12. Murray PJ &amp; Wynn TA Protective and pathogenic functions of macrophage subsets. Nature Reviews Immunology (2011). doi:10.1038/nri3073
- 13. Patente TA et al. Human dendritic cells: Their heterogeneity and clinical application potential in cancer immunotherapy. Front. Immunol (2019). doi:10.3389/fimmu.2018.03176
- 14. Ferlazzo G &amp; Morandi B Cross-talks between natural killer cells and distinct subsets of dendritic cells. Frontiers in Immunology (2014). doi:10.3389/fimmu.2014.00159
- 15. Marshall JS Mast-cell responses to pathogens. Nature Reviews Immunology (2004). doi:10.1038/ nri1460
- 16. Mukai K, Tsai M, Saito H &amp; Galli SJ Mast cells as sources of cytokines, chemokines, and growth factors. Immunological Reviews (2018). doi:10.1111/imr.12634
- 17. Ley K et al. Neutrophils: New insights and open questions. Science Immunology (2018). doi:10.1126/sciimmunol.aat4579
- 18. St. John AL &amp; Abraham SN Innate Immunity and Its Regulation by Mast Cells. J. Immunol (2013). doi:10.4049/jimmunol.1203420
- 19. Veglia F, Perego M &amp; Gabrilovich D Myeloid-derived suppressor cells coming of age reviewarticle. Nature Immunology (2018). doi:10.1038/s41590-017-0022-x
- 20. Bedoui S, Gebhardt T, Gasteiger G &amp; KastenmÃ¼ller W Parallels and differences between innate and adaptive lymphocytes. Nature Immunology (2016). doi:10.1038/ni.3432
- 21. Mortha A &amp; Burrows K Cytokine networks between innate lymphoid cells and myeloid cells. Frontiers in Immunology (2018). doi:10.3389/fimmu.2018.00191
- 22. Geddes-McAlister J &amp; Gadjeva M Mass spectromerty-based quantitative proteomics of murinederived polymorphonuclear neutrophils. Curr. Protoc. Immunol In Press, (2019).
- 23. Aebersold R &amp; Mann M Mass-spectrometric exploration of proteome structure and function. Nature 537, 347-355 (2016). [PubMed: 27629641]
- 24. Mann M, Kulak NA, Nagaraj N &amp; Cox J The Coming Age of Complete, Accurate, and Ubiquitous Proteomes. Mol. Cell 49, 583-590 (2013). [PubMed: 23438854]
- 25. Smith LM &amp; Kelleher NL Proteoform: A single term describing protein complexity. Nature Methods (2013). doi:10.1038/nmeth.2369
- 26. Toby TK, Fornelli L &amp; Kelleher NL Progress in Top-Down Proteomics and the Analysis of Proteoforms. Annu. Rev. Anal. Chem. (Palo Alto. Calif) (2016). doi:10.1146/annurevanchem-071015-041550

- 27. Aebersold R, Bensimon A, Collins BC, Ludwig C &amp; Sabido E Applications and Developments in Targeted Proteomics: From SRM to DIA/SWATH. Proteomics (2016). doi:10.1002/ pmic.201600203
- 28. Mukherjee P &amp; Mani S Methodologies to decipher the cell secretome. Biochimica et Biophysica Acta - Proteins and Proteomics (2013). doi:10.1016/j.bbapap.2013.01.022
- 29. Meissner F &amp; Mann M Quantitative shotgun proteomics: Considerations for a high-quality workflow in immunology. Nature Immunology (2014). doi:10.1038/ni.2781
- 30. Rappsilber J, Mann M &amp; Ishihama Y Protocol for micro-purification, enrichment, pre-fractionation and storage of peptides for proteomics using StageTips. Nat. Protoc 2, 1896-1906 (2007). [PubMed: 17703201]
- 31. Ong S-E et al. Identifying the proteins to which small-molecule probes and drugs bind in cells. Proc. Natl. Acad. Sci 106, 4617-4622 (2009). [PubMed: 19255428]
- 32. Gouw JW, Krijgsveld J &amp; Heck AJR Quantitative proteomics by metabolic labeling of model organisms. Mol. Cell. Proteomics 9, 11-24 (2010). [PubMed: 19955089]
- 33. Thompson A et al. Tandem mass tags: A novel quantification strategy for comparative analysis of complex protein mixtures by MS/MS. Anal. Chem 75, 1895-1904 (2003). [PubMed: 12713048]
- 34. Winter SV et al. EASI-tag enables accurate multiplexed and interference-free MS2-based proteome quantification. Nat. Methods (2018). doi:10.1038/s41592-018-0037-8
- 35. Cox J et al. Accurate Proteome-wide Label-free Quantification by Delayed Normalization and Maximal Peptide Ratio Extraction, Termed MaxLFQ. Mol. Cell. Proteomics 13, 2513-2526 (2014). [PubMed: 24942700]
- 36. Batth TS, Francavilla C &amp; Olsen JV Off-line high-pH reversed-phase fractionation for in-depth phosphoproteomics. J. Proteome Res (2014). doi:10.1021/pr500893m
- 37. Ting YS et al. Peptide-Centric Proteome Analysis: An Alternative Strategy for the Analysis of Tandem Mass Spectrometry Data. Mol. Cell. Proteomics (2015). doi:10.1074/mcp.o114.047035
- 38. Cox J &amp; Mann M MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat. Biotechnol 26, 1367-1372 (2008). [PubMed: 19029910]
- 39. Tsou CC et al. DIA-Umpire: Comprehensive computational framework for data-independent acquisition proteomics. Nat. Methods 12, 258-264 (2015). [PubMed: 25599550]
- 40. Tyanova S et al. The Perseus computational platform for comprehensive analysis of (prote)omics data. Nature Methods 13, 731-740 (2016). [PubMed: 27348712]
- 41. Geyer PE et al. Plasma Proteome Profiling to Assess Human Health and Disease. Cell Syst. 2, 185-195 (2016). [PubMed: 27135364]
- 42. RÃ¶st HL et al. OpenSWATH enables automated, targeted analysis of data-independent acquisition MS data. Nature Biotechnology 32, 219-223 (2014).
- 43. RÃ¶st HL et al. OpenMS: A flexible open-source software platform for mass spectrometry data analysis. Nature Methods (2016). doi:10.1038/nmeth.3959
- 44. Team R R Development Core Team. R A Lang. Environ. Stat. Comput 55, 275-286 (2013).
- 45. Myers SA et al. Streamlined Protocol for Deep Proteomic Profiling of FAC-sorted Cells and Its Application to Freshly Isolated Murine Immune Cells. Mol. Cell. Proteomics (2019). doi:10.1074/ mcp.ra118.001259
- 46. RÃ¸rvig S, Ãstergaard O, Heegaard NHH &amp; Borregaard N Proteome profiling of human neutrophil granule subsets, secretory vesicles, and cell membrane: correlation with transcriptome profiling of neutrophil precursors. J. Leukoc. Biol (2013). doi:10.1189/jlb.1212619
- 47. Borregaard N Neutrophils, from Marrow to Microbes. Immunity (2010). doi:10.1016/ j.immuni.2010.11.011
- 48. Le Cabec V, Cowland JB, Calafat J &amp; Borregaard N Targeting of proteins to granule subsets is determined by timing and not by sorting: The specific granule protein NGAL is localized to azurophil granules when expressed in HL-60 cells. Proc. Natl. Acad. Sci (2002). doi:10.1073/ pnas.93.13.6454
- 49. Hettinger J et al. Origin of monocytes and macrophages in a committed progenitor. Nat. Immunol (2013). doi:10.1038/ni.2638

- 50. Nagala M et al. Expression of Siglec-E Alters the proteome of lipopolysaccharide (LPS)-activated macrophages but does not affect LPS-driven cytokine production or toll-like receptor 4 endocytosis. Front. Immunol (2018). doi:10.3389/fimmu.2017.01926
- 51. Zhang JQ, Biedermann B, Nitschke L &amp; Crocker PR The murine inhibitory receptor mSiglec-E is expressed broadly on cells of the innate immune system whereas mSiglec-F is restricted to eosinophils. Eur. J. Immunol (2004). doi:10.1002/eji.200324723
- 52. McMillan SJ et al. Siglec-E is a negative regulator of acute pulmonary neutrophil inflammation and suppresses CD11b b2-integrin-dependent signaling. Blood (2013). doi:10.1182/ blood-2012-08-449983
- 53. Ross PL et al. Multiplexed protein quantitation in Saccharomyces cerevisiae using amine-reactive isobaric tagging reagents. Mol. Cell. Proteomics (2004). doi:10.1074/mcp.M400129-MCP200
- 54. Scheiter M et al. Proteome Analysis of Distinct Developmental Stages of Human Natural Killer (NK) Cells. Mol. Cell. Proteomics (2013). doi:10.1074/mcp.m112.024596
- 55. Meissner F, Scheltema RA, Mollenkopf HJ &amp; Mann M Direct proteomic quantification of the secretome of activated immune cells. Science (80-.) 340, 475-478 (2013).
- 56. Rieckmann JC et al. Social network architecture of human immune cells unveiled by quantitative proteomics. Nat. Immunol 18, 583-593 (2017). [PubMed: 28263321]
- 57. Piliponsky AM et al. The chymase mouse mast cell protease 4 degrades TNF, limits inflammation, and promotes survival in a model of sepsis. Am. J. Pathol (2012). doi:10.1016/ j.ajpath.2012.05.013
- 58. Shubin NJ et al. Proteome analysis of mast cell releasates reveals a role for chymase in the regulation of coagulation factor XIIIA levels via proteolytic degradation. J. Allergy Clin. Immunol (2017). doi:10.1016/j.jaci.2016.03.051
- 59. Bounab Y et al. Proteomic Analysis of the SH2Domain-containing Leukocyte Protein of 76 kDa (SLP76) Interactome. Mol. Cell. Proteomics (2013). doi:10.1074/mcp.m112.025908
- 60. Jackman JK et al. Molecular cloning of SLP-76, a 76-kDa tyrosine phosphoprotein associated with Grb2 in T cells. J. Biol. Chem (1995). doi:10.1074/jbc.270.13.7029
- 61. Uhlmann J et al. Phosphoglycerate kinase-a novel streptococcal factor involved in neutrophil activation and degranulation. J. Infect. Dis (2016). doi:10.1093/infdis/jiw450
- 62. Stein BM, Keshav S, Harris N &amp; Gordon S Interleukin 4 Potently Enhances Murine Macrophage Mannose Receptor Activity: A Marker of Alternative Immunologic Macrophage Activation By Michael Stein, Satish Keshav, Neil Harris,* and Siamon Gordon. J Exp Med (1992).
- 63. Court M, Petre G, Atifi M EL &amp; Millet, A. Proteomic Signature Reveals Modulation of Human Macrophage Polarization and Functions Under Differing Environmental Oxygen Conditions. Mol. Cell. Proteomics (2017). doi:10.1074/mcp.ra117.000082
- 64. Dai Q &amp; Pruett SB Ethanol suppresses LPS-induced toll-like receptor 4 clustering, reorganization of the actin cytoskeleton, and associated TNFÎ± production. Alcohol. Clin. Exp. Res (2006). doi:10.1111/j.1530-0277.2006.00172.x
- 65. Kamal AHM, Fessler MB &amp; Chowdhury SM Comparative and network-based proteomic analysis of low dose ethanol- and lipopolysaccharide-induced macrophages. PLoS One (2018). doi:10.1371/journal.pone.0193104
- 66. Valadi H et al. Exosome-mediated transfer of mRNAs and microRNAs is a novel mechanism of genetic exchange between cells. Nat. Cell Biol (2007). doi:10.1038/ncb1596
- 67. Xiang X et al. Induction of myeloid-derived suppressor cells by tumor exosomes. Int. J. Cancer (2009). doi:10.1002/ijc.24249
- 68. Gabrilovich DI &amp; Nagaraj S Myeloid-derived suppressor cells as regulators of the immune system. Nature Reviews Immunology (2009). doi:10.1038/nri2506
- 69. Burke M, Choksawangkarn W, Edwards N, Ostrand-Rosenberg S &amp; Fenselau C Exosomes from myeloid-derived suppressor cells carry biologically active proteins. J. Proteome Res (2014). doi:10.1021/pr400879c
- 70. Burke MC, Oei MS, Edwards NJ, Ostrand-Rosenberg S &amp; Fenselau C Ubiquitinated proteins in exosomes secreted by myeloid-derived suppressor cells. J. Proteome Res (2014). doi:10.1021/ pr500854x

- 71. Buschow SI, Liefhebber JMP, Wubbolts R &amp; Stoorvogel W Exosomes contain ubiquitinated proteins. Blood Cells, Mol. Dis (2005). doi:10.1016/j.bcmd.2005.08.005
- 72. Parker KH et al. HMGB1 enhances immune suppression by facilitating the differentiation and suppressive activity of myeloid-derived suppressor cells. Cancer Res. (2014). doi:10.1158/0008-5472.CAN-13-2347
- 73. Geis-Asteggiante L et al. Differential Content of Proteins, mRNAs, and miRNAs Suggests that MDSC and Their Exosomes May Mediate Distinct Immune Suppressive Functions. J. Proteome Res (2018). doi:10.1021/acs.jproteome.7b00646
- 74. Gato-CaÃ±as M et al. A core of kinase-regulated interactomes defines the neoplastic MDSC lineage. Oncotarget (2015). doi:10.18632/oncotarget.4746
- 75. Grabowski P et al. Proteome Analysis of Human Neutrophil Granulocytes From Patients With Monogenic Disease Using Data-independent Acquisition. Mol. Cell. Proteomics (2019). doi:10.1074/mcp.ra118.001141
- 76. WiÅniewski JR, Zougman A, Nagaraj N &amp; Mann M Universal sample preparation method for proteome analysis. Nat. Methods (2009). doi:10.1038/nmeth.1322
- 77. Murray PJ et al. Macrophage Activation and Polarization: Nomenclature and Experimental Guidelines. Immunity (2014). doi:10.1016/j.immuni.2014.06.008
- 78. Gordon S &amp; Mantovani A Diversity and plasticity of mononuclear phagocytes. European Journal of Immunology (2011). doi:10.1002/eji.201141988
- 79. Halu A et al. Context-enriched interactome powered by proteomics helps the identification of novel regulators of macrophage activation. Elife (2018). doi:10.7554/eLife.37059
- 80. O'Donnell CJ &amp; Nabel EG Genomics of Cardiovascular Disease. N. Engl. J. Med (2011). doi:10.1056/nejmra1105239
- 81. Wilkerson EM et al. The peripheral blood eosinophil proteome. J. Proteome Res (2016). doi:10.1021/acs.jproteome.6b00006
- 82. Kugadas A et al. Employing enzymatic treatment options for management of ocular biofilm-based infections. J. Leukoc. Biol https://do, (2019).
- 83. Kragh KN et al. Polymorphonuclear leukocytes restrict growth of Pseudomonas aeruginosa in the lungs of cystic fibrosis patients. Infect. Immun (2014). doi:10.1128/IAI.01969-14
- 84. Mishra M et al. Pseudomonas aeruginosa Psl polysaccharide reduces neutrophil phagocytosis and the oxidative response by limiting complement-mediated opsonization. Cell. Microbiol (2012). doi:10.1111/j.1462-5822.2011.01704.x
- 85. Tseng BS et al. A biofilm matrix-associated protease inhibitor protects Pseudomonas aeruginosa from proteolytic attack. MBio (2018). doi:10.1128/mBio.00543-18
- 86. Ramanjulu JM et al. Design of amidobenzimidazole STING receptor agonists with systemic activity. Nature (2018). doi:10.1038/s41586-018-0705-y
- 87. Yan X &amp; Maier CS Hydrogen/Deuterium Exchange Mass Spectrometry. in (2009). doi:10.1007/978-1-59745-493-3\_15
- 88. Sollberger G et al. Gasdermin D plays a vital role in the generation of neutrophil extracellular traps. Sci. Immunol (2018). doi:10.1126/sciimmunol.aar6689
- 89. Veiseh O, Gunn JW &amp; Zhang M Design and fabrication of magnetic nanoparticles for targeted drug delivery and imaging. Advanced Drug Delivery Reviews (2010). doi:10.1016/j.addr.2009.11.002
- 90. Corbo C et al. Proteomic Profiling of a Biomimetic Drug Delivery Platform. Curr. Drug Targets (2014). doi:10.2174/1389450115666141109211413
- 91. LanÃ§a T &amp; Silva-Santos B The split nature of tumor-infiltrating leukocytes: Implications for cancer surveillance and immunotherapy. OncoImmunology (2012). doi:10.4161/onci.20068
- 92. Galenkamp NS, Soskine M, Hermans J, Wloka C &amp; Maglia G Direct electrical quantification of glucose and asparagine from bodily fluids using nanopores. Nat. Commun (2018). doi:10.1038/ s41467-018-06534-1
- 93. Huang G, Voet A &amp; Maglia G FraC nanopores with adjustable diameter identify the mass of opposite-charge peptides with 44 dalton resolution. Nat. Commun (2019). doi:10.1038/ s41467-019-08761-6

- 94. Spitzer MH &amp; Nolan GP Mass Cytometry: Single Cells, Many Features. Cell (2016). doi:10.1016/ j.cell.2016.04.019
- 95. Lee MV et al. A dynamic model of proteome changes reveals new roles for transcript alteration in yeast. Mol. Syst. Biol (2011). doi:10.1038/msb.2011.48
- 96. Selbach M et al. Widespread changes in protein synthesis induced by microRNAs. Nature (2008). doi:10.1038/nature07228
- 97. SchwanhÃ¼usser B et al. Global quantification of mammalian gene expression control. Nature 473, 337-342 (2011). [PubMed: 21593866]
- 98. Perez-Riverol Y et al. Quantifying the impact of public omics data. Nat. Commun 10, 3512 (2019). [PubMed: 31383865]
- 99. Liu T, Zhang L, Joo D &amp; Sun SC NFÎº B signaling in inflammation. Signal Transduction and Targeted Therapy (2017). doi:10.1038/sigtrans.2017.23
- 100. Chang M, Jin W &amp; Sun SC Peli1 facilitates TRIF-dependent Toll-like receptor signaling and proinflammatory cytokine production. Nat. Immunol (2009). doi:10.1038/ni.1777
- 101. Janeway CA &amp; Medzhitov R Innate Immune Regulation. Annu. Rev. Immunol (2002). doi:10.1146/annurev.immunol.20.083001.084359
- 102. Bouma G et al. Impaired neutrophil migration and phagocytosis in IRAK-4 deficiency: Correspondence. British Journal of Haematology (2009). doi:10.1111/j.1365-2141.2009.07838.x
- 103. Takeuchi O &amp; Akira S Pattern Recognition Receptors and Inflammation. Cell (2010). doi:10.1016/j.cell.2010.01.022
- 104. Yin MJ, Yamamoto Y &amp; Gaynor RB The anti-inflammatory agents aspirin and salicylate inhibit the activity of I Îº B kinaseÎ² . Nature (1998). doi:10.1038/23948
- 105. Glass CK &amp; Natoli G Molecular control of activation and priming in macrophages. Nature Immunology (2016). doi:10.1038/ni.3306
- 106. Platanitis E &amp; Decker T Regulatory networks involving STATs, IRFs, and NF Îº B in inflammation. Frontiers in Immunology (2018). doi:10.3389/fimmu.2018.02542
- 107. Iwasaki A &amp; Medzhitov R Control of adaptive immunity by the innate immune system. Nature Immunology (2015). doi:10.1038/ni.3123

Figure 1: Illustration of broad communication signals and crosstalk among innate immune cells and cascades.

<!-- image -->

The success of innate immunity to recognize a diverse collection of foreign invaders relies on PRRs that sense non-self-molecular motifs (i.e., PAMPs) 6-8 . Upon activation of PRRs, commonly characterized as Toll-like receptors (TLRs), C-type lectins (CLRs), retinoic acidinducible gene (RIG)-I like receptors (RLRs), and nucleotide oligomerization domain (NOD)-like receptors (NLRs), antimicrobial communicational proteins (e.g. cytokines) are manufactured by the cell 99-101 . For this to take place, a series of signaling cascades occur within the cell that include adaptor molecules (e.g. MYD88, TRAM, SYK, and IPS-1) activating intermediate signaling molecules (e.g. MAPK, IKK Î± Î² / , TBK/IKKi) that turn on downstream transcription factors, such as interferon regulatory factors (IRFs) and NFÎº B 8,102,103 . Transcription factors then make their way into the nucleus of activated cells in search for the genes responsible for interferons and pro-inflammatory cytokines 104-106 . Ultimately, the production of these communication proteins is to cue and mobilize a repertoire of effector cells, including natural killer cells, neutrophils, and T cells to the compromised locale 9 . This congregation at the target site shapes the extracellular milieu to an environment promoting effector functions (e.g., NETosis and chemotaxis) and impairing pathogen replication. The response culminates with either the destruction of the pathogen or the innate immune system communicating for further help to the adaptive immune system, a more precise, but initially slower response involving exceptionally selective T and B cells to counter a particular microbe 107 . (Image generated with Servier medical and under the use of the Creative Commons Attribution 3.0 Unported License).

Figure 2: Overview of bottom-up mass spectrometry-based proteomics workflow for innate immune cells.

<!-- image -->

Biological samples are collected from a variety of sources (e.g., human, mouse, cell culture), followed by isolation of material (e.g., serum, blood, media), and purification of innate immune cells (e.g., macrophages, neutrophils, eosinophils, dendritic cells, mast cells, natural killer cells). Proteins are extracted using mechanical (e.g., sonication) or chemical (e.g., sodium dodecyl sulfate) disruption and digested into peptides using sequence-specific proteases (e.g., Lys-C, trypsin). Peptides are separated by high performance liquid chromatography, subjected to electrospray ionization, and measured on the mass spectrometer (e.g., Orbitrap technology, Time-of-Flight). The MS1 scan selects peptides for fragmentation and the MS2 (MS/MS) scan fragments the peptides. Bioinformatic pipelines (examples provided) identify and quantify the proteins in a sample and perform statistical analyses for data processing and interpretation.

Table 1:

An overview of mass spectrometry-based proteomics applications and techniques of innate immune profiling

| Applications                                                      | Protein population                                                                     | Information obtained                                                          | Advantages                                                                                                                                                | Disadvantages                 |
|-------------------------------------------------------------------|----------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
| Intracellular  proteins Membrane-bound  proteins Surface proteins | Proteome remodeling inside  the cell Interface between intra- and  extracellular space | Comprehensive view of the  cell Robust and sensitive  protocol                | Loss of organelle-specific profiles  are lost                                                                                                             | Cellular  proteome            |
| Secreted proteins Released proteins Surface proteins              | Activation state changes Conventional vs.  unconventional secretion                    | Overview of the  extracellular environment Rapid protocol                     | Difficult to distinguish secretion,  release, or cell lysis Highly abundant proteins (e.g.,  albumin) in media interfere with  depth of proteome coverage | Secretome                     |
| Modified proteins                                                 | Protein modifications Signaling cascades Protein turnover and  degradation             | Measure protein  stoichiometry Functional information                         | Low abundance of modified  proteins Requires more starting material  than total proteome analysis                                                         | PTMs                          |
| Intracellular  proteins Extracellular  proteins                   | Protein networks Signaling cascades Communication patterns                             | Information on physical and  functional Interactions Detect protein complexes | Static snapshot of interactions at  specific point in time Non-specific interactors                                                                       | Protein-protein  interactions |
| Organelle-specific  regions                                       | Protein localization Protein-protein interactions                                      | Large dynamic range for  protein abundance Multiple techniques  available     | Proteins identified in multiple  cellular compartments                                                                                                    | Subcellular  fractionation    |