## Proteome Analysis of Resting Human Neutrophils

M.S. Castro , N.M. S‡ , R.P. Gadelha , M.V. Sousa , C.A.O. Ricart , B. Fontes  and W. Fontes a a a a a b a,*

a Brazilian  Center  for  Protein  Research,  Department  of  Cell  Biology,  University  of  Bras'lia,  Brazil; b Laboratory  of Medical Investigation (LIM-62), University of S‹o Paulo, Brazil

Abstract: Neutrophils constitute the first line of host defense against pathogens. In the present study 2-D gel electrophoresis-mass spectrometry technology was employed to analyze the human resting neutrophils proteome. One hundred and two conserved spots were subjected to peptide mass fingerprinting, yielding 22 identifications. Among the identified proteins, nine are related to the inflammatory process, two polypeptides are assigned to metabolic functions and five are classified as structural.

Keywords: Neutrophils, proteome, peptide mass fingerprinting, inflammatory response.

## 1. INTRODUCTION

Neutrophils  (also  called  neutrophilic  granulocytes,  or polymorphonuclear  neutrophilic  leukocytes,  PMNs)  are  the most numerous of the leukocytes and participate  in  a  wide range of normal and pathological responses of the organism to  diverse  stimuli.  In  order  to  help  destroy  invading  pathogens or to participate in the recovery of tissue damage, these cells migrate from the circulatory system through the endothelial cell layer to the site where chemical signals have been emitted [1,2]. Once at the destination site, the activated polymorphonuclear leukocytes attack microorganisms and debris by phagocytosis or by releasing a combination of reactive oxygen species (ROS), enzymes and antimicrobial peptides  [3].  Neutrophils  are  finally  removed  from  the  inflammatory site when cytokine signals indicate that the resolution of the process will take place soon [4].

All of these events are tightly regulated, since cell maturation,  through  margination, priming, chemotaxis, phagocytosis,  degranulation  and  even  apoptosis.  The  regulation  of these processes suffer strong influence of a wide variety of membrane proteins which act both as receptors for cytokines and as ligands to target other cells [2,5-7].

Neutrophils  are  not  only  responsible  for  the  normally regulated  inflammatory  response;  sometimes  they  escape regulation and provoke damage to tissues that were not necessarily  involved  in  the  initial  process.  It  is  known  that  a number  of  pathologies  are  associated  to  tissue damage caused  by  neutrophils,  such  as  the  systemic  inflammatory response  [8];  its  common  consequence,  acute  lung  injury (ALI)  [9];  asthma  [10];  hepatitis  [11],  inflammatory  bowel disease [12], gout [13], rheumatoid arthritis [14] and dermatoses [15].

The pathways involved in the many steps of the inflammatory  process  are  multiple  and  overlapping.  Neutrophil rolling  and  capture  by  the  endothelium  is  mediated  by  P-

selectin  binding  PSGL-1,  followed  by  activation,  generally by contact with chemokines, like interleukin-8, that triggers neutrophil adhesion to the endothelium [16]. Although these events are sufficient to remove neutrophils from the vascular blood flow [17], a real inflammatory response offers a more complex  situation  where  multiple  cytokine  effects  and  the participation  of  other  adhesion  molecules  such  as  PSGL-1 and ESL-1 suggest different and not completely established pathways [18].

Also,  the  events  following  neutrophil  adhesion  to  the endothelium are not completely elucidated [5,19].. In order to generate traction within the extracellular matrix, the neutrophils  need  to  interact  with  surrounding  molecules  and, simultaneously,  break  barriers.  Although  proteases  have  an important  role  in  digesting  collagen  and  laminin,  their  participation  in  the  process  remains  controversial  [5,20].  Neutrophil  binding  to  matrix  components  such  as  collagen,  vitronectin and laminin again involves integrins such as VLA4, VLA-5, VLA-6 and VLA-9 [21]. .

All the above mentioned physiopathological mechanisms are integrated with cellular alterations that reflect the precise regulation  mechanisms  involved.  The  intermediary  state  of such functional transformations, a gradient that ranges from resting  to  fully  activated,  is  usually  called  ÒprimingÓ,  a situation  where  the  neutrophil  response  to  stimuli  is  amplified [2]. Although most of the published reports focus on the enhancement of the respiratory burst by priming, other important functional changes are also induced, such as expression  of  surface  receptors,  actin  polymerization,  increased phagocytosis and inhibition of apoptosis [22].

The possibility  of  interfering  with  the  ability  of  inflammatory cells to perform their many functions from priming and  adhesion  to  the  release  of  free  radicals  during  certain disease  states  can  be  beneficial.  Indeed,  one  of  the  many ameliorative  effects  of  glucocorticoid  or  nonsteroidal  antiinflammatory  drug  treatment  is  to  limit  the  activation  and recruitment of leukocytes [23]. Recently, antagonistic ligands based on the active sites of integrins and selectins are being developed and have proved effective in several animal models of inflammation and autoimmune disease [24,25]. In

addition, the pathways of signal transduction associated with the activation of adhesion molecules have been proposed as targets  for  the  design  of  a  new  class  of  anti-inflammatory agents [26,27].

Although  intensive  and  extensive  research  in  the  last three decades have clarified many relevant events involving the  neutrophils  role  in  the  inflammatory  response,  many questions remain to be answered, and many others have been raised.  Enormous  technologic  advances  have  prompted  research  laboratories  for  these  achievements,  allowing  the presently growing progress in molecular biology. However, many questions  concerning  the  complex  mechanism  of  the role of neutrophils in the inflammatory response remains to be  answered,  such  as  turning  on  and  off  of  the  neutrophil functions  in  the  natural  control  of  inflammation  and  which proteins  in  the  neutrophil,  besides  those  already  characterized, are involved in this mechanism. It seems natural to expect that the answer to these and other questions may contribute  to  the  development  of  new  therapeutic  strategies  to control the deleterious inflammatory response and its pathological sequelae.

The objectives of this study were to identify proteins in resting  human  neutrophils,  that  are  potentially  or  really  involved  in  the  neutrophil  role  in  the  inflammatory  response and to describe an initial  proteome analysis of human neutrophils with the application of 2-D gel electrophoresis-mass spectrometry technology.

## 2. MATERIALS AND METHODS

## 2.1. Sample Preparation

Human neutrophils were obtained from 14 volunteer donors that had no history of previous diseases, nor were using any medications. The donors, aged 25 to 45 years-old, were equally distributed according to sex and ethnical groups. The project  was  approved  by  the  ethical  committee  of  the  HCFMUSP (Brazil) before the individuals were contacted.

Neutrophils  were  purified  as  described  previously  [28] with  a  few  modifications.  Briefly,  blood  was  collected  in heparinized solution, sedimented, centrifuged (10 min, 1000 g)  in  the  presence  of  Ficoll-Paque-Plus  (Amersham  Biosciences),  the  granulocyte  pellet  was  washed  in  water  for residual  erythrocyte  lysis,  saline  and  Hanks  solution.  An aliquot was taken for cell counting and viability analysis by optical  microscopy  using  tripan  blue.  The  purification  was carried out at room temperature to avoid activation [29], except the density gradient step.

Samples yielding at least 10  cells/mL (99% neutrophils, 6 99% viable) were immediately subjected to cell lysis in protein solubilization buffer (7 M urea, 2 M thiourea, 1% (v/v) Triton  X-100,  0.5%  (v/v)  Pharmalyte,  65  mmol/L  DTT) containing  protease  inhibitors  (EDTA,  TLCK,  TPCK  and pepstatin A). The cell lysis was carried out for 10 hours at room temperature,  to  avoid  long  exposure  to  low  temperatures [29] and, after protein solubilization, the solution was frozen in liquid nitrogen until electrophoretic separation.

## 2.2. Two-Dimensional Gel Electrophoresis

Neutrophil  protein  extracts  were  submitted  to  protein quantification [30] and diluted in lysis buffer to a final con-

centration  of  0.86 m g/ m L.  350 m L of this solution were applied to 18 cm IPG gel strips with a linear pH range from 4 to  7  (Amersham  Biosciences)  by  in-gel  sample  rehydration [31].  After  12  h  of  rehydration,  the  strip  was  submitted  to IEF  in  an  IPG-Phor  instrument  (Amersham  Biosciences, UK) using the following conditions: 500 V for 500 Vh in a gradient ramp, 3500 V for 3500 Vh in a gradient ramp and 8000 V for 40000 Vh in one step. The IEF was performed at 20 C and the current limit was set to 75 o m A per strip.

After  focusing,  the  IPG  strips  were  submitted  to  reduction for 20 min in a solution containing 8% (v/v) glycerol, 6 mol/L urea, 65 mmol/L DTT, 50 mmol/L Tris and 3% (v/v) SDS and then applied  to  the  second  dimension  [32].  SDSPAGE was performed on 12% T polyacrylamide gels run on a  Protean  II  system  (Bio-Rad,  Hercules,  CA,  USA)  connected  to  a  Multitemp  II  cooling  bath  (Amersham  Biosciences).  Electrophoresis  was  carried  out  at  5  mA  per  gel and constant current at 15 C until the dye front reached the o lower end of the gel [33]. Proteins were visualized by silverstaining.  Specifically,  gels  were  fixed  in  12%  (w/v)  TCA  , 50% (v/v) methanol for 1h followed by another 1 h incubation in 5% (v/v) acetic acid, 50% (v/v) ethanol. After 2 x 5 min washes in 10% (v/v) ethanol, the gels were incubated in a solution containing 0.89% (w/v) AgNO3, 0.3% (v/v) NH3, 0.08% (v/v) NaOH for 20 min, followed by a 5 min wash in water.  Development  was  done  by  incubation  of  the  gel  in 0.017%  (w/v)  citric  acid  and  0.236%  (v/v)  formaldehyde. The reaction was stopped with 30% (v/v) ethanol, 7% (v/v) acetic acid, 2.5% (v/v) glycerol.

## 2.3. Image Analysis

Silver-stained gels were scanned with a SHARP JX-330 scanner (Tokyo, Japan) at 300 dpi resolution and the tiff images  generated  were  analyzed  with  Image  Master  2D  Elite software (Amersham Biosciences, UK). The authenticity and outline of each spot were validated by visual inspection and edited  where  necessary.  Average  virtual  gels  were  created using all of the 14 samples. The volume values for each spot were  normalized  against  the  total  intensity  of  the  detected spots. The individual gels were compared to each other and only those spots present in all gels were considered for further analysis. Proteins showing a difference of less than 20% in their spot volume among the analyzed gels were assigned as conserved in the normal population and subjected to identification.

## 2.4. Peptide Mass Fingerprinting

The selected spots were excised and destained for 15 min with  a  freshly  prepared  solution  of  15  mmol/L  potassium ferricyanide,  50  mmol/L  sodium  thiosulphate  [34].  The  gel pieces were then rinsed several times with water, treated for 10  min  in  100  mmol/L  NH4CO3  and  then  in  acetonitrile (ACN), vacuum dried, incubated with 10 mmol/L DTT for 1h at 56 o C and subsequently with 55 mmol/L iodoacetamide for  45  min  at  room  temperature,  in  a  dark  vial.  Then  the samples were subjected to alternate 10 min washing cycles with water and ACN and vacuum dried. Gels were reswollen in 5 m L of 50 mmol/L NH4CO3, 5 mmol/L CaCl2 containing 12.5 ng/ m L sequencing grade modified porcine trypsin (Promega, Madison, WI, USA) and incubated at 37 C overo

night. The tryptic peptides were extracted twice with 40 m L of ACN/water/TFA (66:33.9:0.1) solution in a sonicator for 10  min.  The  extracts  were  dried  under  vacuum,  and  then solubilized in 4 m L 0.1% (v/v) TFA [35].

The resulting peptide mixture was analyzed in a MALDITOF  mass  spectrometer (Reflex IV; Bruker Daltonics, Karlsruhe, Germany). Each sample (1 m L) was spotted onto the  sample  plate  followed  by  1 m L  of  matrix  solution  (10 mg/mL a -cyano-4-hydroxycinnamic acid in 0.1%TFA, 50% ACN). The samples were allowed to dry prior to MALDITOF MS. The instrument was used with delayed extraction in reflector mode. Peptide masses (M + H ) in the range of + 800 to 3000 Da were measured. Masses of trypsin peptides occurring at m/z 842.50 and 2211.09 were used for internal calibration.

## 2.5. Database Searching

Protein  identification  was  performed  using  MASCOT software [36]. The peptide mass lists were searched against the NCBI protein database (National Center for Biotechnology  Information,  Bethesda,  USA).  The  mass  tolerance  allowed  was  between  50Ð100  ppm  and  the  protein  mass  restriction values were set according to the Mr values experimentally  determined  for  each  2-DE  spot.  The  search  parameters  were  restricted  to  human  and  allowed  one  missed cleavage, oxidation of methionine, propionamidomethylation and  carbamidomethylation  of  cysteines.  Spots  that  did  not yield significant identification using such settings were further  tested  for  post-translational  modifications  such  as  Nterminal  acetylation  and  pyroglutamic  acid.  The  statistical tools provided by MASCOT as well as the characteristics of the candidate proteins returned by the software were used for result interpretation and protein identification. Only MOWSE scores greater than 63 were considered as significant and at least four matching peptides were necessary for a positive identification.

## 3. RESULTS

The protocols for cell separation, protein extraction and 2D electrophoresis were optimized to allow the detection of up to 1800 spots in neutrophil gels. Compared to other protocols  in  the  literature,  a  few  optimization  steps  were  performed  to  obtain  highly  purified  viable  neutrophils  and  to avoid  activation  or  proteolysis.  The  major  modifications were the use of Ficoll-Paque Plus (Amersham Biosciences, UK), the preparation under refrigeration, the increased washing steps and the use of Teflon vials to hold the cells in all  separation  steps.  The  use  of  urea  and  thiourea  as  chaotropes  and  Triton  X-100  as  detergent  improved  the  2-DE resolution and increased the number of detected spots (data not shown).

Since  neutrophils  contain  a  large  number  of  proteases even in resting state, the addition of protease inhibitors to the lysis buffer was essential as well as decreasing the time used for cell separation to a maximum of 40 minutes from blood collection to cell lysis, leading to a noticeable difference in the  2D  patterns.  The  IEF  steps  were  performed  at  gradient voltages to allow a better focalization without precipitation. On  the  other  hand,  the  alkylation  with  iodoacetamide  was

performed only after spot excision avoiding vertical streaking in some positions of the gel. (Figs. 1A and 1B ).

After standardization, gels were run for all the collected samples  in  the  pH  range  of  4  to  7,  submitted  to  computational comparative analysis and selection of conserved spots. A typical gel is shown in Fig. 1B .  The majority of the proteins focused between pH 4.5 and 6.2, divided in three main molecular mass ranges around 55 kDa, 35 kDa and 15 kDa. To  allow  a  better  individualization  of  low  mass  proteins, different PAGE run times and staining times were used (Fig. 1C ).

<!-- image -->

<!-- image -->

Figure 1. Protein separation optimization. (A) Effect of alkylation before SDS-PAGE and usage of step voltages during focalization producing horizontal and vertical streaking. (B) Optimized result obtained by the use of protease inhibitors and no iodoacetamide alkylation. The vertical lines denote the pI range containing the majority of the proteins. The circles show concentration of proteins in three mass regions. (C) Low molecular mass range of two different gels showing optimization of runtime and staining for this specific gel area. C1 shows a gel that ran 15 minutes after the front marker left the gel, 45 minutes more than the gel in C2. The gel shown in C1 was also stained for 30 seconds longer than the one in C2.

<!-- image -->

One hundred and two spots present in all 14 samples individually analyzed, showing a variability of less than 20% in  volume  are  shown  in  Fig. 2 .  The  identification  of  22  of them is detailed in Table 1 .

Figure 2. Resting human neutrophil 2-DE map. The outlined spots were  matched  to  all  gels  and  presented  a  variability  of  less  than 20%. The arrows indicate spots identified by peptide mass fingerprinting.

<!-- image -->

The  2D  reference  map  generated  is  part  of  an  ongoing project in which other proteins are being identified and will be added to an annotated human neutrophils proteome database that is being built in our group. A complete and updated list of identified proteins can be found at the CBSP database (www.lbqp.unb.br/neutrophil)

## 4. DISCUSSION

A brief review of the literature shows that 2D-PAGE and proteomic  approaches  to  analyze  neutrophil  proteins  have been  performed  since  1981,  when  Watt  and  Burgess  [37] published  electrophoretic  profiles  of  proteins  from  mouse

peritoneal exudate, where they focus on differences of proteins  not  synthesized  by  mature  neutrophils.  Most  studies using  a  similar  approach  analyzed  proteins  differentially expressed  in  either  pathological  conditions  or  between  different  cell  types  [38-43].  More  recently,  other  comparative studies  [44-47]  analyzed  the  neutrophil  response  to  LPS stimulation and the myeloid differentiation program, using a genomic and proteomic approach. The reports from Fessler et al. [45,46] describes a poor concordance between mRNA transcription  and  protein  expression  and  reports  1200  spots with  a  high  variability  among  individuals,  since  only  125 spots were matched in all analyzed gels. The small number of  conserved  proteins  is  in  agreement  with  the  results  presented in this work, where only 102 proteins had low variability in the analyzed population.

The  publication  of  the  proteome  of  rat  polymorphonuclear  leukocytes  by  Piubelli  et  al  in  2002  [48]  shows  the identification  of  52  protein  spots  in  good  quality  gels,  although not showing a large number of spots, probably due to the non-silver staining method that favors mass spectrometric identification in detriment of sensitivity..

All of the previous reports are of major importance. Even those  published  when  current  technological  methods  and databases  were  not  available  showed  impressive  results. Even though, it has not been detected in the reviewed literature a consensus map relating the 2-DE images to the human neutrophil proteins, allowing access to a public database for pattern comparison.

The present report  describes  the  first  2D  reference  map for  the  human  neutrophil.  The  establishment  of  a  reference map for the resting human neutrophil may contribute to further studies. The map will allow future comparisons between primed  and  activated  states.  Furthermore,  it  may  provide new information for comparative studies concerning pathological  states,  leading  to  the  identification  of  diagnostic  or therapeutic targets to more efficiently control the neutrophil functions.

Compared  to  other  2D  electrophoretic  images  already published,  a  few  differences  are  observed  in  the  results  of this  study,  such  as  a  larger  number  of  detected  proteins, which may be due to a more sensitive staining method. Since one of the aims of this study was to detect the largest number

Table 1. Neutrophil Proteins Identified by Peptide Mass Fingerprinting

|   Spot | Accession number   | Name                             |    pI |   Mr (kDa) |   a) Score |   b) C (%) |   Matched peptides |
|--------|--------------------|----------------------------------|-------|------------|------------|------------|--------------------|
|      1 | gi|21221094        | putative serine protease         |  4.78 |        114 |        460 |         47 |                 38 |
|     24 | Q96CS1             | hypothetical protein             |  6.46 |         45 |        118 |         51 |                 15 |
|     25 | gi|20380411        | hypothetical protein             |  6.16 |         60 |        106 |         30 |                 12 |
|     29 | gi|7020414         | unnamed protein                  |  5.24 |         49 |         74 |         35 |                  8 |
|     31 | gi|1695876         | Interleukin-13 receptor          |  5.6  |         49 |         65 |         22 |                  6 |
|     44 | gi|30157637        | phosphoglycerate mutase          |  8.72 |         39 |         76 |         26 |                  7 |
|     45 | gi|11342670        | azurocidin 1 pre-protein         |  9.75 |         27 |         77 |         55 |                  7 |
|     48 | gi|4557293         | autoimmune regulator AIRE        |  6.36 |         36 |         79 |         43 |                 12 |
|     71 | gi|30159334        | ARP2 actin-related protein       |  8.54 |         17 |         70 |         30 |                  7 |
|     78 | gi|4827034         | tumor necrosis factor            |  7.63 |         20 |         73 |         52 |                 11 |
|     79 | gi|3970971         | supported protein of EST         |  4.89 |         19 |         64 |         50 |                  7 |
|     81 | gi|21739673        | hypothetical protein             |  5.04 |         18 |         68 |         42 |                  7 |
|     82 | gi|346339          | pur beta human protein           |  4.96 |         12 |         74 |         39 |                  7 |
|     85 | gi|438069          | thiol antioxidant protein        |  6.84 |         22 |         73 |         58 |                  9 |
|     88 | gi|29739484        | cytoskeletal keratin             |  7.63 |         18 |         79 |         36 |                 13 |
|     89 | gi|7657092         | interleukin-1 protein            |  5.89 |         18 |         75 |         47 |                  9 |
|    114 | gi|11359883        | hypothetical protein             |  4.45 |         55 |        248 |         45 |                 25 |
|    115 | gi|24211557        | chloride channel protein 5       |  4.69 |         46 |        403 |         84 |                 39 |
|    116 | gi|23490080        | hypothetical protein             | 10.9  |         10 |         86 |         40 |                  7 |
|    121 | gi|17438583        | nucleophosmin protein            |  4.54 |         28 |        359 |         90 |                 28 |
|    129 | gi|13477275        | GLUD-1 glutamate dehy- drogenase |  6.71 |         29 |        200 |         80 |                 19 |
|    134 | gi|14278516        | collagenase-3 protein            |  5.16 |         19 |        128 |         38 |                  5 |

a) Score: Mowse score using the Mascot algorithm against the NCBI non redundant database.

b) C (%): percentage of sequence coverage

of proteins in a whole cell analysis, detection was favored in the staining protocols adopted, compromising protein identification. Also the presence of a number of proteins focusing below pH 4.5 is unusual in the reported images of neutrophil proteome. Another finding of this study is the large number of proteins at both limits of molecular mass, since only Kasper et al. [41] presented a gel with more proteins at the high mass range in comparison to other publications. The development  of  an  optimized  methodology  involving  efficient protease inhibition, protein extraction and solubilization may account for these differences.

Up to the submission of this article, twenty two proteins were  identified  (Table 1 ).  Among  these  proteins,  nine  are related  to  the  inflammatory  process,  acting  as  cytokines, proteases,  vasoactive  compounds,  or  protecting  the  neutrophil  against  oxidative  damage.  To  two  polypeptides  are  assigned  metabolic  functions  and  five  ones  are  classified  as structural  or  housekeeping.  Although  the  human  genome  is well characterized, a large number of proteins still remain as hypothetical.

Among those  related  to  the  inflammatory  response,  the serine protease described did not present similarity to serine

proteases found in Homo sapiens , suggesting a different kind of protease in the neutrophil, as revealed by the domain Pep-tidase\_S8 in an Harvester [49] search, showing similarity to human peptidases. The interleukin-13 receptor was identified for the first time in the neutrophil, although strong evidence of its presence had already been published [40]. This identification  also  demonstrates that the methodology is efficient for detection of membrane proteins. The detection of azurocidin,  a  protein  with  antimicrobial  and  chemotactic  activity and a trypsin domain, although proteolytic activity is absent probably due to mutations in the catalytic site [50] was already  expected  in  the  neutrophil.  Actin  related  protein  2 (ARP2) is  part  of  the  ARP2/3  complex,  located  at  the  cell surface and related to dynamic actin filament assembly, being related to the adhesion and migration processes [51]. The TNF receptor identified was also not specifically  described in  the  neutrophil,  although  Gurney et  al. [52]  describe  the presence of the mRNA encoding for this protein in peripheral  leukocytes.  The  antioxidant  protein  peroxiredoxin  2  is assigned to the protection of the cell against its own peroxides and also participates in the cytokines signaling cascades [53].  This  protein  was  described  in  many  different  tissues and  cells (keratinocytes, erythrocytes, colon  carcinoma), having been identified in most of them by proteomic analysis and described here for the first time in neutrophils. The epsilon member of IL-1 family, that activates NF-kappaB by a different way than IL-1 alpha and beta, was also identified in the  neutrophil.  The  collagenase  identified  here  was  not  the expected  neutrophil  collagenase,  but  probably  an  isoform, since  the  assigned  sequence  shares  61%  identity  with  neutrophil  collagenase.  The  autoimmune  regulator  (AIRE),  although classified as a transcription factor, may be involved in immune regulation. This is also the first description of this protein in the neutrophil.

Herein we presented standard procedures to obtain reproducible neutrophil 2D-PAGE profiles and identified 22 proteins as a first step leading to the reference map of the resting human neutrophil, which is available over the internet. Proteins  showing  activity  related  to  the  inflammatory  process, some of them described for the first time in the neutrophil, were  emphasized.  Certainly,  further  studies  involving  the analysis of basic proteins and the use of narrow pH gradient gels will be needed to complete the reference map.

## ACKNOWLEDGMENTS

We thank Dr.  P.  M.  Larsen,  Dr.  S.  Fey  and  Dr.  V.  M. Larsen  (Center  for  Proteome  Analysis,  Odense,  Denmark) for  providing  the  silver  staining  protocol  used  in  this  work and for helpful discussion, Dr. Renato S. Poggetti, Dr. Dario Birolini (University of S‹o Paulo, Brazil) and Dr. M.H. Lima for  the  use  of  their  facilities  and  Dr.  Patr'cia  Carvalho  for language  reviewing.  This  study  was  supported  by  PADCT, CNPq, FINATEC, PIBIC and FUB/UnB.

## REFERENCES

- [1] Gay, J. C., ( 1993 ) J. Cell Physiol , 156 , 1, 189-197
- [2] Swain, S. D., Rohn, T. T., and Quinn, M. T., ( 2002 ) Antioxid Redox Signal , 4 , 1, 69-83
- [3] Hollmann,  M.  W.,  Gross,  A.,  Jelacin,  N.,  and  Durieux,  M.  E., ( 2001 ) Anesthesiology , 95 , 1, 113-122
- [4] Zhang, B., Hirahashi, J., Cullere, X., and Mayadas, T. N., ( 2003 ) J.Biol.Chem. , 278 , 31, 28443-28454
- [5] Wagner,  J.  G.  and  Roth,  R.  A.,  ( 2000 ) Pharmacol..  Rev. , 52 ,  3, 349-374
- [6] Suratt, B. T., Young, S. K., Lieber, J., Nick, J. A., Henson, P. M., and Worthen, G. S., ( 2001 ) Am. J. Physiol Lung Cell Mol. Physiol , 281 , 4, L913-L921
- [7] Mitsui,  T.,  Watanabe,  S.,  Taniguchi,  Y.,  Hanada,  S.,  Ebihara,  Y., Sato,  T.,  Heike,  T.,  Mitsuyama,  M.,  Nakahata,  T.,  and  Tsuji,  K., ( 2003 ) Blood , 101 , 8, 2990-2995
- [8] Kawai, S., ( 2000 ) Rinsho Byori , 48 , 6, 516-20
- [9] Abraham, E., ( 2003 ) Crit Care Med. , 31 , 4 Suppl, S195-S199
- [10] Gabrijelcic, J., Acuna, A., Profita, M., Paterno, A., Chung, K. F., Vignola, A. M., and Rodriguez-Roisin, R., ( 2003 ) Eur. Respir. J. , 22 , 2, 290-297
- [11] Gujral,  J.  S.,  Farhood,  A.,  Bajt,  M.  L.,  and  Jaeschke,  H.,  ( 2003 ) Hepatology , 38 , 2, 355-363
- [12] Cabral, V. L., Miszputen, S. J., and Catapani, W. R., ( 2003 ) Hepatogastroenterology , 50 , 50, 412-415
- [13] Desaulniers,  P.,  Fernandes,  M.,  Gilbert,  C.,  Bourgoin,  S.  G.,  and Naccache, P. H., ( 2001 ) J.Leukoc.Biol. , 70 , 4, 659-668
- [14] Ottonello, L., Frumento, G., Arduino, N., Bertolotto, M., Mancini, M., Sottofattori, E., Dallegri, F., and Cutolo, M., ( 2002 ) Ann.N.Y.Acad.Sci. , 966 , 226-231
- [15] Kobayashi, M., Ito, M., Nakagawa, A., Matsushita, M., Nishikimi, N., Sakurai, T., and Nimura, Y., ( 2000 ) Histopathology , 36 , 4, 362371
- [16] Peveri,  P.,  Walz,  A.,  Dewald,  B.,  and  Baggiolini,  M.,  ( 1988 ) J.Exp.Med. , 167 , 5, 1547-1559
- [17] DiVietro, J. A., Smith, M. J., Smith, B. R., Petruzzelli, L., Larson, R. S., and Lawrence, M. B., ( 2001 ) J.Immunol. , 167 , 7, 4017-4025
- [18] Ley, K., ( 2002 ) Immunol.Rev. , 186 , 8-18
- [19] Lindbom, L. and Werr, J., ( 2002 ) Semin.Immunol. , 14 , 2, 115-121
- [20] Allport, J. R., Ding, H. T., Ager, A., Steeber, D. A., Tedder, T. F., and Luscinskas, F. W., ( 1997 ) J.Immunol. , 158 , 9, 4365-4372
- [21] Shang, T., Yednock, T., and Issekutz, A. C., ( 1999 ) J.Leukoc.Biol. , 66 , 5, 809-816
- [22] Condliffe, A. M., Kitchen, E., and Chilvers, E. R., ( 1998 ) Clin.Sci.(Lond) , 94 , 5, 461-471
- [23] Hofman, P., ( 2004 ) Curr.Drug Targets.Inflamm.Allergy , 3 , 1, 1-9
- [24] Avila, P. C., Boushey, H. A., Wong, H., Grundland, H., Liu, J., and Fahy, J. V., ( 2004 ) Clin.Exp.Allergy , 34 , 1, 77-84
- [25] Vanderslice,  P.,  Biediger,  R.  J.,  Woodside,  D.  G.,  Berens,  K.  L., Holland, G. W., and Dixon, R. A., ( 2004 ) Pulm.Pharmacol.Ther. , 17 , 1, 1-10
- [26] Chavakis, T., Athanasopoulos, A., Rhee, J. S., Orlova, V., SchmidtWoll, T., Bierhaus, A., May, A. E., Celik, I. P., Nawroth, P. P., and Preissner, K. T., ( 2004 ) Blood ,
- [27] Xia, Y. F., Ye, B. Q., Li, Y. D., Wang, J. G., He, X. J., Lin, X., Yao,  X.,  Ma,  D.,  Slungaard,  A.,  Hebbel,  R.  P.,  Key,  N.  S.,  and Geng, J. G., ( 2004 ) J.Immunol. , 173 , 6, 4207-4217
- [28] Ferrante, A. and Thong, Y. H., ( 1978 ) J Immunol Methods , 24 , 3-4, 389-93
- [29] Glasser, L. and Fiederlein, R. L., ( 1990 ) Am J Clin Pathol , 93 , 5, 662-9
- [30] Ramagli,  L.  S.  and  Rodriguez,  L.  V.,  ( 1985 ) Electrophoresis , 6 , 559-563
- [31] Sanchez, J. C., Hochstrasser, D., and Rabilloud, T., ( 1999 ) Methods Mol.Biol. , 112 , 221-225
- [32] Gorg,  A.,  Obermaier,  C.,  Boguth,  G.,  Harder,  A.,  Scheibe,  B., Wildgruber,  R.,  and  Weiss,  W.,  ( 2000 ) Electrophoresis , 21 , 6, 1037-53
- [33] Westermeier,  R.  and  Naven,  T.(2002)  Proteomics  in  Practice  :  A Laboratory Manual of Proteome Analysis, ISBN: 3527303545
- [34] Gharahdaghi, F., Weinberg, C. R., Meagher, D. A., Imai, B. S., and Mische, S. M., ( 1999 ) Electrophoresis , 20 , 3, 601-605
- [35] Katayama,  H.,  Nagasu,  T.,  and  Oda,  Y.,  ( 2001 ) Rapid  Commun Mass Spectrom , 15 , 16, 1416-21
- [36] Perkins,  D.  N.,  Pappin,  D.  J.,  Creasy,  D.  M.,  and  Cottrell,  J.  S., ( 1999 ) Electrophoresis , 20 , 18, 3551-67
- [37] Watt,  S.  M.  and  Burgess,  A.  W.,  ( 1981 ) Biochim.Biophys.Acta , 640 , 2, 583-594
- [38] Gemmell, M. A. and Anderson, N. L., ( 1982 ) Clin.Chem. , 28 , 4 Pt 2, 1062-1066
- [39] Willard, K. E., Thorsrud, A. K., Munthe, E., and Jellum, E., ( 1982 ) Clin.Chem. , 28 , 4 Pt 2, 1067-1073
- [40] Girard,  D.,  Paquin,  R.,  Naccache,  P.  H.,  and  Beaulieu,  A.  D., ( 1996 ) J.Leukoc.Biol. , 59 , 3, 412-419

## Proteome Analysis of Resting Human Neutrophils

- [41] Kasper, B., Thole, H. H., Patterson, S. D., and Welte, K., ( 1997 ) Electrophoresis , 18 , 1, 142-149
- [42] Boussac, M. and Garin, J., ( 2000 ) Electrophoresis , 21 , 3, 665-72
- [43] Yamashita, S., Suzuki, A., Yanagita, T., Hirohata, S., Kamada, M., and Toyoshima, S., ( 2000 ) Biol.Pharm.Bull. , 23 , 5, 519-522
- [44] Lian,  Z.,  Wang,  L.,  Yamaga,  S.,  Bonds,  W.,  Beazer-Barclay,  Y., Kluger,  Y.,  Gerstein,  M.,  Newburger,  P.  E.,  Berliner,  N.,  and Weissman, S. M., ( 2001 ) Blood , 98 , 3, 513-24
- [45] Fessler, M. B., Malcolm, K. C., Duncan, M. W., and Worthen, G. S., ( 2002 ) J Biol Chem , 277 , 35, 31291-302
- [46] Fessler, M. B., Malcolm, K. C., Duncan, M. W., and Worthen, G. S., ( 2002 ) Chest , 121 , 3 Suppl, 75S-76S
- [47] Singh, S., Powell, D. W., Rane, M. J., Millard, T. H., Trent, J. O., Pierce, W. M., Klein, J. B., Machesky, L. M., and McLeish, K. R., ( 2003 ) J.Biol.Chem. , 278 , 38, 36410-36417

Received: ?????????????

Revised: ?????????????

Accepted: ????????????

- [48] Piubelli, C., Galvani, M., Hamdan, M., Domenici, E., and Righetti, P. G., ( 2002 ) Electrophoresis , 23 , 2, 298-310
- [49] Liebel, U., Kindler, B., and Pepperkok, R., ( 2004 ) Bioinformatics. ,
- [50] Morgan,  J.  G.,  Sukiennicki,  T.,  Pereira,  H.  A.,  Spitznagel,  J.  K., Guerra, M. E., and Larrick, J. W., ( 1991 ) J.Immunol. , 147 , 9, 32103214
- [51] Anderson, S. I., Behrendt, B., Machesky, L. M., Insall, R. H., and Nash, G. B., ( 2003 ) Cell Motil.Cytoskeleton , 54 , 2, 135-146
- [52] Gurney, A. L., Marsters, S. A., Huang, R. M., Pitti, R. M., Mark, D.  T.,  Baldwin,  D.  T.,  Gray,  A.  M.,  Dowd,  A.  D.,  Brush,  A.  D., Heldens,  A.  D.,  Schow,  A.  D.,  Goddard,  A.  D.,  Wood,  W.  I., Baker, K. P., Godowski, P. J., and Ashkenazi, A., ( 1999 ) Curr.Biol. , 9 , 4, 215-218
- [53] Rabilloud, T., Heller, M., Gasnier, F., Luche, S., Rey, C., Aebersold,  R.,  Benahmed,  M.,  Louisot,  P.,  and  Lunardi,  J.,  ( 2002 ) J.Biol.Chem. , 277 , 22, 19396-19401