<!-- image -->

<!-- image -->

of June 30, 2021. This information is current as

Reimagine your discoveries Amnis' ImageStream * Mk Il and FlowSight" Imaging Flow Cytometers

<!-- image -->

## Membranes of Human Neutrophils Secretory Vesicle Membranes and Plasma Comparison of Proteins Expressed on

Jog, Richard A. Ward and Kenneth R. McLeish Michael L. Merchant, Timothy D. Cummins, Neelakshi R. Silvia M. Uriarte, David W. Powell, Gregory C. Luerman,

http://www.jimmunol.org/content/180/8/5575 doi: 10.4049/jimmunol.180.8.5575 2008; 180:5575-5581; ; J Immunol

http://www.jimmunol.org/content/suppl/2008/04/01/180.8.5575.DC1

, 25 of which you can access for free at: cites 44 articles This article

Material Supplementary

References

http://www.jimmunol.org/content/180/8/5575.full#ref-list-1

Submit online. ?

The JI Why

- from submission to initial decision Rapid Reviews! 30 days* ¬∑
- Every submission reviewed by practicing scientists No Triage! ¬∑
- 4 weeks from acceptance to publication Fast Publication! ¬∑

average *

Subscription

http://jimmunol.org/subscription is online at: The Journal of Immunology Information about subscribing to

Permissions

http://www.aai.org/About/Publications/JI/copyright.html

Submit copyright permission requests at:

Email Alerts

http://jimmunol.org/alerts Receive free email-alerts when new articles cite this article. Sign up at:

<!-- image -->

## Comparison of Proteins Expressed on Secretory Vesicle Membranes and Plasma Membranes of Human Neutrophils 1

Silvia M. Uriarte,* David W. Powell,* ‚Ä† Gregory C. Luerman, ‚Ä† Michael L. Merchant,* Timothy D. Cummins,* Neelakshi R. Jog, ‚Ä° Richard A. Ward,* and Kenneth R. McLeish * 2 ‚Ä†¬ß

Secretory vesicles are neutrophil intracellular storage granules formed by endocytosis. Understanding the functional consequences of secretory vesicle exocytosis requires knowledge of their membrane proteins. The current study was designed to use proteomic technologies to develop a more complete catalog of secretory vesicle membrane proteins and to compare the proteomes of secretory vesicle and plasma membranes. A total of 1118 proteins were identified, 573 (51%) were present only in plasma membraneenriched fractions, 418 (37%) only in secretory vesicle-enriched membrane fractions, and 127 (11%) in both fractions. Gene Ontology categorized 373 of these proteins as integral membrane proteins. Proteins typically associated with other intracellular organelles, including nuclei, mitochondria, and ribosomes, were identified in both membrane fractions. Ingenuity Pathway Knowledge Base analysis determined that the majority of canonical and functional pathways were significantly associated with proteins from both plasma membrane-enriched and secretory vesicle-enriched fractions. There were, however, some canonical signaling pathways that involved proteins only from plasma membranes or secretory vesicles. In conclusion, a number of proteins were identified that may elucidate mechanisms and functional consequences of secretory vesicle exocytosis. The small number of common proteins suggests that the hypothesis that secretory vesicles are formed from plasma membranes by endocytosis requires more critical evaluation. The Journal of Immunology, 2008, 180: 5575-5581.

C ofphenotypic changes that result in their transition from irculating neutrophils are capable of undergoing a series cells that are poorly responsive to proinflammatory stimuli to become the primary effector cells of innate immunity. These phenotypic changes involve the incorporation of proteins from the membranes of intracellular storage granules into the plasma membrane and the release of proteins stored in granule matrix through regulated exocytosis. Granule exocytosis contributes to enhanced neutrophil tethering and adhesion to vascular endothelial cells at a site of inflammation; enhanced migration across blood vessel walls; chemotaxis to a site of microbial invasion; phagocytosis of invading organisms; and microbicidal activity through a combination of enzymatic degradation, reactive oxygen species generation, and release of microbicidal peptides into phagosomes. Neutrophils contain a heterologous group of storage granules that have been classified into four subsets based on density and composition: azurophil (primary) granules, specific (secondary) granules, gelatinase (tertiary) granules, and secretory vesicles (1). These granule subsets undergo hierarchical stimulated exocytosis, with secretory vesicles the most easily and completely mobilized (2-4). Gelatinase, specific, and azurophil granules are formed from the trans -

Golgi network during neutrophil maturation (1). The intragranule constituents of secretory vesicles include plasma proteins, resulting in the hypothesis that secretory vesicles are formed by endocytosis and that functional changes from their exocytosis are due entirely to incorporation of new molecules into the plasma membrane (1, 5). Thus, to understand the changes in neutrophil functional capability induced by secretory vesicle exocytosis, a comprehensive catalog of membrane proteins is required.

Proteomic techniques, which include methods for protein extraction and separation, protein identification and characterization, and database analysis, provide an unbiased approach to identifying proteins expressed in subcellular compartments (6-9). We recently published a comprehensive proteomic analysis of neutrophil gelatinase, specific, and azurophil granules (10). Using protein separation by two-dimensional gel electrophoresis and two-dimensional HPLC coupled with MALDI-TOF-MS 3 and ESI-MS/MS, 286 proteins were identified on one or more granule subsets, many of which had not been found previously on neutrophil granules. The current study was designed to use similar proteomic technologies to provide a more complete identification of secretory vesicle membrane proteins and to compare those proteins with the proteins expressed on neutrophil plasma membranes. The ability to extract and solubilize membrane proteins is a major limitation to all proteomic approaches. To overcome this limitation, proteins were extracted from membranes using a recently described methanol extraction procedure, followed by two-dimensional HPLC and ESI-MS/MS (11). With this approach, we identified a number of membrane spanning and membrane associated proteins and uncovered significant differences between secretory vesicle-enriched and plasma membrane-enriched proteomes.

## Materials and Methods

Neutrophil isolation

Neutrophils were isolated from healthy donors using plasma-Percoll gradients as described by Haslett et al. (12). Trypan blue staining revealed that at least 97% of cells were neutrophils with &gt; 95% viability. After isolation, neutrophils were suspended in Krebs-Ringer phosphate buffer (pH 7.2) at 4 √ó 10 7 cells/ml and treated with 5 mM diisopropyl fluorophosphate for 10 min on ice to inhibit proteases (13). Extensive evaluation using respiratory burst activity and granule exocytosis indicates that this isolation technique does not prime neutrophils. The Human Studies Committee of the University of Louisville approved the use of human donors.

## Plasma membrane and secretory vesicle membrane enrichment

Neutrophil plasma membrane and secretory vesicle membranes were enriched by centrifugation on a two-layer Percoll density gradient as described by Dahlgren et al. (14). Briefly, isolated neutrophils from single donors (4 √ó 10 7 /ml) were incubated with diisopropyl fluorophosphate, pelleted by centrifugation, and resuspended in disruption buffer containing 100 mM KCl, 1 mM NaCl, 1 mMATPNa2, 3.5 mM MgCl2, 10 mM PIPES, and 0.5 mM PMSF. Cells were disrupted by nitrogen cavitation at 380 p.s.i. and 4¬°C. The cavitate was collected, supplemented with 1.5 mM EGTA, and nuclei and intact cells were removed by centrifugation at 400 √ó g for 5 min. The supernatant membrane suspension was aspirated, placed in a 50-ml conical centrifuge tube, and mixed with an equal volume of a 1.12 g/ml Percoll gradient. A total of 10 ml of the membrane suspension/Percoll gradient was layered under 5 ml of disruption buffer in a 50-ml ultracentrifuge tube. A total of 10 ml of 1.05 g/ml Percoll gradient solution was layered under the membrane suspension, then 5 ml of 1.12 g/ml Percoll gradient solution layered under the 1.05 g/ml solution. The gradient was centrifuged at 37,000 √ó g in a SS-34 fixed angle rotor in a Sorvall RC-5B centrifuge for 30 min at 4¬°C. Following centrifugation, successive 1.5-ml fractions were collected from the top of the gradient.

Protein extraction from plasma membrane and secretory vesicle membranes

Fractions obtained from Percoll gradients were analyzed for alkaline phosphatase in the absence (nonlatent) or presence (latent) of Triton X-100 using p -nitrophenylphosphate as substrate. Briefly, 100:l aliquots of each fraction were added to wells of a 96 flat-well plate with or without 0.3% Triton X-100. Reactions were started by adding 200 ùõç l of reaction buffer containing 5 mM p -nitrophenylphosphate in 100 mM 2-amino-2-methyl1-propanol (pH 10.0). Following incubation for 30 min at 37¬°C, reactions were stopped by addition of 150 ùõç l of 0.04 N NaOH and absorbance was read at 405 nm in a microplate reader. Fractions containing nonlatent, but not latent, alkaline phosphatase were pooled and analyzed as plasma membrane-enriched. Fractions containing latent alkaline phosphatase were pooled and analyzed as secretory vesicle-enriched membrane. Percoll was removed from membranes by ultracentrifugation at 100,000 √ó g for 90 min. Membrane pellets were resuspended in 50 mM ammonium bicarbonate, washed by centrifugation at 100,000 √ó g , then resuspended in 60% methanol in 100 mM ammonium bicarbonate to extract membrane proteins (11). Proteins were digested by incubation with 3 ùõç g each of trypsin and chymotrypsin at 37¬°C overnight. Following centrifugation at 100,000 √ó g for 20 min at 4¬°C, the supernatant was removed for peptide analysis. Peptides were lyophilized and prepared for mass spectrometry analysis using a desalting trap method. Briefly, peptides were resuspended in a 100 ùõç l of 5% acetonitrile and 0.05% formic acid and applied to a peptide microtrap (Michrom BioResources) equilibrated with 1 ml of the same buffer. The trap was then washed twice with 100 ùõç l of resuspension buffer and peptides eluted with 100 ùõç l of 95% acetonitrile, 0.05% formic acid. Eluted peptides were dried in a speed vacuum and resuspended in 5-10 ùõç l of 5% acetonitrile and 0.05% formic acid.

## 2D-LC-MS/MS and computer-assisted data analysis

A modified version of a previously described 2D-LC-MS/MS method was applied (15). Trypsin/Chymotrypsin-generated peptides were loaded onto an analytical 2D microcapillary chromatography column packed with 3-4 cm of 5 ùõç m(pore size) strong cation exchange (SCX) resin (Phenomenex) followed by 2-3 cm of 5 ùõç m(pore size) C18 reversed-phase (RP) resin (Phenomenex). This bi-phasic column was attached to an analytical RP chromatography column (100 √ó 365 ùõç m fused silica capillary with an integrated, laser pulled emitter tip packed with 10 cm of Synergi 4 ùõç m RP80A (Phenomenex)). Peptides were eluted from SCX with seven step gradients of 5, 10, 15, 30, 50, 70, and 100% 500 mM ammonium acetate. Following each SCX elution step, peptides were ionized and sprayed into the mass spectrometer using the following linear RP gradient: 20 min: 0% B, 80 min: 40% B, 90 min: and 60% B at a flow rate of 200 nl/min (mobile phase A: 5% acetonitrile/0.1% formic

acid and mobile phase B: 80% acetonitrile/0.1% formic acid). Spectra were acquired with a LTQ linear ion trap mass spectrometer (Thermo Fisher Scientific). During LC-MS/MS analysis, the mass spectrometer performed datadependent acquisition with a full MS scan between 300 and 2000 mass to change ratio followed by five MS/MS scans (35% collision energy) on the five most intense ions from the preceding MS scan. Data acquisition was performed using dynamic exclusion with a repeat count of 30 and a 1 and 3 min exclusion duration window.

## Mass spectral data interpretations

The acquired mass spectrometric data were searched against a human protein database (human RefSeq) using the Sequest algorithm and a commercial computational platform (SequestSorcerer, Sage-N Research) assuming modifications of oxidation of methionine ( + 15.99) and carbamidomethylation of cysteine ( + 57). High-probability protein identifications were assigned from the Sequest results using the BIGCAT (16) and ProteinProphet (17) statistical platforms (18, 19). Both of these programs eliminate redundant listing by grouping proteins with 100% identity and merging proteins with 100% shared MS/MS spectra. The BIGCAT filter uses Sequest Xcorr cut-offs of 1.5, 2, and 2.5 for + 1, + 2, and + 3 ions, respectively. Proteins were also ranked by relative abundance or enrichment using a protein abundance factor (PAF). The PAF is defined as the total number of nonredundant spectra (spectral counts) that correlate significantly to each respective candidate protein normalized to the protein's m.w. ( √ó 10 4 ). Studies demonstrating linearity between the number of spectral counts and protein concentration provide the framework for this type of label-free quantitative analysis from 2D-LC-MS/MS experiments (20-22). The PAF approach has been highly successful in the development of statistical models based on 2D-LC-MS/MS experimental data (23-25). ProteinProphet gives each protein a ranked probability score, with 1.0 being the highest probability. Our results were fitted into a model where probability scores decreasing from 1.0 were correlated to a predicted false positive identification error rate. Proteins with a probability score greater than 0.65 predicted a false positive error of &lt; 10%, and those proteins were included in the list of candidates.

The proteins were further analyzed using the National Center for Biotechnology Information (NCBI) database. Any proteins that were removed from the database in the process of annotation or that were identified as a bacterial protein were excluded from the final protein list. To identify integral membrane proteins and to determine intracellular location and function, identified proteins were analyzed using the NCBI database, by Gene Ontology (26), and by the Ingenuity Pathways Knowledge Base (IPKB) (Ingenuity Systems) (27). The IPBK is a comprehensive knowledge base of biological findings for genes of human, mouse, and rat origin, which is used to construct pathways and define biological functions (28). The canonical pathways are well-characterized metabolic and cell signaling pathways that have been curated from specific journal articles, review articles, text books, and KEGG Ligand. The functional analysis has three primary categories of functions: molecular and cellular functions; physiological system development and function; and diseases and disorders. There are 85 high-level functional categories that are classified under these categories. The significance value of a given canonical pathway or functional analysis category is a measurement of the likelihood that the pathway or function is associated with the data set by random chance. The value of p is calculated using the right-tailed Fisher Exact Test, and values of p &lt; 0.05 were a priori assumed to be statistically significant.

## Results

Neutrophils from four individual donors were subjected to membrane fractionation, and each set was analyzed separately by mass spectrometry. A total of 1118 proteins were identified, for which there was a &lt; 10% probability that the identification was by chance according to Protein Prophet, and which were confirmed to have mammalian homologues according to the NCBI database. Of the 1118 proteins, 573 were present only in plasma membrane-enriched fractions, 418 only in secretory vesicle-enriched fractions, and 127 in both fractions. The 1118 proteins, together with their probability score, the peptides used to identify them, the membrane fraction from which they were identified, and the Gene Ontology analysis of membrane association, cellular location, and function, are listed in supplemental data Tables I and II. 4

Table I. Distribution of proteins by membrane association and cell location a

| Location                     |   Membrane-Associated |   Membrane-Independent |   Membrane Association Unknown |   Total |
|------------------------------|-----------------------|------------------------|--------------------------------|---------|
| Cytoplasm                    |                     0 |                    151 |                              1 |     152 |
| Cytoskeleton                 |                     0 |                     65 |                              1 |      66 |
| Endoplasmic reticulum        |                    27 |                     13 |                              4 |      44 |
| Endosome/lysosome/peroxisome |                     9 |                      6 |                              1 |      16 |
| Extracellular                |                     0 |                     44 |                              0 |      44 |
| Golgi                        |                     6 |                      4 |                              3 |      13 |
| Mitochondria                 |                    24 |                     31 |                              4 |      59 |
| Nucleus                      |                     9 |                    196 |                              8 |     213 |
| Plasma membrane              |                   268 |                      0 |                              0 |     268 |
| Ribosome                     |                     0 |                     25 |                              0 |      25 |
| Secretory granule            |                    16 |                     17 |                              1 |      34 |
| Unknown                      |                    14 |                      0 |                            170 |     184 |
| Total                        |                   373 |                    552 |                            193 |    1118 |

a All identified proteins were categorized by Gene Ontology as integral membrane proteins (membrane-associated), as not associated with membranes (membrane-independent), and proteins whose membrane association was unknown and then listed by cellular location.

## Recovery of membrane-associated proteins

In the past, protein extraction methods used in proteomic analysis have provided poor identification of transmembrane proteins. To address this limitation, we used a recently described protein extraction method in which 60% methanol was added to 100 mM ammonium bicarbonate, as recently described by Fischer et al. (11). To determine the effectiveness of methanol extraction in the recovery of membrane proteins, each protein was analyzed for known association with cellular membranes using the Gene Ontology database. Of the 1118 proteins identified, 373 were categorized as integral membrane proteins, 552 were not associated with any cellular membranes, and the membrane association of 193 proteins was unknown (Table I). A number of proteins with single transmembrane spanning regions were identified, including Fc ùõÑ RIIa and IIIa, integrins ùõÇ 2b, ùõÇ 3, ùõÇ 4, ùõÇ D, ùõÇ M, ùõÇ X, ùõÉ 1, and ùõÉ 2; TLRs 2 and 8; and complement receptors 1 and 2. Numerous proteins with multiple transmembrane spanning regions were also identified, including seven transmembrane spanning receptors for leukotiene B , formyl peptides, IL-8, C5a, and a number of ion 4 channels and transporters. Table I also shows the cellular location of proteins that were integral to membranes, independent of membranes, or where membrane association was unknown. As expected, all 268 plasma membrane proteins were categorized as membrane-associated. The remainder of the membrane-associated

proteins were primarily localized to the endoplasmic reticulum (27), mitochondria (24), and secretory granules (16). The majority of proteins not associated with a cellular membrane were localized in the cytoplasm (151), nucleus (196), or with the cytoskeleton (65). The majority of proteins for which membrane association was unknown also had an unknown cellular location (169 of 193 proteins).

Cellular location of proteins identified in plasma membraneenriched and secretory vesicle-enriched membrane fractions

Table II shows the cellular location of proteins identified in plasma membrane-enriched and secretory vesicle-enriched fractions. Based on the Gene Ontology database, 25% of proteins were predicted to be from mitochondria, ribosomes, or nuclei. Of the remaining 821 proteins, over half were proteins known to be associated with the cytoskeleton or a cellular membrane compartment, including plasma membrane, secretory granules, endoplasmic reticulum, endosomes, or golgi. Thus, the fractions enriched for plasma membrane and secretory vesicle membranes also contain proteins from a number of other cellular compartments. Another recent analysis reported that proteins from endoplasmic reticulum, mitochondria, and golgi coisolated with neutrophil secretory vesicle-enriched membranes and enriched plasma membranes separated by free flow electrophoresis (29). Of the 1118 proteins identified, 51% were found in plasma membrane-enriched fractions,

Table II. Distribution of proteins by cell location a

| Location                     |   Plasma Membrane |   Secretory Vesicle |   Common to PM and SV |   Total |
|------------------------------|-------------------|---------------------|-----------------------|---------|
| Cytoplasm                    |                70 |                  68 |                    14 |     152 |
| Cytoskeleton                 |                30 |                  22 |                    14 |      66 |
| Endoplasmic reticulum        |                25 |                   9 |                    10 |      44 |
| Endosome/lysosome/peroxisome |                 4 |                   9 |                     3 |      16 |
| Extracellular                |                23 |                  16 |                     5 |      44 |
| Golgi                        |                 5 |                   7 |                     1 |      13 |
| Mitochondria                 |                33 |                  15 |                    11 |      59 |
| Nucleus                      |               113 |                  81 |                    19 |     213 |
| Plasma membrane              |               159 |                  76 |                    33 |     268 |
| Ribosome                     |                 1 |                  22 |                     2 |      25 |
| Secretory granule            |                10 |                  15 |                     9 |      34 |
| Unknown                      |               100 |                  78 |                     6 |     184 |
| Total                        |               573 |                 418 |                   127 |    1118 |

a All proteins identified were categorized by the membrane fraction from which they were identified, plasma membrane (PM), secretory vesicle (SV) membrane, or present in both membrane fractions (common to PM and SV). These proteins were then listed by cell location based on Gene Ontology analysis.

Table III. Distribution of proteins by functional category a

| Function                       |   Plasma Membrane |   Secretory Vesicle |   Common to PM and SV |   Total |
|--------------------------------|-------------------|---------------------|-----------------------|---------|
| Adhesion                       |                31 |                  14 |                     6 |      51 |
| Chaperone/protein folding      |                 7 |                   4 |                     6 |      17 |
| Chromosome organization        |                 1 |                   0 |                     0 |       1 |
| Cytoskeletal regulation        |                29 |                  27 |                    16 |      72 |
| Enzyme/metabolism              |                28 |                  23 |                     8 |      59 |
| GTPase                         |                13 |                  20 |                     5 |      38 |
| Immune/inflammatory response   |                 8 |                   4 |                     5 |      17 |
| Kinase/phosphatase             |                33 |                  15 |                     3 |      49 |
| Membrane trafficking           |                30 |                  22 |                     5 |      57 |
| Protein Modification/targeting |                 7 |                   1 |                     2 |      10 |
| Proteolysis                    |                23 |                  22 |                     5 |      50 |
| Receptor/signal transduction   |                49 |                  27 |                     7 |      83 |
| Redox                          |                 5 |                   0 |                     4 |       9 |
| Structural proteins            |                 2 |                   7 |                     2 |      11 |
| Translation/transcription      |                 9 |                   8 |                     2 |      19 |
| Transport                      |                33 |                  16 |                     3 |      52 |
| Miscellaneous                  |                12 |                  11 |                     5 |      28 |
| Unknown                        |               108 |                  79 |                    11 |     198 |
| Total                          |               426 |                 300 |                    95 |     821 |

a Proteins, excluding those identified as nuclear, mitochondrial, or ribosomal, were categorized by the membrane fraction from which they were identified, plasma membrane (PM), secretory vesicle (SV) membrane, or present in both membrane fractions (common to PM and SV). These proteins were then listed by cellular function based on Gene Ontology analysis.

37% in secretory vesicle-enriched fractions, and 11% were found in both fractions. When proteins were analyzed according to their cellular location, several patterns were observed. Proteins localized to the endoplasmic reticulum, extracellular space, mitochondria, nucleus, plasma membrane, and whose location was unknown were more likely to be present in plasma membrane-enriched fractions. Proteins localized from endosome/lysosome/peroxisome compartments, golgi, ribosomes, and secretory granules were more likely to be present in secretory vesicle-enriched fractions. Proteins defined as cytoplasmic or cytoskeletal were fairly equally distributed between plasma membrane-enriched and secretory vesicleenriched fractions. Surprisingly, the number of proteins present in both fractions was low for all Gene Ontology cell locations, ranging from 3% for proteins of unknown location to 26% for proteins in secretory granules. These results indicate that there was effective separation of cellular membranes based on their density, and the data suggest that proteins from all cellular compartments segregate into distinct membrane compartments of different densities.

## Functional categorization of proteins

To obtain a more accurate assessment of the functions of proteins most likely associated with plasma membrane and secretory ves-

icle, proteins localized to mitochondria, nuclei, and ribosomes were eliminated from the analysis. All proteins identified from plasma membrane-enriched fractions and secretory vesicle-enriched fractions, however, are listed in supplementary Table II. Table III shows the distribution of the remaining 821 proteins from plasma membrane-enriched and secretory vesicle-enriched fractions according to protein function. Functional categories were assigned based on analysis by the Gene Ontology database. Where multiple functions were assigned to a protein, the function most likely to be pertinent to neutrophils was chosen. The majority of proteins were classified into the following functional categories: adhesion, cytoskeletal regulation, enzyme/metabolism, kinase/ phosphatase, membrane trafficking, proteolysis, receptor/signal transduction, and transport. For a given functional class, plasma membrane-enriched fractions and secretory vesicle-enriched fractions contained either an approximately equal number of proteins or there were more proteins in the plasma membrane-enriched fractions. Only a limited number of proteins in each functional category were found in both fractions, ranging from 6% of transport proteins to 22% of cytoskeletal regulatory proteins.

In addition to identification of 57 proteins involved in membrane trafficking, a total of 38 GTPases or their regulatory proteins

Table IV. Pattern of functions of plasma membrane and secretory vesicle proteins a

| Secretory Vesicle Proteins Only   | Secretory Vesicle and Common Proteins                           | Plasma Membrane, Secretory Vesicle, and Common Proteins                                                                                                   | Plasma Membrane, Secretory Vesicle, and Common Proteins                                                                           |
|-----------------------------------|-----------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
| Gene expression Protein folding   | Protein degradation Protein trafficking Nucleic acid metabolism | Cell signaling Cellular movement Cell morphology Tissue development Cellular development Cell growth and proliferation Infectious disease Immune response | Carbohydrate metabolism Cell death Molecular transport Immune system function Immunologic disease Post-translational modification |
|                                   | Free radical scavenging                                         |                                                                                                                                                           |                                                                                                                                   |
|                                   |                                                                 |                                                                                                                                                           | Mineral metabolism                                                                                                                |
|                                   |                                                                 |                                                                                                                                                           | Lipid metabolism                                                                                                                  |
|                                   |                                                                 | Cellular assembly and organization                                                                                                                        | Metabolic disease                                                                                                                 |
|                                   |                                                                 | Cell-to-cell signaling and interaction                                                                                                                    |                                                                                                                                   |
|                                   |                                                                 |                                                                                                                                                           | Hematological disease                                                                                                             |

Table V. Pattern of proteins in canonical pathways a

| PM and SV Proteins and/or Common Proteins    | SV Proteins                          | PM Proteins                              |
|----------------------------------------------|--------------------------------------|------------------------------------------|
| Actin cytoskeleton signaling                 | cAMP-mediated signaling              | Phosphatase and tensin homolog signaling |
| Integrin signaling                           | G-protein coupled receptor signaling | Apoptosis signaling                      |
| Leukocyte extravasation signaling            | Fibroblast growth factor signaling   | NK cell signaling                        |
| Calcium signaling                            | PI3K/AKT signaling                   | TLR signaling                            |
| Complement and coagulation cascade           | Inositol metabolism                  | WNT/ -catenin signaling ùõÉ                |
| Vascular endothelial growth factor signaling | Fc R signaling ùõú                     | TGF- ùõÉ signaling                         |
| Ag presentation pathway                      |                                      | NF- ùõã B signaling                        |
| Protein ubiquitination pathway               |                                      |                                          |
| Oxidative stress response                    |                                      |                                          |
| Epidermal growth factor signaling            |                                      |                                          |
| Platelet-derived growth factor signaling     |                                      |                                          |
| ERK/MAPK signaling                           |                                      |                                          |
| JNK signaling                                |                                      |                                          |

were identified. Thus, 95 proteins were identified that are recognized to participate in regulation of events leading to membrane trafficking. These proteins were equally distributed between plasma membrane-enriched and secretory vesicle-enriched fractions, and only 10 proteins were present in both fractions (Rap1B, Rab1B, Rab5C, Rab7, G ùõÇ i2 , dynein 8, kinesin 27, reticulon 3c, testilin, and secretory carrier membrane protein 2).

Functional and canonical signaling pathways associated with proteins in plasma membrane and secretory vesicle-enriched membrane fractions

To better understand the functional roles of plasma membrane and secretory vesicle proteins, the 821 proteins, excluding those localized to mitochondria, nuclei, and ribosomes, were subjected to IPKB analysis for canonical pathways and functional pathways. Canonical pathways are defined as well-characterized metabolic and cell signaling pathways, whereas functional pathways contain three primary categories of functions: molecular and cellular functions; physiological system development and function; and diseases and disorders. Of the 821 proteins, 743 could be mapped using the IPKB, 368 proteins from the plasma membrane-enriched fractions, 282 proteins from the secretory vesicle-enriched fractions, and 93 protein identified in both fractions. Table IV lists the functional pathways that demonstrated a significant association with proteins identified in each membrane fraction. The vast majority of functions, including cell signaling; cellular movement; hematologic, infectious, immunologic, and inflammatory diseases; and molecular transport, were significantly associated with proteins in all three groups, plasma membrane-enriched fractions, secretory vesicle-enriched fractions, and proteins common to both fractions. Thus, proteins performing these functions were not segregated into any particular membrane fraction. Table V lists the canonical pathways identified by the presence of multiple proteins from each pathway in the membrane fraction. Three patterns were observed. The largest number of canonical pathways contained proteins present in both plasma membrane-enriched and secretory vesicle-enriched fractions, and/or proteins which were common to both membrane fractions. This group of pathways included actin cytoskeletal signaling, integrin signaling, leukocyte extravasation signaling, protein ubiquitination, oxidative stress, ERK or JNK signaling, and growth factor (platelet-derived growth factor and epidermal growth factor) signaling. Canonical pathways containing proteins only identified in secretory vesicle-enriched fractions

included G-protein coupled receptor signaling, fibroblast growth factor signaling, PI3K/AKT signaling, and Fc ùõÜ R signaling. Canonical pathways containing proteins only identified in plasma membrane-enriched fractions included phosphatase and tensin homolog signaling, TLR signaling, TGFùõÉ signaling, and NFùõã B signaling. Thus, plasma membrane-enriched and secretory vesicleenriched fractions contained proteins common to a number of functions and signaling pathways. There were, however, protein components of signaling pathways unique to plasma membrane and to secretory vesicles.

## Discussion

Understanding the functional consequences of secretory vesicle exocytosis in neutrophils requires knowledge of the membrane proteins of this organelle. Previous studies using primarily Abbased methods identified individual proteins from secretory vesicle membranes, including cytochrome b 558 oxidase, CD11b/CD18 adhesion molecules, complement receptor 1 (CR1 or CD35), formyl peptide receptors, Fc(RIIIa) (CD16), membrane metalloendopeptidase (CD10), aminopeptidase N (CD13), the TLR complex molecule CD14, the transmembrane protein tyrosine phosphatase CD45, V-type H + -ATPase, the soluble NSF attachment receptor (SNARE) protein VAMP-2, and the metalloproteinase leukolysin (1). Subcellular fractionation, combined with mass spectrometrybased proteomics, represents a powerful approach to unbiased identification of the protein composition of intracellular organelles (6-9). The current study applied these techniques to develop a more complete catalog of neutrophil secretory vesicle membrane proteins and to compare the proteomes of secretory vesicle membranes and plasma membranes. Over 1100 proteins were identified, the majority of which segregated to either the plasma membrane-enriched fractions or the secretory vesicle-enriched membrane fractions; only 11% of identified proteins were present in both membrane fractions.

Two significant problems affect the interpretation of our data. First, the sensitivity of proteomic analysis makes the ability to obtain highly purified intracellular organelles the limiting factor in establishing the proteome of a specific organelle. Based on the classification of our proteins by cell location and function using Gene Ontology, proteins typically associated with other intracellular organelles, notably nuclei, mitochondria, and ribosomes, were identified in both secretory vesicle-enriched and plasma membrane-enriched fractions. One reason for the presence of those

contaminating proteins is that nitrogen cavitation partially disrupts a number of intracellular compartments, including mitochondria, golgi, endoplasmic reticulum, and nuclei. In the case of neutrophils, this disruption is reported to extend to intracellular storage granules (10, 29-31). Soluble proteins released from these granules likely associate and cosediment with membranes from granule-free fractions, consistent with our finding myeloperoxidase, elastase, and lactoferrin in plasma membrane and secretory vesicle membrane fractions. It is also likely that membranes released by partial disruption of other organelles cosediment in plasma membrane-enriched and secretory vesicle-enriched membrane fractions. Jethwaney et al. (29) identified proteins derived from mitochondria and endoplasmic reticulum in secretory vesicle-enriched fractions derived by free-flow electrophoresis, whereas proteins from these and other organelles were distributed in both membrane fractions in the current study. Whether the presence in both membrane fractions of proteins from a broader range of organelles in our study reflects differences in membrane enrichment or protein identification techniques between the two studies cannot be determined. No direct comparison of free-flow electrophoresis and density-gradient centrifugation enrichment of secretory vesicle-enriched and plasma membrane-enriched fractions has been performed. Although the presence of latent alkaline phosphatase activity indicates that both methods obtain fractions containing secretory vesicles, a comparative study is needed to determine similarities and differences between the two methods.

A total of 73 proteins were assigned by Gene Ontology to endosomes, golgi, or endoplasmic reticulum. The possible localization of those proteins to plasma membranes or secretory vesicles, rather than other intracellular membrane compartments, was not examined further in this study. A recent report, in which proteomic analysis of phagosomes was performed, determined that endoplasmic reticulum fuse with maturing phagosomes in macrophages, suggesting one mechanism by which proteins may be 'shared' by different intracellular compartments (9). Thus, it is likely that some proteins that localize to cellular components, such as endoplasmic reticulum or endosomes, may also be associated with secretory vesicles or plasma membrane.

The second problem with application of proteomic techniques to identification of proteins in secretory and plasma membranes is the difficulty extracting and identifying transmembrane proteins containing ùõÇ helices. Identifying transmembrane proteins is made difficult by the absence of sites for tryptic cleavage in transmembrane regions, variability in the size of exposed hydrophobic regions, low abundance of transmembrane proteins, poor separation by two-dimensional gel electrophoresis of integral membrane proteins, and poor solubility of hydrophobic peptides. Fischer et al. (11) recently reported that tryptic digestion of bacterial membrane proteins extracted in 60% methanol increased identification of integral membrane proteins from 20 to 50% of the total proteins identified. Our results suggest that this approach is useful in membranes from mammalian cells, as one-third of the proteins identified in the present study were classified as integral membrane proteins by Gene Ontology.

The observation that only 11% of proteins identified were common to both plasma membrane-enriched and secretory vesicle-enriched fractions suggests that Percoll density-gradient centrifugation effectively enriched two different membrane populations. Combined with the assays for total and latent alkaline phosphatase, our findings suggest that there are greater differences in the protein content of secretory vesicles and plasma membrane than previously appreciated. Jethwaney et al. (29) used free-flow electrophoresis to separate plasma membrane from secretory vesicles, separated proteins by SDS-PAGE, and identified proteins using HPLC-MS/MS. Similar to our results, these authors identified 30 proteins present in the plasma membrane-enriched fractions and 36

proteins in secretory vesicle-enriched fractions, whereas only 7 proteins (10%) were common to both fractions. Several explanations were considered to account for the low percentage of proteins that were common to both membrane fractions. First, secretory vesicles are not endocytic vesicles derived from the plasma membrane, as has been previously postulated (1, 5). Second, the protein extraction and/or identification techniques failed to identify proteins common to the two membranes. Third, separation of plasma membrane and secretory vesicle membrane by density results in identification of proteins that segregate according to membrane density, rather than by organelle. It is unlikely that the sensitivity of proteomic techniques would vary between the two membrane fractions, resulting in the failure to identify proteins common to both. The localization of latent alkaline phosphatase to the membrane fraction with higher density suggests that proteins associated with secretory vesicles were limited to one group of membrane fractions. Our data do not allow us to distinguish between the possibility that secretory vesicles are not endocytic vesicles or that all membranes contain domains of lighter and heavier density to which distinct sets of proteins localize. However, the data suggest that the hypothesis that secretory vesicles are formed from plasma membranes by endocytosis requires more critical evaluation.

The cell functional classification based on the Gene Ontology database revealed a number of proteins that may elucidate mechanisms of secretory vesicle exocytosis. Examination of the 38 GTPases, the majority of which were identified in secretory vesicle-enriched fractions, revealed a number of Rab proteins known to regulate membrane trafficking. Secretory vesicle-enriched fractions contained Rab11a, Rab14, Rab15, and Rab35; plasma membrane-enriched fractions contained Rab5b and Rab31, and Rab1b, Rab5c, and Rab7 were common to both plasma membrane and secretory vesicle-enriched fractions. Rab5 was reported to play a significant role in chemoattractant receptor endocytosis and fusion of intracellular granules with phagosomes in human neutrophils (32-34). Rab5a was shown to undergo a significant translocation from endosomes and secretory vesicles to the plasma membrane with stimulation of human neutrophils (35), and Rab11 was reported to regulate endocytic-exocytic cycling of integrin molecules (36). The roles of Rab 14, Rab15, Rab31, and Rab35 have not been examined in neutrophils. A total of 57 proteins were identified that play a role in membrane trafficking, 30 were in the plasma membrane, 22 in secretory vesicles, and 5 were common to both. The present study found two SNARE proteins, VAMP-3 and VAMP-8, on secretory vesicle-enriched membranes. VAMP-1, -2, -7, and -8 were reported previously to be present in human neutrophils, and VAMP-2 was identified on secretory vesicles (37-40). VAMP-3 has not been found previously in human neutrophils, although it has been identified in human platelets and human plasma cells (41, 42). Mollinedo et al. (39, 40) reported that VAMP-1 and VAMP-2 mediated exocytosis of specific granules, whereas VAMP-1 and VAMP-7 mediated azurophil granule exocytosis. The SNARE proteins involved in secretory vesicle exocytosis have not been determined. In addition to SNARE proteins, the exocyst complex is a group of eight proteins involved in vesicle targeting and docking at the plasma membrane (43, 44). Two components of the exocyst complex were identified in the current study: exocyst complex component 2 (Sec3) was identified from plasma membrane-enriched fractions, and exocyst complex component 5 (Sec10) was identified from secretory vesicle-enriched fractions. Boyd et al. (45) reported that Sec3p localized to the plasma membrane of Saccharomyces cerevisiae , while Sec10p was found on vesicles. The exocyst complex is an effector for multiple GTPases, including cdc42 and Rab11 (46, 47). Both of these GTPases were identified, cdc42 from plasma membrane-enriched

fractions and Rab11A from secretory vesicle-enriched fractions. These findings suggest that the exocyst complex plays a role in tethering secretory vesicles to the plasma membrane before SNARE-mediated membrane fusion. Consistent with our previous studies which showed large amounts of actin associated with both neutrophil plasma membranes and secretory vesicle membranes (4), 72 cytoskeletal and cytoskeletal regulatory and binding proteins were equally distributed in both membrane fractions. Two groups of actin assembly factors were identified. Formins act in conjunction with profilin to drive nucleation, but not branching, of actin filaments (48). Formin 1 and 2 were identified from plasma membrane-enriched fractions, and profilin 1 was common to both fractions. The Arp2/3 complex acts as a nucleation and branching factor, and members of this complex were identified from secretory vesicle-enriched fractions. Although it was suggested previously that actin and actin-associated proteins reflect cytoskeletal contamination of these membrane fractions (29), the high content of cytoskeletal proteins and the importance of actin reorganization in exocytosis of secretory granules make it more likely that cytoskeletal proteins are associated with plasma and secretory vesicle membranes (4, 10).

## Disclosures

The authors have no financial conflict of interest.

## References

- 1. Faurschou, M., and N. Borregaard. 2003. Neutrophil granules and secretory vesicles in inflammation. Microbes Infect. 5: 1317-1327.
- 2. Sengelov, H., L. Kjeldsen, and N. Borregaard. 1993. Control of exocytosis in early neutrophil activation. J. Immunol. 150: 1535-1543.
- 3. Sengelov, H., P. Follin, L. Kjeldsen, K. Lollike, C. Dahlgren, and N. Borregaard. 1995. Mobilization of granules and secretory vesicles during in vivo exudation of human neutrophils. J. Immunol. 154: 4157-4165.
- 4. Jog, N., M. J. Rane, G. Lominadze, G. C. Luerman, R. A. Ward, and K. R. McLeish. 2007. The actin cytoskeleton regulates exocytosis of all neutrophil granule subsets. Am. J. Physiol. 292: C1690-C1700.
- 5. Borregaard, N., L. Kjeldsen, K. Rygaard, L. Bastholm, M. H. Nielsen, H. Sengelov, O. W. Bjerrum, and A. H. Johnsen. 1992. Stimulus-dependent secretion of plasma proteins from human neutrophils. J. Clin. Invest. 90: 86-96.
- 6. Casey, T., J. L. Meade, and E. W. Hewitt. 2007. Organelle proteomics: identification of the exocytic machinery associated with the natural killer cell secretory lysosome. Mol. Cell. Proteomics. 6: 767-780.
- 7. Brunet, S., P. Thibault, E. Gagnon, P. Kearney, J. J. M. Bergeron, and M. Desjardins. 2003. Organelle proteomics: looking at less to see more. Trends Cell Biol. 13: 629-638.
- 8. Garin, J., R. Diez, S. Kieffer, J. F. Dermine, S. Duclos, E. Gagnon, R. Sadoul, C. Rondeau, and M. Desjardins. 2001. The phagosome proteome: insight into phagosome functions. J. Cell Biol. 152: 165-180.
- 9. Gagnon, E., S. Duclos, C. Rondeau, E. Chevet, P. H. Cameron, O. Steele-Motimer, J. Palement, J. J. M. Bergeron, and M. Desjardins. 2002. Endoplasmic reticulummediated phagocytosis is a mechanism of entry into macrophages. Cell 110: 119-131.
- 10. Lominadze, G., D. W. Powell, G. C. Luerman, A. J. Link, R. A. Ward, and K. R. McLeish. 2005. Proteomic analysis of human neutrophil granules. Mol. Cell. Proteomics 4: 1503-1521.
- 11. Fischer, F., D. Wolters, M. Roegner, and A. Poetsch. 2006. Toward the complete membrane proteome. Mol. Cell. Proteomics 5: 444-453.
- 12. Haslett, C., L. Guthrie, J. Johnston, and P. Henson. 1985. Modulation of multiple neutrophil functions by preparative methods of trace concentrations of bacterial lipopolysaccharide. Am. J. Pathol. 119: 101-110.
- 13. Amrein, P. C., and T. P. Stossel. 1980. Prevention of degradation of human polymorphonuclear leukocyte proteins by diisopropylfluorophospate. Blood 56: 442-447.
- 14. Dahlgren, C., S. R. Carlsson, A. Karlsson, H. Lundqvist, and C. Sjolin. 1995. The lysosomal membrane glycoproteins lamp-1 and lamp-2 are present in mobilizable organelles, but are absent from the azurophil granules of human neutrophils. Biochem. J. 311: 667-674.
- 15. Link, A. J., J. Eng, D. M. Schieltz, E. Carmack, G. J. Mize, D. R. Morris, B. M. Garvik, and J. R. Yates. 1999. Direct analysis of protein complexes using mass spectrometry. Nat. Biotechnol. 17: 676-682.
- 16. http://bigcat.mc.vanderbilt.edu/BIGCAT/index.php
- 17. http://tools.proteomecenter.org/software.php
- 18. McAfee, K. J., D. T. Duncan, M. Assink, and A. J. Link. 2006. Analyzing proteomes and protein function using graphical comparative analysis of tandem mass spectrometry results. Mol. Cell. Proteomics 5: 1497-1513.
- 19. Nesvizhskii, A. I., A. Keller, E. Kolker, and R. Aebersold. 2003. A statistical model for identifying proteins by tandem mass spectrometry. Anal. Chem. 75: 4646-4658.
- 20. Liu, H., R. G. Sadygov, and R. J. Yates. 2004. A model for random sampling and estimation of relative protein abundance in shotgun proteomics. Anal. Chem. 76: 4193-4201.

- 21. Old, W. M., K. Meyer-Arendt, L. Aveline-Wolf, K. G. Pierce, A. Mendoza, J. R. Sevinsky, K. A. Resing, and N. G. Ahn. 2005. Comparison of label-free methods for quantifying human proteins by shotgun proteomics. Mol. Cell. Proteomics 4: 1487-1502.
- 22. Wienkoop, S., E. Larrainzar, M. Niemann, E. M. Gonzalez, U. Lehmann, and W. Weckwerth. 2006. Stable isotope-free quantitative shotgun proteomics combined with sample pattern recognition for rapid diagnostics. J. Sep. Sci. 29: 2793-2801.
- 23. Fleischer, T. C., C. Weaver, K. J. McAfee, J. L. Jennings, and A. J. Link. 2006. Systematic identification and functional screens of uncharacterized proteins associated with eukaryotic ribosomal complexes. Genes Dev. 20: 1294-1307.
- 24. Powell, D. W., C. Weaver, J. C. Jennings, K. J. McAfee, Y. He, P. A. Weil, and A. J. Link. 2004. Cluster analysis of mass spectrometry data reveals a novel component of SAGA. Mol. Cell. Biol. 24: 7249-7259.
- 25. Sanders, S. L., J. Jennings, A. Canutescu, A. Link, and P. A. Weil. 2002. Proteomics of the eukaryotic transcription machinery: identification of proteins associated with components of yeast TFIID by multidimensional mass spectrometry. Mol. Cell. Biol. 22: 4723-4738.
- 26. http://www.geneontology.org
- 27. www.ingenuity.com
- 28. Calvano, S., W. Xiao, D. R. Richards, R. M. Felcano, H. V. Baker, R. J. Cho, R. O. Chen, B. H. Brownstein, J. P. Cobb, S. K. Tschoeke, et al. 2005. A networkbased analysis of systemic inflammation in humans. Nature 437: 1032-1037.
- 29. Jethwaney, D., M. R. Islam, K. G. Leidal, D. B.-V. de Bernabe, K. P. Campbell, W. M. Nauseef, and B. W. Gibson. 2007. Proteomic analysis of plasma membrane and secretory vesicles from human neutrophils. Proteome Sci. 5: 12.
- 30. Kjeldsen, L., H. Sengelov, K. Lollike, M. H. Nielsen, and N. Borregaard. 1994. Isolation and characterization of gelatinase granules from human neutrophils. Blood 83: 1640-1649.
- 31. Borregaard, N., J. M. Heiple, E. R. Simons, and R. A. Clark. 1983. Subcellular localization of the b-cytochrome component of the human neutrophil microbicidal oxidase: translocation during activation. J. Cell Biol. 97: 52-61.
- 32. Perskvist, N., K. Roberg, A. Kulyte, and O. Stendahl. 2001. Rab5a GTPase regulates fusion between pathogen-containing phagosomes and cytoplasmic organelles inhuman neutrophils. J. Cell Sci. 115: 1321-1330.
- 33. Rose, J., J. F. Foley, P. M. Murphy, and S. Venkatesan. 2004. On the mechanism and significance of ligand-induced internalization of human neutrophil chemokine receptors CXCR1 and CXCR2. J. Biol. Chem. 279: 24372-24386.
- 34. Prada-Delgado, A., E. Carrasco-Marin, C. Pena-Macarro, E. del Cerro-Vadillo, M. Fresno-Escudero, F. Leyva-Cobian, and C. Alvarez-Dominguez. 2005. Inhibition of Rab5a exchange activity is a key step for Listeria monocytogenes survival. Traffic 6: 252-265.
- 35. Vita, F., M. R. Soranzo, V. Borelli, P. Bertoncin, P. Bertoncin, and G. Zabucchi. 1996. Subcellular localization of the small GTPase Rab5a in resting and stimulated human neutrophils. Exp. Cell Res. 227: 367-373.
- 36. Fabbri, M., S. D. Meglio, M. C. Gagliani, E. Consonni, R. Molteni, J. R. Bender, C. Tacchetti, and R. Pardi. 2005. Dynamic partitioning into lipid rafts controls the endo-exocytic cycle of the aL/B2 integrin, LFA-1, during leukocyte chemotaxis. Mol. Biol. Cell 16: 5793-54803.
- 37. Brumell, J., A. Volchuk, H. Sengelov, N. Borregaard, A.-M. Cieutat, D. F. Bainton, S. Grinstein, and A. Klip. 1995. Subcellular distribution of docking/fusion proteins in neutrophils, secretory cells with multiple exocytic compartments. J. Immunol. 155: 5750-5759.
- 38. Logan, M., P. Lacy, S. O. Odemuyiwa, M. Steward, F. Davione, H. Kita, and R. Moqbel. 2006. A critical role for vesicle-associated membrane protein-7 in exocytosis from human eosinophils and neutrophils. Allergy 61: 777-784.
- 39. Mollinedo, F., B. Martin-Martin, J. Calafat, S. M. Nabokina, and P. A. Lazo. 2003. Role of vesicle-associated membrane protein-2, through Q-soluble N-ethylmaleimide-sensitive factor attachment protein receptor/R-soluble N-ethylmaleimide-sensitive factor attachment protein receptor interaction, in the exocytosis of specific and tertiary granules in human neutrophils. J. Immunol. 170: 1034-1042.
- 40. Mollinedo, F., J. Calafat, H. Janssen, B. Martin-Martin, J. Canchado, S. M. Nabokina, and C. Gajate. 2006. Combinatorial SNARE complexes modulate the secretion of cytoplasmic granules in human neutrophils. J. Immunol. 177: 2831-2841.
- 41. Polgar, J., S. H. Chung, and G. L. Reed. 2002. Vesicle-associated membrane protein 3 (VAMP-3) and VAMP-8 are present in human platelets and are required for granule secretion. Blood 100: 1081-1083.
- 42. Reales, E., F. Mora-Lopez, V. Rivas, A. Garcia-Poley, J. A. Brieva, and A. Campos-Caro. 2005. Identification of soluble N-ethylmaleimide-sensitive factor attachment protein receptor exocytic machinery in human plasma cells: SNAP-23 is essential for antibody secretion. J. Immunol. 175: 6686-6693.
- 43. Lipschutz, J. H., and K. E. Mostov. 2002. Exocytosis: the many masters of the exocyst. Curr. Biol. 12: R212-R214.
- 44. Munson, M., and P. Novick. 2006. The exocyst defrocked, a framework of rods revealed. Nat. Struct. Mol. Biol. 13: 577-581.
- 45. Boyd, C., T. Hughes, M. Pypaert, and P. Novick. 2004. Vesicles carry most exocyst subunits to exocytic sites marked by the remaining two subunits, Sec3p and Exo70p. J. Cell Biol. 167: 889-901.
- 46. Zhang, X., E. Bi, P. Novick, L. Du, K. G. Kozminski, J. H. Lipschutz, and W. Guo. 2001. Cdc42 interacts with the exocyst and regulates polarized secretion. J. Biol. Chem. 276: 46745-46750.
- 47. Zhang, X. M., S. Ellis, A. Sriratana, C. A. Mitchell, and T. Rowe. 2004. Sec15 is an effector for the Rab11 GTPase in mammalian cells. J. Biol. Chem. 279: 43027-43034.
- 48. Chhabra, E. S., and H. N. Higgs. 2007. The many faces of actin: matching assembly factors with cellular structures. Nat. Cell. Biol. 9: 1110-1121.