## Neutrophil Proteome: Lessons from Different Standpoints

C.F.M. Morris , M.S. Castro 1 1,# and W. Fontes 1,*

1 Brazilian Center for Protein Research, Department of Cell Biology, University of Brasilia, Brazil

Abstract: The present review brings a timeline of some of the major steps given throughout the years towards the development of our knowledge regarding the biology of the neutrophil. The contribution of early articles and their elementary biochemical approach is highlighted. The importance of the development of proteomic techniques is paralleled to the shift in neutrophil research towards high throughput molecular methods. As a last change of standpoint, the study of the neutrophil is presented integrated with other lifesciences technologies such as lipidomics, genomics and systems biology. The paper also brings a perspective/tendency overview  at the same  time that it discusses some of the difficulties encountered in the  research of the neutrophil.

Keywords: Neutrophil, proteome, two-dimensional electrophoresis, mass spectrometry.

<!-- image -->

# Author  Profile: Mariana  S.  Castro  has graduated  in  Biology  (1988-1991),  MSc  in  Molecular  Biology:  Protein  Chemistry (1992-1995) and PhD in Molecular Biology: Protein Chemistry (1995-1999) at University of Brasilia. Adjunct Professor of Physiology at  University  of  Brasilia  since  1996.  Member  of  the  Brazilian  Society  for  Biochemistry  and  Molecular  Biology since 1993. Fields of interest: protein chemistry, venomics and proteomics.

## INTRODUCTION

The neutrophil is widely known for its central role in the mechanism of inflammation in mammals. The importance of this  leukocyte,  member  of  the  granulocyte's  family,  covers that of a professional phagocyte and that of a very important modulator  of  both  local and  systemic  inflammatory  responses through the production and secretion of various immune mediators [1-6]. The neutrophil is generally considered one  of  the  first  cells  to  reach  a  site  of  local  infection  and, once in position, the cell becomes fully activated and competent  to  fulfill  a  series  of  intense  inflammatory  activities,  including  phagocytosis,  exocytosis  of  intracellular  granules, activation of both pro- and anti-inflammatory genes and production of various  types of proteins which  are  important to the recruitment of other immune cells and to the modulation of the initiated inflammatory response [7-14].

Due  to  its  abundant  supply  of  microbicidal  agents  enclosed in intracellular granules, to its capacity for the generation  of  Reactive  Oxygen  Species  and  Intermediates  (ROS and ROI, respectively) and to its ability to synthesize various cytokines  and  chemokines,  the  general  effect  derived  from the actions of the neutrophil is that of up-scaling the immune response [6,15-17]. However, a neutrophil-mediated inflammatory event can become excessively intense or be insufficiently  counterbalanced  by  natural  regulatory  mechanisms, giving way to an overwhelming Systemic Inflammatory  Response  Syndrome  (SIRS)  [18],  that  leads  to  tissue injury  in  vulnerable  organs,  such  as  the  lung,  resulting,  for instance, in the Acute Respiratory Distress Syndrome (ARDS)  [19].  ARDS  may  be  a  consequence  of  traumatic

insults, associated to extensive tissue destruction,  or of ischemia-reperfusion  events.  Other  important  examples  of neutrophil-related  diseases  include  Behcet's  Syndrome  [20] and Asthma [21].

Because of its obvious importance both in the normal and disease-related inflammatory response, the neutrophil and its array  of  proteins  have  been  targets  of  numerous  scientific studies throughout  the  years. When  taken  together, the authors  of  those  studies  used  various  approaches  to  try  to bring to light different mechanisms and molecules that might someday be of diagnostic, prognostic or therapeutic significance  in  clinically  important  inflammatory  events.  A  brief analysis of the literature about neutrophil proteins reveals the complexity of the mechanisms involved in the cell's biology. This complex system cannot be interpreted as a single pathway but, rather,  demands  the  identification of several  components,  most of  them presenting a proteic structure.  Being such  a  complex  system,  studies  of  the  various  mechanisms underlining the physiology of the  many  cellular events that occur in the neutrophil during health or disease have proven themselves to be critically time-consuming. As a result,  the literature is rich in studies focused in quite specific events or components of the neutrophil leukocyte and we think that it is  crucial  to  present  a  comprehensive  and concise reference of what has been done in this field throughout the years, as an attempt to broaden our understanding regarding the biology of the neutrophil as a whole. The present review brings a timeline of some of the major steps taken towards unveiling some important aspects of the neutrophil's proteome during both health and disease, at the same time that it correlates the different  studies with some of the  most  interesting methods used  by  each  group  of  authors  and  the  results  respectively obtained. The literature review itself is divided in three basic parts: the first one deals with the studies performed in a preproteomic  era  when  proteomic  methods  were  either  inexis-

tent or seldom applied; a second part brings the work of several researchers that made use of various proteomic tools to study many aspects of the biology of the neutrophil; a third and last part is dedicated to modern associations that are being  made  between  proteomic  and  non-proteomic  sciences, such as genomics, metallomics, lipidomics and systems biology. It is important to stress that, while we tried to preserve a logical  and  coherent  timeline,  some  temporal  overlap  between  the  three  parts  was  inevitable  even  though  a  linear compilation  of  some  of  the  most  important  studies  herein described can be found in Fig. ( 1 ).  Moreover, we are aware of the fact that due to practical matters it is sadly impossible to give credit to all the researchers that have in any way contributed  to  our  present  understanding  of  the  neutrophil  and we apologize for any relevant references that may have been left out inadvertently.

## LESSONS FROM BACK IN THE DAY

One  of  the  major  concerns  when  working  with  neutrophils in vitro is the establishment of cost-effective cell isola-

tion protocols, which ideally will provide a good number of pure and biologically viable cells. Two of the first people to systematically  isolate  viable  polymorphonuclear  cells  from whole blood samples were Minor and Burnett [22,23]. Using a protocol that would, in light of new information regarding cell activation, be eventually replaced [24-28], their fibrinogen-based isolation method nevertheless represented a landmark  on  the  research  of  the  neutrophil.  Ten  years  later,  a particular author gave immense contribution to the study of the neutrophil. A. M. Woodin published several papers during  the  60's  which,  together,  greatly  clarified  the  composition,  function and overall physiology of the polymorphonuclear  leukocyte  membrane  and  the  biochemistry  of  its  response to bacterial Leucocidin [29-32]. The same author had participated on studies about granule composition and exocytosis  [33-35]. In 1979, Hammerschmidt et  al. measured aggregation and embolization of murine neutrophils when exposed to A23187, N-formyl-methionyl-leucyl-phenyl-alanine (fMLP) and ZAP, stimuli thought to hold a clinical parallel

Figure 1. The figure shows a linear timeline of some of the most important publications revised throughout the text. It is important to note that, during the course of over sixty years, there's a shift from an early and primarily biochemical era towards a proteomic, multilateral and integrated scenario regarding neutrophil research. In addition, it is possible to observe the intensification of proteomic researches after the optimization of MS and LC techniques in the 90's.

<!-- image -->

to  ARDS.  Despite  some  interesting  results,  the  study  was unable to determine a specific mechanism through which the emboli were formed or prevented from doing so [36]. Likewise,  studies  from  the  past  six  decades  (and  some  to  this day)  seldom  explored  the  molecular  biology  of  the  neutrophil in a systematic fashion, but have, rather, used basic tools such as microscopy, regular electrophoresis and various cytological and immunological methods to explore cause-effect relationships [37-44]. However, it is important to remember that  an  increasing number of authors started  to demonstrate the  intention  of  shifting  research  to  a  somewhat  molecular level, although not yet with the use of actual proteomic tools. Two  very  good  examples  of  this  new  inclination  can  be found in Sbarra and Karnovsky's review from 1959 regarding the biochemical mechanisms of phagocytosis [45] and in Abraham  J.  Botha's  review  in  1994  where  he  consistently stresses  the  importance  of  the  understanding  of  the  neutrophil physiology during both health and disease on a molecular  level,  with  particular  focus  on  the  mechanism  of  tissue injury  caused  by glyph&lt;c=1,font=/IHCEOP+Symbol&gt; 2 O production  and  release  [46].  Similar subjects  and inclinations constantly surfaced throughout the next decade, a fact that is corroborated by Swain's review of the various agents that can cause neutrophil priming [47]. In this  scenario,  yet  a  third  kind  of  approach  started  to  take shape  with  the  advent  of  2-DE  associated  to  Mass  Spectrometry  (MS)  and,  more  recently,  Liquid  Chromatography (LC). These techniques are widely accepted as today's most important tools to study a large set of proteins  in a system, but it is  important to remember that they were in their early stage of development throughout the 50's, 60's, 70's and part of the 80's [48-54]. In 1975 the first High Resolution 2-DE method was introduced by O'Farrel [55]. However, the technique was optimized during the next two decades or so [5660] and only with the introduction of more simplified procedures, along with the introduction of commercially available 2-DE supplies (e.g.  immobilized pH strips, IEF buffers and staining kits) [60-62], and the association with cost-effective MS devices and protocols in the 90's, the 2-DE method rose to  the  position  of  a  pivotal  tool  for  analyzing  complex protein mixtures.  The end-result of this association is exemplified  in  Fig.  ( 1 )  that  demonstrates  the  intensification of proteomic studies just before the turn of the century. Nevertheless, the good representation of highly hydrophobic proteins, poorly soluble peptides and low abundance proteins is still a methodological  challenge  in  2-DE  technology.  These  and other limitations of classic 2-DE methods were partially addressed  by  the  development  of  modern  chromatographic tools  and  state-of-the-art  MS  [63-69]  (i.e.  Matrix-assisted Laser Desorption/Ionization-Time of Flight/MS or MALDITOF/MS, Electrospray Ionization-Time of Flight/MS or ESITOF/MS,  Surface-enhanced Laser Desorption/IonizationTime  of  Flight  or  SELDI-TOF/MS,  Quadrupole-Time  of Flight/MS  or  Q-TOF/MS,  Fourier-transform  Ion  Cyclotron Resonance  Mass  Spectrometry  or  FT-ICR-MS  and  Topdown Proteomics). In addition, classic proteomic approaches in general  (e.g.  2-DE/MS  and  LC/MS  methods)  present many intrinsic  limitations  when  quantitative  or  relative  expression studies are pursued. As an attempt to mitigate some of these limitations, various techniques available today

greatly enhanced the reproducibility and efficiency of modern quantitative proteomics. The introduction of Differential In-gel  Electrophoresis  (DIGE)  technology,  that  has  the  capacity to dramatically reduce the time for comparison of up to three different protein samples or experimental conditions by use of differentially labeled pools of proteins [49,70,71], is one example. Accordingly, the Isotope-coded Affinity Tag (ICAT)  and  the  Cleavable  Isotope-coded  Affinity  Tag  (CICAT)  labeling  methods,  associated  to  either  LC/MS  or  2DE/MS, also represented an important step towards improving the efficiency for the quantification of relative expression of some proteins in two sets of samples or experimental conditions [71-75]. Lastly, the quite new Isobaric Tag for Relative  and  Absolute  Quantitation  (iTRAQ)  technology,  that allows one to globally tag up to four sets of samples or experimental  conditions,  represented  a  sensitive  and  effective way  of  measuring  relative  expression  without  provoking mass alterations on the original peptides [71,76-78].

## LESSON FROM A PROTEOMIC UNIVERSE

The first trace of proteomic work performed with neutrophils was perhaps the one conducted by Watt and Burgess in 1981  with  the  use  of  a  murine  model.  This  early  study brought excellent results with the use of 2-DE methodology available  at  the  time,  and  included  two-dimensional  maps with up to 450 spots detected by autoradiography [79]. The lack  of  MS  techniques  and  the  respective  bioinformatics tools impaired the precise identification of the detected spots, but the study is nevertheless a cornerstone of the proteomic research of  the neutrophil. In 1982 Gemmell and Anderson compared  the  whole-cell  proteome  of  granulocytes,  monocytes  and  lymphocytes.  The  high-resolution  2-DE  method used  in  the  study  was  able  to  demonstrate  the  presence  of specific proteins for each cell type [80]. In the same edition of Clinical Chemistry, Willard et al .  demonstrated the presence of a group of proteins present exclusively in the 2-DE maps of leukocytes from patients with rheumatoid arthritis. The so called Rheuma Proteins group had various members of  known  granulocytic  origin  [81].  Over  ten  years  later,  a Canadian group explored some of the effects of IL-13 in the human neutrophil. An important modulation role was attributed to this molecule due to its effects on protein phosphorylation, mRNA syntheses and IL-8 production [82]. In 1997, Kasper et al ., using an immobilized pH gradient for the IEF step  of  the  2-DE,  discovered  three  proteins  exclusive  of stimulated neutrophils from patients presenting a complex of myeloid diseases called Severe Chronic Neutropenia (SCN). After  sequencing,  a  protein  of  32.4  kDa  was  identified  as being a member of the tropomyosin family, possibly a previously  unknown isoform [83]. These were some of the first efforts in neutrophil-proteome science in the sense of identifying molecular markers for a given pathological condition. Three years later, a Japanese group led by Shozo Yamashita studied the differences regarding the proteomic maps of neutrophils  from  healthy  individuals  and  those  from  patients with  Behcet's  Syndrome.  This  group  observed  a  series  of proteins  ranging  from  30-55  kDa  and  pI  4.8-6.2  that  were differentially expressed between the two conditions. This set of proteins included a p40 protein thought to be a truncated form  of  actin  [20].  The  authors  suggest  that  this  structural

difference  might  be  one  of  the  mechanisms  that  allow  the hyperactivity  seen  in  the  neutrophils  from  patients  with Behcet's Syndrome. Also in 2000, a French group identified 19  soluble  granular  proteins  through  2-DE  and  MALDITOF/MS after inducing neutrophil degranulation with iomycin [84]. For the first time in the literature, Annexin XI and Chitotriosidase were proved to be secreted by activated neutrophils and the secretion of HC gp-39 and Calgranulins by this granulocyte was further confirmed [84]. In 2005, Lominadze et al. made use of a dual approach of 2-DE coupled to MALDI-TOF/MS and of HPLC and tandem ESI-MS/MS in order to specifically study the proteome of human neutrophil granules.  More  than  280  proteins  (e.g.  GTPases,  receptors, housekeeping proteins, kinases, phosphatases,  and signaling molecules)  were  identified  as  granular  or  granule-related molecules [85]. The same group is responsible for publishing a  summarized manual for the proteomic analysis of the human neutrophil by 2-DE and tandem MS [86]. In the following year our group published an optimized 2-DE protocol for the study of the human neutrophil. After a modified ammonium silver staining procedure, more  than 1.800 spots were distinctly  visible  in  the  gels  (pI  4-7).  Over  100  spots  were present, with a volume variation of less than 20%, in all the 14  individuals  used  in  the  study.  Unfortunately  this  same protocol proved not to be MS-friendly and only 22 proteins spanning  from  inflammatory  to  housekeeping  molecules, were identified by MALDI-TOF/MS at the time [27]. Latter that year, Burlak et al . studied the proteome of the neutrophil phagosome.  The  work  associated  several  proteomic  tools such as 2-DE, MALDI-TOF/MS  and  LC-MS/MS  and yielded more than 400 identified proteins, being 198 unique to the phagosome [87]. One of the major features of that paper was the discovery of crucial differences between the neutrophil  and  the  macrophage  phagosomes  which  were  once thought to be similar [2]. Another surprising study came in 2006 as a group lead by Kerstin Puellmann and Wolfgang E Kaminski  discovered  a  T-Cell  Receptor  (TCR)-based  immunoreceptor in a subpopulation of human and murine neutrophils,  along  with  the  expression  of  the  RAG-1/RAG-2 recombinase complex [88]. That discovery shed light on the novel possibility of  combination between  a variable  and an innate mechanism in the cellular response of the mammalian neutrophil.

In 2007, a Japanese group implicated, by means of 2-DDIGE  and  MALDI-TOF/TOF  technology,  the  Ca -binding + protein S100-A9 in the cell signaling triggered by the covalent  linkage  of  CD-69  [89].  The  CD-69  glycoprotein  is thought  to  be  involved  in  some  disease  processes,  particularly those  of  the sinovial joint and  the  referred  study brought S100-A9 to attention as a possible therapeutic target. Our  group  has  recently  conducted  a  proteomic  study  that showed  the  decrease  of  intracellular  S100-A9  after  neutrophils were incubated with Platelet Aggregation Factor (PAF) [90].  When  taken  together,  these  results  corroborate  to  the notion  of  the  importance  of  S100-A9  as  an in  vivo and in vitro chemotactic  agent  for  neutrophils  [91,92].  Latter  that same year, DiMartino et  al .  somewhat  clarified  the  mechanism  by  which  the  Vitamin  D-binding  Protein  (DBP)  gets attached  to  the  neutrophil.  In  that  study,  the  authors  implicated  general  neutrophil  activation  along  with  azurophilic degranulation  as  two  important  effectors  for  the  ligation  of

DBP [93]. Also in 2007, a British group discovered the relevance  of  Proteinase-3  (PR-3)  as  a  possible  therapeutic  and prognostic target for human diseases due to its immense importance  in  the  cleavage  of  Annexin-1  [94].  Also  in  2007, Jethwaney et al . published the first work comparing the proteome of secretory vesicles  and plasma membrane vesicles. A total of 80 proteins were identified by the association of 2DE, HPLC, MALDI-TOF/MS and FFE (Free Flow Electrophoresis), from which two proteins hadn't yet been described in these organelles: 5-Lypoxigenase Activating Protein (FLAP)  and  Dysferlin,  which  are  important  for  leukotriene synthesis  and  in  the  pathogenesis  of  muscle  dystrophies, respectively [95].

Domestic species have also been investigated for the proteome of their neutrophils. As said before,  Watt  &amp;  Burgess were  the  first  to  systematically  study  the  proteome  of  the murine neutrophil  in  1981  [79].  Almost  twenty  years  later, Piubelli et al. used the association between 2-DE coupled to Mass Spectrometry and LC coupled to ESI-TOF/MS for the first  time  in  the  study  of  the  bovine  neutrophil.  The  group identified 52 proteins, including housekeeping and cytoskeleton  molecules,  granules  proteins,  and  phagocytosis, adhesion and chemotaxis-related proteins [96]. A paper from 2005 described a SDS-PAGE methodology associated with a LC device coupled to a Q-TOF/MS apparatus for the study of the neutrophil of this very species. This particular method yielded the identification of 250 proteins of the bovine neutrophil, many of which were NADPH oxidase components or apoptotic  proteins  and  metabolic  enzymes  [97].  In  2006,  a group lead by Ambatipudi started a novel work studying the proteome of neutrophils from a marsupial model, the Tammar  Wallaby.  Proteomic  tools,  such  as  2-DE,  LC  and  MS, yielded  100  well-resolved  spots  and  53  high-confidence identifications  [98].  Possible  sequence  differences  between human and marsupial neutrophils were noted, being that the lack  of  the  genome  sequence  for  the  latter  species  represented the major challenge of the study. Two years latter, the same group published the first specific data about the subcellular proteome of the Tammar Wallaby neutrophil granules. The  latter  work  brought  identification  of  approximately  50 granular  proteins  of  various  functions,  after  the  appropriate subcellular  fractioning  [99].  The  identified  proteins'  functions spanned from housekeeping to microbicidal.

## LESSONS FROM AN INTEGRATED PERSPECTIVE

In 2001, an elegant work by Lian et al. used a novel approach  for  studying  the  developmental  biology  of  the  neutrophil.  In  this  study  the  authors  used  a  Mouse  Promyelocytic  Cell  Line  (MPRO)  differentiated  into  neutrophils  by use of AllTrans Retinoic Acid (ATRA) and monitored the expression of mRNA for several genes concurrently with the generation of 2-DE maps showing the state of the proteome of  the  neutrophil  during  its  differentiation.  In  accordance with previous works in yeast [100] and human liver cells, the authors  found  poor  correspondence  between  the  variations on the  expression of mRNA when compared to the expression  variation  on  the  proteins  they  encoded,  in  the  2-DE maps [101]. In addition, the same authors proposed that the lack of correspondence of the results from the two different approaches  was  probably  due  to  differential  translational regulation  during  cellular  differentiation,  alterations  of  the

mRNA syntheses during in vitro experiments, posttranslational modifications of proteins and selective degradation or excretion of certain proteins. Another study associating  a  genomic  and  a  proteomic  approach  also  found  poor correspondence between mRNA and the respective proteins concentrations on neutrophils stimulated with LPS [102]. A different  paper  published  by  Fessler's  group  later  that  year further described the effects of LPS on the neutrophil and the participation of the p38 MAPK in the process, also through a genomic and proteomic approach. This last work found that 100  genes  were  up-regulated  and  another  56  were  downregulated  after  exposure  of  the  cell  to  LPS  [103].  Furthermore, the authors described the up- and down-regulation of the proteins  encoded by  those genes, but  in an  independent and unrelated fashion. These findings were in accordance to the previous studies that found poor correspondence between mRNA and  their  correspondent  proteins  concentrations.  In the same paper, Fessler's group implicates the p38 MAPK as a  regulatory  factor  of  genetic  transcription  for  some  of  the genes  studied  (particularly  of  annexin  III  and glyph&lt;c=1,font=/IHCHCH+TimesNewRomanPSMT&gt; -enolase) [103].

Another  interesting  use  of  proteomics  is  its  association with  lipidomics  methodologies.  The  first  study  to  establish this  link  in  neutrophil  research  was  the  one  published  by Bannenberg et al .  in  2005. In this paper, the authors, members  of  a  laboratory  from  which  the  discovery  of  novel  inflammation-resolving mediators (i.e. resolvins, docosatrienes and  neuroprotectines)  had  originated  [104-106],  presented the  first  temporal  profile  of  the  events  of  resolution  of  inflammation at the same time as they establish a series of indexes in order to allow an objective and cytological evaluation  of  the  resolution  phase  of  the  inflammatory  process [107].  The  study  also  describes  the  temporal  variation  of several  cytokines  and  eicosanoids  during  the  inflammatory event and implicates TGFglyph&lt;c=2,font=/IHCHCH+TimesNewRomanPSMT&gt; as an important molecule during the resolution period [107].

Another recent approach for the study of the neutrophil is the use of systems biology as an attempt to keep up with the dynamicity  of  the  neutrophil's  biology  [108].  This  type  of study moves away from very specific approaches and seems particularly  suitable  for  studying  complex  cell,  such  as  the neutrophil.

## CONCLUDING REMARKS

The present review showed the remarkable development of  neutrophil  research  throughout  half  a  century  of  studies, papers  and  discoveries.  It  is  interesting  to  note  that,  being such a special  cell, research on the neutrophil was polished and ameliorated synchronically to technical improvements in the proteomic science itself. Accordingly, after the exclusion of  a  few  exceptions,  different  ways  of  studying  the  neutrophil can be clustered by each period of time. Fifty years ago neutrophil research, with some  exceptions, made use of  the biochemical tools available to generate specific results exploring restrict point-features of the neutrophil biology. As time  went  by,  technologies  like  2-DE,  LC,  MS  and  DIGE were gradually and smoothly incorporated by research groups interested in the neutrophil, which culminated in the modern multilateral  approaches  that can be observed  today. It  is  very  pleasing  to  contemplate  the  enormous  amount  of

data  that  has  been  generated  concerning  the  biology  of  the neutrophil and, more specifically, its proteome, with far over 700  proteins  identified  altogether.  On  the  other  hand,  researchers are now presented to an almost overwhelming array  of  protocols,  methods  and  hypothesis  for  studying  this cell.  That  is  why  we  feel  that  reviews  like  the  present  are crucial for optimizing and directing the work of researchers and research groups everywhere. In order to achieve such a goal, we are sure of the necessity of other reliable, broad and experiment-oriented  sources  of  information  on  various  aspects of neutrophil physiology, such as metabolic pathways, apoptosis, cytokine generation and function, granular release and function, and others.

Moreover,  the  present  paper  was  able  to  demonstrate some  of  the  challenges  in  neutrophil  research  at  the  same time that it brings a report on some of the new tendencies in the study of this cell. Whether it is the lack of a sequenced genome database for the study of a given species, or methodological  incompatibilities  between  protein  separation  and analytical  methods,  or  problems  in  analyzing  a  system  presenting both high- and low-abundance proteins, or the difficulties  in  acquiring  pure  and  abundant  samples,  there  is surely  still  a  lot  of  room  for  development  inside  neutrophil science and research.

## ACKNOWLEDGMENTS

The authors would like to thank Dr. Belchor Fontes, for his criticism regarding the medical portion of the manuscript, Ms. Larissa B. V. Pfeilsticker for the kind help with the illustration and FUB/UnB and CNPq for financial support.

## REFERENCES

| [1]      | Burg, N.D.; Pillinger, M.H. ( 2001 )  Clin. Immunol., 99,  7-17.                                                                                                                                                             |
|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| [2]      | Lee, W.L.; Harrison, R.E.; Grinstein, S. ( 2003 )  Microbes Infect. ,  5,  1299-1306.                                                                                                                                        |
| [3]  [4] | Bank, U.; Ansorge, S. ( 2001 )  J. Leukocyte Biol., 69,  197-206.  Ethuin, F.; Delarche, C.; Benslama, S.; Gougerot-Pocidalo, M.-A.;  Jacob,  L.;  Chollet-Martin,  S.  ( 2001 )  J.  Leukocyte  Biol.,  70,  439-           |
| [5]      | 446.  Cloutier, A.; Ear, T.; Blais-Charron, E.; Dubois, C.M.; McDonald,  P.P. ( 2007 )  J. Leukocyte Biol., 81,  567-577.                                                                                                    |
| [6]      | Cloutier, A.; McDonald, P.P. ( 2003 ) In: Cassatella, M.A.; Ed.  The  neutrophil:  and  emerging  regulator  of  inflammatory  and  immune  response.  Switzerland, Karger, pp. 1-23.                                        |
| [7]      | Herlaar, E.; Brown, Z. ( 1999 )  Mol. Med. Today ,  5,  439-447.                                                                                                                                                             |
| [8]      | Zychlinsky, A.; Weinrauch, Y.; Weiss, J. ( 2003 )  Microbes Infect.,  5,  1289-1291.                                                                                                                                         |
| [9]      | Stockelman, M.G.; Lorenz, J.N.; Smith, F.N.; Boivin, G.P.; Sahota,  A.;  Tischfield,  J.A.;  Stambrook,  P.J.  ( 1998 )  Am. J.  Physiol. ,  275,  F154-163.  J.                                                             |
| [10]     | Molesworth-Kenyon,  S.J.;  Oakes,  J.E.;  Lausch,  R.N.  ( 2005 )                                                                                                                                                            |
|          | Leukocyte Biol., 77,  552-559.                                                                                                                                                                                               |
| [11]     | Cunha, R.B.; Barbaro, K.C.; Muramatsu, D.; Portaro, F.C.; Fontes,  W.; de Sousa, M.V. ( 2003 )  J. Protein Chem. ,  22,  135-146.                                                                                            |
| [12]     | Machado, L.F.; Laugesen, S.; Botelho, E.D.; Ricart, C.A.; Fontes,  W.; Barbaro, K.C.; Roepstorff, P.; Sousa, M.V. ( 2005 )  Proteomics ,  5,  2167-2176.                                                                     |
| [13]     | Maciel,  N.M.;  Schwartz,  C.A.;  Rodrigues,  Pires,  Junior,  O.;  Seb- ben, A.; Castro, M.S.; Sousa, M.V.; Fontes, W.; Ferroni, Schwartz,  E.N. ( 2003 )  Comp. Biochem. Physiol. B. Biochem. Mol. Biol. ,  134,  641-649. |
| [14]     | Nascimento,  A.C.;  Fontes,  W.;  Sebben,  A.;  Castro,  M.S.  ( 2003 )  Protein Pept. Lett. ,  10,  227-238.                                                                                                                |
| [15]     | Cassatella, M.A. ( 1999 )  Adv. Immunol., 73,  369-509.  Fontes,  B.;  Fontes,  W.;  Utiyama,  E.M.;  Birolini,  D.  (                                                                                                       |
| [16]     | 1988 )  Dis.  Colon Rectum. ,  31,  298-302.                                                                                                                                                                                 |

- [17] Fontes,  W.;  Sousa,  M.V.;  Aragao,  J.B.;  Morhy,  L.  ( 1997 ) Arch. Biochem. Biophys. , 347, 201-207.
- [18] Kaye,  A.D.;  Hoover,  J.M.;  Baluch,  A.R.  ( 2005 ) Middle  East  J. Anesthesiol., 18, 273-292.
- [19] Ware, L.B. ( 2006 ) Semin. Respir. Crit. Care Med., 27, 337-349.
- [20] Yamashita, S.; Suzuki, A.; Yanagita, T.; Hirohata, S.; Kamada, M.; Toyoshima, S. ( 2000 ) Biol. Pharm. Bull., 23, 519-522.
- [21] MacDowell, A.L.; Peters,  S.P.  ( 2007 ) Curr. Allergy Asthma Rep., 7, 464-468.
- [22] Minor, A.H.; Burnett, L. ( 1949 ) Blood , 4, 667-669.
- [23] Minor, A.H.; Burnett, L. ( 1948 ) Blood , 3, 799-802.
- [24] Pember, S.O.;  Barnes,  K.C.;  Brandt,  S.J.;  Kinkade,  J.M.J.  ( 1983 ) Blood , 61, 1105-1115.
- [25] Haslett,  C.;  Guthrie,  L.A.;  Kopaniak,  M.M.;  Johnston,  R.B.,  Jr.; Henson, P.M. ( 1985 ) Am. J. Pathol. , 119, 101-110.
- [26] Szücs, S.; Varga, C.; Ember, I.; Kertai, P. ( 1994 ) J. Immunol. Methods , 167, 245-251.
- [27] de,  Souza,  Castro,  M.;  de,  Sa,  N.M.;  Gadelha,  R.P.;  de,  Sousa, M.V.;  Ricart,  C.A.;  Fontes,  B.;  Fontes,  W.  ( 2006 ) Protein  Pept. Lett. , 13, 481-487.
- [28] Sugawara,  T.;  Miyamoto,  M.;  Takayama,  S.;  Kato,  M.  ( 1995 ) J. Pharmacol. Toxicol. Methods, 33, 91-100.
- [29] Woodin, A.M. ( 1961 ) Biochem. J., 80, 562-572.
- [30] Woodin, A.M.; Wieneke, A.A. ( 1966 ) Biochem. J., 99, 493-500.
- [31] Woodin, A.M.; Wieneke, A.A. ( 1963 ) Biochem. J., 87, 480-487.
- [32] Woodin, A.M.; Wieneke, A.A. ( 1966 ) Biochem. J., 99, 479-492.
- [33] Woodin, A.M.; Wieneke, A.A. ( 1964 ) Biochem. J., 90, 498-509.
- [34] Woodin, A.M.; Wieneke, A.A. ( 1963 ) Biochem. J., 87, 487-495.
- [35] Woodin,  A.M.;  French,  J.E.;  Marchesi,  V.T.  ( 1963 ) Biochem.  J., 87, 567-571.
- [36] Hammerschmidt,  D.E.;  White,  J.G.;  Craddock,  P.R.;  Jacob,  H.S. ( 1979 ) J. Clin. Invest., 63, 798-803.
- [37] Fruhman, G.J. ( 1960 ) Blood , 16, 1753-1761.
- [38] Albrethsen, J.; Bogebo, R.; Gammeltoft, S.; Olsen, J.; Winther, B.; Raskov, H. ( 2005 ) BMC Cancer , 5.
- [39] Brewer, D.B. ( 1972 ) Br. Med. J., 2, 396-400.
- [40] Jacobs, A. ( 1964 ) J. Clin. Pathol., 17, 185-187.
- [41] Sramkova, L.; Kouba, K.; Bendova, N. ( 1965 ) Blood , 26, 479-488.
- [42] Brogan, T.D. ( 1966 ) Immunology 10, , 137-148.
- [43] Tullis, J.L. ( 1947 ) J. Clin. Invest., 26, 1098-1108.
- [44] Paape,  M.J.;  Pearson,  R.E.;  Wergin,  W.P.;  Guidry,  A.J.  ( 1977 ) J. Dairy Sci., 60, 53-62.
- [45] Sbarra,  A.J.;  Karnovsky,  M.L.  ( 1959 ) J.  Biol.  Chem.,  234, 13551362.
- [46] Botha,  A.J.;  Moore,  F.A.;  Moore,  E.E.;  Fontes,  B.;  Banerjee,  A.; Peterson, V.M. ( 1994 ) Shock , 3, 157-166.
- [47] Swain, S.D.R.T.T. and, Quinn, M.T. ( 2002 ) Antioxid. Red. Signal. , 4, 69-83.
- [48] Westermeier,  R. Electrophoresis  in  practice. Freiburg:  WileyVCH, ( 2005 ).
- [49] Westermeier, R.; Navem, T. Preoteomics , in, practice: a laboratory manual of proteome analysis. Freiburg: Wiley-VCH, ( 2002 ).
- [50] Nebelin, E.; Das, B.C. ( 1979 ) FEBS Letts. , 107, 254-258.
- [51] Horning, E.C.; Carroll, D.I.; Dzldic, I.; Haegele, K.D.; Lin, S.-N.; Oertli, C.U.; Stillwell, R.N. ( 1977 ) Clin. Chem., 23, 13-21.
- [52] Bowers, L.D. ( 1989 ) Clin. Chem., 35, 1282-1287.
- [53] Fowler, K.T.; Hugh-Jones, P. ( 1957 ) Br. Med. J., 1, 1205-1211.
- [54] Arnott,  D.;  Shabanowitz,  J.;  Hunt,  D.F.  ( 1993 ) Clin.  Chem.,  39, 2005-2010.
- [55] O'Farrel, P.H. ( 1975 ) J. Biol. Chem., 250, 4007-4021.
- [56] Oh-Ishi,  M.;  Satoh,  M.;  Maeda,  T.  ( 2000 ) Electrophoresis, 21, 1653-1669.
- [57] Pasquali,  C.;  Fialka,  I.;  Huber,  L.A.  ( 1997 ) Electrophoresis , 18, 2582-2590.
- [58] Cupo, J.F.; Lidgard, G.P.; Liehtman, W.F. ( 1990 ) Electrophoresis , 11, 500-504.
- [59] Hirano,  H.;  Kawasaki,  H.;  Sassa,  H.  ( 2000 ) Electrophoresis , 21, 440-445.
- [60] Gorg,  A.;  Obermaier,  C.;  Boguth,  G.;  Harder,  A.;  Scheibe,  B.; Wildgruber, R.; Weiss, W. ( 2000 ) Electrophoresis , 21, 1037-1053.
- [61] Hoving,  S.;  Gerrits,  B.;  Voshol,  H.;  Müller,  D.;  Roberts,  R.C.; Oostrum, J.V. ( 2002 ) Electrophoresis , 2, 127-134.
- [62] Berkelman, T.; Stenstedt, T. 2-D electrophoresis using immobilized pH gradients: principles and methods : Amershan Pharmacia Biotech, ( 1998 ).
- [63] Salzano, A.M.; Crescenzi, M. ( 2005 ) Ann Istituto Sup. Sanità , 41, 443-450.
- [64] Ens, W.; Standing, K.G. ( 2005 ) Methods Enzimol., 4 002, 49-78.
- [65] Cañas, B.; López-Ferrer, D.; Ramos-Fernández, A.; Camafeita, E.; Calvo, E. ( 2006 ) Brief. Funct. Genomic. Proteomic., 4, 295-320.
- [66] Qian, W.-J.; Jacobs, J.M.; Liu, T.; II, D.G.C.; Smith,  R.D. ( 2006 ) Mol. Cell. Proteomics , 5, 1727-1744.
- [67] Siuzdak, G. ( 1994 ) Proc. Nat. Acad. Sci. USA, 91, 11290-11297.
- [68] Listgarten, J.; Emili, A. ( 2005 ) Mol. Cell. Proteomics , 4, 419-434.
- [69] Mann, M.; Hendrickson, R.C.; Pandey, A. ( 2001 ) Annu. Rev. Biochem. 70, , 437-473.
- [70] Ünlü, M.; Morgan, M.E.; Minden, J.S. ( 1997 ) Electrophoresis , 18, 2071-2077.
- [71] Li, X.; Liu, X.; Lou, Z.; Duan, X.; Wu, H.; Liu, Y.; Rao, Z. ( 2004 ) Protein Sci. , 13, 2845-2851.
- [72] Smolka, M.; Zhou, H.; Aebersold, R. ( 2002 ) Mol. Cell. Proteomics , 1, 19-29.
- [73] Gygi, S.P.; Rist, B.; Gerber, S.A.; Turecek, F.; Gelb, M.H.; Aebersold, R. ( 1999 ) Nat. Biotechnol., 17, 994-999.
- [74] Cannataro,  M.;  Cuda,  G.;  Gaspari,  M.;  Greco,  S.;  Tradigo,  G.; Veltri, P. ( 2007 ) BMC Bioinformatics, 8, 255.
- [75] Vaughn,  C.P.;  Crockett, D.K.; Lim, M.S.; Elenitoba-Johnson, K.S.J. ( 2006 ) J. Mol. Diagn., 8, 513-520.
- [76] Boehm,  A.M.;  Pütz,  S.;  Altenhöfer,  D.;  Sickmann,  A.;  Falk,  M. ( 2006 ) BMC Bioinformatics, 8, 214.
- [77] Shadforth,  I.P.;  Dunkley,  T.P.J.;  Lilley,  K.S.;  Bessant,  C.  ( 2005 ) BMC Genomics 6, , 145.
- [78] Mantegazza, A.R.; Barrio, M.M.; Moutel, S.; Bover, L.; Weck, M.; Brossart,  P.;  Teillaud,  J.L.;  Mordoh,  J.  ( 2004 ) Blood , 104, 11831190.
- [79] Watt,  S.M.;  Burgess,  A.W.  ( 1981 ) Biochim.  Biophys.  Acta , 640, 583-594.
- [80] Gemmell,  M.A.;  Anderson,  N.L.  ( 1982 ) Clin.  Chem.,  28, 10621066.
- [81] Willard, K.E.; Thorsrud, A.K.; Munthe, E.; Jellum, E. ( 1982 ) Clin. Chem., 28, 1067-1073.
- [82] Girard,  D.;  Paquin,  R.;  Naccache,  P.H.;  Beaulieu,  A.D.  ( 1996 ) J. Leukocyte Biol., 59, 412-419.
- [83] Kasper, B.; Thole, H.H.; Patterson, S.D.; Welte, K. ( 1997 ) Electrophoresis , 18, 142-149.
- [84] Boussac, M.; Garin, J. ( 2000 ) Electrophoresis , 21, 665-672.
- [85] Lominadze, G.; Powell, D.W.;  Luerman, G.C.; Link, A.J.;  Ward, R.A.; McLeish, K.R. ( 2005 ) Mol. Cell. Proteomics , 4, 1503-1521.
- [86] Lominadze, G.; Ward, R.A.; Klein, J.B.; McLeish, K.R. In: Ali, H.; Bodduluri,  H.;  Eds. Methods in Molecular Biology. Totowa:  Humana Press, ( 2005 ) pp. 343-357.
- [87] Burlak, C.; Whitney, A.R.; Mead, D.J.; Hackstadt, T.; DeLeo, F.R. ( 2006 ) Mol. Cell. Proteom. , 5, 620-634.
- [88] Puellmann, K.; Kaminski, W.E.; Vogel, M.; Nebe, C.T.; Schroeder, J.; Wolf, H.; Beham, A.W. ( 2006 ) Proc. Nat. Acad. Sci. USA, 103, 14441-14446.
- [89] Shimada, S.; Nakurama, M.; Tanaka, Y.; Tsutsumi, K.; Katano, M.; Masuko,  K.;  Yudoh,  K.;  Koizuka,  I.;  Kato,  T.  ( 2007 ) Microbiol. Immunol., 51, 87-98.
- [90] Aquino,  E.N.  Análise  proteômica  comparativa  entre  neutrófilos quiescentes  e  estimulados  com  Fator  de  Agregação  Plaquetária (PAF)  Departamento  de  Biologia  Celular,  Brasília:  UnB,  ( 2008 ) pp. 140.
- [91] Ryckman, C.; Vandal, K.; Rouleau, P.; Talbot, M.v.; Tessier, P.A. ( 2003 ) J. Immunol., 170, 3233-3242.
- [92] Vandal,  K.;  Rouleau,  P.;  Boivin,  A.;  Ryckman,  C.;  Talbot,  M.v.; Tessier, P.A. ( 2003 ) J. Immunol., 171, 2602-2609.
- [93] DiMartino,  S.J.; Trujillo, G.;  McVoy, L.A.; Zhang, J.; Kew, R.R. ( 2007 ) Mol. Immunol., 44, 2370-2377.
- [94] Vong, L.; D'Acquisto, F.;  Pederzoli-Ribeil,  M.; Lavagno, L.;  Flower, R.J.; Witko-Sarsat, V.; Perretti, M. ( 2007 ) J. Biol. Chem., 282, 19998-30004.
- [95] Jethwaney,  D.;  Islam,  M.R.;  Leidal,  K.G.;  de,  Bernabe,  D.B.; Campbell,  K.P.;  Nauseef,  W.M.;  Gibson,  B.W.  ( 2007 ) Proteome Sci. , 5, 12.
- [96] Piubelli,  C.;  Galvani,  M.;  Hamdan,  M.;  Domenici,  E.;  Righetti, P.G. ( 2002 ) Electrophoresis , 23, 298-310.
- [97] Lippolis,  J.D.;  Reinhardt,  T.A.  ( 2005 ) Vet.  Immunol.  Immunopathol., 103, 53-65.
- [98] Ambatipudi,  K.;  Old,  J.;  Guilhaus,  M.;  Raftery,  M.;  Hinds,  L.; Deane, E. ( 2006 ) Comp. Biochem. Physiol., 4, 283-291.

## Neutrophil Proteome

- Mol. Immunol., 45,
- [100] Gygi,  S.P.;  Rochon,  Y.;  Franza,  B.R.;  Aebersold,  R.  ( 1999 ) Cell. Biol., 19, 1720-1730.
- [99] Ambatipudi, K.; Deane, E.M. ( 2008 ) 690-700. Mol.
- [101] Lian,  Z.;  Wang,  L.;  Yamaga,  S.;  Bonds,  W.;  Beazer-Barclay,  Y.; Kluger,  Y.;  Gerstein,  M.;  Newburger,  P.E.;  Berliner,  N.;  Weissman, S.M. ( 2001 ) Blood , 98, 513-524.
- [102] Fessler, M.B.;  Malcolm,  K.C.;  Duncan,  M.W.;  Worthen,  G.S. ( 2002 ) Chest  Supplement) 121, ( 75S-76S.
- [103] Fessler, M.B.;  Malcolm,  K.C.;  Duncan,  M.W.;  Worthen,  G.S. ( 2002 ) J. Biol. Chem., 277, 31291-31302.
- [104] Marcheselli, V.L.; Hong, S.; Lukiw, W.J.; Tian, X.H.; Gronert, K.; Musto, A.;  Hardy,  M.;  Gimenez,  J.M.;  Chiang,  N.;  Serhan,  C.N.; Bazan, N.G. ( 2003 ) J. Biol. Chem., 278, 51974-43817.

Received:  March 06, 2008

Revised: May 14, 2008

Accepted: June 02, 2008

- [105] Serhan,  C.N.;  Clish,  C.B.;  Brannon,  J.;  Colgan,  S.P.;  Chiang,  N.; Gronert, K. ( 2000 ) J. Exp. Med., 192, 1197-1204.
- [106] Hong, S.; Gronert, K.; Devchand, P.R.; Moussignac, R.-L.; Serhan, C.N. ( 2003 ) J. Biol. Chem., 278, 14677-14687.
- [107] Bannenberg, G.L.; Chiang, N.; Ariel, A.; Arita,  M.; Tjonahen, E.; Gotlinger, K.H.; Hong, S.; Serham, C.N. ( 2005 ) J. Immunol., 174, 4345-4355.
- [108] Theilgaard-Monch,  K.;  Porse,  B.T.;  Borregaard,  N.  ( 2006 ) Curr. Opin. Immunol. , 18, 54-60.