## Gene expression in mature neutrophils: early responses to inflammatory stimuli

Xueqing Zhang,* Yuval Kluger, ‚Ä† Yasuhiro Nakayama, ‚Ä† Ranjana Poddar, ‚Ä† Constance Whitney,* Adam DeTora,* Sherman M. Weissman, ‚Ä† and Peter E. Newburger* ,1

* Department of Pediatrics and Cancer Center, University of Massachusetts School of Medicine, Worcester; and the ‚Ä† Department of Genetics, Boyer Center for Molecular Medicine, Yale University School of Medicine, New Haven, Connecticut

Abstract: Neutrophils provide an essential defense against bacterial and fungal infection and play a major role in tissue damage during inflammation. Using oligonucleotide microarrays, we have examined the time course of changes in gene expression induced by stimulation with live, opsonized Escherichia coli , soluble lipopolysaccharide, and the chemoattractant formyl-methionyl-leucylphenylalanine. The results indicate that activated neutrophils generate a broad and vigorous set of alterations in gene expression. The responses included changes in the levels of transcripts encoding 148 transcription factors and chromatin-remodeling genes and 95 regulators of protein synthesis or stability. Clustering analysis showed distinct temporal patterns with many rapid changes in gene expression within the first hour of exposure. In addition to the temporal clustering of genes, we also observed rather different profiles associated with each stimulus, suggesting that even a nonvirulent organism such as E. coli is able to play a dynamic role in shaping the inflammatory response. Principal component analysis of transcription factor genes demonstrated clear separation of the neutrophil-response clusters from those of resting and stimulated human monocytes. The present study indicates that combinatorial transcriptional regulation including alterations of chromatin structure may play a role in the rapid changes in gene expression that occur in these terminally differentiated cells. J. Leukoc. Biol. 75: 358-372; 2004.

Key Words: mRNA /H18528 transcription /H18528 transcription factor /H18528 chromatin

## INTRODUCTION

Neutrophils provide a critical host defense against bacterial and fungal infections. They are present in large numbers in the circulation, through which they rapidly transit en route to tissue, where they deploy to ingest and kill invading microorganisms [1, 2]. As potent agents of the inflammatory response, they also play a major role in the inflammation and tissue

damage of noninfectious diseases such as arthritis, inflammatory bowel disease, and ischemia-reperfusion injury [3, 4].

A widespread but incorrect view of the neutrophil portrays it as a short-lived, terminally differentiated cell with a highly condensed nucleus and hence, unable to induce gene expression. However, in spite of its dense chromatin and paucity of total mRNA and ribosomes, the neutrophil has long been known to be capable of transcription-dependent synthesis of heat-shock protein [5], to express mRNA-encoding phagocytic receptors [6], and to modulate RNA synthesis in response to lectin stimulation or glucocorticoid treatment [7]. Neutrophils can up-regulate genes involved in phagocytic function, such as the genes encoding gp91phox and p22phox , the two components of the phagocyte cytochrome b [8, 9]. In addition, a variety of stimuli, defined and complex, up-regulates c-fos expression [10, 11].

Neutrophils not only respond to cytokines but also participate in the 'cytokine network' by secretion of interleukin (IL) and colony-stimulating factor (CSF) molecules, with regulation at the level of gene expression demonstrated in some studies. Stimulation with bacterial lipopolysaccharide (LPS) induces neutrophils to synthesize IL-1 ùõÇ and -1 ùõÉ [12, 13], as well as IL-1 receptor (IL-1R) antagonist [14]. Granulocyte macrophage (GM)-CSF induces neutrophil expression of IL-6 [15], and 5 ' lipoxygenase [16]. Tumor necrosis factorùõÇ (TNFùõÇ ) expression by neutrophils occurs in response to LPS, GM-CSF, and G-CSF [17], and the secreted cytokine creates a feedback loop for further neutrophil activation. In other positive-feedback loops, neutrophils augment their own recruitment by up-regulating expression of IL-8 and G-CSF [13, 18]. Conversely, incubation of neutrophils with G-CSF induces expression of interferonùõÇ (IFNùõÇ ) [19], which provides negative feedback to diminish myelopoiesis and neutrophil activation.

Several recent high-throughput studies have expanded the scope of neutrophil gene expression by showing a wide range of mRNAs present in unstimulated neutrophils and dramatic changes in transcript levels induced by stimulation with bacteria and other agonists [20-23]. Our previous studies using

cDNA-display analysis identified ‚àº 600 known genes and expressed sequence tags differentially expressed by human neutrophils exposed to bacteria and encoding a variety of cytokines, receptors, apoptosis-regulating products, and membrane-trafficking regulators [20].

Fessler et al. [21] performed genomic and proteomic analyses of neutrophil activation by LPS, noting the up-regulation of 100 distinct genes and focusing on the attenuation of the response by pretreatment with a p38 kinase inhibitor. Kobayashi et al. [22, 24] used oligonucleotide arrays to compare changes in human neutrophil transcript levels over a 6-h time-course, accompanying phagocytosis of opsonized particles mediated by Fc receptors for immunoglobulin G (IgG) versus complement receptors. Many differentially expressed genes related to at least three distinct apoptotic pathways, in contrast with a more limited number of genes involved in host defense. The onset of apoptosis was marked by changes in expression of nearly 200 genes involved in membrane trafficking and cell metabolism, including enzymes necessary for energy metabolism and antioxidant defenses [24]. Yang et al. [23] used microarray analysis to examine neutrophil activation by antineutrophil cytoplasmic antibodies from kidney disease patients, focusing on the induction of human differentiationdependent gene 2.

In the present study, we have examined in detail the timecourse of changes in gene expression during neutrophil activation and have compared the expression profiles induced by stimulation with live, opsonized Escherichia coli K-12 enterobacteria, soluble E. coli LPS, and the bacterially derived chemoattractant formyl-methionyl-leucyl-phenylalanine (fMLP). In particular, our data analysis has focused on the early-response genes, which should represent the most direct effects of exogenous stimuli and include the transcriptional regulators of subsequent changes in expression of the effector genes described in previous studies.

## MATERIALS AND METHODS

## Stimuli

Opsonized bacteria were prepared as described previously [20]. Briefly, cultures of E. coli K-12 strain R594supO grown overnight in Luria-Bertani medium were diluted 1:100 and further incubated for 2 h at 37¬∞C. The bacteria were then opsonized with 20% (v/v) complement factor C7-deficient human serum (Sigma Chemical Co., St. Louis, MO) in Hanks' balanced salt solution (HBSS) without calcium and magnesium (Life Technologies, Grand Island, NY) at 37¬∞C for 20 min. Serum deficient in C7 was used to avoid complementinduced cell lysis. Opsonized bacteria were washed twice in HBSS with 10% heat-inactivated, low-endotoxin fetal calf serum (HyClone, Logan, UT) and were resuspended at 1 √ó 10 10 /mL in HBSS. LPS (from E. coli 026:B6; Sigma Chemical Co.) and fMLP (Sigma Chemical Co.) were dissolved in HBSS.

## Isolation and stimulation of leukocytes

Neutrophils were isolated from venous blood (freshly drawn at 8-9 AM) of healthy volunteers, using dextran sedimentation and centrifugation through Ficoll-Paque Plus (Pharmacia, Uppsala, Sweden), as described previously [20]. Morphologically, the neutrophil preparations were more than 99% pure except for the presence of 1-3% eosinophils and &lt; 0.75% monocytes. Freshly purified neutrophils, ‚àº 2.5 √ó 10 8 cells for each time point, were suspended at ‚àº √ó 2 10 6 ml -1 in 80-100 ml RPMI medium (Life Technologies), supplemented with 10% heat-inactivated donor serum (from a previous blood draw, frozen after

heat-inactivation at 56¬∞C for 30 min). Neutrophils were equilibrated to room temperature, then stimulated by addition of opsonized E. coli (20:1 ratio of E. coli to neutrophils), fMLP (10 -7 M), or LPS (10 ng/mL), and incubated at 37¬∞C with gentle agitation in a temperature-controlled water bath. For each timecourse, a time-zero control was harvested before stimulation and subsequent samples at 10, 20, 30, and 60 min of incubation, plus a 120-min time-point for E. coli and LPS. Rapid chilling and then centrifugation at 1100 g at 4¬∞C terminated the incubation. Because of inter-donor differences in gene expression [25], we adopted a repeated-measures design of the experiments to treat the data as within-subjects variables to isolate each set of time-points for each neutrophil isolation from each donor. Each stimulation time-course was repeated at least three times with different donors.

Human monocytes were prepared and stimulated as described previously [26]. In brief, peripheral blood mononuclear cells were separated from leukopheresis samples by Ficoll-Paque density gradient centrifugation and allowed to adhere onto serum-coated, plastic, six-well, tissue-culture dishes (2 √ó 10 6 cells/well) for 2 h at 37¬∞C in a 5% CO 2 incubator. Nonadherent cells were removed by gentle washes, and half the wells with the purified monocytes adhering to the plastic were harvested for total RNA using RNAeasy (Qiagen, Valencia, CA). Monocytes were immunostained with anti-CD-14 antibody to assure ‚â• 95% purity.

The manufacturer certified all reagents-serum, buffers, media, and containers-as nonpyrogenic. Peripheral blood and leukopheresis samples were obtained from normal adult volunteers. The University of Massachusetts Medical Center Committees on Protection of Human Subjects in Research (Worcester) and Yale University Human Investigation Committee (New Haven, CT) approved all procedures and consent forms at their respective sites.

## RNA sampling and oligonucleotide array hybridization

Total RNA was extracted from neutrophils by a guanidine HCl method, as described previously [27, 28]. RNA samples were digested with DNase I at 37¬∞C for 30 min and cleaned by passage through RNeasy mini-spin columns (Qiagen). Running aliquots on native agarose gels monintored the quality of RNA. Each RNA sample (10 ùõç g) was used as starting material for cRNA preparation, as described previously [29]. In vitro transcription by T7 RNA polymerase (Ambion, Austin, TX) was performed in a nucleotide triphosphate mixture containing the biotinylated reagents Bio-11-CTP and Bio-16-UTP (Enzo Life Sciences, Famingdale, NY). Labeled cRNA samples and fragmentations were checked on native agarose gels, and 15 ùõç g fragmentation products were hybridized to HG\_U95A version 2 GeneChip arrays (Affymetrix, Santa Clara, CA), according to the manufacturer's instructions.

## Data analysis

Affymetrix MicroArray Suite version 5.0 managed raw data output, with average intensity (target signal) set as 150. The absolute expression (detection) analysis used the statistical algorithm of Microarray Suite 5.0. To reduce noise in E. coli time-course samples as a result of bacterial RNA contamination, raw signals were fitted to the Li and Hung [30] and Li and Wong [31] noise model, implemented in DNA chip-analyzer software (dChip; http://www.dchip.org/). The probe-sensitivity index file generated from all of the nonE. coli timecourse samples was applied to all data, including the E. coli samples, to generate model-based expression index (MBEI) values. The Log2-transformed MBEIs were first filtered by a flooring filter; that is, at least one group (stimulus and time) needed to have signal values up to the average signal level of marginal calls [Log2(MBEI) = 7 or MicroArray Suite 5.0 signal = 128 (for details, see Table S1 in the online supplementary material http://www.jleukbio. org/cgi/full/jlb.0903412/DC1)]. The floored MBEI data were used for an F-test of each time-course by significance analysis of microarrays (SAM) software [32], a multiple testing approach based on permutation resampling and designed for microarray analysis involving small numbers of replicates. The MBEI data without log transformation were used for a group-comparison analysis, implemented in dChip, which calculates weighted group means to reduce bias and uses a lower 90% confidence bound of relative change for filtering. The 'Affy' package of the R project BioConductor (http://www.bioconductor.org/) [29, 33], which is based on different assumptions and uses a different normalization approach, was used to generate the median polish estimates of expression levels. The data normalized by Affy were subjected to filtering with same floor filter as described above and to the SAM multiclass test using F statistics with the same setting.

Genes were classified as showing a significant change in expression level if they met the following criteria for an arbitrary threshold of relative change in comparisons of group means and a P -value threshold in statistical hypothesis testing: For at least for one time-point and stimulus, the weighted mean of the gene's expression level had a relative change ‚â• 1.8-fold (lower 90% confidence bound-of-fold change) and an absolute difference ‚â• 64 compared with its corresponding time-zero group; and for at least one stimulation time-course, the gene must be rejected by SAM using F statistics with a false discovery rate set at 0.01.

Genes with significantly different levels of expression over the time-course, as identified by the approach described above, were then subjected to hierarchical clustering to group the genes according to similarity defined by correlation coefficient. The k-means cluster was used to test the stability of the hierarchical clusters. The functional annotations of significant genes were organized with GoSurfer software (http://biosun1.harvard.edu/complab/ gosurfer). Small, relevant branches and unresolved genes (with limited or ambiguous Gene Ontology annotation) were merged manually, based on information in National Center for Biotechnology Information (http://www.ncbi. nlm.nih.gov/) databases.

## Real-time reverse transcriptase-polymerase chain reaction (RT-PCR)

RT and PCR analysis using real-time PCR verified microarray findings for multiple genes [34]. Target regions for PCR primers were selected from the human oligonucleotide set for use in microarray and PCR measures of RNA designed by the Harvard-Lipper Center for Computational Genetics [35] (http://arep.med.harvard.edu/probes.htm) and based on the human UniGene dataset Build #158 (http://www.ncbi.nlm.nih.gov/); or from Affymetrix probe selection regions within the target sequence of the Affymetrix HG\_U133 set (http://www.affymetrix.com/analysis/download\_center.affx), based on human UniGene Build #133. Primers were designed to yield amplicons of 100-150 bp.

Aliquots of the same RNA samples used for microarray analyses served as templates for the RT reactions. First-strand cDNA was synthesized from 50 ng DNaseI-treated, total RNA using SuperScript II RT (Life Technologies, Rockville, MD). Quantitative PCR was performed on a Smart Cycler (Cepheid, Sunnyvale, CA), using SYBR Green I fluorescent dye (QuantiTech SYBR Green PCR, Qiagen). A comparative threshold cycle (C ) method was used to t process the real-time PCR data [36]. A panel of six constitutively expressed genes, most from the Affymetrix HG-U133A set normalization controls (http:// www.affymetrix.com/support/technical/byproduct.affx?product = hgu133), was chosen as reference genes for normalization. Results were calculated as the normalized difference in Ct for a given time-point relative to the time-zero baseline ( ùö´ùö´ Ct ), calculated by subtracting the C t of the reference gene from the Ct of the target gene for each time point ( ùö´ Ct ) and then subtracting the ùö´ Ct,cb of time zero from the ùö´ Ct,q of time-point. Amplicon size and specificity were verified by agarose gel electrophoresis.

## Principal component analysis of transcription factors

To analyze the differences and similarities between neutrophils and monocytes in resting or activated states, we examined the expression profiles of 474 transcription factor genes that had at least one sample with an Affymetrix 'present call'. We performed a principal component analysis among 31 datasets representing resting neutrophils, neutrophils stimulated with E. coli for 30 or 120 min, resting monocytes, and monocytes stimulated with E. coli for 120 min. As a preliminary step in searching for differentially expressed transcription factor genes among the five different cell types, we excluded 180 transcription factor genes for which we did not have complete data or genes that have Affymetrix 'absent calls' in all samples. To verify that the 31 samples are separable into five distinctive groups, despite the expected sample similarities among related cell types, we visualized their distribution in a reduced, two-dimensional space. To reduce the dimensionality from 474 (the number of transcription factors used in profiling each sample), we used a variant of principal components analysis. This procedure allows projection of the data onto a two-dimensional space spanned by the two leading principal components, which are the linear combinations of the normalized expression values of the 474 transcription factors that capture most of the variance of the complete data.

The projections of the samples onto the leading principal components are computed by applying the singular value decomposition technique on the transformed data matrix. The latter is obtained by a preprocessing, normalization step, as described recently [37]. We selected a normalization procedure based on the concept that two genes and likewise, two samples, whose expression profiles differ only by a multiplicative constant of proportionality, are really behaving in the same way. That is, we consider two genes (each represented by a row of the matrix) or two samples (represented by the columns of the matrix) to be equivalent if their expression profiles differ by a multiplicative constant. This adjustment involves independent rescaling of the rows and columns as described previously [37].

To identify the differentially expressed transcription factors that have at least one cell type with a mean normalized-expression value significantly different from means of samples of other cell types, we applied a one-way ANOVA [38] and its nonparametric Kruskal-Wallis test [39]. Transcription factors that return P values &lt; 0.0001 using the classical ANOVA or &lt; 0.01 using its nonparametric version cast doubt on the null hypothesis that the means of all cell types are equal. Small P values suggest that at least one sample mean is significantly different than the other sample means. By applying the classical one-way ANOVA, we identified a subset of 96 transcription factors with P values &lt; 0.0001. In the ANOVA test, it is assumed that all sample populations have equal variance and are normally distributed (in addition to the assumption that all observations are mutually independent); therefore, we also applied the less-restrictive Kruskal-Wallis test on this subset of 96 genes. We found that 90% of genes have P values &lt; 0.001, indicating that both tests select similar lists of differentially expressed genes. Once we identified a differentially expressed transcription factor, we determined which pairs of the normalized expression means of the different cell types are significantly different using a multiple comparison procedure [40] based on Tukey's honestly significant difference criterion as a conservative test. This procedure is designed to provide an upper bound on the probability that any comparison will be incorrectly found significant. The output contains the estimated difference in means of each pair of cell types and an adjustable confidence interval for the difference, which we fixed at 99.99%. If the confidence interval does contain zero, the difference would not be significant at the 0.0001 level. Thus, if the confidence interval of the difference of means of two cell types does not contain zero, we considered the transcription factor to have significant differential expression between this pair of cell types.

## RESULTS

## The neutrophil transcriptome

The set of all gene transcripts in a specific cell, termed a transcriptome, includes a subset of mRNAs common to most cells in the organism plus a subset associated with the cell's particular phenotype and function. Both subsets contain constitutively expressed and differentially regulated genes. Although the present study focused on the latter, it also provided an opportunity for quantitative assessment of the entire transcriptome of resting and activated neutrophils.

Although there is no consensus approach for the identification of a transcriptome, an absolute detection analysis by the microarray manufacturer's procedure provides a good estimate. For Affymetrix HG\_U95Av2 chips, which display ‚àº 10,000 transcripts represented by 12,561 probe sets, the total number of neutrophil transcripts eliciting a present signal constituted ‚àº 4400 probe sets (35% of the total), representing ‚àº 3500 unique entries (for details, see Table S1 in the online supplementary material). The numbers of transcripts detected as present in resting and stimulated neutrophils varied only slightly within a narrow range and correspond closely to a previous measure of neutrophil gene expression using similar microarrays [22] and to an alternative calculation of the present data by dChip software. Microarray measurements of transcrip-

ize in other cell systems have reported similar proportions of positive signals [41-44].

Using criteria detailed above, 976 unique entries in the UniGene (Build #95) database were identified as differentially expressed in response to stimulation by three microbe-derived stimuli: particulate-opsonized E. coli , soluble LPS, and soluble chemoattractant fMLP (see Fig. 1 ). Of these identified entries, 126 represent genes that currently have no functional annotation in the public databases and so were not included in further analysis. Among the 850 annotated response genes, only 16

Fig. 1. Hierarchical clustering of genes expressed in human neutrophils and responsive to stimulation by opsonized E. coli , LPS, and fMLP. The 'heat map' presents up- and down-regulation of mRNA levels detected on human genome HG95 microarrays, normalized and filtered as described in Materials and Methods. Rows correspond to different genes, and the columns represent the various time-points after exposure to each stimulus [time 0 (unstimulated) and times 30, 60, 90, and 120 min for E. coli and LPS; times 30, 60, and 90 min for fMLP, from left to right, respectively). (a) The final clustering tree, with omission of the initial branch between the upper, down-regulated cluster (blue dendogram) and the lower, up-regulated clusters. (Purple and red dendograms indicate early/transient and late up-regulation, respectively.) (b) The expanded cluster shows transient responses in greater detail, and gene symbols are indicated to the right of each row. The scales below the heat maps indicate the relative changes in gene expression represented by the range of colors.

<!-- image -->

<!-- image -->

Fig. 2. Unique and overlapping responses to neutrophil stimuli. The Venn diagram presents the number of genes that showed significant changes in expression in response to the indicated stimuli. (a) Repressed genes; (b) up-regulated genes.

<!-- image -->

genes had group mean signals below the average absent level (30.68) at time zero (resting), and 84 genes had a group mean signal under the average marginal level (96.73). The paucity of low or undetectable signals indicates that most changes occurred within a detectable range, rather than representing absolute 'on/off' switching, and suggests that most of the cellular response is derived from the active transcriptome, through wide-ranging, quantitative fine-tuning of gene expression.

## General features of gene-expression responses in neutrophils

The 850-member annotated dataset of significant responses consisted of 665 genes responding to E. coli , 402 to LPS, and 253 to fMLP (represented by each circle in Fig. 2 ), among which there was considerable overlapping of responses. The intersection of responses to three different classes of stimuli (illustrated in the Venn diagram in Fig. 2) indicates a possible set of 'core' immunostimulatory-response genes, as also observed in other cell types of the innate-immune system [4547]. This set includes many of the basic early-response genes of circulating neutrophils to the gram-negative bacteria, which included a high proportion of transcription regulators (50/122) as well as small protein-conjugating enzymes and chaperones. However, all major functional categories are represented in the core set.

The set of annotated, differentially expressed genes can be functionally organized by 'gene ontology' (http://www. geneontology.org/) into the following major categories: transcriptional regulation, translation, cytokines and growth factors (including receptors and signal transduction), cell motility and cytoskeleton, apoptosis, regulation of cell proliferation, host defense response, Rho family small GTPases, transport, and enzymes. A complete list of genes in this classification scheme is available as Table S2 in the online supplementary materials. Categories of particular interest are discussed below.

The largest group, surprisingly, is a collection of 245 genes involved in the multiple layers regulating gene expression, including 148 transcription-related genes ( Table 1 ) encoding transcription factors, transcription activators, chromatin remodeling and modifying proteins, DNA helicases and other DNA-binding proteins, as well as 95 regulators of protein synthesis or stability ( Table 2 ), including translation factors, ubiquitination machinery or small protein-conjugating enzymes, and chaperones. The two processes of transcription and

TABLE 1. Differentially Expressed Genes Involved in Transcriptional Regulation

|                                                                                               |              |              | Expression* fMLP LPS   | Expression* fMLP LPS   | Expression* fMLP LPS   |
|-----------------------------------------------------------------------------------------------|--------------|--------------|------------------------|------------------------|------------------------|
| Functional category /gene name                                                                | Symbol       | LocusLink ID | E. coli                |                        |                        |
| Transcription factors                                                                         |              |              |                        |                        |                        |
| (SMAD7) MAD, mothers against decapentaplegic homolog 7 (Drosophila)                           | MADH7        | 4092         | -                      | u                      | -                      |
| activating transcription factor 3                                                             | ATF3         | 467          | u                      | u                      | -                      |
| activating transcription factor 4 (tax-responsive enhancer element B67)                       | ATF4         | 468          | d                      | d                      | t                      |
| basic helix-loop-helix domain containing, class B, 2                                          | BHLHB2       | 8553         | u                      | u                      | -                      |
| bromodomain containing 2                                                                      | BRD2         | 6046         | -                      | -                      | l                      |
| BTB and CNC homology 1, basic leucine zipper transcription factor 1                           | BACH1        | 571          | d                      | -                      | -                      |
| BTG family, member 2                                                                          | BTG2         | 7832         | t                      | t                      | t                      |
| cAMP response element-binding protein (CRE-BPa)                                               | H\_GS165L15.1 | 9586         | d                      | -                      | -                      |
| CRE-BP3 (luman)                                                                               |              | 10488        | -                      | -                      | l                      |
|                                                                                               | CREB3        |              | u                      |                        |                        |
| CRE modulator                                                                                 | CREM         | 1390         |                        | u                      | u                      |
| CCAAT/enhancer-binding protein (C/EBP), ùõÖ                                                     | CEBPD        | 1052         | d                      | -                      | -                      |
| CCCTC-binding factor (zinc finger protein)                                                    | CTCF         | 10664        | d                      | -                      | -                      |
| CCR4-NOT transcription complex, subunit 3                                                     | CNOT3        | 4849         | d                      | -                      | d                      |
| c-myc promoter-binding protein                                                                | IRLB         | 10260        | l                      | -                      | -                      |
| cytoskeleton-associated protein 4 (p63)                                                       | CKAP4        | 10970        | -                      | -                      | l                      |
| early growth response 1                                                                       | EGR1         | 1958         | t                      | t                      | -                      |
| early growth response 2 (Krox-20 homolog, Drosophila)                                         | EGR2         | 1959         | t                      | t                      | t                      |
| early growth response 3                                                                       | EGR3         | 1960         | u                      | u                      | t                      |
| ELL-related RNA polymerase II, elongation factor                                              | ELL2         | 22936        | u                      | u                      | u                      |
| Ets2 repressor factor                                                                         | ERF FOSB     | 2077 2354    | - u                    | u                      | -                      |
| FBJ murine osteosarcoma viral oncogene homolog B forkhead box O1A (rhabdomyosarcoma)          | FOXO1A       | 2308         | d                      | u                      | -                      |
| FOS-like antigen 2                                                                            | FOSL2        | 2355         | l                      | - -                    | d -                    |
| Friend leukemia virus integration 1                                                           | FLI1         | 2313         | d                      | -                      | -                      |
| gem (nuclear organelle)-associated protein 4                                                  | GEMIN4       | 50628        | u                      | u                      | u                      |
| general transcription factor II, i                                                            | GTF2I        | 2969         | d                      | -                      | -                      |
| general transcription factor IIB                                                              | GTF2B        | 2959         | u                      | u                      | -                      |
| general transcription factor IIE, polypeptide 1, ùõÇ 56 kDa                                     | GTF2E1       | 2960         | d                      | -                      | -                      |
| general transcription factor IIH, polypeptide 1, 62 kDa                                       | GTF2H1       | 2965         | u                      | u                      | u                      |
| hairy homolog (Drosophila)                                                                    | HES1         | 3280         | u                      | -                      | -                      |
| hematopoietically expressed homeobox                                                          | HHEX         | 3087         | d                      | -                      | d                      |
| human immunodeficiency virus type I enhancer-binding protein 1                                | HIVEP1       | 3096         | -                      | -                      | l                      |
| interleukin enhancer-binding factor 3, 90 kDa                                                 | ILF3         | 3609         | -                      | u                      | -                      |
| jun-B proto-oncogene                                                                          | JUNB         | 3726         | t                      | t                      | -                      |
| jun-D proto-oncogene                                                                          | JUND         |              |                        |                        |                        |
| NF- ùõã B 1 (p105)                                                                              | NFKB1        | 3727         | -                      | -                      | d                      |
| NF- ùõã B 2 (p49/p100)                                                                          |              | 4790         | l                      | -                      | l                      |
|                                                                                               | NFKB2        | 4791         | -                      | -                      | u                      |
| nuclear factor (erythroid-derived 2)-like 2                                                   | NFE2L2       | 4780         | u                      | u                      | u                      |
| nuclear factor of activated T cells, calcineurin-dependent 3                                  | NFATC3       | 4775         | -                      | -                      | d                      |
| nuclear receptor subfamily 1, group D, member 1                                               | NR1D1        | 9572         | l                      | -                      | -                      |
| nuclear receptor subfamily 4, group A, member 1                                               | NR4A1        | 3164         | u                      | t                      | -                      |
| nuclear receptor subfamily 4, group A, member 2                                               | NR4A2        | 4929         | d                      |                        |                        |
| nuclear receptor subfamily 4, group A, member 3                                               | NR4A3        | 8013         | u                      | t u                    | t u                    |
| nuclear transcription factor Y, ùõÇ (CAAT-box-binding protein subunit A)                        | NFYA         | 4800         | u                      | u                      | t                      |
| pleiomorphic adenoma gene-like 2                                                              | PLAGL2       | 5326         | -                      | -                      | u                      |
| polymerase (RNA) II (DNA-directed) polypeptide B, 140 kDa                                     | POLR2B       | 5431         | d                      | -                      | -                      |
| repressor of estrogen receptor activity                                                       | REA          | 11331        | d                      | -                      | d                      |
| runt-related transcription factor 3                                                           | RUNX3        | 864          | u                      |                        | u                      |
| serum response factor (c-fos serum response element-binding)                                  | SRF          | 6722         |                        | u                      | -                      |
| signal transducer and activator of transcription 1, 91 kDa                                    | STAT1        | 6772         | - d                    | u -                    | -                      |
| skeletal muscle-abundant protein                                                              | BRD8 SNAPC1  | 10902        | d                      | d                      | d                      |
| small nuclear RNA-activating complex, polypeptide 1, 43 kDa                                   |              | 6617         | l                      | -                      | -                      |
| TIA1 cytotoxic granule-associated RNA-binding protein-like 1                                  | TIAL1        | 7073         | d                      | -                      | -                      |
| transcription factor 8 (represses interleukin-2 expression)                                   | TCF8         | 6935         | t                      | t                      | t                      |
| transcription factor-like 4                                                                   | TCFL4        | 6945         | d                      | -                      | d                      |
| transcription factor-like 5 (basic helix-loop-helix)                                          | TCFL5        | 10732        | -                      | -                      | l                      |
| transcriptional activator of the c-fos promoter                                               | CROC4        | 10485        | l                      | -                      | -                      |
| v-ets erythroblastosis virus E26 oncogene homolog 2 (avian)                                   | ETS2         | 2114 2353    | u t                    | u t                    | u t                    |
| v-fos FBJ murine osteosarcoma viral oncogene homolog                                          | FOS VDR      | 7421         | -                      | u                      | u                      |
| vitamin D (1,25-dihydroxyvitamin D3) receptor v-jun sarcoma virus 17 oncogene homolog (avian) | JUN          | 3725         | t                      | t                      | -                      |
| v-maf musculoaponeurotic fibrosarcoma oncogene homolog F (avian)                              | MAFF         | 23764        | u                      | u                      | u                      |
| zinc finger protein 238                                                                       | ZNF238       | 10472        | d                      | -                      | -                      |

|                                                                                                                                                      |                |              | Expression* fMLP LPS   | Expression* fMLP LPS   | Expression* fMLP LPS   |
|------------------------------------------------------------------------------------------------------------------------------------------------------|----------------|--------------|------------------------|------------------------|------------------------|
| Functional category /gene name                                                                                                                       | Symbol         | LocusLink ID | E. coli                |                        |                        |
| Transcription cofactors                                                                                                                              |                |              |                        |                        |                        |
| calcineurin-binding protein 1                                                                                                                        | CABIN1         | 23523        | d                      | -                      | d                      |
| arginine-glutamic acid dipeptide (RE) repeats                                                                                                        | RERE           | 473          | d                      | -                      | d                      |
| B-cell CLL/lymphoma 3                                                                                                                                |                | 602          | -                      | -                      | u                      |
| c-myc-binding protein                                                                                                                                | BCL3 MYCBP     | 26292        | d                      | -                      | -                      |
| cyclin D-binding myb-like transcription factor 1                                                                                                     | DMTF1          | 9988         | d                      | -                      | -                      |
| ùõÖ sleep-inducing peptide, immunoreactor                                                                                                              |                |              | d                      | -                      | d                      |
| DNA damage-inducible transcript 3 (C/EBP                                                                                                             | DSIPI          | 1831         | u                      | u                      |                        |
| ùõá ) HMT1 hnRNP methyltransferase-like 1 (                                                                                                            | DDIT3          | 1649         | d                      |                        | -                      |
| Saccharomyces cerevisiae ) human immunodeficiency virus type 1 enhancer-binding protein 2                                                            | HRMT1L1 HIVEP2 | 3275 3097    | -                      | - -                    | - l                    |
|                                                                                                                                                      |                |              | d                      | -                      |                        |
| Kruppel-like factor 7 (ubiquitous)                                                                                                                   | KLF7           | 8609         |                        |                        | -                      |
| LIM domain only 4                                                                                                                                    | LMO4           | 8543         | d                      | -                      | d                      |
| MAD, mothers against decapentaplegic homolog 4 (Drosophila)                                                                                          | MADH4          | 4089         | d                      | -                      | -                      |
| MAX dimerization protein 1                                                                                                                           | MAD            | 4084         | d                      | d                      | d                      |
| metal-regulatory transcription factor 1                                                                                                              | MTF1           | 4520         | -                      | -                      | l                      |
| NGFI-A-binding protein 1 (EGR1-binding protein 1)                                                                                                    | NAB1           | 4664         | -                      | -                      | l                      |
| nuclear factor (erythroid-derived 2), 45 kDa                                                                                                         | NFE2           | 4778         | d                      | d                      | d                      |
| nuclear factor, interleukin-3-regulated                                                                                                              | NFIL3          | 4783         | u                      | u                      | -                      |
| nuclear receptor coactivator 4                                                                                                                       | NCOA4          | 8031         | d                      | -                      | -                      |
| sex comb on midleg-like 2 (Drosophila)                                                                                                               | SCML2          | 10389        | u                      | u                      | -                      |
| SMART/HDAC1-associated repressor protein                                                                                                             | SHARP          | 23013        | d                      | -                      | -                      |
| TGFB-inducible early growth response                                                                                                                 | TIEG           | 7071         | u                      | u                      | u                      |
| TGFB-induced factor (TALE family homeobox)                                                                                                           | TGIF           | 7050         | u                      | u                      | u                      |
| TGFB-induced factor 2 (TALE family homeobox)                                                                                                         | TGIF2          | 60436        | -                      | -                      | u                      |
| transcriptional regulator interacting with the PHS-bromodomain 2                                                                                     | TRIP-Br2       | 9792         | t                      | t                      | t                      |
| ubinuclein 1                                                                                                                                         | UBN1           | 29855        | d                      | -                      | d                      |
| v-rel reticuloendotheliosis viral oncogene homolog (avian)                                                                                           | REL            | 5966         | -                      | -                      | u                      |
| v-rel reticuloendotheliosis viral oncogene homolog A, NF- ùõã B 3, p65 (avian) v-rel reticuloendotheliosis viral oncogene homolog B, NF- ùõã B 3 (avian) | RELA RELB      | 5970 5971    | - -                    | - -                    | l u                    |
| Wilms' tumor1-associating protein                                                                                                                    |                | 9589         | l                      | -                      | l                      |
| zinc finger protein 131 (clone pHZ-10)                                                                                                               | WTAP ZNF131    | 7690         | -                      | u                      | -                      |
| zinc finger protein 165                                                                                                                              | ZNF165         | 7718         | u                      | u                      | -                      |
| zinc finger protein 216                                                                                                                              | ZNF216         | 7763         | -                      | -                      | d                      |
|                                                                                                                                                      |                |              | d                      |                        |                        |
| zinc finger protein 217                                                                                                                              | ZNF217         | 7764         |                        | d                      | d                      |
| zinc finger protein 263                                                                                                                              | ZNF263         | 10127        | l                      | -                      | -                      |
| zinc finger protein 267                                                                                                                              | ZNF267         | 10308        | u                      | u                      | u                      |
| zinc finger protein 313                                                                                                                              | ZNF313         | 55905        | d                      | -                      | -                      |
| zinc finger protein 36, C3H type-like 2                                                                                                              | ZFP36L2        | 678          | d                      | -                      | -                      |
| zinc finger protein 384                                                                                                                              | ZNF384         | 171017       | d                      | -                      | -                      |
| zinc finger protein 44 (KOX 7)                                                                                                                       | ZNF44          | 7595         | -                      | u                      | -                      |
| Chromatin binding and remodeling                                                                                                                     |                |              |                        |                        |                        |
| bromodomain adjacent to zinc finger domain, 2A                                                                                                       | BAZ2A          | 11176        | d                      | -                      | -                      |
| chromatin assembly factor 1, subunit A (p150)                                                                                                        | CHAF1A         | 10036        | -                      | -                      | l                      |
| chromobox homolog 1 (HP1 ùõÉ homolog Drosophila)                                                                                                       | CBX1           | 10951        | d                      | -                      | d                      |
| chromodomain helicase DNA-binding protein 1                                                                                                          | CHD1           | 1105         | u                      | u                      | u                      |
| enhancer of zeste homolog 1 (Drosophila)                                                                                                             | EZH1           | 2145         | d                      | -                      | d                      |
| high-mobility group nucleosomal-binding domain 4                                                                                                     | HMGN4          | 10473        | d                      | -                      | d                      |
|                                                                                                                                                      |                | 3148         | d                      | -                      | d                      |
| high-mobility group box 2                                                                                                                            | HMGB2          | 23119        | -                      | u                      | -                      |
| hypermethylated in cancer 2 lamin B receptor                                                                                                         | HIC2 LBR       | 3930         | d                      | -                      | -                      |
| male-specific lethal 3-like 1 (Drosophila)                                                                                                           | MSL3L1         | 10943        | d                      | -                      | -                      |
| PHD finger protein 2                                                                                                                                 | PHF2           | 5253         | d                      | -                      | d                      |
| PHD finger protein 3                                                                                                                                 | PHF3           | 23469        | d                      | -                      | -                      |
| suppressor of Ty 6 homolog ( S. cerevisiae )                                                                                                         | SUPT6H         | 6830         | u                      | u                      | u                      |
| DNA helicase and other DNA binding                                                                                                                   |                |              |                        |                        |                        |
| Ac-like transposable element                                                                                                                         | ALTE           | 9189         | d                      | -                      | -                      |
| BRAF35/HDAC2 complex (80 kDa)                                                                                                                        | BHC80          | 51317        | -                      | -                      | d                      |
| cutaneous T cell lymphoma-associated tumor antigen se20-4                                                                                            | SE20-4         | 64061        | -                      | u                      | -                      |
| H1 histone family, member X                                                                                                                          | H1FX           | 8971         | -                      | -                      | l                      |
| H2.0-like homeo box 1 (Drosophila)                                                                                                                   | HLX1           | 3142         | d                      | d                      | d                      |
| H2A histone family, member L                                                                                                                         | HIST1H2AC      | 8334         | -                      | -                      | d                      |
| H2A histone family, member O                                                                                                                         | HIST2H2AA      | 8337         | u                      | u                      | -                      |
| H3 histone family, member B                                                                                                                          | HIST1H3D       | 8351         | d                      | -                      | d                      |
| helicase with SNF2 domain 1                                                                                                                          | HELSNF1        | 57680        | d                      | -                      | -                      |

|                                                                           |         |              | Expression*   | Expression*   | Expression*   |
|---------------------------------------------------------------------------|---------|--------------|---------------|---------------|---------------|
| Functional category /gene name                                            | Symbol  | LocusLink ID | E. coli       | fMLP          | LPS           |
| high-mobility group AT-hook 1                                             | HMGA1   | 3159         | u             | u             | u             |
| histone deacetylase 2                                                     | HDAC2   | 3066         | -             | -             | l             |
| makorin, ring finger protein, 1                                           | MKRN1   | 23608        | d             | -             | d             |
| methyl-CpG-binding domain protein 4                                       | MBD4    | 8930         | d             | -             | d             |
| muscleblind-like (Drosophila)                                             | MBNL1   | 4154         | d             | -             | -             |
| mixed-lineage leukemia (trithorax homolog, Drosophila); translocated to 2 | MLLT2   | 4299         | -             | -             | d             |
| MYST histone acetyltransferase 1                                          | MYST1   | 84148        | d             | -             | -             |
| polymerase (DNA-directed) ùõî                                               | POLS    | 11044        | -             | u             | -             |
| rearranged L-myc fusion sequence                                          | RLF     | 6018         | u             | u             | -             |
| Ret transforming gene                                                     | RFP     | 5987         | d             | d             | d             |
| runt-related transcription factor-binding protein 2                       | RUNXBP2 | 7994         | d             | -             | -             |
| single-stranded DNA-binding protein Mssp-1                                | RBMS1   | 5937         | d             | -             | d             |
| small optic lobes homolog (Drosophila)                                    | SOLH    | 6650         | d             | -             | -             |
| SWI/SNF-related, regulator of chromatin, subfamily a, member 2            | SMARCA2 | 6595         | d             | -             | -             |
| topoisomerase (DNA) I                                                     | TOP1    | 7150         | -             | u             | -             |
| topoisomerase (DNA) I pseudogene 2                                        | TOP1P2  | 7152         | -             | u             | -             |
| topoisomerase (DNA) II-binding protein                                    | TOPBP1  | 11073        | d             | -             | -             |
| tripartite motif-containing 33                                            | TRIM33  | 51592        | d             | -             | d             |
| tumor protein p53-binding protein                                         | TP53BPL | 10210        | d             | -             | -             |
| X-box-binding protein 1                                                   | XBP1    | 7494         | u             | -             | u             |
| xeroderma pigmentosum, complementation group C                            | XPC     | 7508         | d             | d             | d             |
| X-ray repair complementing defective repair in Chinese hamster cells 5    | XRCC5   | 7520         | d             | -             | -             |
| Polyamine metabolism                                                      |         |              |               |               |               |
| ornithine decarboxylase antizyme 2                                        | OAZ2    | 4947         | d             | -             | -             |
| ornithine decarboxylase antizyme inhibitor                                | OAZIN   | 51582        | u             | u             | u             |

protein synthesis/stability are not completely distinct. For example, polyamine metabolism alters DNA-protein interactions [48] but also contributes to an autoregulatory feedback loop controlled by translational frameshifting [49]. Conversely, chaperone proteins not only facilitate proper folding of polypeptides and protect fully translated proteins from aggregation but also stimulate the activity of histone-deacetylase complexes [50], which are essential for remodeling chromatin structure and transcriptional accessibility [51].

The second largest group of 124 genes was classified into the category of signaling pathway genes [such as A2A adenosine receptor (ADORA2A), G protein-coupled receptor (GPR)65, mitogen-activated protein kinase (MAPK)6, dual-specificity phosphatase 2 (DUSP2), and protein tyrosine phosphatase Œµ (PTPRE)], which were up-regulated by all stimuli. The other major groups include 80 cell migration and cytoskeletal-regulation genes [such as intercellular adhesion molecule-1 (ICAM1), urokinase plasminogen activator receptor (PLAUR), and ùõÑ -actin (ACTG)1], 57 host defense genes [such as Toll/IL1R-containing molecule domain-containing adaptor-inducing IFNùõÉ (TRIF), CD83, and oxidative stress response 1 (OSR1), induced in all responses], and 55 cytokine and cytokinesignaling molecules [such as CXC chemokine ligand (CXCL)1, IL1B, TNF, and nuclear factor (NF)KB immediate-early (IE)]. One of the host-defense genes, CD83, encodes an Ig supergene family member, best known as a marker for mature dendritic cells [52], but has also been identified as a differentially regulated gene in neutrophil and macrophage-transcription profiles [22, 47, 53]. This gene underwent early and sustained up-regulation from marginal expression in resting cells to a

more than 20-fold higher level in cells responding to all three stimuli. CD83 surface antigen has been reported previously in neutrophils cultured in vitro with IFNùõÑ or GM-CSF [54] or harvested during acute bacterial infection in vivo [55].

Neutrophils exposed to the exogenous microbe-derived stimuli also showed distinct, temporal patterns in the resultant changes in gene expression. By hierarchical clustering, fourtime clusters were identified, including early down, transient up, early up, and late up clusters ( Fig. 3 ). The early downregulated or repressed cluster, consisting mainly of E. coli response genes, showed a rapid declining pattern that occurred within the first hour of stimulation. In addition, within the 2-h time course, a group of ‚àº 60 genes, listed in Table 3 , exhibited a particularly finely tuned pattern of transient up-regulation and then a return to the original level in response to one or more of the stimuli. Some genes in this cluster [such as prostaglandin-endoperoxide synthase-2 (PTGS2) oncostatin M, (OSM), and TOB1] showed no change or a sustained response in other studies [21, 22]. However, the transient pattern in our system was confirmed by real time RT-PCR analysis (see below).

## Stimulus-specific features of gene-expression responses in neutrophils

As shown in Figure 2, almost two-thirds (433/655) of the E. coli response genes was in the down-regulated cluster, in contrast to the fMLP response that included far fewer significantly repressed genes (47/253). In fact, E. coli similarly affected most of the genes down-regulated in response to fMLP and LPS, and all repression occurred within the first hour. Notably,

TABLE 2. Differentially Expressed Genes Involved in Protein Synthesis and Stability

|                                                                                                        |                |              | Expression* fMLP LPS   | Expression* fMLP LPS   | Expression* fMLP LPS   |
|--------------------------------------------------------------------------------------------------------|----------------|--------------|------------------------|------------------------|------------------------|
| Functional category /gene name                                                                         | Symbol         | LocusLink ID | E. coli                |                        |                        |
| Translation factor                                                                                     |                |              |                        |                        |                        |
| ATP-binding cassette, sub-family F (GCN20), member 1                                                   | ABCF1          | 23           | u                      | u                      | u                      |
| basic leucine zipper and W2 domains 1                                                                  | BZW1           | 9689         | -                      | u                      | -                      |
| eukaryotic translation elongation factor 2                                                             | EEF2           | 1938         | -                      | u                      | -                      |
| eukaryotic translation-initiation factor 3, subunit 2 ùõÉ , 36 kDa                                       | EIF3S2         | 8668         | -                      | u                      | -                      |
| eukaryotic translation-initiation factor 4A, isoform 1                                                 | EIF4A1         | 1973         | u                      | u                      | u                      |
| eukaryotic translation-initiation factor 4A, isoform 2                                                 | EIF4A2         | 1974         | t                      | -                      | -                      |
| eukaryotic translation-initiation factor 5                                                             | EIF5           | 1983         | d                      | d                      | t                      |
| eukaryotic translation-termination factor 1                                                            | ETF1           | 2107         | u                      | u                      | u                      |
|                                                                                                        |                | 10209        | u                      | u                      | -                      |
| putative translation-initiation factor                                                                 | SUI1           |              |                        |                        |                        |
| zinc finger protein 259                                                                                | ZNF259         | 8882         | u                      | u                      | u                      |
| RNA helicase                                                                                           |                |              |                        |                        |                        |
| DEAD/H (Asp-Glu-Ala-Asp/His) box polypeptide 21                                                        | DDX21          | 9188         | u                      | u                      | u                      |
| DEAD/H (Asp-Glu-Ala-Asp/His) box polypeptide 27                                                        | DDX27          | 55661        | -                      | u                      | u                      |
| DEAD/H (Asp-Glu-Ala-Asp/His) box polypeptide 3                                                         | DDX3           | 1654         | -                      | u                      | -                      |
| DEAD/H (Asp-Glu-Ala-Asp/His) box polypeptide 39                                                        | DDX39          | 10212        | u                      | u                      | -                      |
| DEAD/H (Asp-Glu-Ala-Asp/His) box polypeptide 5 (RNA helicase, 68 kDa)                                  | DDX5           | 1655         | u                      | u                      | -                      |
| DEAD/H (Asp-Glu-Ala-Asp/His) box polypeptide 8 (RNA helicase)                                          | DDX8           | 1659         | d                      | -                      | -                      |
| DEAD/H (Asp-Glu-Ala-Asp/His) box polypeptide 8 (RNA helicase)                                          | DDX8           | 1659         | d                      | -                      | -                      |
| prp28, U5 snRNP 100 kD protein                                                                         | U5-100K        | 9416         | d                      | -                      | -                      |
| U5 snRNP-specific protein, 200 kD                                                                      | U5-200KD       | 23020        | d                      | -                      | -                      |
| Other RNA binding                                                                                      |                |              |                        |                        |                        |
| adenosine deaminase, RNA-specific                                                                      | ADAR           | 103          | d                      | -                      | -                      |
| chromosome 20 open-reading frame 14 (ANT-1)                                                            | C20orf14       | 24148        | d                      | -                      | -                      |
| cryptochrome 1 (photolyase-like)                                                                       | CRY1           | 1407         | -                      | u                      | -                      |
| CUG triplet repeat, RNA-binding protein 1                                                              | CUGBP1         | 10658        | u                      | -                      | u                      |
| cysteinyl-tRNA synthetase                                                                              | CARS           | 833          | d                      | -                      | -                      |
| Dicer1, Dcr-1 homolog (Drosophila)                                                                     | DICER1         | 23405        | d                      | -                      | d                      |
| dual-specificity phosphatase 11 (RNA/RNP complex 1-interacting)                                        | DUSP11         |              | d                      | -                      | -                      |
| fusion, derived from t(12;16) malignant liposarcoma                                                    | FUS            | 2521         | u                      | u                      | -                      |
| Glycyl-tRNA synthetase, mRNA sequence                                                                  | GARS           | 2617         | l                      | -                      | -                      |
| heterogeneous nuclear ribonucleoprotein A2/B1                                                          | HNRPA2B1       | 3181         | u                      | -                      | -                      |
| heterogeneous nuclear ribonucleoprotein A3                                                             | HNRPA3         | 10151        | -                      | d                      |                        |
| NP220 nuclear protein                                                                                  | NP220          | 27332        | d                      | -                      | d -                    |
| NS1-associated protein 1                                                                               | NSAP1          | 10492        | -                      | u                      | -                      |
| poly(A)-binding protein, cytoplasmic 1                                                                 | PABPC1         | 26986        | -                      | u                      | -                      |
| protein phosphatase 1, regulatory subunit 10                                                           | PPP1R10        | 5514         | u                      | -                      | -                      |
| pumilio homolog 2 (Drosophila)                                                                         | PUM2           | 23369        | d                      | -                      | -                      |
| regulator of nonsense transcripts 1 ribophorin I                                                       | RENT1 RPN1     | 5976 6184    | d                      | -                      | -                      |
| RNA (guanine-7-) methyltransferase                                                                     | RNMT           | 8731         | d                      | -                      | -                      |
| RNA-binding motif protein 5                                                                            | RBM5           |              | u                      | u                      | -                      |
|                                                                                                        |                | 10181        | d                      | -                      | -                      |
| RNA-binding motif, single-stranded-interacting protein 1, pseudogene                                   | RBMS1P         | 5938         | d                      | -                      | -                      |
| seryl-tRNA synthetase                                                                                  | SARS SRP54     | 6301         | d                      | -                      | -                      |
| signal-recognition particle 54 kDa Sjogren syndrome antigen A1 (ribonucleoprotein autoantigen SS-A/Ro) | SSA1           | 6729 6737    | l d                    | - -                    | -                      |
|                                                                                                        | SCA1           |              |                        |                        | -                      |
| spinocerebellar ataxia 1 (olivopontocerebellar ataxia 1, ataxin 1)                                     |                | 6310         | d                      | -                      | -                      |
| splicing factor 3b, subunit 2, 145 kDa                                                                 | SF3B2          | 10992        | d                      | -                      | -                      |
| splicing factor proline/glutamine-rich (polypyrimidine-binding protein assoc)                          | SFPQ           | 6421         | u                      | u                      | -                      |
| splicing factor, arginine/serine-rich 10 (transformer 2 homolog, Drosophila)                           | SFRS10         | 6434         | d                      | d                      | t                      |
| splicing factor, arginine/serine-rich 2                                                                | SFRS2 SFRS3    | 6427         | u                      | t                      | -                      |
| splicing factor, arginine/serine-rich 3                                                                |                | 6428         | d                      | d                      | t                      |
| splicing factor, arginine/serine-rich 4                                                                | SFRS4          | 6429         | d                      | -                      | d                      |
| splicing factor, arginine/serine-rich 5                                                                | SFRS5          | 6430         | t                      | t                      | t                      |
| squamous cell carcinoma antigen recognized by T cells                                                  | SART1          | 9092         | d                      | -                      | -                      |
| stem-loop (histone)-binding protein                                                                    | SLBP           | 7884         | d                      | -                      | d                      |
| TIA1 cytotoxic granule-associated RNA-binding protein                                                  | TIA1           | 7072         | d                      | -                      | -                      |
| zinc finger protein 36, C3H type, homolog (mouse) Small protein-conjugating enzyme                     | ZFP36          | 7538         | u                      | t                      | -                      |
| ariadne homolog, ubiquitin-conjugating enzyme E2-binding protein, 1                                    | ARIH1          | 25820        | -                      | u                      | -                      |
| F-box-only protein 9                                                                                   | FBXO9          | 26268        | d                      | -                      | d                      |
| highly charged protein Nedd4-binding protein 1                                                         | D13S106E N4BP1 | 10208 9683   | - -                    | u -                    | - l                    |

|                                                                              |              |              | Expression*   | Expression*   | Expression*   |
|------------------------------------------------------------------------------|--------------|--------------|---------------|---------------|---------------|
| Functional category /gene name                                               | Symbol       | LocusLink ID | E. coli       | fMLP          | LPS           |
| pVHL-interacting deubiquitinating enzyme 1                                   | VDU1         | 23032        | d             | -             | -             |
| sequestosome 1                                                               | SQSTM1       | 8878         | l             | -             | l             |
| ubiquitin B                                                                  | UBB          | 7314         | u             | u             | -             |
| ubiquitin carrier protein                                                    | E2-EPF       | 27338        | d             | d             | t             |
| ubiquitin protein ligase                                                     | UBE3B        | 89910        | d             | -             | -             |
| ubiquitin-specific protease 1                                                | USP1         | 7398         | d             | -             | -             |
| ubiquitin-specific protease 15                                               | USP15        | 9958         | d             | -             | d             |
| ubiquitin-specific protease 4 (proto-oncogene)                               | USP4         | 7375         | d             | -             | -             |
| ubiquitination factor E4A (UFD2 homolog, yeast)                              | UBE4A        | 9354         | d             | -             | -             |
| ubiquitination factor E4A (UFD2 homolog, yeast)                              | UBE4A        | 9354         | d             | -             | -             |
| ubiquitination factor E4B (UFD2 homolog, yeast)                              | UBE4B        | 10277        | d             | -             | -             |
| ubiquitin-conjugating enzyme E2, J1 (UBC6 homolog, yeast)                    | UBE2J1       | 51465        | d             | -             | -             |
| ubiquitin-conjugating enzyme E2E 1 (UBC4/5 homolog, yeast)                   | UBE2E1       | 7324         | l             | -             | l             |
| ubiquitin-conjugating enzyme E2H (UBC8 homolog, yeast)                       | UBE2H        | 7328         | l             | -             | l             |
| ubiquitin-conjugating enzyme E2N (UBC13 homolog, yeast)                      | UBE2N        | 7334         | -             | u             | -             |
| Chaperone                                                                    |              |              |               |               |               |
| ceroid-lipofuscinoiss, neuronal 3, juvenile (Batten Spielmeyer-Vogt disease) | CLN3         | 1201         | -             | u             | -             |
| DNAJ heat shock protein (Hsp40) homolog, subfamily A, member 1               | DNAJA1       | 3301         | u             | u             | u             |
| DNAJ (Hsp40) homolog, subfamily B, member 6                                  | DNAJB6       | 10049        | -             | u             | -             |
| DNAJ (Hsp40) homolog, subfamily B, member 9                                  | DNAJB9       | 4189         | d             | d             | t             |
| DNAJ (Hsp40) homolog, subfamily C, member 8                                  | DNAJC8       | 22826        | l             | -             | -             |
| DNAJ (Hsp40) homolog, subfamily B, member 1                                  | DNAJB1       | 3337         | u             | u             | t             |
| heat shock 10 kDa protein 1 (chaperonin 10)                                  | HSPE1        | 3336         | u             | u             | u             |
| heat shock 27 kDa protein 1                                                  |              |              | u             | -             | -             |
| heat shock 60 kDa protein 1 (chaperonin)                                     | HSPB1        | 3315         |               |               |               |
|                                                                              | HSPD1        | 3329         | u             | u             | -             |
| heat shock 70 kDa protein 1A                                                 | HSPA1A       | 3303         | u             | u             | -             |
| heat shock 70 kDa protein 1B                                                 | HSPA1B       | 3304         | u             | u             | -             |
| heat shock 70 kDa protein 1-like                                             | HSPA1L       | 3305         | d             | -             | -             |
| heat shock 70 kDa protein 6 (HSP70B ) '                                      | HSPA6        | 3310 3312    | d u           | d u           | d             |
| heat shock 70 kDa protein 8                                                  | HSPA8 HSPA9B | 3313         | u             | u             | - -           |
| heat shock 70 kDa protein 9B (mortalin-2) heat shock 90 kDa protein 1, ùõÇ     | HSPCA        | 3320         | u             | u             | u             |
| heat shock 90 kDa protein 1, ùõÉ                                               | HSPCB        | 3326         | u             | u             | u             |
|                                                                              | HSPA2        | 3306         | u             | u             | u             |
| heat shock protein, 70 kDa                                                   |              | 6767         | -             | u             | -             |
| suppression of tumorigenicity 13 (Hsp70-interacting protein)                 | ST13         |              | u             | u             |               |
| t-complex 1 tubulin-specific chaperone c                                     | TCP1 TBCC    | 6950 6903    | d             | -             | u -           |

however, many of the genes that were significantly downregulated only by E. coli did show a response to the other stimuli that trended in the same direction but did not reach statistical significance. Thus, the difference in response was more quantitative than qualitative.

It is also noteworthy that many of the down-regulated genes function as repressors. For example, the transcriptional corepressor MAX dimerization protein 1 (MAD) [56] was downregulated by all three stimuli. Neutrophils also down-regulated the genes encoding chromobox homologue 1 (CBX1), a major component of heterochromatin [57], and SMART/histone deacetylase (HDAC)1-associated repressor protein (SHARP), which acts as a transcriptional corepressor through the recruitment of histone deacetylase complexes [58]. Both were rapidly down-regulated in response to E. coli ; CBX1 was also significantly down-regulated by LPS. Thus, the diminished expression of these genes could paradoxically contribute to the net activation response of the cell.

Compared with the down-regulated responses, the sets of up-regulated genes appeared to be somewhat more specific to

each stimulus. Although the number of genes up-regulated by all three stimuli was larger than the set that all three repressed, the temporal patterns of down-regulation were not identical. In addition, each stimulus induced expression of a unique group of genes: 60 by E. coli , 54 by fMLP, and 103 by LPS. This divergence may reflect differences in receptors and signaltransduction pathways activated by each stimulus, although many of the pathways eventually converge on the same cytoplasmic transcription factors, such as NFùõã B, Smad, and signal transducer and activator of transcription (STAT) family members.

## Validation of microarray data by real-time quantitative PCR

Thirty well-annotated genes with significant changes in expression level were picked for validation by real-time quantitative RT-PCR (a complete list is available as Table S3 in the online supplementary material). These included up-regulated or transiently up-regulated genes (including OSM, IL8, DNAJB1, PTGS2, and CD69) and down-regulated genes (including

Fig. 3. Temporal clustering of responses. The bar graphs show the numbers of differentially expressed genes, divided by temporal pattern (indicated above each graph) and functional category of the genes (indicated in the left margin). The numbers of responding genes are indicated in the scale at the top of each graph. Colors represent the different stimuli, as shown below.

<!-- image -->

HLX1, CNOT3, FBXO9, FOXO1A, and BRD8). Each target gene was examined in 15 reactions using samples from three stimulation time-course experiments. The specificity of each reaction was confirmed by melting-curve analysis and agarosegel visualization of the PCR products.

Figure 4 presents a plot of real-time quantitative PCR data, expressed as ùö´ùö´ Ct (where the change in transcript abundance is a logarithmic function of 2 -ùö´ùö´ CT ), versus the linear change in oligonucleotide array signal. The data were filtered to show only points with a greater than 1.25-fold change from time zero. Almost all of the interrogations are located in the first and third quadrants, indicating good concordance between the results from real-time PCR and from oligonucleotide arrays for upregulated and down-regulated changes (91% and 88% concordance, respectively, with correlation R 2 = 0.49). As expected, we observed greater changes in expression by real-time quantitative PCR (along the logarithmic x -axis) than by oligonucleotide array (on the linear y -axis). RT-PCR has a larger dynamic range and sensitivity than microarray analysis hybridization and tends to generate data showing a greater magnitude of change. Thus, the technique is generally used to validate observed trends but is not expected to duplicate the fold changes obtained in chip experiments [59, 60].

## Comparison with other leukocytes

To examine the possible differences between cell types in their patterns of response for transcription factor genes, we performed a principal component analysis to determine whether well-separated clusters could be identified among 31 datasets representing the following five cell types, represented by dif-

ferent symbols in Figure 5 : resting neutrophils, neutrophils stimulated with E. coli for 30 or 120 min, resting monocytes, and monocytes stimulated with E. coli for 120 min. To determine whether the 31 samples are separable into distinctive groups, we applied a variant of principal-components analysis that allows projection of the highly multidimensional data onto a two-dimensional space spanned by the two leading principal components, represented by the two axes of the graph. As shown in the scatter plot in Figure 5, most of the samples of the different cell types are separable. Projection of the samples onto the second principal component explains 30% of the variability of the data and is sufficient for partitioning the neutrophil and monocyte data points. Moreover, the third principal component captures 14% of the data variability, and its lower values are associated with activation of neutrophils and monocytes. Analyses of transcription factor expression profiles from other human leukocyte cell types produced clusters with equal or greater separation from the neutrophil samples (data not shown).

## DISCUSSION

Overall, the results of this study indicate that activated neutrophils not only generate the long-recognized phagocytic response [61, 62] but also a remarkably vigorous and broad array of alterations in gene expression. Among these responses were changes in the levels of transcript, not only for cytokines, receptors, host defense, and apoptosis-related genes and but also a large number of transcription factors and chromatinremodeling genes. Clustering analysis of the data showed distinct temporal patterns with many rapid and transient changes in gene expression occurring within the first hour of exposure to each stimulus.

The total number of genes identified in this study is comparable with that reported in microarray analyses by Kobayashi et al. [22, 53], of which 40% of the genes were found to be differentially expressed under the various experimental conditions of the previously published and present studies. In particular, down-regulation of critical inflammatory response genes, as first described by Kobayishi et al. [53], was also observed in our experiments using E. coli stimulation. For example, the bacterial agonist caused down-regulation of platelet/endothelial cell adhesion molecule 1 and IL-17R, but the changes in these genes were not significant in fMLP and LPS experiments.

The diversity of the other sets of genes identified in the different experiments probably derives from differences in experimental design, such as preparation of neutrophils, time points for examination of expression profiles, and diversity of stimuli. The current studies focused on the early transcriptional changes and comparisons of 'complete' particulate bacterial stimuli and with two of its soluble components. In comparison with the previous findings, we identified fewer metabolism and vesicle-transport genes but more chromatin-binding genes and translation regulators. Most importantly, 75% of the transiently regulated genes found in our study (mostly at early time points) were not identified in other work on the neutrophil response. As expected, we did not detect events observed by

TABLE 3. Transiently Up-Regulated Genes

|                                                                                                       |                 |           | Expression*   | Expression*   | Expression*   |
|-------------------------------------------------------------------------------------------------------|-----------------|-----------|---------------|---------------|---------------|
| Name                                                                                                  | Symbol          | LocusLink | E. coli       | fMLP          | LPS           |
| transcriptional regulator interacting with the PHS-bromodomain 2                                      | TRIP-Br2        | 9792      | -             | ,             | ,             |
| splicing factor, arginine/serine-rich 5                                                               | SFRS5           | 6430      | ,             | ,             | ,             |
| v-fos FBJ murine osteosarcoma viral oncogene homolog                                                  | FOS             | 2353      | ,             | ,             | ,             |
| transducer of ERBB2, 1                                                                                | TOB1            | 10140     | ,             | ,             | ,             |
| BTG family, member 2                                                                                  | BTG2            | 7832      | ,             | ,             | ,             |
| transcription factor 8 (represses interleukin-2 expression)                                           | TCF8            | 6935      | ,             | ,             | ,             |
| early growth response 2 (Krox-20 homolog, Drosophila)                                                 | EGR2            | 1959      | ,             | ,             | ,             |
| transcriptional regulator interacting with the PHS-bromodomain 2                                      | TRIP-Br2        | 9792      | ,             | ,             | ,             |
| protein transport protein SEC61 ùõÇ subunit isoform 1                                                   | SEC61A1         | 29927     | ,             | ,             | -             |
| amphiregulin (schwannoma-derived growth factor)                                                       |                 |           | ,             | ,             | -             |
| parvin, ùõÉ                                                                                             | AREG            | 374       | ,             | ,             | -             |
|                                                                                                       | PARVB           | 29780     | ,             |               |               |
| jun-B proto-oncogene immediate early protein                                                          | JUNB            | 3726      | ,             | , ,           | -             |
| v-jun sarcoma virus 17 oncogene homolog (avian)                                                       | ETR101 JUN      | 9592 3725 | ,             | ,             | -             |
| early growth response 1                                                                               | EGR1            | 1958      | ,             | ,             | -             |
|                                                                                                       |                 |           |               |               | -             |
| dual-specificity phosphatase 1                                                                        | DUSP1           | 1843      | ,             | -             | ,             |
| Bcl-2-associated transcription factor dual-specificity phosphatase 11 (RNA/RNP complex 1-interacting) | BTF             | 9774      | ,             | -             | -             |
|                                                                                                       | DUSP11          | 8446      | ,             | -             | -             |
| chromosome 14 open-reading frame 11                                                                   | C14orf11 EIF4A2 | 55837     | , ,           | - -           | -             |
| eukaryotic translation-initiation factor 4A, isoform 2 oncostatin M                                   | OSM             | 1974 5008 | -             | ,             | - ,           |
| nuclear receptor subfamily 4, group A, member 2                                                       | NR4A2           | 4929      | -             | ,             | ,             |
| period homolog 1 (Drosophila)                                                                         | PER1            | 5187      | -             | ,             | ,             |
| CD69 antigen (p60, early T cell activation antigen)                                                   | CD69            | 969       | -             | ,             | ,             |
| prostaglandin-endoperoxide synthase 2 (hCox-2)                                                        | PTGS2           | 5743      | -             | ,             | ,             |
| dual-specificity phosphatase 2                                                                        | DUSP2           | 1844      | -             | ,             | ,             |
| zinc finger protein 36, C3H type, homolog (mouse)                                                     | ZFP36           | 7538      | -             | ,             | -             |
| nuclear receptor subfamily 4, group A, member 1                                                       | NR4A1           | 3164      | -             | ,             | -             |
| myosin, light polypeptide 6, alkali, smooth muscle and nonmuscle                                      | MYL6            | 4637      | -             | ,             | -             |
| chemokine (C-C motif) ligand 3                                                                        | CCL3            | 6348      | -             | ,             | -             |
| diphtheria toxin receptor                                                                             | DTR             | 1839      | -             | ,             | -             |
| splicing factor, arginine/serine-rich 2                                                               | SFRS2           | 6427      | -             | ,             | -             |
| nuclear factor of ùõã light polypeptide gene enhancer in B cells inhibitor, ùõÇ                           | NFKBIA          | 4792      | -             | ,             | -             |
| regulator of G-protein signaling 1                                                                    | RGS1            | 5996      | -             | -             | ,             |
| phosphatidylserine receptor                                                                           | PTDSR           | 23210     | -             | -             | ,             |
| heterogeneous nuclear ribonucleoprotein A3                                                            | HNRPA3          | 10151     | -             | -             | ,             |
| eukaryotic translation-initiation factor 5                                                            | EIF5            | 1983      | -             | -             | ,             |
| splicing factor, arginine/serine-rich 3                                                               | SFRS3           | 6428      | -             | -             | ,             |
| activating transcription factor 4 (tax-responsive enhancer element B67)                               | ATF4            | 468       | -             | -             | ,             |
| splicing factor, arginine/serine-rich 10 (transformer 2 homolog, Drosophila)                          | SFRS10          | 6434      |               |               |               |
| phorbol 12-myristate-13-acetate-induced protein 1                                                     | PMAIP1          | 5366      | -             | -             | ,             |
| ubiquitin carrier protein                                                                             | E2-EPF          |           | -             | -             | ,             |
| pleckstrin homology, Sec7 and coiled/coil domains, binding protein                                    | PSCDBP          | 27338     | -             | -             | , ,           |
| isopentenyl-diphosphate ùõÖ isomerase                                                                   | IDI1            | 9595 3422 | - -           | - -           | ,             |
| 6-pyruvoyltetrahydropterin synthase                                                                   | PTS             | 5805      | -             |               | ,             |
| serine/threonine kinase 17b (apoptosis-inducing)                                                      | STK17B          | 9262      | -             | - -           | ,             |
| DNAJ (Hsp40) homolog, subfamily B, member 9                                                           | DNAJB9          | 4189      | -             | -             | ,             |
| chromosome 21 open-reading frame 80                                                                   | C21orf80        | 23275     | -             | -             | ,             |
| DNAJ (Hsp40) homolog, subfamily B, member 1                                                           | DNAJB1          | 3337      | -             | -             | ,             |
| nuclear transcription factor Y, ùõÇ                                                                     | NFYA            | 4800      | -             | -             | ,             |
| phosphoprotein regulated by mitogenic pathways                                                        | C8FW            | 10221     | -             |               | ,             |
| protein tyrosine phosphatase, receptor type, E                                                        | PTPRE           | 5791      | -             | -             | ,             |
| Heat shock protein, 70 kDa                                                                            | HSPA8           | 3312      | -             | -             |               |
| PCTAIRE protein kinase 1                                                                              | PCTK1           | 5127      | -             | -             | ,             |
| adipose differentiation-related protein                                                               | ADFP            | 123       | -             | -             | , ,           |
| early growth response 3                                                                               | EGR3            | 1960      |               | -             | ,             |
| unr-interacting protein                                                                               |                 | 11171     | - -           | -             | ,             |
| mago-nashi homolog, proliferation-associated (Drosophila)                                             | UNRIP MAGOH     | 4116      | -             | - -           | ,             |
| putative lymphocyte G0/G1 switch gene                                                                 | G0S2            | 50486     | -             | -             | ,             |

Fig. 4. Comparison of changes in gene expression measured by real-time quantitative PCR versus oligonucleotide array. Each point on the scatter plot represents the difference in expression level of a given gene relative to the time-zero expression level, as measured in the same RNA sample by the two techniques. The PCR data are expressed as normalized differences in ùö´ùö´ Ct , representing the difference between time zero (base line) and a specific time point, normalized to a reference gene that showed the closest relative amplification efficiency within a panel of constantly expressed normalization control genes; thus, the change in transcript abundance is a logarithmic function of 2 -ùö´ùö´ CT . Microarray hybridization signals are expressed as fold changes, representing the ratios of weighted group means generated by the 'combine comparisons' function of dChip software.

<!-- image -->

DeLeo's laboratory [22, 24, 53] at later time points, such as down-regulation of IL-8R ùõÉ (IL8RB) and IL-10R ùõÇ (IL10RA) at 3-6 h after stimulation. Overall, the differences between the sets of genes identified in the previous and current studies probably reflect temporal changes in the transcription of earlyand late-response genes as well as the diversity of the responses to complex particulate stimuli compared with the individual priming or activating components. Thus, the response to opsonized bacteria appears to represent a complex set of interacting up- and down-regulating changes rather than a simple summation of responses to individual agonist molecules.

In our functional classification of identified genes, the category of regulators and mediators of gene expression included genes encoding molecules ranging from general RNA polymerase II transcription factors to protein chaperones. Some of the transcription factors, such as members of the fos, jun, and NFùõã B families, were predictable findings, as they have been

studied individually in phagocytic cells. However, the present studies confirm and considerably expand our previous analysis [20] that indicated the operation of a much broader array of transcriptional regulators in neutrophils.

Other differentially expressed, regulatory genes included the protein kinase CK2-phosphorylated HDAC2 (class I HDAC) and MYST histone acetyltransferase 1 (MYST1), which allow chromatins to fluctuate between the condensed and decondensed states [63, 64]. HDAC1 and -2 are components of the large, multisubunit complexes Sin3 and NuRD, which are recruited by transcription factors such as Mad [65]. Ubiquitination mediators, other small protein-conjugating enzymes, and chaperones have not been reported in previous transcription profiles of cells in the innate-immune system. They not only direct proteasome-mediated degradation of proteins but also affect chromatin structure and thus serve as members of the multilayered regulation mechanism that coordinates gene expression with other integral biological processes [66, 67].

In addition to the temporal clustering of genes, we also observed rather different profiles associated with each stimulus. The responses of neutrophils to soluble and particulate stimuli are initiated by multiple surface receptors and mediated by a web of signal-transduction pathways [68, 69]. The three agonists used in the current study use quite different signaling mechanisms. Opsonized, particulate stimuli such as bacteria are recognized by antibody and complement receptors, and the live organisms also engage the cell surface through fimbrial proteins and fimbriae-mediated delivery of LPS [7072]. LPS molecules vary in structure and form of delivery among bacteria, which can lead to recruitment of different pattern-recognition molecules and signaling pathways [73, 74]. Thus, the differences in responses may reflect the additional receptors engaged by the opsonized bacteria and the differences in structure and presentation of LPS in soluble versus bacterial surface forms. Formylated peptides such as fMLP engage a single receptor in the seven-transmembrane helix family but activate multiple intracellular signaling proteins including p38 MAPK, phosphatidylinositol-3 kinase/Akt, and extracellular-regulated kinase [75]. Our previous study of neutrophil responses to virulent and nonvirulent gram-negative bacterial species showed that Yersinia pestis was able to suppress many host-defense genes compared with E. coli [20]; the present studies indicate that even the nonvirulent organism is able to play a more dynamic role in shaping the inflammatory response than do simple soluble agonists.

The large array of transcription factors in the neutrophil's early-response repertoire far exceeds the small number investigated in most previous studies of myeloid function and development, which generally focused on the analysis of the promoters of single genes or on gene knockouts in mice. The total number was comparable with that reported in microarray analyses by Kobayashi et al. [22, 53]. Equally striking was the prominence of chromatin-regulatory genes in this repertoire. Gene regulation at the level of chromatin structure has recently been demonstrated for short-term NFùõã B-induced changes of cytokine and major histocompatibility complex gene expression in T and B cells [76-78]. The present study indicates that alterations of histone acetylation and chromatin structure may also play a role in the innate-immune system by mediating the

Fig. 5. Principal component analysis of transcription-factor expression by neutrophils and monocytes. The graph shows the projection of 31 samples onto a subspace spanned by the two leading principal components. Using a normalized matrix, projections of the samples onto the second and third principal components for normalized profiles of each cell type are shown by the symbols indicated in the figure insert. Clustering of samples is evident, although the identities of the samples were not used in performing the projection.

<!-- image -->

rapid changes in gene expression that we have observed in neutrophils. The role and interplay of these transcription factors and chromatin-remodeling elements in normal myeloid development and in the response of normal neutrophils to activation are ripe for further intensive study.

## ACKNOWLEDGMENTS

NIH Grant DK54369, the John H. Pierce Cancer Research fund, and a Cancer Bioinformatics fellowship from the Anna Fuller fund supported this work.

## REFERENCES

- 1. Dinauer, M. C. (2003) The phagocyte system and disorders of granulopoiesis and granulocyte function. In Hematology of Infancy &amp; Childhood (D. G. Nathan, S. H. Orkin, A. T. Look, D. Ginsberg, eds.), Philadelphia, PA, Saunders, 923-1010.
- 2. Baehner, R. L. (2000) Normal neutrophil structure and function. In Hematology: Basic Principles and Practice (R. Hoffman, E. J. Benz Jr., S. J. Shattil, B. Furie, H. J. Cohen, L. E. Silberstein, P. McGlave, eds.), New York, NY, Churchill Livingstone, 667-685.
- 3. Babior, B. M. (2000) Phagocytes and oxidative stress. Am. J. Med. 109, 33-44.
- 4. Edwards, S. W., Hallett, M. B. (1997) Seeing the wood for the trees: the forgotten role of neutrophils in rheumatoid arthritis. Immunol. Today 18, 320-324.
- 5. Eid, N. S., Kravath, R. E., Lanks, K. W. (1987) Heat-shock protein synthesis by human polymorphonuclear cells. J. Exp. Med. 165, 14481452.
- 6. Jack, R. M., Fearon, D. T. (1988) Selective synthesis of mRNA and proteins by human peripheral blood neutrophils. J. Immunol. 140, 42864293.
- 7. Granelli-Piperno, A., Vassalli, J. D., Reich, E. (1979) RNA and protein synthesis in human peripheral blood polymorphonuclear leukocytes. J. Exp. Med. 149, 284-289.
- 8. Newburger, P. E., Ezekowitz, R. A. B., Whitney, C., Wright, J., Orkin, S. H. (1988) Induction of phagocyte cytochrome b heavy chain gene expression by interferon gamma. Proc. Natl. Acad. Sci. USA 85, 52155219.

- 9. Newburger, P. E., Dai, Q., Whitney, C. (1991) In vitro regulation of human phagocyte cytochrome b heavy and light chain gene expression by bacterial lipopolysaccharide and recombinant human cytokines. J. Biol. Chem. 266, 16171-16177.
- 10. Colotta, F., Wang, J. M., Polentarutti, N., Mantovani, A. (1987) Expression of c-fos protooncogene in normal human peripheral blood granulocytes. J. Exp. Med. 165, 1224-1229.
- 11. Itami, M., Kuroki, T., Nose, K. (1987) Induction of c-fos proto-oncogene by a chemotactic peptide in human peripheral granulocytes. FEBS Lett. 222, 289-292.
- 12. Lord, P. C., Wilmoth, L. M., Mizel, S. B., McCall, C. E. (1991) Expression of interleukin-1 ùõÇ and ùõÉ genes by human blood polymorphonuclear leukocytes. J. Clin. Invest. 87, 1312-1321.
- 13. Lindemann, A., Riedel, D., Oster, W., Ziegler-Heitbrock, H. W., Mertelsmann, R., Herrmann, F. (1989) Granulocyte-macrophage colony-stimulating factor induces cytokine secretion by human polymorphonuclear leukocytes. J. Clin. Invest. 83, 1308-1312.
- 14. Lloyd, A. R., Oppenheim, J. J. (1992) Poly's lament: the neglected role of the polymorphonuclear neutrophil in the afferent limb of the immune response. Immunol. Today 13, 169-172.
- 15. Melani, C., Mattia, G. F., Silvani, A., Care ', A., Rivoltini, L., Parmiani, G., Colombo, M. P. (1993) Interleukin-6 expression in human neutrophil and eosinophil peripheral blood granulocytes. Blood 81, 2744-2749.
- 16. Stankova, J., Rola-Pleszczynski, M., Dubois, C. M. (1995) Granulocytemacrophage colony-stimulating factor increases 5lipoxygenase gene transcription and protein expression in human neutrophils. Blood 85, 3719-3726.
- 17. Dubravec, D. B., Spriggs, D. R., Mannick, J. A., Rodrick, M. L. (1990) Circulating human peripheral blood granulocytes synthesize and secrete tumor necrosis factor ùõÇ . Proc. Natl. Acad. Sci. USA 87, 6758-6761.
- 18. Bazzoni, F., Cassatella, M. A., Rossi, F., Ceska, M., Dewald, B., Baggiolini, M. (1991) Phagocytosing neutrophils produce and release high amounts of the neutrophil-activating peptide 1/interleukin 8. J. Exp. Med. 173, 771-774.
- 19. Shirafuji, N., Matsuda, S., Ogura, H., Tani, K., Kodo, H., Ozawa, K., Nagata, S., Asano, S., Takaku, F. (1990) Granulocyte colony-stimulating factor stimulates human mature neutrophilic granulocytes to produce interferonùõÇ . Blood 75, 17-19.
- 20. Subrahmanyam, Y. V., Yamaga, S., Prashar, Y., Lee, H. H., Hoe, N. P., Kluger, Y., Gerstein, M., Goguen, J. D., Newburger, P. E., Weissman, S. M. (2001) RNA expression patterns change dramatically in human neutrophils exposed to bacteria. Blood 97, 2457-2468.
- 21. Fessler, M. B., Malcolm, K. C., Duncan, M. W., Worthen, G. S. (2002) A genomic and proteomic analysis of activation of the human neutrophil by lipopolysaccharide and its mediation by p38 mitogen-activated protein kinase. J. Biol. Chem. 277, 31291-31302.
- 22. Kobayashi, S. D., Voyich, J. M., Buhl, C. L., Stahl, R. M., DeLeo, F. R. (2002) Global changes in gene expression by human polymorphonuclear leukocytes during receptor-mediated phagocytosis: cell fate is regulated at the level of gene expression. Proc. Natl. Acad. Sci. USA 99, 6901-6906.
- 23. Yang, J. J., Preston, G. A., Alcorta, D. A., Waga, I., Munger, W. E., Hogan, S. L., Sekura, S. B., Phillips, B. D., Thomas, R. P., Jennette, J. C., Falk, R. J. (2002) Expression profile of leukocyte genes activated by anti-neutrophil cytoplasmic autoantibodies (ANCA). Kidney Int. 62, 1638-1649.
- 24. Kobayashi, S. D., Voyich, J. M., Somerville, G. A., Braughton, K. R., Malech, H. L., Musser, J. M., DeLeo, F. R. (2003) An apoptosis-differentiation program in human polymorphonuclear leukocytes facilitates resolution of inflammation. J. Leukoc. Biol. 73, 315-322.
- 25. Whitney, A. R., Diehn, M., Popper, S. J., Alizadeh, A. A., Boldrick, J. C., Relman, D. A., Brown, P. O. (2003) Individuality and variation in gene expression patterns in human blood. Proc. Natl. Acad. Sci. USA 100, 1896-1901.
- 26. Raghunathan, A., Sivakamasundari, R., Wolenski, J., Poddar, R., Weissman, S. M. (2001) Functional analysis of B144/LST1: a gene in the tumor necrosis factor cluster that induces formation of long filopodia in eukaryotic cells. Exp. Cell Res. 268, 230-244.
- 27. Subrahmanyam, Y. V. B. K., Baskaran, N., Newburger, P. E., Weissman, S. M. (1999) A modified method for the display of 3 -end ' restriction fragments of cDNAs: molecular profiling of gene expression in neutrophils. Methods Enzymol. 303, 272-297.
- 28. Subrahmanyam, Y. V. B. K., Yamaga, S., Prashar, Y., Lee, H. H., Hoe, N. P., Kluger, Y., Gerstein, M., Goguen, J. D., Newburger, P. E., Weissman, S. M. (2001) RNA expression patterns change dramatically in human neutrophils exposed to bacteria. Blood 97, 2457-2468.
- 29. Coller, H. A., Grandori, C., Tamayo, P., Colbert, T., Lander, E. S., Eisenman, R. N., Golub, T. R. (2000) Expression analysis with oligonucleotide microarrays reveals that MYC regulates genes involved in growth, cell cycle, signaling, and adhesion. Proc. Natl. Acad. Sci. USA 97, 3260-3265.
- 30. Li, C., Hung, W. W. (2001) Model-based analysis of oligonucleotide arrays: model validation, design issues and standard error application. Genome Biol. 2, RESEARCH0032.1-RESEARCH0032.11.
- 31. Li, C., Wong, W. H. (2001) Model-based analysis of oligonucleotide arrays: expression index computation and outlier detection. Proc. Natl. Acad. Sci. USA 98, 31-36.
- 32. Tusher, V. G., Tibshirani, R., Chu, G. (2001) Significance analysis of microarrays applied to the ionizing radiation response. Proc. Natl. Acad. Sci. USA 98, 5116-5121.
- 33. Irizarry, R. A., Bolstad, B. M., Collin, F., Cope, L. M., Hobbs, B., Speed, T. P. (2003) Summaries of Affymetrix GeneChip probe level data. Nucleic Acids Res. 31, e15.
- 34. Heid, C. A., Stevens, J., Livak, K. J., Williams, P. M. (1996) Real time quantitative PCR. Genome Res. 6, 986-994.
- 35. Wright, M. A., Church, G. M. (2002) An open-source oligomicroarray standard for human and mouse. Nat. Biotechnol. 20, 1082-1083.
- 36. Livak, K. J., Schmittgen, T. D. (2001) Analysis of relative gene expression data using real-time quantitative PCR and the 2(ùõÖùõÖ C(T)) method. Methods 25, 402-408.
- 37. Kluger, Y., Basri, R., Chang, J. T., Gerstein, M. (2003) Spectral biclustering of microarray data: coclustering genes and conditions. Genome Res. 13, 703-716.
- 38. Hogg, R. V., Ledolter, J. (1987) Engineering Statistics , New York, NY, MacMillan.
- 39. Hollander, M., Wolfe, D. A. (1973) Nonparametric Statistics , New York, NY, Wiley.
- 40. Hochberg, Y., Tamhane, A. C. (1987) Multiple Comparison Procedures , New York, NY, Wiley.
- 41. Tadjali, M., Seidelin, J. B., Olsen, J., Troelsen, J. T. (2002) Transcriptome changes during intestinal cell differentiation. Biochim. Biophys. Acta 1589, 160-167.
- 42. Bakay, M., Zhao, P., Chen, J., Hoffman, E. P. (2002) A web-accessible complete transcriptome of normal human and DMD muscle. Neuromuscul. Disord. 12 (Suppl. 1), S125-S141.
- 43. McLean, D. J., Friel, P. J., Pouchnik, D., Griswold, M. D. (2002) Oligonucleotide microarray analysis of gene expression in follicle-stimulating hormone-treated rat Sertoli cells. Mol. Endocrinol. 16, 2780-2792.
- 44. Wells, C. A., Ravasi, T., Sultana, R., Yagi, K., Carninci, P., Bono, H., Faulkner, G., Okazaki, Y., Quackenbush, J., Hume, D. A., Lyons, P. A. (2003) Continued discovery of transcriptional units expressed in cells of the mouse mononuclear phagocyte lineage. Genome Res. 13, 1360-1365.
- 45. Boldrick, J. C., Alizadeh, A. A., Diehn, M., Dudoit, S., Liu, C. L., Belcher, C. E., Botstein, D., Staudt, L. M., Brown, P. O., Relman, D. A. (2002)
- Stereotyped and specific gene expression programs in human innate immune responses to bacteria. Proc. Natl. Acad. Sci. USA 99, 972-977.
- 46. Huang, Q., Liu, D., Majewski, P., Schulte, L. C., Korn, J. M., Young, R. A., Lander, E. S., Hacohen, N. (2001) The plasticity of dendritic cell responses to pathogens and their components. Science 294, 870-875.
- 47. Nau, G. J., Richmond, J. F., Schlesinger, A., Jennings, E. G., Lander, E. S., Young, R. A. (2002) Human macrophage activation programs induced by bacterial pathogens. Proc. Natl. Acad. Sci. USA 99, 15031508.
- 48. Thomas, T., Thomas, T. J. (2001) Polyamines in cell growth and cell death: molecular mechanisms and therapeutic applications. Cell. Mol. Life Sci. 58, 244-258.
- 49. Ivanov, I. P., Gesteland, R. F., Atkins, J. F. (2000) Antizyme expression: a subversion of triplet decoding, which is remarkably conserved by evolution, is a sensor for an autoregulatory circuit. Nucleic Acids Res. 28, 3185-3196.
- 50. Johnson, C. A., White, D. A., Lavender, J. S., O'Neill, L. P., Turner, B. M. (2002) Human class I histone deacetylase complexes show enhanced catalytic activity in the presence of ATP and co-immunoprecipitate with the ATP-dependent chaperone protein Hsp70. J. Biol. Chem. 277, 95909597.
- 51. Fry, C. J., Peterson, C. L. (2001) Chromatin remodeling enzymes: who's on first? Curr. Biol. 11, R185-R197.
- 52. te Velde, A. A., van Kooyk, Y., Braat, H., Hommes, D. W., Dellemijn, T. A., Slors, J. F., van Deventer, S. J., Vyth-Dreese, F. A. (2003) Increased expression of DC-SIGN + IL-12 + IL-18 + and CD83 + IL-12-IL-18- dendritic cell populations in the colonic mucosa of patients with Crohn's disease. Eur. J. Immunol. 33, 143-151.
- 53. Kobayashi, S. D., Voyich, J. M., Braughton, K. R., DeLeo, F. R. (2003) Down-regulation of proinflammatory capacity during apoptosis in human polymorphonuclear leukocytes. J. Immunol. 170, 3357-3368.
- 54. Iking-Konert, C., Cseko, C., Wagner, C., Stegmaier, S., Andrassy, K., Hansch, G. M. (2001) Transdifferentiation of polymorphonuclear neutrophils: acquisition of CD83 and other functional characteristics of dendritic cells. J. Mol. Med. 79, 464-474.
- 55. Iking-Konert, C., Wagner, C., Denefleh, B., Hug, F., Schneider, M., Andrassy, K., Hansch, G. M. (2002) Up-regulation of the dendritic cell marker CD83 on polymorphonuclear neutrophils (PMN): divergent expression in acute bacterial infections and chronic inflammatory disease. Clin. Exp. Immunol. 130, 501-508.
- 56. Ayer, D. E., Kretzner, L., Eisenman, R. N. (1993) Mad: a heterodimeric partner for Max that antagonizes Myc transcriptional activity. Cell 72, 211-222.
- 57. Cheutin, T., McNairn, A. J., Jenuwein, T., Gilbert, D. M., Singh, P. B., Misteli, T. (2003) Maintenance of stable heterochromatin domains by dynamic HP1 binding. Science 299, 721-725.
- 58. Oswald, F., Kostezka, U., Astrahantseff, K., Bourteele, S., Dillinger, K., Zechner, U., Ludwig, L., Wilda, M., Hameister, H., Knochel, W., Liptay, S., Schmid, R. M. (2002) SHARP is a novel component of the Notch/ RBP-J ùõã signalling pathway. EMBO J. 21, 5417-5426.
- 59. Wurmbach, E., Yuen, T., Ebersole, B. J., Sealfon, S. C. (2001) Gonadotropin-releasing hormone receptor-coupled gene network organization. J. Biol. Chem. 276, 47195-47201.
- 60. Mutch, D. M., Berger, A., Mansourian, R., Rytz, A., Roberts, M. A. (2002) The limit fold change model: a practical approach for selecting differentially expressed genes from microarray data. BMC Bioinformatics 3, 17.
- 61. Metchnikoff, E. (1883) Untersuchungen u ber die intracellulare Verdauung bei wirbellosen Thieren. Arb. Zoologischen Inst. Univ. Wien 5, 141-168.
- 62. Dinauer, M. C., Nauseef, W. M., Newburger, P. E. (2001) Inherited disorders of phagocytic cells. In The Metabolic and Molecular Bases of Inherited Disease (C. R. Scriver, A. L. Beaudet, W. S. Sly, D. Valle, eds.), New York, NY, McGraw-Hill, 4857-4890.
- 63. Katan-Khaykovich, Y., Struhl, K. (2002) Dynamics of global histone acetylation and deacetylation in vivo: rapid restoration of normal histone acetylation status upon removal of activators and repressors. Genes Dev. 16, 743-752.
- 64. Walia, H., Chen, H. Y., Sun, J. M., Holth, L. T., Davie, J. R. (1998) Histone acetylation is required to maintain the unfolded nucleosome structure associated with transcribing DNA. J. Biol. Chem. 273, 1451614522.
- 65. Li, J., Lin, Q., Wang, W., Wade, P., Wong, J. (2002) Specific targeting and constitutive association of histone deacetylase complexes during transcriptional repression. Genes Dev. 16, 687-692.
- 66. Freiman, R. N., Tjian, R. (2003) Regulating the regulators: lysine modifications make their mark. Cell 112, 11-17.
- 67. Conaway, R. C., Brower, C. S., Conaway, J. W. (2002) Emerging roles of ubiquitin in transcription regulation. Science 296, 1254-1258.

- 68. Berton, G., Yan, S. R., Fumagalli, L., Lowell, C. A. (1996) Neutrophil activation by adhesion: mechanisms and pathophysiological implications. Int. J. Clin. Lab. Res. 26, 160-177.
- 69. Hanada, T., Yoshimura, A. (2002) Regulation of cytokine signaling and inflammation. Cytokine Growth Factor Rev. 13, 413-421.
- 70. Hedlund, M., Duan, R. D., Nilsson, A., Svensson, M., Karpman, D., Svanborg, C. (2001) Fimbriae, transmembrane signaling, and cell activation. J. Infect. Dis. 183 (Suppl. 1), S47-S50.
- 71. Hajishengallis, G., Martin, M., Sojar, H. T., Sharma, A., Schifferle, R. E., DeNardin, E., Russell, M. W., Genco, R. J. (2002) Dependence of bacterial protein adhesins on Toll-like receptors for proinflammatory cytokine induction. Clin. Diagn. Lab. Immunol. 9, 403-411.
- 72. Frendeus, B., Wachtler, C., Hedlund, M., Fischer, H., Samuelsson, P., Svensson, M., Svanborg, C. (2001) Escherichia coli P fimbriae utilize the Toll-like receptor 4 pathway for cell activation. Mol. Microbiol. 40, 37-51.
- 73. Kirikae, T., Nitta, T., Kirikae, F., Suda, Y., Kusumoto, S., Qureshi, N., Nakano, M. (1999) Lipopolysaccharides (LPS) of oral black-pigmented

- bacteria induce tumor necrosis factor production by LPS-refractory C3H/ HeJ macrophages in a way different from that of Salmonella LPS. Infect. Immun. 67, 1736-1742.
- 74. Triantafilou, M., Brandenburg, K., Gutsmann, T., Seydel, U., Triantafilou, K. (2002) Innate recognition of bacteria: engagement of multiple receptors. Crit. Rev. Immunol. 22, 251-268.
- 75. Heit, B., Tavener, S., Raharjo, E., Kubes, P. (2002) An intracellular signaling hierarchy determines direction of migration in opposing chemotactic gradients. J. Cell Biol. 159, 91-102.
- 76. Edelstein, L. C., Lagos, L., Simmons, M., Tirumalai, H., Gelinas, C. (2003) NFùõã B-dependent assembly of an enhanceosome-like complex on the promoter region of apoptosis inhibitor Bfl-1/A1. Mol. Cell. Biol. 23, 2749-2761.
- 77. Smale, S. T., Fisher, A. G. (2002) Chromatin structure and gene regulation in the immune system. Annu. Rev. Immunol. 20, 427-462.
- 78. Agarwal, S., Viola, J. P., Rao, A. (1999) Chromatin-based regulatory mechanisms governing cytokine gene transcription. J. Allergy Clin. Immunol. 103, 990-999.