<!-- image -->

## Post-Translational Modification Control of Innate Immunity

Juan Liu, 1 Cheng Qian, 1 and Xuetao Cao 1,2, *

1 National Key Laboratory of Medical Immunology &amp; Institute of Immunology, Second Military Medical University, Shanghai 200433, China 2 National Key Laboratory of Medical Molecular Biology &amp; Department of Immunology, Institute of Basic Medical Sciences, Chinese Academy of Medical Sciences, Beijing 100005, China

*Correspondence: caoxt@immunol.org

http://dx.doi.org/10.1016/j.immuni.2016.06.020

A coordinated balance between the positive and negative regulation of pattern-recognition receptor (PRR)initiated innate inflammatory responses is required to ensure the most favorable outcome for the host. Posttranslational modifications (PTMs) of innate sensors and downstream signaling molecules influence their activity and function by inducing their covalent linkage to new functional groups. PTMs including phosphorylation and polyubiquitination have been shown to potently regulate innate inflammatory responses through the activation, cellular translocation, and interaction of innate receptors, adaptors, and downstream signaling molecules in response to infectious and dangerous signals. Other PTMs such as methylation, acetylation, SUMOylation, and succinylation are increasingly implicated in the regulation of innate immunity and inflammation. In this review, we focus on the roles of PTMs in controlling PRR-triggered innate immunity and inflammatory responses. The emerging roles of PTMs in the pathogenesis and potential treatment of infectious and inflammatory immune diseases are also discussed.

## Introduction

The innate immune response provides the critical first line of host defense against invading pathogens via dynamic and complex interactions among its numerous cellular and molecular components. Discrimination of ''non-self'' pathogens from ''self'' components is the fundamental function of the innate immune system. In the last two decades, much attention has been paid to the signaling pathways initiated by pattern-recognition receptors (PRRs). These serve as sensors for ''non-self'' infectious and danger signals, initiating and regulating innate inflammatory responses. Their dysfunction has been linked to the inflammatory immune disorders. The germline-encoded PRRs detect a wide range of conserved pathogen-associated molecular patterns (PAMPs) originating from infectious microorganisms, as well as danger-associated molecular patterns (DAMPs) derived from injured host cells. Toll-like receptors (TLRs), retinoic-acid-inducible gene I (RIG-I)-like receptors (RLRs), and nucleotide-binding domain and leucine-rich repeat containing molecules (NLRs) are the most-well studied PRRs that induce potent host innate immunity in response to variable microbial stimuli (Figure 1). Other PRRs include c-type lectin receptors (CLRs), formyl peptide receptors, complement receptors, signaling lymphocyte activation molecules (SLAMs), and numerous innate sensors for cytoplasmic nucleic acids (Takeuchi and Akira, 2010; Iwasaki and Medzhitov, 2015). Following ligand engagement, these PRRs induce divergent and highly interacting signaling networks that generally converge on nuclear factork B (NFk B)-dependent expression of proinflammatory cytokines or chemokines, such as interleukin-6 (IL-6), tumor necrosis factor (TNF), C-C motif ligand 1 (CCL1), and the interferon regulatory factor (IRF)-dependent expression of type I interferons (IFNs), leading to the activation of innate immune pathways with subsequent effects on adaptive immune response (Figure 1).

<!-- image -->

Increasing attention has been paid to the pathways and mechanisms that detect nucleic acids, which are the primary immunostimulatory component of viruses for the innate immune system. TLRs (TLR3, TLR7) in endosomal compartments, and numerous other cytoplasmic DNA and RNA sensors have been identified to recognize nucleic acids from viruses (Paludan and Bowie, 2013; Barrat et al., 2016). Recognition of DNA or RNA by these sensors generally elicits antiviral responses via activation of transcriptional factors IRF and NFk B or inflammasomes. Cytoplasmic DNA sensors such as cyclic GMP-AMP synthase (cGAS) trigger downstream antiviral signaling generally via the adaptor protein stimulator of interferon genes (STING, also known as MITA, ERIS, or MPYS)-TANK-binding kinase 1 (TBK1) axis (Sun et al., 2013; Li et al., 2013). Meanwhile, RIG-I, melanoma differentiation-associated protein 5 (MDA5), and several other cytoplasmic RNA sensors trigger downstream signals generally via the caspase activation and recruitment domains (CARD)-containing adaptor mitochondrial antiviralsignaling protein (MAVS, also known as IPS-1, VISA, or Cardif) for induction of antiviral innate defense (Figure 1).

Inflammasomes are large multiprotein platforms typically triggered by various NLR members such as NLR family pyrin domain (PYD)-containing protein 3 (NLRP3) and NLR family CARD domain-containing protein 4 (NLRC4), which are capable of recognizing a wide range of PAMPs and DAMPs. Inflammasomes can also be activated by non-self DNA and RNA via AIM2 (Hornung et al., 2009; Bu rckstu mmer et al., 2009), RIG-I (Poeck et al., 2010), (Asp-Glu-Ala-His) box helicase 33 (DHX33) (Mitoma et al., 2013; Chakrabarti et al., 2015), and RIP1-RIP3DRP1 (Wang et al., 2014c) pathways (Figure 1). Activation of AIM2 inflammasomes by cytosolic bacteria such as Francisella tularensis subspecies novicida (F. novicida) requires type I IFNinduced guanylate-binding protein (GBPs) that mediate bacterium lysis for release of pathogenic DNA (Man et al., 2015;

Figure 1. Major Signaling Cascades Triggered by PRRs

<!-- image -->

<!-- image -->

Toll-like receptor (TLR), RIG-I-like receptors (RLRs), nucleotide-binding domain and leucinerich repeat containing molecules (NLRs), and numerous nucleic acid sensors trigger a complex signaling network converging on NFk B-dependent proinflammatory cytokine production and interferon regulatory factor (IRF)-dependent IFN production. (1) TLR (shown in light blue)-dependent signaling: TLRs are mostly located on the plasma membrane such as TLR4, TLR5, and TLR2 (TLR2 forms a heterodimer with TLR1 or TLR6) while TLR3, TLR7, TLR8, and TLR9 are present on endosomes. Most of TLRs activate NF- B via k myeloid differentiation primary response protein 88 (MyD88)-dependent pathway with the exception of endosomal TLR3, which activates IRF3 via TIR-domain-containing adaptor protein inducing interferon b (TRIF)-dependent pathway. TLR4 is also internalized into endosomes to mediate TRIF signaling via TRIF-related adaptor molecule (TRAM). For the MyD88-dependent pathway, TLR4 and TLR2 signaling involves MyD88 adaptor-like (Mal) facilitating the interaction between MyD88. MyD88 in turn activates E3 ligase TNF receptor-associated factor 6 (TRAF6), inducing the formation of kinase complexes composed of transforming growth factor beta b -activated kinase 1 (TAK1) and its ubiquitinbinding subunits, TAK1-associated binding pro-

tein 2 (TAB2), TAB3, and subsequently activating the kinase complex composed of I k B kinase a (IKK a ), IKK b , and the ubiquitin-binding subunits IKK g . Activated IKK complex promotes NFk B translocation to nucleus to drive the expression of proinflammatory cytokines. For the TRIF-dependent pathway, TRIF activates TRAF3/TANK-binding kinase 1 (TBK1) axis to promote phosphorylation of IRF3, which induces the expression of type I IFN. (2) RLR (shown in half green as being RLRsandhalf dark blue as being cytoplasmic RNA sensors)-dependent signaling: Retinoic acid-inducible gene I (RIG-I) and melanoma differentiation-associated protein 5 (MDA5) activate IRF3 via mitochondrial antiviral-signaling protein (MAVS)/TBK1 signaling; RIG-I also activates NFk B via caspase recruitment domaincontaining protein 9 (CARD9)/B cell lymphoma/leukemia 10 (Bcl-10) for pro-interleukin-1 b (pro-IL-1 b ) transcription and activates inflammasomes through binding to apoptosis-associated speck-like protein containing a CARD (ASC) for IL-1 b maturation. (3) NLR (shown in purple)-dependent signaling: Nucleotide-binding oligomerization domain-containing protein 1 (NOD1) and NOD2 activate NFk B via the receptor-interacting protein 2 (RIP2)/TAK1 pathway. Several NLR family members trigger inflammasome activation, and those inflammasome-dependent signaling pathways are introduced below in (6). (4) Cytoplasmic DNA sensors (shown in dark red)-triggered signaling: Cyclic GMP-AMP synthase (cGAS), DNA-dependent activator of IRFs (DAI), IFN-inducible protein-16 (IFI16), DEAD (AspGlu-Ala-Asp) box helicase 41 (DDX41), DNA-dependent protein kinase (DNA-PK), and meiotic recombination 11 homolog A (MRE11) activate IRF3 via STING/ TBK1 signaling; (Asp-Glu-Ala-His) box helicase 9 (DHX9) and DHX36 activate NFk B and IRF7 via MyD88 in plasmacytoid dendritic cells; DDX60 and DNAdependent RNA polymerase III (Pol III) promote RIG-I-mediated signaling; Ku70 activates IRF1 and IR7 to produce type III IFN; leucine-rich repeat flightlessinteracting protein 1 (LRRFIP1) activates IRF3 via b -catenin; Sry (Sex Determining Region Y)-box 2 (SOX2) activates NFk B via TAK1; DNA repair protein Rad50 activates NFk B-dependent pro-IL-1 b transcription via MAVS/CARD9/Bcl-10 signaling; and AIM2 and nuclear IFI16 activate inflammasome-dependent IL-1 b maturation. (5) Cytoplasmic RNA sensors (shown in dark blue)-triggered signaling: DDX3, DHX9, DHX33, and DHX15 activate IRF3 via association with MAVS; a DDX1-DDX21-DHX36 complex activates IRF3 and NFk Bvia TRIF; IFN-induced protein with tetratricopeptide repeats 1 (IFIT1) inhibits growth and pathogenicity of viruses by sequestering 5 -triphosphate viral RNA. (6) Inflammasome signaling: NLR members such as NLR family pyrin domain (PYD)-containing protein 3 0 (NLRP3) and NLR family CARD domain-containing protein 4 (NLRC4), RIG-I (which detects RNA), and AIM2 and nuclear IFI16 (which detect DNA), all activate apoptosis-associated speck-like protein containing a CARD (ASC)/caspase-1 inflammasomes leading to IL-1 b maturation. Activation of AIM2 inflammasome by cytosolic bacteria requires guanylate-binding proteins (GBPs)-mediated bacterium lysis for release of pathogenic DNA. GBPs is induced by type I IFN in a manner dependent on type I IFN receptor 1 (IFNAR1). RNA viruses also initiate the assembly of the RIP1-RIP3 complex, causing mitochondrial damage and activation of the NLRP3 inflammasome via the GTPase dynamin-related protein1 (DRP1). In addition, DHX33 binds to dsRNA and activates the NLRP3 inflammasome after endoribonuclease ribonuclease L cleavage of activated RNA.

Meunier et al., 2015). GBPs induced by cytosolic bacterial infection are encoded by interferon-stimulated genes (ISGs) in a cGAS-STING pathway and type I IFN receptor 1 (IFNAR1)dependent manner. Upon cellular infection or dangerous stress, assembled inflammasomes recruit the adaptor apoptosis-associated speck-like protein containing a CARD (ASC) to activate caspase-1, which mediates the maturation and secretion of IL1 b , IL-18, and initiates pyroptosis, contributing to anti-pathogen inflammatory responses. ASC particles could also be released to extracellular space for continued activation of caspase-1 and to amplify inflammatory responses in adjacent cells after being endocytosed (Franklin et al., 2014; Baroja-Mazo et al., 2014). While caspase-1 is critical for canonical inflammasome activation, some non-canonical inflammasomes induced by caspase-8 during fungal and mycobacterial infection or caspase-11 during

gram-negative bacterial infection are also reported (Gringhuis et al., 2012; Kayagaki et al., 2011; Rathinam et al., 2012). Lipopolysaccharide (LPS) from gram-negative bacteria is shown to be directly detected by caspase-11 and human caspase-4 or caspase-5 (Shi et al., 2014). Inflammasome-dependent caspase-1 and LPS-dependent caspase-4, -5, and -11 trigger pyroptosis via cleavage of another crucial component of inflammasomes, gasdermin D (Gsdmd) (Kayagaki et al., 2015; Shi et al., 2015; He et al., 2015).

In order to avoid harmful pathology caused by uncontrolled innate inflammatory responses, the immune system has evolved comprehensive and multi-level mechanisms to tightly regulate PRR signaling, including (1) transcriptional regulation of chromatin state, histone modification, DNA modifications, and transcriptional factor or co-regulator activity (Medzhitov and Horng,

<!-- image -->

2009; A ' lvarez-Errico et al., 2015; Zhang et al., 2015); (2) posttranscriptional regulation of mRNA splicing, export, and stability (Carpenter et al., 2014); (3) translational regulation of protein production and abundance (Piccirillo et al., 2014); and (4) posttranslational regulation of protein function and activity (Deribe et al., 2010). These successive molecular events together regulate the initiation, activation, and termination of PRR responses and determine the strength of the inflammatory response against microbial infection. While the first three layers of regulation set the critical checkpoint for DNA transcription and mRNA translation, the post-translational level of regulation creates an equally important defense to ensure an appropriate duration and magnitude of inflammation.

Post-translational modifications (PTMs) are important in regulating the activity, stability, and folding of proteins by inducing their covalent linkage to new functional groups, such as phosphate, methyl groups, and acetate (Deribe et al., 2010). Various PTMs have emerged as critical regulators of immunity through their effects on the activation, survival, proliferation, differentiation, and migration of immune cells. Conventional PTMs such as phosphorylation and polyubiquitination, as well as unconventional PTMs such as methylation and acetylation, have been shown to influence PRR-dependent inflammatory responses via their targeting of innate sensors and downstream signaling molecules, including receptors, adaptors, enzymes, and transcriptional factors (Deribe et al., 2010; Mowen and David, 2014; Cao, 2016). Moreover, PTMs dynamically alter the compartmentalization, trafficking, and physical interaction of key molecules that regulate immunological processes. In this review, we summarize the roles of PTMs in the positive and negative regulation of innate inflammatory responses and the implications for pathogenesis and translational studies of the inflammatory immune diseases.

## Protein Phosphorylation in the Control of Innate Immunity and Inflammatory Responses

Phosphorylation is the most extensively investigated PTM type in innate immunity, which is inversely regulated by kinases and phosphatases (Karin and Ben-Neriah, 2000). Protein phosphorylation and dephosphorylation plays an important role in innate immune responses by regulating the activation and deactivation of multiple TLR-dependent signaling molecules, such as MAPKs, NF- Binhibitor (I k k B) kinase a (IKK a ), IKK b , I k B , and IRF3 (Arthur a and Ley, 2013). Phosphorylation of innate adaptor proteins MAVS, STING, and Toll-interleukin-1 receptor (TIR)-domaincontaining adaptor protein inducing interferonb (TRIF) occurring at a conserved p x L IS motif is required for recruitment of IRF3 and production of type I IFN, which indicates an essential role for protein phosphorylation in the activation of antiviral innate immune signaling (Liu et al., 2015). Similarly, self-association and autophosphorylation of TBK1 mediated by glycogen synthase kinase 3 b (GSK3 ) is critical for virus-induced IRF3 activab tion (Lei et al., 2010). Moreover, receptor-interacting protein 2 (RIP2) undergoes autophosphorylation, which is necessary for effective nucleotide-binding oligomerization domain-containing protein 2 (NOD2) signaling and is absent from the most common Crohn's disease-associated NOD2 allele (Tigno-Aranjuez et al., 2010). Phosphorylation of ASC by kinases Syk and Jnk promotes inflammasome activity by promoting ASC specks formation and

caspase-1 activation (Hara et al., 2013). Phosphorylation of NLRC4 is also critical for inflammasome activation by promoting inflammasome specks assembly, and biochemical purification and screening implicates protein kinase Cd (PKC- ) as a potend tial NLRC4 kinase (Qu et al., 2012). Whether additional adaptor proteins are recruited or regulated by phosphorylated inflammasome components and their potential role in the dynamical regulation of inflammasome assembly and activation requires further investigation.

Accordingly, numerous proteins have been identified that limit PRR signaling via regulating protein (de)phosphorylation, such as Src homology (SH2)-containing protein tyrosine phosphatase 1 (SHP1) (An et al., 2008), mammalian STE20-like protein kinase 4 (MST4) (Jiao et al., 2015), NLRC5 (Cui et al., 2010), NLR family member X1 (NLRX1) (Xia et al., 2011), NLRP6 (Anand et al., 2012), and protein phosphatase 2A (PP2A) (Long et al., 2014) (Figure 2 and Table 1). As an example, MST4 kinase suppresses TLR-triggered inflammatory responses by directly phosphorylating the adaptor tumor necrosis factor receptor-associated factor 6 (TRAF6) to prevent its oligomerization and autoubiquitination. MST4 knockdown mice show exacerbated inflammation and death in septic shock, a phenotype that is reversed by heterozygous deletion of Traf6 (Jiao et al., 2015). Interestingly, long non-coding RNAs (lncRNAs) also participate in innate immune cell differentiation via regulating protein PTM. Lnc-DC is exclusively expressed in human conventional dendritic cells (DCs) and is required for DC differentiation by directly binding to phosphorylated STAT3 in the cytoplasm to prevent its dephosphorylation (Wang et al., 2014a). This study adds mechanistic insights into how lncRNAs directly bind signaling molecules in the cytoplasm to regulate their interactions and PTM, thus suggesting a novel functional model for cytoplasmic lncRNAs-mediated PTM of signaling molecule in biological processes, highlighting the importance of chromatin modifiers as critical regulators of innate immunity and inflammation (Zhang and Cao, 2015).

## Protein Ubiquitination in the Control of Innate Immunity and Inflammatory Responses

Protein ubiquitination is inversely regulated by E1, E2, and E3 enzymes or deubiquitylation enzymes (DUBs) and plays critical roles in a wide variety of immunological processes such as antigen processing, cell differentiation, immune defense, and inflammation (Jiang and Chen, 2012). Lysine 63 (K63)-linked polyubiquitination is involved in the transduction of signaling pathways, while K48-linked polyubiquitination induces the proteasomal degradation of targeted proteins (Jiang and Chen, 2012; Heaton et al., 2016). Other types of polyubiquitination such as K6, K11, K27, K29, K33, and linear-linked polyubiquitination are being increasingly investigated for their divergent roles in immunity and inflammation (Jiang and Chen, 2012; Heaton et al., 2016).

In the TLR pathway, K63-linked polyubiquitination of TRAF6, TAB2/3, NEMO, and TRAF3 are required for NFk B and IRF3 activation. K48-linked polyubiquitination induces the proteasomal degradation of I k B, which allows the nuclear translocation of NFk B. Linear ubiquitin assembly complex (LUBAC)-mediated linear polyubiquitination of NEMO is also important for NFk B activation (Tokunaga et al., 2009). In addition, multiple ubiquitination-dependent processes are critical for the production of

Figure 2. Negative Regulation of PRR Signaling Pathways by Post-Translational modifications

<!-- image -->

<!-- image -->

Numerous proteins negatively regulate PRR signaling by affecting the post-translational modifications (PTMs) of signaling molecules. Color differences indicate various strategies the proteins use on their targets. A detailed list of the target proteins, regulators for PTMs, PTMs, and signaling functions are included in Table 1.

Abbreviations for signaling molecules are as follows: TLR, Toll-like receptors; MyD88, myeloid differentiation primary response protein 88; Mal, MyD88 adaptor-like; IRAK, interleukin 1 receptor associated kinase; TRAF6/3, tumor necrosis factor receptor-associated factor 6/3; TAK1, transforming growth factor beta b -activated kinase1; TAB2/3, TAK1-associated binding protein 2/3; I k B, NFk B inhibitor; IKK a b g / / , I k B kinase a b g / / ; NF- B, k nuclear factork B; NOD2, nucleotidebinding oligomerization domain 2; RIP2, receptorinteracting protein 2; TRIF, Toll/interleukin-1 receptor (TIR)-domain-containing adaptor protein inducing interferon b ; TRAM, TRIF-related adaptor molecule; RIG-I, retinoic acid-inducible gene I; MDA5, melanoma differentiation-associated protein 5; MAVS, mitochondrial antiviral-signaling protein; TBK1, TANK-binding kinase 1; IRF3, interferon regulatory factor 3; cGAS, cyclic GMP-

AMP synthase; DDX41, DEAD (Asp-Glu-Ala-Asp) box helicase 41; STING, stimulator of interferon genes.

Abbreviations for regulators are as follows: SOCS1/3, suppressor of cytokine signaling 1/3; Nrdp1, neuroregulin receptor degradation protein 1; Smurf, smad ubiquitin regulator factor proteins; TANK, TRAF family member-associated NFk B activator; SHP, small heterodimer partner; MHC-I, major histocompatibility complex class I molecules; FBXL2/11, F-box and leucine-rich repeat protein 2/11; NLRC3/5, NLR family CARD domain-containing protein 3/5; Rhbdd3, rhomboid domain containing 3; PDLIM2, PDZ and LIM domain 2; ARIH2, ariadne RBR E3 ubiquitin protein ligase 2; Triad3A, Triad domain-containing protein 3, variant A; WWP2, WW domain-containing protein 2; DUBA, de-ubiquitinating enzyme A; RING finger protein 125/5, RNF125/5; Siglec-G/1, sialic acid binding immunoglobulin-like lectin G/1; CYLD, cylindromatosis; CHIP, carboxyl terminus of constitutive heat shock cognate 70-interacting protein; PCBP2, poly(rC) binding protein 2; MARCH5, membrane-associated RING finger protein 5; NLRP4/6, NLR family pyrin domain (PYD)-containing protein 4/6; TRIP, TRAF-interacting protein; RBCK1, RBCC protein interacting with PKC1; Pin1, peptidylprolyl cis /trans isomerase, NIMA-interacting 1; TRIM21/27, tripartite-motif protein 21/27; SHP1, Src homology (SH2)-containing protein tyrosine phosphatase 1; MST4, mammalian STE20-like protein kinase 4 ; NLRX1, NLR family member X1; PP2A, protein phosphatase 2A; SENP6/2, SUMO-specific protease 6/2; PIAS3, protein inhibitor of activated STAT3; SETD6, SET domain containing protein 6.

type I IFN in the antiviral signaling pathway, including K63-linked polyubiquitination of RIG-I mediated by tripartite-motif protein 25 (TRIM25) (Gack et al., 2007) and by Riplet (Oshiumi et al., 2009), MAVS ubiquitination and degradation mediated by TRIM25 (Castanier et al., 2012), K63-linked polyubiquitination of STING by TRIM56 (Tsuchida et al., 2010), K27-linked polyubiquitination of STING by autocrine motility factor receptor (AMFR) (Wang et al., 2014b), and K11-linked polyubiquitination of STING by RING finger protein 26 (RNF26) that competes for RNF5-mediated K48-linked polyubiquitination and degradation (Qin et al., 2014; Zhong et al., 2009). Moreover, K63-linked polyubiquitination of RIP2 catalyzed by Pellino3 is critical for the NOD2-induced activation of NF- B (Yang et al., k 2013a). Deubiquitination of NLRP3 by deubiquitinating enzyme BRCC3 and linear ubiquitination of ASC by LUBAC are important for NLRP3 inflammasome assembly and activation (Py et al., 2013; Rodgers et al., 2014). The differential regulation of inflammasome sensors by different types of polyubiquitinations, their respective (or shared) ubiquitination sites, structural basis, and signaling regulatory roles in innate immunity still warrant further investigation.

Accordingly, targeting ubiquitination emerges as an essential and common regulatory mechanism for TLR signaling inhibition: e.g., by suppressor of cytokine signaling 1 (SOCS1) (Mansell et al., 2006), neuroregulin receptor degradation protein 1 (Nrdp1) (Wang et al., 2009), smad ubiquitin regulator factor pro-

teins (Smurf) (Lee et al., 2011), A20 (Boone et al., 2004), TRAF family member-associated NFk B activator (TANK) (Kawagoe et al., 2009), b -arrestin (Wang et al., 2006), SHP (Yuk et al., 2011), major histocompatibility complex (MHC) class I molecules (Xu et al., 2012), F-box and leucine-rich repeat protein 2 (Fbxl2) (Chen et al., 2013a), NLRC3 (Schneider et al., 2012), rhomboid domain containing 3 (Rhbdd3) (Liu et al., 2014), PDZ and LIM domain 2 (PDLIM2) (Tanaka et al., 2007), ariadne RBR E3 ubiquitin protein ligase 2 (ARIH2) (Lin et al., 2013), Triad domain-containing protein 3, variant A (Triad3A) (Chuang and Ulevitch, 2004), WW domain-containing protein 2 (WWP2) (Yang et al., 2013b), de-ubiquitinating enzyme A (DUBA) (Kayagaki et al., 2007), and serine/threonine kinase 38 (Stk38) (Wen et al., 2015) (Figure 2 and Table 1). Using the rhomboid family of proteases as an example, Rhbdd3 inhibits TLR-triggered activation of NF- B and IL-6 production in DCs through binding to K27-linked k ubiquitination of NFk Bessential modifier (NEMO, also known as IKK g ) and subsequently recruiting A20 via K27-linked ubiquitination on Rhbdd3 itself, which facilitates A20-mediated K63-linked deubiquitination of IKK g (Liu et al., 2014). NEMO is capable of binding various types of ubiquitin chains, including K63-, K48-, and K27-linked ubiquitin chains, as well as linear-linked ubiquitin chains (Jiang and Chen, 2012). It will be intriguing to investigate the cross-regulation of different types of polyubiquitination on the same substrate such as NEMO. Absence of some regulators such as A20 (Boone et al., 2004; Hammer et al., 2011; Vande

<!-- image -->

Walle et al., 2014), Tank (Kawagoe et al., 2009), Rhbdd3 (Liu et al., 2014), or Arih2 (Lin et al., 2013) renders mice susceptible to multi-organ inflammatory and autoimmune diseases, indicating a critical role of ubiquitin modification in the pathogenesis and potential treatment of PRR signaling-associated inflammatory immune diseases (Liu and Cao, 2015).

Polyubiquitination is also a key regulatory mechanism for post-translational regulation of host antiviral responses by RNF125 (Arimoto et al., 2007), sialic acid binding immunoglobulin-like lectin G (Siglec-G) (Chen et al., 2013c), cylindromatosis (CYLD) (Friedman et al., 2008), CHIP (carboxyl terminus of constitutive heat shock cognate 70-interacting protein) (Zhao et al., 2016), poly(rC) binding protein 2 (PCBP2) (You et al., 2009), membrane-associated RING finger protein 5 (MARCH5) (Yoo et al., 2015), NLRP4 (Cui et al., 2012), TRAF-interacting protein (TRIP) (Zhang et al., 2012), Siglec-1 (Zheng et al., 2015), RBCC protein interacting with PKC1 (RBCK1) (Zhang et al., 2008), peptidylprolyl cis-trans isomerase, NIMA-interacting 1 (Pin1) (Saitoh et al., 2006), TRIM21 (Zhang et al., 2013), and RNF5(Zhonget al., 2009) (Figure 2 and Table 1). In the NLR pathways, SOCS3and TRIM27 induce polyubiquitination and proteasomal degradation of NOD2 (Lee et al., 2012; Zurek et al., 2012). Moreover, dopamine D1 receptor (DRD1) signaling by neurotransmitter dopamine (DA) inhibits the NLRP3 inflammasome by promoting its K48-linked ubiquitination and degradation via the E3 ubiquitin ligase MARCH7. Mice deficient in Drd1 show exaggerated neurotoxin-induced neuroinflammation and LPSinduced systemic inflammation due to NLRP3 overactivation. This study indicates the critical role of DA-DRD1 signaling and the subsequent ubiquitination of NLRP3 in the control of inflammasome activity and systemic inflammatory response and suggests DRD1 as a potential drug target for NLRP3 inflammasome-related diseases (Yan et al., 2015).

## Unconventional PTMs in the Control of Innate Immunity and Inflammatory Responses

The involvement of unconventional PTM in innate immune responses is being increasingly shown (Mowen and David, 2014). Methylation of lysine or arginine residues inversely regulated by methyltransferases or demethylases is implicated in NF- B-dependent k innate immune responses. For example, methyltransferase SET domain-containing protein 6 (SETD6) mediates monomethylation of NFk B p65 subunit at K310 and inhibits p65-driven inflammatory responses under basal conditions (Levy et al., 2011). By contrast, the lysine demethylase F-box and the leucine-rich repeat protein 11 (FBXL11) limit NFk B activity by mediating lysine demethylation of p65 at K218 and K221 (Lu et al., 2010) (Figure 2 and Table 1). Moreover, direct methylation of the cytoplasmic integrin adaptor talin by histonelysine N-methyltransferase enhancer of zeste homolog 2 (Ezh2) promotes the turnover of adhesion structures by disrupting the binding of talin to F-actin, thereby promoting the extravasation and motility of DC and neutrophils under inflammatory conditions (Gunawan et al., 2015).

Acetylation of lysine residues, which is inversely regulated by acetyltransferases and deacetylases, plays a role in diverse cellular processes, such as chromatin remodeling, cell division, nuclear transport, and cell metabolism (Choudhary et al., 2009). Several studies have highlighted an important role for pro-

tein acetylation in the regulation of TLR, RLR, and inflammasome-dependent innate immune responses. TLR stimulation induces acetylation of MAPK phosphatase 1 (MKP1), which increases MKP1 interaction with p38 MAPK and decreases phosphorylation of p38 MAPK, leading to negative control of the TLR-triggered inflammatory response (Cao et al., 2008). In the antiviral immunity, histone deacetylase 9 (HDAC9) is upregulated by DNA methyltransferase Dnmt3a via epigenetic mechanisms, and the upregulated HDAC9 in turn deacetylases the kinase TBK1 for activation, contributing to enhanced IFN production (Li et al., 2016). This study highlights an important role for the interplay between epigenetic modifications and PTMs in the regulation of PRR-triggered antiviral innate immune responses. Histone deacetylase-mediated regulation of antiviral immunity was also supported by the finding showing deacetylation of RIG-I by HDAC6 in the presence of viral RNAs and the increased susceptibility to RNA virus infections in Hdac6 knockout mice (Choi et al., 2016). In addition, deacetylation of a -tubulin by protein deacetylase Sirtuin 2 (Sirt2) is induced by calorie restriction, which impedes the transport of mitochondria and inhibits NLRP3 inflammasome assembly. Metabolic stress induced by higher concentrations of glucose, fatty acids, cholesterol, and uric acid crystals results in aberrant mitochondrial homeostasis to inactivate Sirt2 (Misawa et al., 2013).

SUMOylation, which is regulated by specific small ubiquitinrelated modifier (SUMO) E1, E2, and E3 cascades, and desumoylases, has been shown to exert both suppressive and activating roles in TLR- and RLR-triggered innate immunity and inflammatory responses by altering protein stability, interactions, or activity. SUMOylation of RIG-I and of MDA5 via PIAS2 b increases antiviral type I IFN responses (Mi et al., 2010; Fu et al., 2011). By contrast, the SUMOylation of the p65 subunit of NFk B by E3 SUMO ligase protein inhibitor of activated STAT 3 (PIAS3) and of (c-Fos:c-Jun)-AP-1 suppress their activation (Liu et al., 2012; Tempe ' et al., 2014). In addition, SUMO-specific protease 6 (SENP6) and SENP2 respectively inhibit TLR inflammatory responses and cellular antiviral responses by catalyzing the de-SUMOylation of IKK g and IRF3 (Liu et al., 2013; Ran et al., 2011) (Figure 2 and Table 1). Thus SUMOylation might exert multiple roles in innate immunity via targeting various substrates. A more recent study shows that SUMOylation serves as a master regulator of both inflammatory and anti-viral responses in a manner dependent on the type I IFN receptor. Mechanistic study suggests SUMOylation targets Ifnb1 gene promoters to silence both basal and stimulus-induced activity of the Ifnb1 , further expanding the functional model of SUMOylation. Because inflammatory responses are not completely dependent of IFN signaling, future investigations are required for additional mechanistic explanation for SUMOylation-mediated universal inhibition of inflammatory and antiviral responses (Decque et al., 2016).

Protein succinylation through the addition of succinyl groups to lysine residues is a critical process in mitochondrial energy generation and is emerging as a potent regulator of immunity and inflammation (Mills and O'Neill, 2014). The tricarboxylicacid cycle intermediate succinate activates intracellular calcium mobilization, migratory responses and TLR-dependent proinflammatory cytokines in DCs via the G protein-coupled receptor, GPR91 (Rubic et al., 2008). A later study reveals an inflammatory

<!-- image -->

<!-- image -->

Table 1. Negative Regulation of PRR Signaling Pathways by Post-Translational Modifications

| Target Proteins   | Regulators for PTMs                     | PTMs              | Signaling Function                                                                                                                                                     |
|-------------------|-----------------------------------------|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Mal               | Signaling regulator, SOCS1              | Ubiquitination    | Promoting Mal ubiquitination and degradation, inhibiting TLR/NF- k B signaling                                                                                         |
| MyD88 and TBK1    | E3 ubiquitin ligase, Nrdp1              | Ubiquitination    | Promoting MyD88 and TBK1 ubiquitination, inducing degradation of MyD88 and activation of TBK1, inhibition of TLR/NF- k B signaling but promotion of TLR/IRF3 signaling |
| MyD88             | E3 ubiquitin ligase, Smurf              | Ubiquitination    | Mediating TGF- b -induced K48-linked ubiquitination and degradation of MyD88, inhibiting TLR4 signaling                                                                |
| IRAKs             | Tyrosine phosphatase, SHP-1             | Dephosphorylation | Inhibiting IRAK phosphorylation and activation, inhibiting TLR signaling                                                                                               |
| TRAF6 and IKK g   | Ubiquitin-editing enzyme, A20           | Deubiquitination  | Inhibiting K63-linked ubiquitination of TRAF6 and IKK g , inhibiting TLR4 signaling                                                                                    |
| TRAF6             | Multifunctional protein, b -arrestin    | Deubiquitination  | Directly binding to TRAF6 to prevent its autoubiquitination, inhibiting TLR-IL-1R signaling                                                                            |
| TRAF6             | TRAF-binding protein, TANK              | Deubiquitination  | Inhibiting TRAF6 ubiquitination by directly binding to TRAF6, inhibiting TLR signaling                                                                                 |
| TRAF6             | Orphan nuclear receptor, SHP            | Deubiquitination  | Inhibiting TRAF6 ubiquitination and repressing transactivation of the NF- k B subunit p65, inhibiting TLR signaling                                                    |
| TRAF6             | MHC class I molecules                   | Deubiquitination  | Inhibiting TRAF6 ubiquitination via the Fps-SHP-2 pathway, inhibiting TLR signaling                                                                                    |
| TRAF6             | NLR family member, NLRC3                | Deubiquitination  | Inhibiting K63-linked but keeping K48-linked ubiquitination of TRAF6, promoting TRAF6                                                                                  |
|                   | F box protein, FBXL2                    | Ubiquitination    | Interacting with and targets TRAFs for ubiquitination and degradation, suppressing TRAF protein levels and activation, inhibiting TLR4 signaling                       |
| TRAF1-6 TRAF6     | Kinase, MST4                            | Phosphorylation   | Directly interacting with and phosphorylating TRAF6 to prevent its oligomerization and autoubiquitination, inhibiting TLR4 signaling                                   |
| IKK g             | Rhomboid family member, Rhbdd3          | Deubiquitination  | Binding to K27-linked ubiquitination of IKK g and subsequently recruiting A20 to induce K63-linked deubiquitination of IKK g , inhibiting TLR/NF- k B                  |
| IKK g             | SUMO-specific protease, SENP6           | De-SUMOylation    | Catalyzing the de-SUMOylation of IKK g , inhibiting TLR signaling                                                                                                      |
| IKK a b /         | NLR family member, NLRC5                | Dephosphorylation | Interacting with IKK a and IKK b and blocking their phosphorylation, inhibiting NF- k B signaling                                                                      |
| IKK a b /         | NLR family member, NLRX1                | Dephosphorylation | Binding to the IKK complex to inhibit IKK a and IKK b phosphorylation, inhibiting TLR/NF- k B signaling                                                                |
| ERK and I k B     | NLR family member, NLRP6                | Dephosphorylation | Inhibiting TLR-mediated ERK and NF- k B phosphorylation and activation                                                                                                 |
|                   |                                         |                   | Binding to NF- k ubiquitination and degradation, inhibiting NF- k                                                                                                      |
| NF- B k           | E3 ligase, PDLIM2                       | Ubiquitination    | B p65 subunit and promoting p65 B activation                                                                                                                           |
| NF- B k           | Protein lysine methyltransferase, SETD6 | Methylation       | Mediating lysine methylation of nuclear NF- k B p65 subunit at K310, inhibiting p65-driven transcriptional programs                                                    |
| NF- B k           | Lysine demethylase, FBXL11              | Demethylation     | Mediating lysine demethylation of NF- k B p65 subunit at K218 and K221, inhibiting p65-driven transcriptional programs                                                 |
| NF- B k           | E3 SUMO ligase, PIAS3                   | SUMOylation       | Mediating SUMOylation of NF- k B p65 subunit, inhibiting NF- k B activation                                                                                            |

( Continued on next page )

## Immunity

## Review

| Table 1. Continued Target Proteins   | Regulators for PTMs                    | PTMs              | Signaling Function                                                                                                                                              |
|--------------------------------------|----------------------------------------|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
| I k B b                              | E3 ligase, ARIH2                       | Ubiquitination    | Binding to nuclear hypophosphorylated I k B , leading b to its ubiquitination and degradation, inhibiting the transcriptional activity of p65                   |
| TLR4                                 | E3 ligase, Triad3A                     | Ubiquitination    | Enhances ubiquitination and proteolytic degradation of some TLRs (TLR4 and TLR9), inhibiting TLR signaling                                                      |
| TRIF                                 | E3 ligase, WWP2                        | Ubiquitination    | Mediating K48-linked ubiquitination and degradation of TRIF, inhibiting TLR3-mediated NF- k B and IRF activation                                                |
| TRAF3                                | Deubiquitinating enzyme, DUBA          | Deubiquitination  | Selectively inhibiting K63-linked ubiquitination of TRAF3, inhibiting type I IFN response                                                                       |
| RIG-I                                | E3 ligase, RNF125                      | Ubiquitination    | Promoting ubiquitination and proteasomal degradation of RIG-I, inhibiting RIG-I-dependent antiviral signaling                                                   |
| RIG-I                                | Deubiquitinating enzyme, CYLD          | Deubiquitination  | Inhibiting ubiquitination of RIG-I, inhibiting RIG-I-dependent antiviral signaling                                                                              |
| RIG-I                                | Lectin family member, Siglec-G         | Ubiquitination    | Recruiting E3 ubiquitin ligase c-Cbl to promote RIG-I degradation via K48-linked ubiquitination at K813, inhibiting RIG-I-dependent antiviral signaling         |
| RIG-I                                | E3 ligase, CHIP                        | Ubiquitination    | Mediating proteasomal degradation of RIG-I via K48-linked ubiquitination, inhibiting RIG-I-dependent antiviral signaling                                        |
| MAVS                                 | RNA-binding protein, PCBP2             | Ubiquitination    | Recruiting the HECT domain-containing E3 ligase AIP4 to induce ubiquitination and degradation of MAVS, inhibiting MAVS-mediated antiviral signaling             |
| MAVS                                 | E3 ligase, MARCH5                      | Ubiquitination    | Promoting ubiquitination of MAVS at K7 and K500 and its proteasomal degradation, inhibiting MAVS-dependent antiviral signaling                                  |
| DDX41                                | E3 ligase, TRIM21                      | Ubiquitination    | Mediating K48-linked ubiquitination of DDX41 at K9 and K115 and its degradation, inhibiting dsDNA-triggered innate signaling                                    |
| STING                                | E3 ligase, RNF5                        | Ubiquitination    | Promoting ubiquitination of STING at K150 and its degradation, inhibiting IRF3-dependent antiviral signaling                                                    |
| TBK1                                 | NLR family member, NLPR4               | Ubiquitination    | Recruiting the E3 ligase DTX4 to induce K48-linked ubiquitination of TBK1 at K670 and its degradation, inhibiting type I IFN response                           |
| TBK1                                 | E3 ligase, TRIP                        | Ubiquitination    | Directly binding to and promoting K48-linked ubiquitination of TBK1 and its proteasomal degradation, inhibiting TLR3/4- and RIG-I-dependent antiviral signaling |
| TBK1                                 | Lectin family member, Siglec-1         | Ubiquitination    | Recruiting E3 ligase TRIM27 to induce K48-linked ubiquitination TBK1 at K251 and K372 and its degradation, inhibiting antiviral signaling                       |
| IRF3                                 | Prolyl isomerase, Pin1                 | Ubiquitination    | Mediating ubiquitination and proteasomal degradation of IRF3, inhibiting IRF3-dependent antiviral signaling                                                     |
| IRF3                                 | E3 ligase, RBCK1                       | Ubiquitination    | Mediating ubiquitination and proteasomal degradation of IRF3, inhibiting IRF3-dependent antiviral signaling                                                     |
| IRF3                                 | Serine and threonine phosphatase, PP2A | Dephosphorylation | Mediating dephosphorylation and deactivation of IRF3, inhibiting IRF3-type I IFN signaling                                                                      |
| IRF3                                 | SUMO-specific protease, SENP2          | De-SUMOylation    | Mediating de-SUMOylation of IRF3 and conditioning it for ubiquitination and degradation, inhibiting IRF3-dependent antiviral signaling                          |
| NOD2                                 | Signaling regulator, SOCS3             | Ubiquitination    | Inducing ubiquitination and proteasomal degradation of NOD2, inhibiting NOD2 signaling                                                                          |
| NOD2                                 | E3 ligase, TRIM27                      | Ubiquitination    | Inducing ubiquitination and proteasomal degradation of NOD2, inhibiting NOD2 signaling                                                                          |

Figure 3. Regulation of Protein Compartmentalization in Innate Immunity and Inflammation by Post-Translational modifications

<!-- image -->

<!-- image -->

Post-translational modifications (PTMs) and posttranslational processing of proteins regulate the outcome of innate immune responses in an organelle-selective manner. The distribution of Toll-like receptor 4 (TLR4) and TLR9 from the endoplasmic reticulum (ER) to the plasma membrane or endosomes, respectively, is regulated by protein unc-93 homolog B1 (UNC93B1) and glycosylated heat shock protein glycoprotein 96 (gp96). Internalization of TLR4 to endosomal compartments is regulated by p110 d isoform of phosphatidylinositol-3-OH kinase (PI3K d ). Hepatocyte growth factor-regulated tyrosine kinase substrate (Hrs) and lysosome-associated small guanosine triphosphatase (GTPase) protein Rab7b promote TLR4 degradation in a lysosomedependent manner, while TLR4 and TLR9 also undergo ubiquitination and proteosomal degradation mediated by Triad domain-containing protein 3, variant A (Triad3A). Inactive rhomboid protein 2 (iRhom2) is required for TNFa -converting enzyme (TACE) exit from the ER to promote tumor necrosis factor (TNF) shedding, while cathepsin B and golgin promote TNF release from the Golgi

complex. Intracellular DNA activates stimulator of interferon genes (STING) translocation from the ER to perinuclear vesicles in a manner dependent on K27linked polyubiquitination mediated by autocrine motility factor receptor (AMFR). Autophagy-related gene 9a (Atg9a) interacts with STING to block the translocation of STING and its assembly with TBK1. Intracellular dsRNA activates K63-linked polyubiquitination of mitochondrial antiviral-signaling protein (MAVS) and its translocation to mitochondria and mitochondrial-associated endoplasmic reticulum membranes (MAM). Mitofusin 1 (MFN1) redistributes MAVS to specklelike aggregates to facilitate MAVS-dependent antiviral responses, but MFN2 inhibits MAVS activation on mitochondria via direct interaction. NLR family pyrin domain (PYD)-containing protein 3 (NLRP3) is recruited to mitochondria from the cytosol upon activation, which is regulated by MAVS.

role for succinate in innate immune signaling. Succinate is significantly increased in LPS-stimulated macrophages, and in turn stabilizes the transcription factor hypoxia-inducible factor 1 a (HIF1 a ) to promote the production of the proinflammatory cytokine IL-1 b (Tannahill et al., 2013).

Notably, PTMs, especially methylation and acetylation, of histone substrates are emerging as potent regulators of innate immunity via epigenetic modulation of chromatin status and DNA transcription (Alvarez-Errico et al., 2015). For example, protein ' lysine methyltransferase SET domain, bifurcated 2 (Setdb2) is upregulated upon viral infection and then occupies and induces the repressive H3K9me3 of Cxcl1 gene promoters, leading to reduced neutrophil infiltration and as a result attenuated host defense against secondary bacterial infection (Schliehe et al., 2015). The molecular basis for the selectivity of histone and non-histone substrates for PTMs and their biological relevance and functional cross-regulation still remain elusive and require further investigations.

## PTMs in the Control of Compartmentalization and Trafficking of Innate Sensors and Signaling Molecules

Distinct subcellular compartments, such as the endoplasmic reticulum (ER), endosomes, lysosomes, mitochondria, and proteasomes, provide unique biochemical and biological environments that potentially affect the quality of signaling transduction. The organelle-specific regulation of PRR signaling molecules by PTMs serves as an indispensable gatekeeping step that determines the outcome of innate immune responses (Brubaker et al., 2015). In the following section, we focus on how PTMs affect protein compartmentalization and trafficking during different phases of innate inflammatory responses (Figure 3).

The ER is the compartment where proteins undergo folding and transport to the Golgi apparatus. The trafficking of TLRs to specific destinations, such as nucleotide-sensing TLR9 from the ER to endolysosomes, and LPS-detecting TLR4 to the plasma membrane, are crucial for the proper initiation of TLR signaling. This is mediated by ER proteins such as protein unc93 homolog B1 (UNC93B1) (Brinkmann et al., 2007; Kim et al., 2008) and heat shock protein glycoprotein 96 (gp96) (Yang et al., 2007). Both TLR4 and TLR9 preferentially interact with glycosylated gp96, and this interaction was significantly inhibited by an inhibitor of N-glycosylation, tunicamycin. Thus, gp96 chaperones TLR4 and TLR9 in a manner that is dependent on N-linked glycosylation. In addition, post-translational modifications are essential for the processing and secretion of cytokines. A polytopic ER protein, inactive rhomboid protein 2 (iRhom2), is specifically upregulated in the ER of macrophages in response to LPS stimulation and promotes the transfer of TNFa -converting enzyme (TACE) from the ER to the cell surface, thereby facilitating LPS-induced TNF shedding. iRhom2 deficiency renders mice unable to control Listeria monocytogenes infection and prevents LPS-induced liver pathology by inhibiting TNF production (Adrain et al., 2012; McIlwain et al., 2012). TACE matures by acquiring Golgi N-glycans and phosphorylation (Dı 'az-Rodrı 'guez et al., 2002), and it would be intriguing to discover whether PTM of iRhom2 could be important for the TNF shedding activity. A lysosomal cysteine proteinase, cathepsin B, and a trans-Golgi network golgin are both required for optimal post-translational processing of TNF-containing vesicles to the plasma membrane in response to LPS (Ha et al., 2008; Lieu et al., 2008). Furthermore, detection of cytosolic DNA can trigger STING translocation from the ER to the Golgi complex and finally to the

<!-- image -->

perinuclear microsome compartment for the activation of type I IFN-dependent innate immunity via TBK1 (Ishikawa et al., 2009). The trafficking of STING from the ER to perinuclear vesicles is promoted by E3 ubiquitin ligase AMFR, which catalyzes the K27-linked polyubiquitination of STING (Wang et al., 2014b). By contrast, an essential autophagy protein, autophagy-related gene 9a (Atg9a), negatively regulates dsDNA-triggered innate immunity by inhibiting the translocation of STING and its assembly with TBK1 (Saitoh et al., 2009).

Endosomes are classically linked with the internalization, sorting and degradation of proteins before they reach the lysosomes. The p110 d isoform of phosphatidylinositol-3-OH kinase (PI(3)K), PI3K , enhances internalization of TLR4 from the plasma memd brane to the endosomal compartment, and thus induces dissociation of the TIR domain containing adaptor protein (TIRAP, also known as myeloid differentiation primary response protein 88 [MyD88] adaptor-like [Mal]) from the plasma membrane. This leads to the suppression of TIRAP-mediated proinflammatory cytokine production but the promotion of TRIF-related adaptor molecule (TRAM)-dependent anti-inflammatory cytokines, IL-10 and IFNb , via the phosphorylation of the Akt pathway. Consistently, p110 -deficient d mice show increased susceptibility to endotoxin shock, indicating the importance of TLR4 subcellular compartmentalization in regulating the overall balance of TLR4induced inflammatory responses (Aksoy et al., 2012).

Lysosomes have powerful hydrolytic enzymes that are capable of digesting unwanted cellular debris. The lysosomal degradation of proteins is another critical regulatory process in PRR signaling. TLR4 is associated with the ubiquitin-binding endosomal sorting protein hepatocyte growth factor-regulated tyrosine kinase substrate (Hrs), which directs TLR4 to lysosomes for degradation (Husebye et al., 2006). Similarly, the lysosomeassociated small guanosine triphosphatase (GTPase) protein Rab7b, a member of the RAS oncogene family, acts as a negative regulator of TLR4 signaling by promoting the degradation of lysosomal TLR4 (Wang et al., 2007). The role of Rab7b in negative regulation of TLR4 signaling is related to its GTPase activity. Additionally, another study showed that TLR4 and TLR9 are subjected to polyubiquitination and proteasomal degradation by the Triad3A ubiquitin ligase (Chuang and Ulevitch, 2004).

Mitochondria are energy-producing organelles involved in diverse cellular processes, including mitophagy and autophagy, apoptosis, and metabolism. Mitochondrial oxidative phosphorylation is essential for the activation of the NLRP3 inflammasome. The compartmentalization of inflammasomes in the cytosol or mitochondria is regulated by another mitochondria-associated molecule, MAVS, which promotes the recruitment of NLRP3 to mitochondria and the production of IL-1 b (Subramanian et al., 2013). Mitochondria are also regarded as critical platforms for antiviral signaling due to MAVS. Localization of MAVS to mitochondria by its C-terminal transmembrane domain is required for downstream antiviral signal transduction following viral infection (Seth et al., 2005). Mitochondria-associated ER membrane (MAM) acts as an important subcellular compartment for MAVS signaling by linking the ER to mitochondria (Horner et al., 2011). Mitochondrial fusion proteins become involved in MAVS-mediated mitochondrial antiviral responses through different mechanisms. Mitofusin 1 (MFN1) redistributes MAVS to speckle-like aggregates and thus facilitates MAVS-dependent

antiviral responses (Onoguchi et al., 2010). By contrast, MFN2 prevents MAVS activation via direct interactions on the mitochondrial outer membrane and not via its role in mitochondrial dynamics (Yasukawa et al., 2009).

## PTMs in the Control of Protein Interactions during Innate Immunity and Inflammatory Responses

Physical interaction and assembly bridge the formation of activating or inhibitory signaling complexes under steady-state and inflammatory conditions, and are frequently accompanied by various PTMs. Inflammasomes are multicomponent signaling complexes composed of sensor proteins ASC and caspase-1. The adaptor protein ASC connects sensor proteins and caspase-1 through its CARD and PYD domains to form a prionlike protein aggregate structure (Lu et al., 2014; Cai et al., 2014). MAVS also displays a prion-like polymerization through its CARD domain, which is required for RIG-I triggered antiviral immune responses (Hou et al., 2011; Xu et al., 2014). Therefore prion-like aggregation of signaling proteins represents an important mechanism for PRR-triggered signaling activation.

Interestingly, without the need for particular PTMs, the sequestration or disruption of activating complexes directly mediates the silencing of TLR signaling through SOCS1 (Nakagawa et al., 2002), interleukin-1 receptor associated kinase M (IRAK-M) (Kobayashi et al., 2002), TRAM adaptor with GOLD domain (TAG) (Palsson-McDermott et al., 2009), sterile alpha and TIR motif containing protein (SARM) (Carty et al., 2006), and SH2-containing protein tyrosine phosphatase 2 (SHP2) (An et al., 2006) (Figure 4 and Table 2). For example, IRAK-M prevents the dissociation of IRAK and IRAK-4 from MyD88 and inhibits the formation of IRAK-TRAF6 complexes, contributing to decreased inflammatory responses to bacterial infection (Kobayashi et al., 2002). Similar strategies were used by some proteins to limit PRR-triggered antiviral responses, such as NLRC5 (Cui et al., 2010), NLRX1 (Moore et al., 2008), IFN-stimulated gene 56 (ISG56) (Li et al., 2009), and NLRC3 (Zhang et al., 2014a) (Figure 4 and Table 2). Moreover, several CARD-only proteins (COPs) and PYRIN domain-only proteins (POPs) participate in the negative regulation of inflammasome signaling by blocking the formation of the inflammasome complex via a direct sequestration mechanism (Le and Harton, 2013). The PYRIN domain (PYD)-only protein POP3 competes with ASC for interaction with AIM2-like receptors (ALRs), to inhibit ALRinflammasome activation by dsDNA viruses without affecting NLRP1, NLRP3, and NLRC4 inflammasomes (Khare et al., 2014). Interestingly, POP3 could also upregulate IFNb production, possibly as a consequence of inflammasome-dependent regulation of IFNb production or by directly targeting DNA-dependent signaling molecules such as STING, TBK1, or IRF3. Although PTM does not appear to be required for this model of negative regulation, it is reasonable to assume that their protein targets have already undergone certain PTMs for primary signal transduction. Whether PTMs are present or not, protein sequestration or the competitive inhibition of their interactions and assembly sets an essential layer of post-translational regulation of innate immunity and inflammation.

## The Role of PTMs in Inflammatory Immune Disorders

Variations in genes encoding innate receptors, adaptors, regulators and cytokines have been widely identified in patients with

Figure 4. Negative Regulation of PRR Signaling Pathways by Direct Physical Interaction

<!-- image -->

infectious and inflammatory diseases. Aberrant activation of PRRs leads to the overproduction of proinflammatory cytokines such as IL-1 a , IL-1 b , IL-6, IL-12, and TNF, which exert variable and overlapping effects on immune and non-immune cells and eventually directs the development of immunological pathogenesis. For example, TNF and IL-1 play essential roles in driving the pathogenesis of rheumatoid arthritis, not only by inducing innate immune cell activation and cytokine release but also by promoting the production of matrix degrading enzymes by synovial fibroblasts, leading to cartilage degradation (McInnes et al., 2016).

General defects in PTMs include the absence or mutation of its mediators, regulators or targets. While these are widely associated with the development of a variety of diseases (Karve and Cheema, 2011), we herein focus on how alterations in protein PTMs are directly linked with tissue pathology and dysfunction in infectious diseases, autoimmune diseases, neurological diseases, and cardiovascular diseases.

Because PRR-dependent IFN production is critical for the establishment of host antiviral state, viruses have developed a variety of mechanisms to evade host innate immunity. Specifically, they can alter the PTMs of host signaling molecules to impair innate response, so as to facilitate viral entry into host cells and allow viral replication and spreading. For example, influenza A virus nonstructural protein 1 (NS1) specifically targets the ubiquitin ligase TRIM25 to suppress RIG-I polyubiquitination, thus inhibiting the RIG-I-mediated host IFN response (Gack et al., 2009). Proteases derived from Enterovirus 71 act by inducing cleavage of MAVS for suppression of antiviral immunity in a proteolytic manner (Wang et al., 2013a). The hepatitis B virus (HBV) polymerase takes advantage of host PKCd and importina 5 to inhibit IFN-activated STAT1 phosphorylation, suggesting possible mechanisms for HBV resistance to IFN therapy (Chen

<!-- image -->

Numerous proteins negatively regulate Toll-like receptor (TLR) and RIG-I-like receptors (RLRs) signaling via direct physical interaction with the signaling molecules or by blocking the formation of activation complexes. A detailed list of the target proteins, regulators (physical interactors), and signaling functions are included in Table 2. Abbreviations for signaling molecules are the

same as Figure 2.

Abbreviations for regulators: SOCS1, suppressor of cytokine signaling 1; IRAK-M, interleukin 1 receptor associated kinase M; TAG; TRAM adaptor with GOLD domain; SARM, sterile alpha and TIR motif containing protein; SHP2, Src homology (SH2)-containing protein tyrosine phosphatase 2; NLRC5/3, NLR family CARD domain-containing protein5/3; NLRX1, NLR family member X1; ISG56, IFN-stimulated gene 56.

et al., 2013b). Ebola Zaire virus exploits the cellular SUMOylation machinery to overcome host innate defenses (Chang et al., 2009). In addition, Mycobacterium tuberculosis (Mtb) has also developed variable strategies to target key innate signaling pathways for their successful persistence. For example, Mtb-induced

TNF activates a series of signaling cascades, which induce phosphorylation of FLIP(S) and the subsequent interaction with the E3 ubiquitin ligase c-Cbl for proteasomal degradation of FLIP(S). This TNF- and c-Cbl-dependent FLIP(S) degradation pathway promotes the activation of caspase-8 and apoptosis, which are needed for Mtb-triggered macrophage cell death (Kundu et al., 2009). PtpA, a Mtb secreted protein tyrosine phosphatase required for Mtb survival and pathogenicity within host macrophages, can dephosphorylate human vacuolar protein sorting 33B (VPS33B) to inhibit phagosome-lysosome fusion, and also dephosphorylate GSK3 a to inhibit anti-apoptotic activity of GSK3 a (Hmama et al., 2015). These studies expand our knowledge of the complex interactions between invading microorganisms and the host innate immune system, shedding new light into the pathogenesis and possible interventions in acute and chronic infectious diseases.

Disruptions in innate immune signaling characterized by abnormal PTMs are also associated with the pathogenesis of autoimmune disease. Single nucleotide polymorphisms (SNPs) in the human A20 locus (also known as TNFAIP3 ) have been shown to be strongly associated with several human autoimmune diseases, consistent with the universal regulatory role of A20 in inhibiting multiple innate inflammatory signaling pathways (Ma and Malynn, 2012). Loss-of-function mutations in TNFAIP3 have been recently identified in early-onset systemic inflammation, which is associated with enhanced NFk Bsignaling, defective removal of K63-linked ubiquitin from TRAF6, IKK g , and RIP1, and spontaneous NLRP3 activation (Zhou et al., 2016). A mutation in the protein tyrosine phosphatase nonreceptor type 22 (PTPN22), PTPN22W (encoded by the disease susceptibility allele, 1858T), reduces the TLR-triggered K63-linked polyubiquitination of TRAF3 and type I IFN production, and as a result

<!-- image -->

Table 2. Negative Regulation of PRR Signaling Pathways by Direct Physical Interaction

| Target Protein/Pathway   | Regulators (Physical Interactors)   | Signaling Function                                                                                                    |
|--------------------------|-------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
| IRAK                     | Signaling regulator, SOCS1          | Directly inhibiting LPS signal pathways by binding to IRAK                                                            |
| IRAKs-TRAF6              | IRAK family inactive kinase, IRAK-M | Preventing dissociation of IRAK and IRAK-4 from MyD88 and formation of IRAK-TRAF6 complexes, inhibiting TLR signaling |
| TRIF                     | TIR adaptor, SARM                   | Directly interacting with TRIF, inhibiting TRIF-dependent TLR signaling                                               |
| TRAM-TRIF                | TRAM adaptor, TAG                   | Displacing the adaptor TRIF from TRAM, inhibiting MyD88-independent TLR4 signaling                                    |
| RIG-I and MDA5           | NLR family member, NLRC5            | Interacting with RIG-I and MDA5, inhibiting RLR-mediated type I IFN responses                                         |
| RIG-I-MAVS               | NLR family member, NLRX1            | Disrupting the interaction between RIG-I and MAVS                                                                     |
| TBK1                     | Phosphatase, SHP2                   | Directly binding to TBK1, inhibiting TRIF-dependent TLR signaling                                                     |
| STING/TBK1               | Interferon induced gene, ISG56      | Disrupting the interactions between STING and TBK1, inhibiting antiviral signaling                                    |
| STING/TBK1               | NLR family member, NLRC3            | Interacting with both STING and TBK1 and impeding the interaction between STING and TBK1                              |

fails to suppress inflammatory arthritis (Wang et al., 2013b). Further investigation is needed to address how PTPN22 selectively targets K63-linked polyubiquitination of TRAF3 or whether there are additional substrates for PTPN22 that might contribute to the control of systemic autoimmunity. In addition, two new gain-of-function mutations of NLRC4, Val341Ala, and Thr337Ser, are associated with the development of severe spontaneous autoimmune syndromes in humans, but it is unclear whether these mutations lead to inflammasome activation by altering NLRC4 oligomerization or phosphorylation (Canna et al., 2014; Romberg et al., 2014).

Dysregulated PTMs such as ubiquitination, phosphorylation, acetylation, and SUMOylation are also widely linked to various neurological and cardiac diseases (Karve and Cheema, 2011). Genetic mutations in PARK2 result in impaired ubiquitin ligase function and are linked to autosomal recessive Parkinson's disease (Shimura et al., 2000). Acetylation of tau at K174 is critical for tau stability and toxicity in early stages of Alzheimer's disease. Administration of salsalate after frontotemporal dementia onset inhibits p300 activity, reducing the total tau level and the acetylation of tau at K174, thus halting tau-induced neurodegeneration and cognitive deficits (Min et al., 2015). The lysosomal cysteine proteinase asparagine endopeptidase (AEP) induced proteolytical cleavage, phosphorylation, and aggregation of tau also contribute to neurodegeneration and cognitive deficits (Zhang et al., 2014b). In addition, alterations in SUMOylation have emerged as a new feature and potential therapeutic target of heart failure. SUMOylation of two lysine residues is essential for preserving the activity and stability of SERCA2a ATPase. While failing hearts show decreased levels of SUMO1 and SERCA2a, restitution of SERCA2a or SUMO could reverse heart failure (Kho et al., 2011).

These studies highlight the critical role of disordered PTMs in the initiation, progression, and outcome of various diseases. The increasing understanding of molecular mechanisms and the biological relevance of diverse PTMs has also promoted the development of novel therapeutic strategies that specifically and selectively modulate inflammatory pathways for many lifethreatening immunological disorders. For example, small mole-

cule inhibitors targeting MAPK, IKK, and IKK-related kinases (IKK ε and TBK1) are being actively investigated as potential drugs for autoimmune and inflammatory diseases (Cohen, 2009; Llona-Minguez et al., 2013). Future investigations are necessary to design and characterize new recombinant proteins, antagonists, antibodies, and genetic manipulating products that can effectively and safely modulate specific immune response for treatment of human inflammatory immune disorders.

## Conclusions and Perspectives

Despite the remarkable advances in our understanding of the functions and mechanisms of PTMs in regulating inflammation and immunity, several intriguing and important aspects of this field still remain elusive and represent important topics for further investigation.

While ubiquitination, phosphorylation, methylation, and acetylation are increasingly associated with immunological responses, the roles of others PTMs, such as SUMOylation, succinylation, carbonylation, glycosylation, glycation, hydroxylation, citrullination, nitration, palmitoylation, and sulfation in immunity and inflammation are still poorly understood. Meanwhile, whether additional forms of PTMs might also exist and carry potential immunological functions are unknown. In this regard, advanced technological platforms for comprehensive high-resolution mass spectrometric measurement and analysis are required for specific and sensitive detection of new protein alterations.

Different forms of PTMs of innate signaling molecules are observed in distinct cellular organelles and in response to different pathogenic stimuli, e.g., RNA viruses induce K63-linked polyubiquitination of MAVS and its aggregation in mitochondria while DNA viruses induce K27-linked polyubiquitination of STING and its translocation from the ER to perinuclear vesicles. The detailed molecular mechanisms behind this, especially their respective binding partners and interaction dynamics, and the crosstalk between different PTMs need to be further clarified. In this regard, development and application of new ''omics'' for an integrated and dynamic understanding of the innate immune system are required, e.g., interactomics and fluxomics are two

powerful tools for the detection of protein interactions and dynamic changes at an organismal, tissue, or cellular level. Meanwhile, in vivo imaging techniques makes it possible to track the molecular interaction during inflammatory processes in vivo and thereby is becoming an indispensable tool for immunological study.

In sum, further investigations of the immunological relevance of the known and perhaps additional unknown PTMs, their regulators and targets, their expression profiles, molecular interactions, and biological functions during infection and dangerous stress, and their contributions to the pathogenesis of immunological disorders will not only expand our understanding of the cellular and molecular mechanisms of innate immune responses and regulation but will also outline novel diagnostic markers and therapeutic targets of related immune diseases.

## ACKNOWLEDGMENTS

This work was supported by grants from the National Key Basic Research Program of China (2013CB530502) and National Natural Science Foundation of China (31400777, 31390431).

## REFERENCES

Adrain, C., Zettl, M., Christova, Y., Taylor, N., and Freeman, M. (2012). Tumor necrosis factor signaling requires iRhom2 to promote trafficking and activation of TACE. Science 335 , 225-228.

Aksoy, E., Taboubi, S., Torres, D., Delbauve, S., Hachani, A., Whitehead, M.A., Pearce, W.P., Berenjeno, I.M., Nock, G., Filloux, A., et al. (2012). The p110 d isoform of the kinase PI(3)K controls the subcellular compartmentalization of TLR4 signaling and protects from endotoxic shock. Nat. Immunol. 13 , 10451054.

A ' lvarez-Errico, D., Vento-Tormo, R., Sieweke, M., and Ballestar, E. (2015). Epigenetic control of myeloid cell differentiation, identity and function. Nat. Rev. Immunol. 15 , 7-17.

An, H., Zhao, W., Hou, J., Zhang, Y., Xie, Y., Zheng, Y., Xu, H., Qian, C., Zhou, J., Yu, Y., et al. (2006). SHP-2 phosphatase negatively regulates the TRIF adaptor protein-dependent type I interferon and proinflammatory cytokine production. Immunity 25 , 919-928.

An, H., Hou, J., Zhou, J., Zhao, W., Xu, H., Zheng, Y., Yu, Y., Liu, S., and Cao, X. (2008). Phosphatase SHP-1 promotes TLR- and RIG-I-activated production of type I interferon by inhibiting the kinase IRAK1. Nat. Immunol. 9 , 542-550.

Anand, P.K., Malireddi, R.K., Lukens, J.R., Vogel, P., Bertin, J., Lamkanfi, M., and Kanneganti, T.D. (2012). NLRP6 negatively regulates innate immunity and host defence against bacterial pathogens. Nature 488 , 389-393.

Arimoto, K., Takahashi, H., Hishiki, T., Konishi, H., Fujita, T., and Shimotohno, K. (2007). Negative regulation of the RIG-I signaling by the ubiquitin ligase RNF125. Proc. Natl. Acad. Sci. USA 104 , 7500-7505.

Arthur, J.S., and Ley, S.C. (2013). Mitogen-activated protein kinases in innate immunity. Nat. Rev. Immunol. 13 , 679-692.

Baroja-Mazo, A., Martı 'n-Sa ' nchez, F., Gomez, A.I., Martı 'nez, C.M., Amores-Iniesta, J., Compan, V., Barbera ' -Cremades, M., Yagu e, J., Ruiz-Ortiz, E., Anto ' n, J., et al. (2014). The NLRP3 inflammasome is released as a particulate danger signal that amplifies the inflammatory response. Nat. Immunol. 15 , 738-748.

Barrat, F.J., Elkon, K.B., and Fitzgerald, K.A. (2016). Importance of nucleic acid recognition in inflammation and autoimmunity. Annu. Rev. Med. 67 , 323-336.

Boone, D.L., Turer, E.E., Lee, E.G., Ahmad, R.C., Wheeler, M.T., Tsui, C., Hurley, P., Chien, M., Chai, S., Hitotsumatsu, O., et al. (2004). The ubiquitin-modifying enzyme A20 is required for termination of Toll-like receptor responses. Nat. Immunol. 5 , 1052-1060.

Brinkmann, M.M., Spooner, E., Hoebe, K., Beutler, B., Ploegh, H.L., and Kim, Y.M. (2007). The interaction between the ER membrane protein UNC93B and TLR3, 7, and 9 is crucial for TLR signaling. J. Cell Biol. 177 , 265-275.

<!-- image -->

Brubaker, S.W., Bonham, K.S., Zanoni, I., and Kagan, J.C. (2015). Innate immune pattern recognition: a cell biological perspective. Annu. Rev. Immunol. 33 , 257-290.

Bu rckstummer, T., Baumann, C., Blu ml, S., Dixit, E., Du rnberger, G., Jahn, H., Planyavsky, M., Bilban, M., Colinge, J., Bennett, K.L., and Superti-Furga, G. (2009). An orthogonal proteomic-genomic screen identifies AIM2 as a cytoplasmic DNA sensor for the inflammasome. Nat. Immunol. 10 , 266-272.

Cai, X., Chen, J., Xu, H., Liu, S., Jiang, Q.X., Halfmann, R., and Chen, Z.J. (2014). Prion-like polymerization underlies signal transduction in antiviral immune defense and inflammasome activation. Cell 156 , 1207-1222.

Canna, S.W., de Jesus, A.A., Gouni, S., Brooks, S.R., Marrero, B., Liu, Y., DiMattia, M.A., Zaal, K.J., Sanchez, G.A., Kim, H., et al. (2014). An activating NLRC4 inflammasome mutation causes autoinflammation with recurrent macrophage activation syndrome. Nat. Genet. 46 , 1140-1146.

Cao, X. (2016). Self-regulation and cross-regulation of pattern-recognition receptor signalling in health and disease. Nat. Rev. Immunol. 16 , 35-50.

Cao, W., Bao, C., Padalko, E., and Lowenstein, C.J. (2008). Acetylation of mitogen-activated protein kinase phosphatase-1 inhibits Toll-like receptor signaling. J. Exp. Med. 205 , 1491-1503.

Carpenter, S., Ricci, E.P., Mercier, B.C., Moore, M.J., and Fitzgerald, K.A. (2014). Post-transcriptional regulation of gene expression in innate immunity. Nat. Rev. Immunol. 14 , 361-376.

Carty, M., Goodbody, R., Schro der, M., Stack, J., Moynagh, P.N., and Bowie, A.G. (2006). The human adaptor SARM negatively regulates adaptor protein TRIF-dependent Toll-like receptor signaling. Nat. Immunol. 7 , 1074-1081.

Castanier, C., Zemirli, N., Portier, A., Garcin, D., Bide ' re, N., Vazquez, A., and Arnoult, D. (2012). MAVS ubiquitination by the E3 ligase TRIM25 and degradation by the proteasome is involved in type I interferon production after activation of the antiviral RIG-I-like receptors. BMC Biol. 10 , 44.

Chakrabarti, A., Banerjee, S., Franchi, L., Loo, Y.M., Gale, M., Jr., Nu ' n ˜ ez, G., and Silverman, R.H. (2015). RNase L activates the NLRP3 inflammasome during viral infections. Cell Host Microbe 17 , 466-477.

Chang, T.H., Kubota, T., Matsuoka, M., Jones, S., Bradfute, S.B., Bray, M., and Ozato, K. (2009). Ebola Zaire virus blocks type I interferon production by exploiting the host SUMO modification machinery. PLoS Pathog. 5 , e1000493.

Chen, B.B., Coon, T.A., Glasser, J.R., McVerry, B.J., Zhao, J., Zhao, Y., Zou, C., Ellis, B., Sciurba, F.C., Zhang, Y., and Mallampalli, R.K. (2013a). A combinatorial F box protein directed pathway controls TRAF adaptor stability to regulate inflammation. Nat. Immunol. 14 , 470-479.

Chen, J., Wu, M., Zhang, X., Zhang, W., Zhang, Z., Chen, L., He, J., Zheng, Y., Chen, C., Wang, F., et al. (2013b). Hepatitis B virus polymerase impairs interferona -induced STA T activation through inhibition of importina 5 and protein kinase Cd . Hepatology 57 , 470-482.

Chen, W., Han, C., Xie, B., Hu, X., Yu, Q., Shi, L., Wang, Q., Li, D., Wang, J., Zheng, P., et al. (2013c). Induction of Siglec-G by RNA viruses inhibits the innate immune response by promoting RIG-I degradation. Cell 152 , 467-478.

Choi, S.J., Lee, H.C., Kim, J.H., Park, S.Y., Kim, T.H., Lee, W.K., Jang, D.J., Yoon, J.E., Choi, Y.I., Kim, S., et al. (2016). HDAC6 regulates cellular viral RNA sensing by deacetylation of RIG-I. EMBO J. 35 , 429-442.

Choudhary, C., Kumar, C., Gnad, F., Nielsen, M.L., Rehman, M., Walther, T.C., Olsen, J.V., and Mann, M. (2009). Lysine acetylation targets protein complexes and co-regulates major cellular functions. Science 325 , 834-840.

Chuang, T.H., and Ulevitch, R.J. (2004). Triad3A, an E3 ubiquitin-protein ligase regulating Toll-like receptors. Nat. Immunol. 5 , 495-502.

Cohen, P. (2009). Targeting protein kinases for the development of anti-inflammatory drugs. Curr. Opin. Cell Biol. 21 , 317-324.

Cui, J., Zhu, L., Xia, X., Wang, H.Y., Legras, X., Hong, J., Ji, J., Shen, P., Zheng, S., Chen, Z.J., and Wang, R.F. (2010). NLRC5 negatively regulates the NF-kappaB and type I interferon signaling pathways. Cell 141 , 483-496.

Cui, J., Li, Y., Zhu, L., Liu, D., Songyang, Z., Wang, H.Y., and Wang, R.F. (2012). NLRP4 negatively regulates type I interferon signaling by targeting the kinase TBK1 for degradation via the ubiquitin ligase DTX4. Nat. Immunol. 13 , 387-395.

<!-- image -->

Decque, A., Joffre, O., Magalhaes, J.G., Cossec, J.C., Blecher-Gonen, R., Lapaquette, P., Silvin, A., Manel, N., Joubert, P.E., Seeler, J.S., et al. (2016). Sumoylation coordinates the repression of inflammatory and anti-viral geneexpression programs during innate sensing. Nat. Immunol. 17 , 140-149.

Deribe, Y.L., Pawson, T., and Dikic, I. (2010). Post-translational modifications in signal integration. Nat. Struct. Mol. Biol. 17 , 666-672.

Dı 'az-Rodrı 'guez, E., Montero, J.C., Esparı 's-Ogando, A., Yuste, L., and Pandiella, A. (2002). Extracellular signal-regulated kinase phosphorylates tumor necrosis factor a -converting enzyme at threonine 735: a potential role in regulated shedding. Mol. Biol. Cell 13 , 2031-2044.

Franklin, B.S., Bossaller, L., De Nardo, D., Ratter, J.M., Stutz, A., Engels, G., Brenker, C., Nordhoff, M., Mirandola, S.R., Al-Amoudi, A., et al. (2014). The adaptor ASC has extracellular and 'prionoid' activities that propagate inflammation. Nat. Immunol. 15 , 727-737.

Friedman, C.S., O'Donnell, M.A., Legarda-Addison, D., Ng, A., Ca ' rdenas, W.B., Yount, J.S., Moran, T.M., Basler, C.F., Komuro, A., Horvath, C.M., et al. (2008). The tumour suppressor CYLD is a negative regulator of RIG-Imediated antiviral response. EMBO Rep. 9 , 930-936.

Fu, J., Xiong, Y., Xu, Y., Cheng, G., and Tang, H. (2011). MDA5 is SUMOylated by PIAS2 b in the upregulation of type I interferon signaling. Mol. Immunol. 48 , 415-422.

Gack, M.U., Shin, Y.C., Joo, C.H., Urano, T., Liang, C., Sun, L., Takeuchi, O., Akira, S., Chen, Z., Inoue, S., and Jung, J.U. (2007). TRIM25 RING-finger E3 ubiquitin ligase is essential for RIG-I-mediated antiviral activity. Nature 446 , 916-920.

Gack, M.U., Albrecht, R.A., Urano, T., Inn, K.S., Huang, I.C., Carnero, E., Farzan, M., Inoue, S., Jung, J.U., and Garcı 'a-Sastre, A. (2009). Influenza A virus NS1 targets the ubiquitin ligase TRIM25 to evade recognition by the host viral RNA sensor RIG-I. Cell Host Microbe 5 , 439-449.

Gringhuis, S.I., Kaptein, T.M., Wevers, B.A., Theelen, B., van der Vlist, M., Boekhout, T., and Geijtenbeek, T.B. (2012). Dectin-1 is an extracellular pathogen sensor for the induction and processing of IL-1 b via a noncanonical caspase-8 inflammasome. Nat. Immunol. 13 , 246-254.

Gunawan, M., Venkatesan, N., Loh, J.T., Wong, J.F., Berger, H., Neo, W.H., Li, L.Y., La Win, M.K., Yau, Y.H., Guo, T., et al. (2015). The methyltransferase Ezh2 controls cell adhesion and migration through direct methylation of the extranuclear regulatory protein talin. Nat. Immunol. 16 , 505-516.

Ha, S.D., Martins, A., Khazaie, K., Han, J., Chan, B.M., and Kim, S.O. (2008). Cathepsin B is involved in the trafficking of TNF-alpha-containing vesicles to the plasma membrane in macrophages. J. Immunol. 181 , 690-697.

Hammer, G.E., Turer, E.E., Taylor, K.E., Fang, C.J., Advincula, R., Oshima, S., Barrera, J., Huang, E.J., Hou, B., Malynn, B.A., et al. (2011). Expression of A20 by dendritic cells preserves immune homeostasis and prevents colitis and spondyloarthritis. Nat. Immunol. 12 , 1184-1193.

Hara, H., Tsuchiya, K., Kawamura, I., Fang, R., Hernandez-Cuellar, E., Shen, Y., Mizuguchi, J., Schweighoffer, E., Tybulewicz, V., and Mitsuyama, M. (2013). Phosphorylation of the adaptor ASC acts as a molecular switch that controls the formation of speck-like aggregates and inflammasome activity. Nat. Immunol. 14 , 1247-1255.

He, W.T., Wan, H., Hu, L., Chen, P., Wang, X., Huang, Z., Yang, Z.H., Zhong, C.Q., and Han, J. (2015). Gasdermin D is an executor of pyroptosis and required for interleukin-1 b secretion. Cell Res. 25 , 1285-1298.

Heaton, S.M., Borg, N.A., and Dixit, V.M. (2016). Ubiquitin in the activation and attenuation of innate antiviral immunity. J. Exp. Med. 213 , 1-13.

Hmama, Z., Pen ˜ a-Dı 'az, S., Joseph, S., and Av-Gay, Y. (2015). Immunoevasion and immunosuppression of the macrophage by Mycobacterium tuberculosis. Immunol. Rev. 264 , 220-232.

Horner, S.M., Liu, H.M., Park, H.S., Briley, J., and Gale, M., Jr. (2011). Mitochondrial-associated endoplasmic reticulum membranes (MAM) form innate immune synapses and are targeted by hepatitis C virus. Proc. Natl. Acad. Sci. USA 108 , 14590-14595.

Hornung, V., Ablasser, A., Charrel-Dennis, M., Bauernfeind, F., Horvath, G., Caffrey, D.R., Latz, E., and Fitzgerald, K.A. (2009). AIM2 recognizes cytosolic dsDNA and forms a caspase-1-activating inflammasome with ASC. Nature 458 , 514-518.

Hou, F., Sun, L., Zheng, H., Skaug, B., Jiang, Q.X., and Chen, Z.J. (2011). MAVS forms functional prion-like aggregates to activate and propagate antiviral innate immune response. Cell 146 , 448-461.

Husebye, H., Halaas, Ø., Stenmark, H., Tunheim, G., Sandanger, Ø., Bogen, B., Brech, A., Latz, E., and Espevik, T. (2006). Endocytic pathways regulate Toll-like receptor 4 signaling and link innate and adaptive immunity. EMBO J. 25 , 683-692.

Ishikawa, H., Ma, Z., and Barber, G.N. (2009). STING regulates intracellular DNA-mediated, type I interferon-dependent innate immunity. Nature 461 , 788-792.

Iwasaki, A., and Medzhitov, R. (2015). Control of adaptive immunity by the innate immune system. Nat. Immunol. 16 , 343-353.

Jiang, X., and Chen, Z.J. (2012). The role of ubiquitylation in immune defence and pathogen evasion. Nat. Rev. Immunol. 12 , 35-48.

Jiao, S., Zhang, Z., Li, C., Huang, M., Shi, Z., Wang, Y., Song, X., Liu, H., Li, C., Chen, M., et al. (2015). The kinase MST4 limits inflammatory responses through direct phosphorylation of the adaptor TRAF6. Nat. Immunol. 16 , 246-257.

Karin, M., and Ben-Neriah, Y. (2000). Phosphorylation meets ubiquitination: the control of NF-[kappa]B activity. Annu. Rev. Immunol. 18 , 621-663.

Karve, T.M., and Cheema, A.K. (2011). Small changes huge impact: the role of protein posttranslational modifications in cellular homeostasis and disease. J. Amino Acids 2011 , 207691.

Kawagoe, T., Takeuchi, O., Takabatake, Y., Kato, H., Isaka, Y., Tsujimura, T., and Akira, S. (2009). TANK is a negative regulator of Toll-like receptor signaling and is critical for the prevention of autoimmune nephritis. Nat. Immunol. 10 , 965-972.

Kayagaki, N., Phung, Q., Chan, S., Chaudhari, R., Quan, C., O'Rourke, K.M., Eby, M., Pietras, E., Cheng, G., Bazan, J.F., et al. (2007). DUBA: a deubiquitinase that regulates type I interferon production. Science 318 , 1628-1632.

Kayagaki, N., Warming, S., Lamkanfi, M., Vande Walle, L., Louie, S., Dong, J., Newton, K., Qu, Y., Liu, J., Heldens, S., et al. (2011). Non-canonical inflammasome activation targets caspase-11. Nature 479 , 117-121.

Kayagaki, N., Stowe, I.B., Lee, B.L., O'Rourke, K., Anderson, K., Warming, S., Cuellar, T., Haley, B., Roose-Girma, M., Phung, Q.T., et al. (2015). Caspase-11 cleaves gasdermin D for non-canonical inflammasome signalling. Nature 526 , 666-671.

Khare, S., Ratsimandresy, R.A., de Almeida, L., Cuda, C.M., Rellick, S.L., Misharin, A.V., Wallin, M.C., Gangopadhyay, A., Forte, E., Gottwein, E., et al. (2014). The PYRIN domain-only protein POP3 inhibits ALR inflammasomes and regulates responses to infection with DNA viruses. Nat. Immunol. 15 , 343-353.

Kho, C., Lee, A., Jeong, D., Oh, J.G., Chaanine, A.H., Kizana, E., Park, W.J., and Hajjar, R.J. (2011). SUMO1-dependent modulation of SERCA2a in heart failure. Nature 477 , 601-605.

Kim, Y.M., Brinkmann, M.M., Paquet, M.E., and Ploegh, H.L. (2008). UNC93B1 delivers nucleotide-sensing toll-like receptors to endolysosomes. Nature 452 , 234-238.

Kobayashi, K., Hernandez, L.D., Gala ' n, J.E., Janeway, C.A., Jr., Medzhitov, R., and Flavell, R.A. (2002). IRAK-M is a negative regulator of Toll-like receptor signaling. Cell 110 , 191-202.

Kundu, M., Pathak, S.K., Kumawat, K., Basu, S., Chatterjee, G., Pathak, S., Noguchi, T., Takeda, K., Ichijo, H., Thien, C.B., et al. (2009). A TNF- and c-Cbl-dependent FLIP(S)-degradation pathway and its function in Mycobacterium tuberculosis-induced macrophage apoptosis. Nat. Immunol. 10 , 918-926.

Le, H.T., and Harton, J.A. (2013). Pyrin- and CARD-only Proteins as Regulators of NLR Functions. Front. Immunol. 4 , 275.

Lee, Y.S., Park, J.S., Kim, J.H., Jung, S.M., Lee, J.Y., Kim, S.J., and Park, S.H. (2011). Smad6-specific recruitment of Smurf E3 ligases mediates TGFb 1induced degradation of MyD88 in TLR4 signalling. Nat. Commun. 2 , 460.

Lee, K.H., Biswas, A., Liu, Y.J., and Kobayashi, K.S. (2012). Proteasomal degradation of Nod2 protein mediates tolerance to bacterial cell wall components. J. Biol. Chem. 287 , 39800-39811.

Lei, C.Q., Zhong, B., Zhang, Y., Zhang, J., Wang, S., and Shu, H.B. (2010). Glycogen synthase kinase 3 b regulates IRF3 transcription factor-mediated antiviral response via activation of the kinase TBK1. Immunity 33 , 878-889.

Levy, D., Kuo, A.J., Chang, Y., Schaefer, U., Kitson, C., Cheung, P., Espejo, A., Zee, B.M., Liu, C.L., Tangsombatvisit, S., et al. (2011). Lysine methylation of the NFk B subunit RelA by SETD6 couples activity of the histone methyltransferase GLP at chromatin to tonic repression of NFk Bsignaling. Nat. Immunol. 12 , 29-36.

Li, Y., Li, C., Xue, P., Zhong, B., Mao, A.P., Ran, Y., Chen, H., Wang, Y.Y., Yang, F., and Shu, H.B. (2009). ISG56 is a negative-feedback regulator of virus-triggered signaling and cellular antiviral response. Proc. Natl. Acad. Sci. USA 106 , 7945-7950.

Li, X.D., Wu, J., Gao, D., Wang, H., Sun, L., and Chen, Z.J. (2013). Pivotal roles of cGAS-cGAMP signaling in antiviral defense and immune adjuvant effects. Science 341 , 1390-1394.

Li, X., Zhang, Q., Ding, Y., Liu, Y., Zhao, D., Zhao, K., Shen, Q., Liu, X., Zhu, X., Li, N., et al. (2016). Methyltransferase Dnmt3a upregulates HDAC9 to deacetylate the kinase TBK1 for activation of antiviral innate immunity. Nat. Immunol. 17 , 806-815, http://dx.doi.org/10.1038/ni.3464.

Lieu, Z.Z., Lock, J.G., Hammond, L.A., La Gruta, N.L., Stow, J.L., and Gleeson, P.A. (2008). A trans-Golgi network golgin is required for the regulated secretion of TNF in activated macrophages in vivo. Proc. Natl. Acad. Sci. USA 105 , 3351-3356.

Lin, A.E., Ebert, G., Ow, Y., Preston, S.P., Toe, J.G., Cooney, J.P., Scott, H.W., Sasaki, M., Saibil, S.D., Dissanayake, D., et al. (2013). ARIH2 is essential for embryogenesis, and its hematopoietic deficiency causes lethal activation of the immune system. Nat. Immunol. 14 , 27-33.

Liu, J., and Cao, X. (2015). Regulatory dendritic cells in autoimmunity: A comprehensive review. J. Autoimmun. 63 , 1-12.

Liu, Y., Bridges, R., Wortham, A., and Kulesz-Martin, M. (2012). NFk Brepression by PIAS3 mediated RelA SUMOylation. PLoS ONE 7 , e37636.

Liu, X., Chen, W., Wang, Q., Li, L., and Wang, C. (2013). Negative regulation of TLR inflammatory signaling by the SUMO-deconjugating enzyme SENP6. PLoS Pathog. 9 , e1003480.

Liu, J., Han, C., Xie, B., Wu, Y., Liu, S., Chen, K., Xia, M., Zhang, Y., Song, L., Li, Z., et al. (2014). Rhbdd3 controls autoimmunity by suppressing the production of IL-6 by dendritic cells via K27-linked ubiquitination of the regulator NEMO. Nat. Immunol. 15 , 612-622.

Liu, S., Cai, X., Wu, J., Cong, Q., Chen, X., Li, T., Du, F., Ren, J., Wu, Y.T., Grishin, N.V., and Chen, Z.J. (2015). Phosphorylation of innate immune adaptor proteins MAVS, STING, and TRIF induces IRF3 activation. Science 347 , aaa2630.

Llona-Minguez, S., Baiget, J., and Mackay, S.P. (2013). Small-molecule inhibitors of I k Bkinase (IKK) and IKK-related kinases. Pharm. Pat. Anal. 2 , 481-498.

Long, L., Deng, Y., Yao, F., Guan, D., Feng, Y., Jiang, H., Li, X., Hu, P., Lu, X., Wang, H., et al. (2014). Recruitment of phosphatase PP2A by RACK1 adaptor protein deactivates transcription factor IRF3 and limits type I interferon signaling. Immunity 40 , 515-529.

Lu, T., Jackson, M.W., Wang, B., Yang, M., Chance, M.R., Miyagi, M., Gudkov, A.V., and Stark, G.R. (2010). Regulation of NF-kappaB by NSD1/FBXL11dependent reversible lysine methylation of p65. Proc. Natl. Acad. Sci. USA 107 , 46-51.

Lu, A., Magupalli, V.G., Ruan, J., Yin, Q., Atianand, M.K., Vos, M.R., Schro der, G.F., Fitzgerald, K.A., Wu, H., and Egelman, E.H. (2014). Unified polymerization mechanism for the assembly of ASC-dependent inflammasomes. Cell 156 , 1193-1206.

Ma, A., and Malynn, B.A. (2012). A20: linking a complex regulator of ubiquitylation to immunity and human disease. Nat. Rev. Immunol. 12 , 774-785.

Man, S.M., Karki, R., Malireddi, R.K., Neale, G., Vogel, P., Yamamoto, M., Lamkanfi, M., and Kanneganti, T.D. (2015). The transcription factor IRF1 and guanylate-binding proteins target activation of the AIM2 inflammasome by Francisella infection. Nat. Immunol. 16 , 467-475.

Mansell, A., Smith, R., Doyle, S.L., Gray, P., Fenner, J.E., Crack, P.J., Nicholson, S.E., Hilton, D.J., O'Neill, L.A., and Hertzog, P.J. (2006). Suppressor of

<!-- image -->

cytokine signaling 1 negatively regulates Toll-like receptor signaling by mediating Mal degradation. Nat. Immunol. 7 , 148-155.

McIlwain, D.R., Lang, P.A., Maretzky, T., Hamada, K., Ohishi, K., Maney, S.K., Berger, T., Murthy, A., Duncan, G., Xu, H.C., et al. (2012). iRhom2 regulation of TACE controls TNF-mediated protection against Listeria and responses to LPS. Science 335 , 229-232.

McInnes, I.B., Buckley, C.D., and Isaacs, J.D. (2016). Cytokines in rheumatoid arthritis -shaping the immunological landscape. Nat. Rev. Rheumatol. 12 , 63-68.

Medzhitov, R., and Horng, T. (2009). Transcriptional control of the inflammatory response. Nat. Rev. Immunol. 9 , 692-703.

Meunier, E., Wallet, P., Dreier, R.F., Costanzo, S., Anton, L., Ru hl, S., Dussurgey, S., Dick, M.S., Kistner, A., Rigard, M., et al. (2015). Guanylate-binding proteins promote activation of the AIM2 inflammasome during infection with Francisella novicida. Nat. Immunol. 16 , 476-484.

Mi, Z., Fu, J., Xiong, Y., and Tang, H. (2010). SUMOylation of RIG-I positively regulates the type I interferon signaling. Protein Cell 1 , 275-283.

Mills, E., and O'Neill, L.A. (2014). Succinate: a metabolic signal in inflammation. Trends Cell Biol. 24 , 313-320.

Min, S.W., Chen, X., Tracy, T.E., Li, Y., Zhou, Y., Wang, C., Shirakawa, K., Minami, S.S., Defensor, E., Mok, S.A., et al. (2015). Critical role of acetylation in tau-mediated neurodegeneration and cognitive deficits. Nat. Med. 21 , 11541162.

Misawa, T., Takahama, M., Kozaki, T., Lee, H., Zou, J., Saitoh, T., and Akira, S. (2013). Microtubule-driven spatial arrangement of mitochondria promotes activation of the NLRP3 inflammasome. Nat. Immunol. 14 , 454-460.

Mitoma, H., Hanabuchi, S., Kim, T., Bao, M., Zhang, Z., Sugimoto, N., and Liu, Y.J. (2013). The DHX33 RNA helicase senses cytosolic RNA and activates the NLRP3 inflammasome. Immunity 39 , 123-135.

Moore, C.B., Bergstralh, D.T., Duncan, J.A., Lei, Y., Morrison, T.E., Zimmermann, A.G., Accavitti-Loper, M.A., Madden, V.J., Sun, L., Ye, Z., et al. (2008). NLRX1 is a regulator of mitochondrial antiviral immunity. Nature 451 , 573-577.

Mowen, K.A., and David, M. (2014). Unconventional post-translational modifications in immunological signaling. Nat. Immunol. 15 , 512-520.

Nakagawa, R., Naka, T., Tsutsui, H., Fujimoto, M., Kimura, A., Abe, T., Seki, E., Sato, S., Takeuchi, O., Takeda, K., et al. (2002). SOCS-1 participates in negative regulation of LPS responses. Immunity 17 , 677-687.

Onoguchi, K., Onomoto, K., Takamatsu, S., Jogi, M., Takemura, A., Morimoto, S., Julkunen, I., Namiki, H., Yoneyama, M., and Fujita, T. (2010). Virus-infection or 5'ppp-RNA activates antiviral signal through redistribution of IPS-1 mediated by MFN1. PLoS Pathog. 6 , e1001012.

Oshiumi, H., Matsumoto, M., Hatakeyama, S., and Seya, T. (2009). Riplet/ RNF135, a RING finger protein, ubiquitinates RIG-I to promote interferonbeta induction during the early phase of viral infection. J. Biol. Chem. 284 , 807-817.

Palsson-McDermott, E.M., Doyle, S.L., McGettrick, A.F., Hardy, M., Husebye, H., Banahan, K., Gong, M., Golenbock, D., Espevik, T., and O'Neill, L.A. (2009). TAG, a splice variant of the adaptor TRAM, negatively regulates the adaptor MyD88-independent TLR4 pathway. Nat. Immunol. 10 , 579-586.

Paludan, S.R., and Bowie, A.G. (2013). Immune sensing of DNA. Immunity 38 , 870-880.

Piccirillo, C.A., Bjur, E., Topisirovic, I., Sonenberg, N., and Larsson, O. (2014). Translational control of immune responses: from transcripts to translatomes. Nat. Immunol. 15 , 503-511.

Poeck, H., Bscheider, M., Gross, O., Finger, K., Roth, S., Rebsamen, M., Hannesschla ger, N., Schlee, M., Rothenfusser, S., Barchet, W., et al. (2010). Recognition of RNA virus by RIG-I results in activation of CARD9 and inflammasome signaling for interleukin 1 beta production. Nat. Immunol. 11 , 63-69.

Py, B.F., Kim, M.S., Vakifahmetoglu-Norberg, H., and Yuan, J. (2013). Deubiquitination of NLRP3 by BRCC3 critically regulates inflammasome activity. Mol. Cell 49 , 331-338.

<!-- image -->

Qin, Y., Zhou, M.T., Hu, M.M., Hu, Y.H., Zhang, J., Guo, L., Zhong, B., and Shu, H.B. (2014). RNF26 temporally regulates virus-triggered type I interferon induction by two distinct mechanisms. PLoS Pathog. 10 , e1004358.

Qu, Y., Misaghi, S., Izrael-Tomasevic, A., Newton, K., Gilmour, L.L., Lamkanfi, M., Louie, S., Kayagaki, N., Liu, J., Ko mu ves, L., et al. (2012). Phosphorylation of NLRC4 is critical for inflammasome activation. Nature 490 , 539-542.

Ran, Y., Liu, T.T., Zhou, Q., Li, S., Mao, A.P., Li, Y., Liu, L.J., Cheng, J.K., and Shu, H.B. (2011). SENP2 negatively regulates cellular antiviral response by deSUMOylating IRF3 and conditioning it for ubiquitination and degradation. J. Mol. Cell Biol. 3 , 283-292.

Rathinam, V.A., Vanaja, S.K., Waggoner, L., Sokolovska, A., Becker, C., Stuart, L.M., Leong, J.M., and Fitzgerald, K.A. (2012). TRIF licenses caspase11-dependent NLRP3 inflammasome activation by gram-negative bacteria. Cell 150 , 606-619.

Rodgers, M.A., Bowman, J.W., Fujita, H., Orazio, N., Shi, M., Liang, Q., Amatya, R., Kelly, T.J., Iwai, K., Ting, J., and Jung, J.U. (2014). The linear ubiquitin assembly complex (LUBAC) is essential for NLRP3 inflammasome activation. J. Exp. Med. 211 , 1333-1347.

Romberg, N., Al Moussawi, K., Nelson-Williams, C., Stiegler, A.L., Loring, E., Choi, M., Overton, J., Meffre, E., Khokha, M.K., Huttner, A.J., et al. (2014). Mutation of NLRC4 causes a syndrome of enterocolitis and autoinflammation. Nat. Genet. 46 , 1135-1139.

Rubic, T., Lametschwandtner, G., Jost, S., Hinteregger, S., Kund, J., Carballido-Perrig, N., Schwa rzler, C., Junt, T., Voshol, H., Meingassner, J.G., et al. (2008). Triggering the succinate receptor GPR91 on dendritic cells enhances immunity. Nat. Immunol. 9 , 1261-1269.

Saitoh, T., Tun-Kyi, A., Ryo, A., Yamamoto, M., Finn, G., Fujita, T., Akira, S., Yamamoto, N., Lu, K.P., and Yamaoka, S. (2006). Negative regulation of interferon-regulatory factor 3-dependent innate antiviral response by the prolyl isomerase Pin1. Nat. Immunol. 7 , 598-605.

Saitoh, T., Fujita, N., Hayashi, T., Takahara, K., Satoh, T., Lee, H., Matsunaga, K., Kageyama, S., Omori, H., Noda, T., et al. (2009). Atg9a controls dsDNAdriven dynamic translocation of STING and the innate immune response. Proc. Natl. Acad. Sci. USA 106 , 20842-20846.

Schliehe, C., Flynn, E.K., Vilagos, B., Richson, U., Swaminathan, S., Bosnjak, B., Bauer, L., Kandasamy, R.K., Griesshammer, I.M., Kosack, L., et al. (2015). The methyltransferase Setdb2 mediates virus-induced susceptibility to bacterial superinfection. Nat. Immunol. 16 , 67-74.

Schneider, M., Zimmermann, A.G., Roberts, R.A., Zhang, L., Swanson, K.V., Wen, H., Davis, B.K., Allen, I.C., Holl, E.K., Ye, Z., et al. (2012). The innate immune sensor NLRC3 attenuates Toll-like receptor signaling via modification of the signaling adaptor TRAF6 and transcription factor NFk B. Nat. Immunol. 13 , 823-831.

Seth, R.B., Sun, L., Ea, C.K., and Chen, Z.J. (2005). Identification and characterization of MAVS, a mitochondrial antiviral signaling protein that activates NF-kappaB and IRF 3. Cell 122 , 669-682.

Shi, J., Zhao, Y., Wang, Y., Gao, W., Ding, J., Li, P., Hu, L., and Shao, F. (2014). Inflammatory caspases are innate immune receptors for intracellular LPS. Nature 514 , 187-192.

Shi, J., Zhao, Y., Wang, K., Shi, X., Wang, Y., Huang, H., Zhuang, Y., Cai, T., Wang, F., and Shao, F. (2015). Cleavage of GSDMD by inflammatory caspases determines pyroptotic cell death. Nature 526 , 660-665.

Shimura, H., Hattori, N., Kubo, Si., Mizuno, Y., Asakawa, S., Minoshima, S., Shimizu, N., Iwai, K., Chiba, T., Tanaka, K., and Suzuki, T. (2000). Familial Parkinson disease gene product, parkin, is a ubiquitin-protein ligase. Nat. Genet. 25 , 302-305.

Subramanian, N., Natarajan, K., Clatworthy, M.R., Wang, Z., and Germain, R.N. (2013). The adaptor MAVS promotes NLRP3 mitochondrial localization and inflammasome activation. Cell 153 , 348-361.

Sun, L., Wu, J., Du, F., Chen, X., and Chen, Z.J. (2013). Cyclic GMP-AMP synthase is a cytosolic DNA sensor that activates the type I interferon pathway. Science 339 , 786-791.

Takeuchi, O., and Akira, S. (2010). Pattern recognition receptors and inflammation. Cell 140 , 805-820.

Tanaka, T., Grusby, M.J., and Kaisho, T. (2007). PDLIM2-mediated termination of transcription factor NF-kappaB activation by intranuclear sequestration and degradation of the p65 subunit. Nat. Immunol. 8 , 584-591.

Tannahill, G.M., Curtis, A.M., Adamik, J., Palsson-McDermott, E.M., McGettrick, A.F., Goel, G., Frezza, C., Bernard, N.J., Kelly, B., Foley, N.H., et al. (2013). Succinate is an inflammatory signal that induces IL-1 b through HIF1 a . Nature 496 , 238-242.

Tempe ' , D., Vives, E., Brockly, F., Brooks, H., De Rossi, S., Piechaczyk, M., and Bossis, G. (2014). SUMOylation of the inducible (c-Fos:c-Jun)/AP-1 transcription complex occurs on target promoters to limit transcriptional activation. Oncogene 33 , 921-927.

Tigno-Aranjuez, J.T., Asara, J.M., and Abbott, D.W. (2010). Inhibition of RIP2's tyrosine kinase activity limits NOD2-driven cytokine responses. Genes Dev. 24 , 2666-2677.

Tokunaga, F., Sakata, S., Saeki, Y., Satomi, Y., Kirisako, T., Kamei, K., Nakagawa, T., Kato, M., Murata, S., Yamaoka, S., et al. (2009). Involvement of linear polyubiquitylation of NEMO in NF-kappaB activation. Nat. Cell Biol. 11 , 123-132.

Tsuchida, T., Zou, J., Saitoh, T., Kumar, H., Abe, T., Matsuura, Y., Kawai, T., and Akira, S. (2010). The ubiquitin ligase TRIM56 regulates innate immune responses to intracellular double-stranded DNA. Immunity 33 , 765-776.

Vande Walle, L., Van Opdenbosch, N., Jacques, P., Fossoul, A., Verheugen, E., Vogel, P., Beyaert, R., Elewaut, D., Kanneganti, T.D., van Loo, G., and Lamkanfi, M. (2014). Negative regulation of the NLRP3 inflammasome by A20 protects against arthritis. Nature 512 , 69-73.

Wang, Y., Tang, Y., Teng, L., Wu, Y., Zhao, X., and Pei, G. (2006). Association of beta-arrestin and TRAF6 negatively regulates Toll-like receptor-interleukin 1 receptor signaling. Nat. Immunol. 7 , 139-147.

Wang, Y., Chen, T., Han, C., He, D., Liu, H., An, H., Cai, Z., and Cao, X. (2007). Lysosome-associated small Rab GTPase Rab7b negatively regulates TLR4 signaling in macrophages by promoting lysosomal degradation of TLR4. Blood 110 , 962-971.

Wang, C., Chen, T., Zhang, J., Yang, M., Li, N., Xu, X., and Cao, X. (2009). The E3 ubiquitin ligase Nrdp1 'preferentially' promotes TLR-mediated production of type I interferon. Nat. Immunol. 10 , 744-752.

Wang, B., Xi, X., Lei, X., Zhang, X., Cui, S., Wang, J., Jin, Q., and Zhao, Z. (2013a). Enterovirus 71 protease 2Apro targets MAVS to inhibit anti-viral type I interferon responses. PLoS Pathog. 9 , e1003231.

Wang, Y., Shaked, I., Stanford, S.M., Zhou, W., Curtsinger, J.M., Mikulski, Z., Shaheen, Z.R., Cheng, G., Sawatzke, K., Campbell, A.M., et al. (2013b). The autoimmunity-associated gene PTPN22 potentiates toll-like receptor-driven, type 1 interferon-dependent immunity. Immunity 39 , 111-122.

Wang, P., Xue, Y., Han, Y., Lin, L., Wu, C., Xu, S., Jiang, Z., Xu, J., Liu, Q., and Cao, X. (2014a). The STAT3-binding long noncoding RNA lnc-DC controls human dendritic cell differentiation. Science 344 , 310-313.

Wang, Q., Liu, X., Cui, Y., Tang, Y., Chen, W., Li, S., Yu, H., Pan, Y., and Wang, C. (2014b). The E3 ubiquitin ligase AMFR and INSIG1 bridge the activation of TBK1 kinase by modifying the adaptor STING. Immunity 41 , 919-933.

Wang, X., Jiang, W., Yan, Y., Gong, T., Han, J., Tian, Z., and Zhou, R. (2014c). RNA viruses promote activation of the NLRP3 inflammasome through a RIP1RIP3-DRP1 signaling pathway. Nat. Immunol. 15 , 1126-1133.

Wen, M., Ma, X., Cheng, H., Jiang, W., Xu, X., Zhang, Y., Zhang, Y., Guo, Z., Yu, Y., Xu, H., et al. (2015). Stk38 protein kinase preferentially inhibits TLR9activated inflammatory responses by promoting MEKK2 ubiquitination in macrophages. Nat. Commun. 6 , 7167.

Xia, X., Cui, J., Wang, H.Y., Zhu, L., Matsueda, S., Wang, Q., Yang, X., Hong, J., Songyang, Z., Chen, Z.J., and Wang, R.F. (2011). NLRX1 negatively regulates TLR-induced NFk B signaling by targeting TRAF6 and IKK. Immunity 34 , 843-853.

Xu, S., Liu, X., Bao, Y., Zhu, X., Han, C., Zhang, P., Zhang, X., Li, W., and Cao, X. (2012). Constitutive MHC class I molecules negatively regulate TLR-triggered inflammatory responses via the Fps-SHP-2 pathway. Nat. Immunol. 13 , 551-559.

Xu, H., He, X., Zheng, H., Huang, L.J., Hou, F., Yu, Z., de la Cruz, M.J., Borkowski, B., Zhang, X., Chen, Z.J., and Jiang, Q.X. (2014). Structural basis for the prion-like MAVS filaments in antiviral innate immunity. eLife 3 , e01489.

Yan, Y., Jiang, W., Liu, L., Wang, X., Ding, C., Tian, Z., and Zhou, R. (2015). Dopamine controls systemic inflammation through inhibition of NLRP3 inflammasome. Cell 160 , 62-73.

Yang, Y., Liu, B., Dai, J., Srivastava, P.K., Zammit, D.J., Lefranc ¸ ois, L., and Li, Z. (2007). Heat shock protein gp96 is a master chaperone for toll-like receptors and is important in the innate function of macrophages. Immunity 26 , 215-226.

Yang, S., Wang, B., Humphries, F., Jackson, R., Healy, M.E., Bergin, R., Aviello, G., Hall, B., McNamara, D., Darby, T., et al. (2013a). Pellino3 ubiquitinates RIP2 and mediates Nod2-induced signaling and protective effects in colitis. Nat. Immunol. 14 , 927-936.

Yang, Y., Liao, B., Wang, S., Yan, B., Jin, Y., Shu, H.B., and Wang, Y.Y. (2013b). E3 ligase WWP2 negatively regulates TLR3-mediated innate immune response by targeting TRIF for ubiquitination and degradation. Proc. Natl. Acad. Sci. USA 110 , 5115-5120.

Yasukawa, K., Oshiumi, H., Takeda, M., Ishihara, N., Yanagi, Y., Seya, T., Kawabata, S., and Koshiba, T. (2009). Mitofusin 2 inhibits mitochondrial antiviral signaling. Sci. Signal. 2 , ra47.

Yoo, Y.S., Park, Y.Y., Kim, J.H., Cho, H., Kim, S.H., Lee, H.S., Kim, T.H., Sun Kim, Y., Lee, Y., Kim, C.J., et al. (2015). The mitochondrial ubiquitin ligase MARCH5resolves MAVS aggregates during antiviral signalling. Nat. Commun. 6 , 7910.

You, F., Sun, H., Zhou, X., Sun, W., Liang, S., Zhai, Z., and Jiang, Z. (2009). PCBP2 mediates degradation of the adaptor MAVS via the HECT ubiquitin ligase AIP4. Nat. Immunol. 10 , 1300-1308.

Yuk, J.M., Shin, D.M., Lee, H.M., Kim, J.J., Kim, S.W., Jin, H.S., Yang, C.S., Park, K.A., Chanda, D., Kim, D.K., et al. (2011). The orphan nuclear receptor SHP acts as a negative regulator in inflammatory signaling triggered by Tolllike receptors. Nat. Immunol. 12 , 742-751.

Zhang, Y., and Cao, X. (2015). Long noncoding RNAs in innate immunity. Cell. Mol. Immunol. 2015 , 17.

Zhang, M., Tian, Y., Wang, R.P., Gao, D., Zhang, Y., Diao, F.C., Chen, D.Y., Zhai, Z.H., and Shu, H.B. (2008). Negative feedback regulation of cellular antiviral signaling by RBCK1-mediated degradation of IRF3. Cell Res. 18 , 10961104.

<!-- image -->

Zhang, M., Wang, L., Zhao, X., Zhao, K., Meng, H., Zhao, W., and Gao, C. (2012). TRAF-interacting protein (TRIP) negatively regulates IFNb production and antiviral response by promoting proteasomal degradation of TANK-binding kinase 1. J. Exp. Med. 209 , 1703-1711.

Zhang, Z., Bao, M., Lu, N., Weng, L., Yuan, B., and Liu, Y.J. (2013). The E3 ubiquitin ligase TRIM21 negatively regulates the innate immune response to intracellular double-stranded DNA. Nat. Immunol. 14 , 172-178.

Zhang, L., Mo, J., Swanson, K.V., Wen, H., Petrucelli, A., Gregory, S.M., Zhang, Z., Schneider, M., Jiang, Y., Fitzgerald, K.A., et al. (2014a). NLRC3, a member of the NLR family of proteins, is a negative regulator of innate immune signaling induced by the DNA sensor STING. Immunity 40 , 329-341.

Zhang, Z., Song, M., Liu, X., Kang, S.S., Kwon, I.S., Duong, D.M., Seyfried, N.T., Hu, W.T., Liu, Z., Wang, J.Z., et al. (2014b). Cleavage of tau by asparagine endopeptidase mediates the neurofibrillary pathology in Alzheimer's disease. Nat. Med. 20 , 1254-1262.

Zhang, Q., Zhao, K., Shen, Q., Han, Y., Gu, Y., Li, X., Zhao, D., Liu, Y., Wang, C., Zhang, X., et al. (2015). Tet2 is required to resolve inflammation by recruiting Hdac2 to specifically repress IL-6. Nature 525 , 389-393.

Zhao, K., Zhang, Q., Li, X., Zhao, D., Liu, Y., Shen, Q., Yang, M., Wang, C., Li, N., and Cao, X. (2016). Cytoplasmic STAT4 promotes antiviral type I IFN production by blocking CHIP-mediated degradation of RIG-I. J. Immunol. 196 , 1209-1217.

Zheng, Q., Hou, J., Zhou, Y., Yang, Y., Xie, B., and Cao, X. (2015). Siglec1 suppresses antiviral innate immune response by inducing TBK1 degradation via the ubiquitin ligase TRIM27. Cell Res. 25 , 1121-1136.

Zhong, B., Zhang, L., Lei, C., Li, Y., Mao, A.P., Yang, Y., Wang, Y.Y., Zhang, X.L., and Shu, H.B. (2009). The ubiquitin ligase RNF5 regulates antiviral responses by mediating degradation of the adaptor protein MITA. Immunity 30 , 397-407.

Zhou, Q., Wang, H., Schwartz, D.M., Stoffels, M., Park, Y.H., Zhang, Y., Yang, D., Demirkaya, E., Takeuchi, M., Tsai, W.L., et al. (2016). Loss-of-function mutations in TNFAIP3 leading to A20 haploinsufficiency cause an early-onset autoinflammatory disease. Nat. Genet. 48 , 67-73.

Zurek, B., Schoultz, I., Neerincx, A., Napolitano, L.M., Birkner, K., Bennek, E., Sellge, G., Lerm, M., Meroni, G., So derholm, J.D., and Kufer, T.A. (2012). TRIM27 negatively regulates NOD2 by ubiquitination and proteasomal degradation. PLoS ONE 7 , e41255.