<!-- image -->

<!-- image -->

<!-- image -->

OPENACCESS

Citation: Petretto A, Bruschi M, Pratesi F, Croia C, Candiano G, Ghiggeri G, et al. (2019) Neutrophil extracellular traps (NET) induced by different stimuli: A comparative proteomic analysis. PLoS ONE 14(7): e0218946. https://doi.org/10.1371/ journal.pone.0218946

Editor: Nades Palaniyar, Hospital for Sick Children, CANADA

Received:

January 15, 2019

Accepted:

June 12, 2019

Published:

July 8, 2019

Copyright: © 2019 Petretto et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.

Data Availability Statement: All relevant data are within the manuscript.

Funding: The authors received no specific funding for this work.

Competing interests: The authors have declared that no competing interests exist.

## RESEARCHARTICLE

## Neutrophil extracellular traps (NET) induced by different stimuli: A comparative proteomic analysis

Andrea Petretto 1 ☯ , Maurizio Bruschi 2 ☯ , Federico Pratesi , Cristina Croia , 3 3 Giovanni Candiano 2 , Gianmarco Ghiggeri , Paola MiglioriniID 4 3 *

- 1 Core Facilities-Proteomics Laboratory, Istituto Giannina Gaslini, Genoa, Italy, 2 Laboratory of Molecular Nephrology, Istituto Giannina Gaslini, Genoa, Italy, 3 Clinical Immunology Unit, Department of Clinical and Experimental Medicine, University of Pisa, Pisa, Italy, 4 Division of Nephrology, Dialysis, and Transplantation, Scientific Institute for Research and Health Care (IRCCS), Istituto Giannina Gaslini, Genoa, Italy
- ☯ These authors contributed equally to this work.
- * paola.migliorini@med.unipi.it

## Abstract

Neutrophil extracellular traps (NET) formation is part of the neutrophil response to infections, but excessive or inappropriate NETosis may trigger the production of autoantibodies and cause organ damage in autoimmune disorders. Spontaneously netting neutrophils are not frequent and induction of NET in vitro by selected stimuli is necessary to investigate their structure. In the present work, the protein composition and post-translational modifications of NET produced under different stimuli have been studied by means of proteomic analysis. Neutrophils from healthy donors were stimulated by PMA, A23187, Escherichia coli LPS or untreated; after three hours, cells were washed, treated with DNase and supernatants collected for mass spectrometry. Data were analyzed by unsupervised hierarchical clustering analyses. We identified proteins contained in NETs of any source or exclusive of one stimulus: LPS-induced and spontaneous NET diverge in protein composition, while PMA- and A23187-induced NET appear more similar. Among the post-translational modifications we examined, methionine sulfoxidation is frequent especially in PMA- and LPS-induced NETs. Myeloperoxidase is the protein more extensively modified. Thus, proteomic analysis indicates that NETs induced by different stimuli are heterogeneous in terms of both protein composition and post-translational modifications, suggesting that NET induced in different conditions may have different biological effects.

## Introduction

Neutrophil extracellular traps (NET) formation (NETosis) is a mechanism of defense that neutrophils deploy as an alternative to phagocytosis, to constrain the spread of fungi, large bacteria and several other microorganisms [1,2]. During NETosis, nucleus decondenses and intracellular membranes disintegrate; nuclear and cytoplasmic content mixes and plasma membrane

permeabilizes, allowing the extrusion of a tangle of chromatin fibers decorated with granule proteins. NET trap and kill infectious agents due to the high local concentration of histones, anti-microbial peptides and other anti-microbial agents. Neutrophils are induced to release NET by the engagement of a variety of receptors by microbial products, immune complexes and crystals. In the best characterized pathway leading to NET extrusion, ERK signaling leads to the activation of NADPH oxidase and the production of superoxide radicals that are converted into oxygen peroxide by superoxide dismutase [3]. Myeloperoxidase (MPO) transforms hydrogen peroxide in hypochlorous acid, activating neutrophil elastase (NE). NE is responsible for degradation of cytoskeleton and nuclear membrane, allowing the mixing of nuclear content and cytoplasm [4]. Histone deimination by activated peptidylarginine deiminase (PAD) and proteolytic cleavage by MPO and NE lead to chromatin decondensation [5]. Chromatin fibers associate with granule and cytoplasmic proteins and are then released extracellularly. Transcriptional firing at multiple locations contributes to chromatin decondensation [6], but NET formation proceeds independently of translation [7].

Reactive oxygen species (ROS) production is the key event in NETosis and both mitochondrial respiratory chain and NADPH oxidase independently contribute to their generation. Several different receptors trigger NET formation activating NADPH oxidase (NOX), in the classical 'suicidal' NETosis [8]. Similarly, phorbol myristate acetate (PMA), resembling diacylglycerole, activates classical and conventional phosphokinase C (PKC) [9] and ERK signaling, mimicking NET induction by bacteria and fungi. PMA is in fact a very efficient and very frequently used stimulus, even if experimental protocols of NET induction via PMA may substantially differ [10].

NOXindependent NETosis relies on mitochondrial ROS production, facilitated by alkaline pH that increases Ca influx [11]. Triggering of a calcium-activated small conductance potassium (SK) channel member SK3 is critical for calcium activated NOX-independent NETosis [12].

PAD4 activation is prominent in this condition and citrullination of histones is extensive. Calcium ionophores like ionomycin (product of some Gram-positive bacteria like Streptomyces) and A23187 activate PKCz and PAD4 [9], inducing NOX-independent NETosis. In some conditions, NOX-independent nuclear and mitochondrial DNA release takes place from vital neutrophils. It has been shown that immune complexes, acting on normal or low-density granulocytes, induce mitochondrial ROS production and extrusion of NET containing mitochondrial DNA from alive cells [13]. Similarly, activated platelets in sepsis patients adhere to neutrophils and induce the extrusion of NET from live cells [14].

Spontaneously netting neutrophils, even in condition where NETosis is increased, are not so frequent to allow NET isolation and characterization. Thus, induction of NET in vitro from isolated neutrophils is a pre-requisite to investigate their physiological and pathological role. To such extent, different stimuli have been employed, such as bacterial products (lipopolysaccharide-LPS), calcium ionophores or PMA (recently reviewed by Hoppenbrowers et al , 10).

Twenty-five different NETosis inducers have been reported, and many of them at different concentrations or for different times [10]. Subtle procedural differences may lead to different results, complicating any attempt to build up a common core of knowledge on NETosis [15]. Moreover, non-overlapping pathways of NET induction are triggered by different stimuli, as recently shown by means of selective inhibitors [16]. Despite a different role of ROS production and activity of granule enzymes in NET induction, the different pathways lead to the production of NETs endowed with a similar bactericidal activity [16].

Nonetheless, the possibility that NET induced by different stimuli differ in protein composition has never been thoroughly analyzed.

The aim of the present work is to investigate the protein composition and post-translational modifications of NET produced under different stimuli by means of proteomic analysis.

## Results

## Proteins composition of NETs

Weanalyzed the protein composition of NETs, produced in vitro by isolated neutrophils, either spontaneously or after stimulation with lipopolysaccharide (LPS), calcium ionophore A23187 and phorbol 12-myristate 13-acetate (PMA).

PMAwas the most effective stimulus for the induction of NETs (S1 Fig) and no stimulus induced neutrophil apoptosis (S2 Fig).

Atotal of 330 proteins were identified as NET constituents. Among these, 74 proteins (22%) were detected in NETs of any source while 30 (9.1%), 7 (2.1%), 27 (8.2%) and 40 (12.1%) proteins were exclusive, respectively, of spontaneous NETs, or NETs obtained with LPS, A23187 and PMA (Fig 1A and S1 Table).

The majority of NET proteins corresponded to proteins annotated to inflammation (183), autoimmunity disease (196) and in particular to SLE (109) in several databases (UniProt, Open Target and Atlas) (Fig 1B).

Cellular origin of NET constituents was very similar in NETs of any origin, with a range of 1.6-2.1% (extracellular), 19.9-20.9% (membrane), 34.3-36% (cytoplasm/cytoskeleton), 27.829.7% (organelle) and 13.6-14.3% (nucleus) (Fig 2).

<!-- image -->

Fig 1. Venn diagram of proteins identified in NETs. A. Venn diagram of overlapped and unique NETs proteins identified by means of mass spectrometry. Number of proteins in the overlapping and not-overlapping areas is shown. B. Venn diagram of NETs proteins grouped on the basis of database annotation. Number of proteins in the overlapping and not-overlapping areas is shown.

<!-- image -->

https://doi.org/10.1371/journal.pone.0218946.g001

## Spontaneous

## PMA

The vast majority of NET proteins are shared by spontaneous NETs and by NETs obtained under PMA, LPS and A23187 stimulation. However, by clustering analysis a core of proteins allowing a good discrimination between spontaneous and stimuli-induced NETs can be identified (Fig 3).

To better describe the differences between spontaneous and stimuli-induced NETs, univariate statistical analysis, PLS-DA and SVM were performed. A total of 25 proteins were highlighted. Among these, 9, 5 and 6 proteins are enriched respectively in PMA-, LPS- and A23187-induced NETs (right side of each volcano plot, Fig 4).

The proteome profile of the 25 highlighted proteins is shown by means of heat map after Zscore (Fig 5).

These 25 proteins were classified according the available gene ontology (GO) signatures in cellular component (CC), molecular function (MF), and biological process (BP). Among CC proteins, 50% were annotated as cytoplasm/cytoskeleton, 20% as nucleus, 19% as membrane and 11% as extracellular. Based on MF, 50% proteins were classified as 'binding proteins', 23% as 'catalytic activity', 20% as 'structural molecule' and 6% as 'antioxidant activity'. In the BP, most proteins were involved in cellular component rearrangement, development process and response to external stimuli. The diversity of the protein expression profile between spontaneous and stimuli-induced NETs suggests an involvement of these proteins in different biological pathways/processes. To assess this, we performed gene ontology (GO) analysis based on gene annotation extracted from different open source databases. Using all identified proteins and their abundance, we highlighted the signatures of each condition. To summarize the results of GO analysis we drew a heat map of enriched gene signatures (Fig 6).

<!-- image -->

https://doi.org/10.1371/journal.pone.0218946.g002

<!-- image -->

Fig 3. Spearman's correlogram associated to unsupervised hierarchical clustering analysis of NETs. In Spearman's correlogram the coefficient values are depicted by a pseudo color scale from 0.8 (blue) to 0.9 (red); moreover the tree dendrogram displays the results of an unsupervised hierarchical clustering analysis placing similar Spearman's coefficient values near each other.

<!-- image -->

https://doi.org/10.1371/journal.pone.0218946.g003

This diagram reports the average of proteins expression associated to each highlighted biological process (rows) in the different stimuli (columns). In PMA- and LPS-induced NETs, proteins associated with cellular response to external stimuli, immune system response, neutrophil degranulation, cytoskeleton rearrangement, interleukin signalling and in JAK/STAT pathwaywere positively enriched. Proteins involved in the response to elevated Ca 2+ are instead up regulated in A23187-induced NETs.

## Post translational modifications (PTM) of NET components

Weevaluated a group of PTM potentially relevant in NETosis: methionine oxidation (sulfoxide), thiol alkylation (addition of iodoacetamide), deamination of N and Q amino acids, deimination of arginine (citrullination) and all other types of PTM grouped as 'others'. A total of 812 peptides with at least one PTM were identified. The majority of PTM were found in PMA-

Fig 4. Volcano plot of univariate statistical analysis of NETs samples. Volcano plot based on fold change (Log2) and P value (-Log10) of all the proteins identified in PMA(4a), LPS (4b) and A23187 (4c) as compared with spontaneous NETs. White circles indicate the proteins displaying statistically significant changes.

<!-- image -->

<!-- image -->

<!-- image -->

Log2

a

https://doi.org/10.1371/journal.pone.0218946.g004

induced NETs (484) followed by A23187 (442), LPS (289) and spontaneous NETs (297) (Fig 7 and S2 Table).

Most of these PTM are shared by different stimuli, but a few were exclusive for each condition. In particular, 78 (9.6%), 59 (7.3%), 137 (16.9%) and 151 (18.6%) PTM were exclusive respectively for spontaneous, LPS-, A23187- and PMA-induced NETs (Fig 7).

Spontaneous NETs display a prevalence of thiol alkylation, whereas PMA- and LPSinduced NETs exhibit a prelevance of methionine sulfoxidation and N and Q deamination.

To better describe the differences between spontaneous and stimuli-induced PTM of NET proteins, univariate statistical analysis, PLS-DA and SVM were performed.

Atotal of 11 peptides with at least one PTM, corresponding to 7 proteins, were significantly enriched across the different conditions (Fig 8).

The intensity profile of these highlighted peptides bearing PTM are show by means of heat map after Z-score (Fig 9).

Visual inspection of heat map shows the presence of two clusters composed by spontaneous and LPS-induced NETs versus A23187- and PMA-induced NETs. A23187, in particular, shows the highest intensity of peptides bearing PTM. The 7 highlighted proteins classified as CC components were equally distributed between extracellular and organelle proteins. Based on MF, the majority of proteins were classified as catalytic/antioxidant activity (40%), binding

<!-- image -->

Fig 5. Heat map of NETs highlighted proteins. Heat map of proteome profile of 25 proteins highlight through the combined use of univariate statistical analysis, support vector machine and partial least square discriminant analysis. In heat map, each row represents a protein, and each column corresponds to each condition. Normalized Z-score of proteins abundance are depicted by a pseudo color scale with red indicating positive expression, white equal expression, and blue negative expression compared to each proteins values, whereas the tree dendrogram displays the results of an unsupervised hierarchical clustering analysis placing similar proteome profile values near each other. Visual inspection of the dendrogram and heat map demonstrate the possibility to distinguish among the different condition of NETs induction.

<!-- image -->

https://doi.org/10.1371/journal.pone.0218946.g005

<!-- image -->

Fig 6. Heat map of NETs gene ontology enrichment analysis. Heat map of biological process enrichment of different NETs stimuli compared to no stimulated condition. In the heat map, each row represents a protein, and each column corresponds to each condition. Normalized Z-score of proteins abundance are depicted by a pseudo color scale with red indicating positive expression, white equal expression, and blue negative expression compared to each proteins values, whereas the tree dendrogram displays the results of an unsupervised hierarchical clustering analysis placing similar proteome profile values near each other. Visual inspection of the dendrogram and heat map demonstrate the ability of these biological processes to distinguish between the A23187 and PMA/LPS stimuli.

<!-- image -->

https://doi.org/10.1371/journal.pone.0218946.g006

proteins and structural molecules (20% each). The principals BP involved were cellular component rearrangement, immune system process and response to external stimuli.

Interestingly, methionine sulfoxide is the most frequent PTM across all conditions of NET formation (82%) and the majority of modified peptides (5/11, 45%) derive from MPO. The position of the oxidized amino acid depends on the type of stimulus but all oxidized peptides are localized on the surface of molecule (Fig 10), suggesting that in the different pathways triggered by each stimulus MPO can be oxidized in different spots.

In NETs obtained with different stimuli, proteins differ also in PTM. PMA- and A23187induced NETs are again similar, bearing a higher number of PTM when compared to LPSinduced and spontaneous NETs. These latter are characterized by a lower degree of oxidation: thiol alkylation, indicative of a free SH group, is in fact more frequent in spontaneous NETs. No major differences are observed in the frequency of the other PTM we studied, but individual proteins bearing PTM may be differentially enriched in NET obtained with the various stimuli: oxidized MPO and cathepsin G are abundant in PMA- and A23187-induced NET, oxidized calprotectin in A23187- and LPS-induced NET, while oxidized actin is highly represented in A23187-induced NET.

<!-- image -->

Fig 7. Venn diagram of total NETs peptides with at least one post translational modification identified by means of mass spectrometry. Venn diagram shows overlapped and unique peptides. Numbers represent the distinct peptides/post translational modifications in the overlapping and not-overlapping areas.

<!-- image -->

https://doi.org/10.1371/journal.pone.0218946.g007

## Discussion

Despite the growing interest in NETosis as a double-edged sword in innate immunity, several issues are still obscure. Multiple receptors are engaged in NETosis and redundant signaling pathways are involved, thus allowing a fine tuning of this potentially dangerous tool.

Each of the stimuli we used for NET induction recapitulates the pathways physiologically triggered in neutrophils. The different pathways, however, converge and the final steps (degradation of nuclear membrane, chromatin decondensation, extrusion of fibers) are shared. As a consequence, NET produced by all stimuli are proteolytically active and kill bacteria [16]. Thus, it is conceivable that the vast majority of NET proteins are identical across stimuli and come from the same cell compartments. Proteomic analysis of NETs induced by different stimuli shows in fact a common core of proteins, which characterizes any type of NETs we studied and have been previously described as NET components [17, 18]. The presence in NETs of proteins contained in cytoplasm, nucleus, and membranes is consistent with the process of NET formation, which involves multiple pathways in several cellular compartments. Components from granules and cytoplasm in general are more abundant; histones represent the main nuclear proteins.

Fig 8. Volcano plot of univariate statistical analysis of spontaneous and induced NETs. Volcano plot based on fold change (Log2) and P value (-Log10) of all the peptides with at least one PTM identified in PMA (8a), LPS (8b) and A23187 (8c) as compared with spontaneous NETs. White circles indicate the peptides displaying statistically significant changes.

<!-- image -->

https://doi.org/10.1371/journal.pone.0218946.g008

However, in the different conditions proteins differentially expressed were also detected, that can be ascribed to only partially overlapping pathways, related to the process of NET formation.

The heat map and the Volcano plot clearly depict these differences. On the whole, LPSinduced and spontaneous NET widely diverge in protein composition, while PMA- and A23187-induced NET appear more similar. In fact, ROS production is observed also in NOXindependent pathways [16], thus explaining why end products are similar in NOX dependent (PMA) or independent stimuli (A23187).

Taking into account individual components, cytoskeleton proteins such as actin and tubulin are enriched in spontaneous NET, while the dynein protein DNAH5 and the heat shock protein HSPA1B are highly represented in PMA-induced NET. Calprotectin S100A9 and the ribosomal protein RPS27 are shared by PMA- and A23187-induced NET, while LAMP2 (lysosomal associated membrane protein 2) is enriched in LPS-induced NET.

NETosis is critical in innate immunity and the failure to form NET is associated with a severe form of immunodeficiency [19]. Even if NETosis has been discovered and studied as a critical step in protection from infections, available data on the interaction of neutrophils and pathogens in NET formation are not conclusive. The role of LPS, a major component of the outer membrane of Gram-negative bacteria, as signaling molecule has been debated and conflicting data have been published. Recently, it has been suggested that only a few bacteria express LPS able to induce directly NETosis, by a TLR4 independent signaling [20]. LPS from

<!-- image -->

Fig 9. Heat map of highlight peptides. Heat map of peptide profile of NETs samples. In heat map, each row represents a modified peptide, and each column corresponds to each stimulus. Normalized Z-score of peptides abundance are depicted by a pseudo color scale with red indicating positive expression, white equal expression, and blue negative expression compared to each peptide value, whereas the tree dendrogram displays the results of an unsupervised hierarchical clustering analysis placing similar peptide profile values near each other.

<!-- image -->

https://doi.org/10.1371/journal.pone.0218946.g009

a strain of E coli . and Pseudomonas aeruginosa induces NOX-dependent suicidal NETosis while infection from the other Gram-negative bacteria leads to LPS interaction with platelets, platelet activation and the process of vital NETosis. However, NET induction by LPS seems to be at least in part dose-dependent and LPS from different sources may induce NETs when used at appropriate concentrations [21].

On the other side, excessive or inappropriate NETosis may trigger the production of autoantibodies and directly cause organ damage in autoimmune disorders [22, 23]. Excessive NET load due to increased production and/or defective removal may elicit interferon (IFN) I production and contribute to the induction of anti-chromatin antibodies in systemic lupus

Fig 10. Three-dimensional (3D) model of MPO. 3D-model of MPO based on its crystal structure (PDB codes: 5 MFA). The model is colored in function of second structure i.e. gray, red and cyan respectively for coil, alpha helix and beta-strand. In yellow methionine sulfoxide is shown.

<!-- image -->

<!-- image -->

## Modified Aminoacids

https://doi.org/10.1371/journal.pone.0218946.g010

erythematosus (SLE) [24-27]. Netting neutrophils have been detected in renal biopsies of patients affected by systemic lupus or ANCA-associated vasculitis (AAV) [28]. In AAV, often associated with a necrotizing pauci-immune glomerulonephritis, NETosis is an important mechanism of endothelial damage. Autoantibodies specific for neutrophil granule-associated proteins such as MPO, lactoferrin, cathepsin G, alpha enolase are a hallmark of AAV [29, 30]. These autoantigens have all been detected in NETs induced by any stimuli. Another NET constituent relevant for AAV is hLAMP-2 (Lysosome associated membrane protein-2), a protein that shuttles between lysosomes and cell membrane.Anti-hLAMP-2 antibodies, as anti-MPO antibodies, are in fact strongly related with disease activity, being detected mainly in the active phases of AAV [31].

Other features of NET constituents that may influence their fate as autoantigens are post translational modifications (PTMs), that may render immunogenic otherwise tolerated

antigens. Oxidation and citrullination, disrupting the structure of proteins and increasing the susceptibility to proteolytic degradation, allow antigen presentation and exposure of cryptic epitopes, steps that are all involved in the induction of autoimmune responses.

In NETs obtained with different stimuli, proteins differ also in PTM. PMA- and A23187-induced NETs are again similar, bearing a higher number of PTM when compared to LPS-induced and spontaneous NETs. These latter are characterized by a lower degree of oxidation: thiol alkylation, indicative of a free SH group, is in fact more frequent in spontaneous NETs. No major differences are observed in the frequency of the other PTM we studied, but individual proteins bearing PTM may be differentially enriched in NET obtained with the various stimuli: oxidized MPO and cathepsin G are abundant in PMA- and A23187-induced NET, oxidized calprotectin in A23187- and LPS-induced NET, while oxidized actin is highly represented in A23187-induced NET.

MPOand histones are representative examples of post translationally modified autoantigens contained in NETS; citrullinated histone H4 from NETs has been described as target of anti-citrullinated antibodies in RA, potentially involved in the induction of this class of autoantibodies [32,33]. Another critical histone PTM is acetylation, that contributes to chromatin unfolding [34], increases the immune stimulatory potential of NETs [35] and creates targets for autoantibodies in SLE[36]. The above described autoantigens are present in NETs produced under any stimulus, together with cathelicidin, a defense peptide that may induce maturation of plasmacytoid dendritic cells permitting antigen presentation and possibly autoreactive B cell expansion [37]. Thus, the protein composition of NETs (autoantigens together with danger signals such as cathelicidin) is consistent with their role in induction and expansion of autoantibody production.

In conclusion, proteomic analysis of NET indicates that NETs induced by different stimuli are heterogeneous in terms of protein composition and post-translational modifications. Many of the differentially expressed or differentially modified proteins are targets of autoantibodies.

This observation adds further complexity to a very complex biological phenomenon, suggesting that NET induced in different conditions by different pathways may have different biological effects.

## Materials and methods

## Isolation of neutrophils and 'in vitro' NETs induction

Neutrophils were obtained from buffy coat of healthy blood donors and isolated using discontinuous gradient centrifugation according to English et al . [38]. Briefly, 30 ml of buffy coat preparation were diluted to 150ml with D-PBS, stratified over a double Histopaque 1.077/ 1.119 gradient and centrifuged at 400g for 30 min, with low acceleration and without brake. Granulocyte ring was recovered, cells were washed twice with cold D-PBS and counted in Burker chamber.

Atotal of 10-15 million neutrophils were seeded in 10mm Petri dishes in Hank's balanced salt solution (HBSS) at 10 × 10 6 cells/plate and treated with a) no stimulus; b) 100 nM phorbol 12-myristate 13-acetate (PMA); c) 4 μ MA23187; d) 1 μ g/ml LPS from E coli . (all from Sigma Aldrich) for 3 hours at 37˚C.

After removing the medium, the wells were gently washed 2 × 10 min with 10ml Dulbeccomodified phosphate buffer saline (D-PBS) and incubated for 20 min at 37˚C with 10 U/ml DNase I (Sigma) in HBSS + CaCl2 2 mM. DNase activity was stopped by adding EDTA 5 mM (final concentration). The samples were then centrifuged 10' at 3000g to remove intact cells

<!-- image -->

and intact nuclei; the supernatants containing NET proteins were processed as described below.

In parallel, neutrophils were seeded on cover slides (400000/slide) and incubated for 3 h with medium only or with PMA, A23187, LPS or an hypertonic solution (PBS, NaCl 509 mM). Cells were washed in PBS and then fixed in PFA 4% for 10' RT. After fixation cells were incubated with protein block (Dako) for 10' and then with the proper primary antibody: antiMPO(Dako, 1:300) to visualise NETs, or anti-cleaved Caspase 3 (Cell Signalling Technology, 1:250) to visualise apoptotic cells. After a PBS wash, cells were incubated with goat anti-rabbit Alexa488 secondary antibody (Invitrogen, 1:300) for 30'. To visualise the nucleus, cells were incubated for 10' with DAPI. After a wash in PBS, cover slides were mounted with Mowiol on a slide and analysed with a fluorescence microscope.

Counts of stained cells were performed in five 40x microscopic fields per section.

The study was approved by the local Ethics Committee, Comitato Etico per la Sperimentazione Clinica Area Vasta Nord Ovest (CEAVNO)- protocol 3661/2012. No informed consent was obtained because the data were analyzed anonymously.

## Sample preparation for mass spectrometry and mass spectrometer setup

The supernatant containing NET proteins was precipitated with acetone and NET pellets were re-suspended in 25 μ l of lysis buffer (6M GdmCl, 10mM TCEP, 40mM CAA, and 100 mM Tris pH8.5). The samples were reduced, alkylated and lastly digested in a single step and then loaded into StageTip. Peptides were analyzed by nano-UHPLC-MS/MS using an Ultimate 3000 RSLC with EASY spray column (75 μ mx500 mm, 2 μ mparticle size, Thermo Scientific) and with a 180 minute non-linear gradient of 5-45% solution B (80% CAN and 20% H2O, 5% DMSO, 0.1% FA) at a flow rate of 250 nl/min. Eluting peptides were analyzed using an Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific Instruments, Bremen, Germany). Orbitrap detection was used for both MS1 and MS2 measurements at resolving powers of 120 Kand 30 K (at m/z 200), respectively. Data dependent MS/MS analysis was performed in top speed mode with a 2 sec. cycletime, during which precursors detected within the range of m/z 375 -1500 were selected for activation in order of abundance. Quadrupole isolation with a 1.4 m/z isolation window was used, and dynamic exclusion was enabled for 45 s. Automatic gain control targets were 2.5 × 10 5 for MS1 and 5 × 10 4 for MS2, with 50 and 60 ms maximum injection times, respectively. The signal intensity threshold for MS2 was 1 × 10 . HCD was per4 formed using 30% normalized collision energy. One microscan was used for both MS1 and MS2 events. For all the MS1 scans the option ETD internal Calibration was selected.

MaxQuant software, version 1.5.5.30, was used to process the raw data, setting a false discovery rate (FDR) of 0.01 for the identification of proteins, peptides and PSM (peptide-spectrum match), moreover a minimum length of 6 amino acids for peptide identification was required. Andromeda engine, incorporated into MaxQuant software, was used to search MS/ MSspectra against Uniprot human database (release UP000005640\_9606 February 2016).

Two different elaborations were made to identify the PTMs in order to limit the false positives. In the first processing, variable modifications were Acetylation (Protein N-Term), Oxidation (M), Deamidation (NQR) and Carbamido-methylation (C). Finally, in order to overcome the common limitations of search engine based PTM analysis, we used the unbiased PTMDependent Peptide (DP) search option, taking advantage of high mass accuracy data collected in high resolution mode with an internal calibration (MS1 error &lt; 1 ppm). ). The DP peptide table was extracted from allPeptides.txt by Plugin Dependent Peptides of Perseus (https://github.com/jdrudolph/PluginDependentPeptides) and filtered by score &gt; 80. The intensity values were extracted and statistically evaluated using the different Site Table, DP

<!-- image -->

table or Protein Groups table. Algorithm MaxLFQbwas chosen for the protein quantification with the activated option 'match between runs'to reduce the number of the missing proteins.

The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE [39] partner repository under the Project Name 'NET induced by different stimuli: a comparative proteomic analysis' and the dataset identifier PXD012951.

## Statistics and bioinformatic analysis

After normalization, data obtained from mass spectrometry were analyzed using unsupervised hierarchical clustering analyses, i.e. Multidimensional Scaling (MDS) and Spearman's Correlation, to identify outlier and samples dissimilarity. Differences in proteins and peptides expression between spontaneous and stimuli-induced NET samples were detected using a nonparametric U-Mann Whitney test and p-values were adjusted using the Benjamini-Hochberg method. Results were considered significant with two-fold change and adjusted for Pvalue � 0.05. Volcano plot was used to visualize the statistical differences, in which case the cutoff lines were established using the function y = c/(x-x0). Non-linear support vector machine (SVM) and partial least squares discriminant analysis (PLS-DA) were utilized to identify maximal discrimination among groups. In SVM, a cross-validation approach a 4-fold increment limit was applied to estimate the accuracy of classification and make a ranked proteins list. The results of these analyses were summarized by mean of heat map graph.

Cytoscape software was utilized for gene ontology and pathway analysis. The list of identified proteins was validated according to their gene ontology (GO) annotation extracted from the UniProt, Reactome, KEGG and ClueGO database. The results of GO analysis were shown by mean of heat map. All statistical analysis were performed using software package R last version available at the time of experiments.

## Supporting information

S1 Fig. MPO/DAPI IF on neutrophils untreated (A-C) or treated with PMA (D-F), A23187 (G-I) or LPS (J-L). A graph in M summarizes the results. (TIF)

S2 Fig. Cleaved Caspase 3/DAPI IF on neutrophils untreated (A, B), or treated with PMA (C-E), A23187 (F-H), LPS (I-K) or hypertonic solution (J-N) treated. A graph in O summarizes the results.

(TIF)

S1 Table. List of all proteins identified by mass spectrometry. The symbol '+' identify the proteins present in each experimental condition (Fig 1A). (DOCX)

S2 Table. List of all post-translational modifications identified by mass spectrometry using dependent peptide (DP) and variable modification (VM) methods. The symbol '+' identify the post-translational modifications present in each experimental condition (Fig 7). (DOCX)

## Author Contributions

Conceptualization: Federico Pratesi, Gianmarco Ghiggeri, Paola Migliorini.

Data curation:

Maurizio Bruschi.

Formal analysis: Andrea Petretto, Giovanni Candiano.

<!-- image -->

Investigation: Andrea Petretto, Maurizio Bruschi, Federico Pratesi, Cristina Croia.

Methodology: Giovanni Candiano.

Resources: Gianmarco Ghiggeri.

Supervision: Paola Migliorini.

Writing - original draft: Federico Pratesi, Paola Migliorini.

Writing - review &amp; editing: Paola Migliorini.

## References

- 1. Brinkmann V, Reichard U, Goosmann C, Fauler B, Uhlemann Y, Weiss DS, et al. Neutrophil extracellular traps kill bacteria. Science 2004; 303: 1532-5. https://doi.org/10.1126/science.1092385 PMID: 15001782
- 2. Branzk N, Lubojemska A, Hardison SE, Wang Q, Gutierrez MG, Brown GD, et al. Neutrophils sense microbe size and selectively release neutrophil extracellular traps in response to large pathogens. Nat Immunol. 2014; 15: 1017-25. https://doi.org/10.1038/ni.2987 PMID: 25217981
- 3. Delgado-Rizo V, Martı 'nez-Guzma 'n MA, I ñ iguez-Gutierrez L, Garcı 'a-Orozco A, Alvarado-Navarro A, Fafutis-Morris M. Neutrophil Extracellular Traps and Its Implications in Inflammation: An Overview. Front Immunol. 2017; 8: 81. https://doi.org/10.3389/fimmu.2017.00081 PMID: 28220120
- 4. Papayannopoulos V, Metzler KD, Hakkim A, Zychlinsky A. Neutrophil elastase and myeloperoxidase regulate the formation of neutrophil extracellular traps. J Cell Biol. 2010; 191(3): 677-91. https://doi. org/10.1083/jcb.201006052 PMID: 20974816
- 5. WangY, Li M, Stadler S, Correll S, Li P, Wang D, et al. Histone hypercitrullination mediates chromatin decondensation and neutrophil extracellular trap formation. J Cell Biol. 2009; 184(2): 205-13. https:// doi.org/10.1083/jcb.200806072 PMID: 19153223
- 6. Khan MA, Palaniyar N. Transcriptional firing helps to drive NETosis. Sci Rep. 2017; 7: 41749. https:// doi.org/10.1038/srep41749 PMID: 28176807
- 7. Sollberger G, Amulic B, Zychlinsky A. Neutrophil Extracellular Trap Formation Is Independent of De Novo Gene Expression. PLoS One. 2016 Jun 16; 11(6): e0157454. https://doi.org/10.1371/journal. pone.0157454 PMID: 27310721
- 8. Yang H, Biermann MH, Brauner JM, Liu Y, Zhao Y, Herrmann M. New Insights into Neutrophil Extracellular Traps: Mechanisms of Formation and Role in Inflammation. Front Immunol. 2016; 7: 302. https:// doi.org/10.3389/fimmu.2016.00302 PMID: 27570525
- 9. Neeli I, Radic M. Opposition between PKC isoforms regulates histone deimination and neutrophil extracellular chromatin release. Front Immunol. 2013; 4: 38. https://doi.org/10.3389/fimmu.2013.00038 PMID: 23430963
- 10. Hoppenbrouwers T, Autar ASA, Sultan AR, Abraham TE, van Cappellen WA, Houtsmuller AB, et al. In vitro induction of NETosis: Comprehensive live imaging comparison and systematic review. PLoS One. 2017; 12(5): e0176472. https://doi.org/10.1371/journal.pone.0176472 PMID: 28486563
- 11. Naffah de Souza C, Breda LCD, Khan MA, de Almeida SR, Ca ˆmara NOS, Sweezey N, et al. Alkaline pH Promotes NADPH Oxidase-Independent NeutrophilExtracellular Trap Formation: A Matter of Mitochondrial Reactive Oxygen Species Generation and Citrullination and Cleavage of Histone. Front Immunol. 2018; 8: 1849. https://doi.org/10.3389/fimmu.2017.01849 PMID: 29375550
- 12. Douda DN, Khan MA, Grasemann H, Palaniyar N. SK3 channel and mitochondrial ROS mediate NADPHoxidase-independent NETosis induced by calcium influx. Proc Natl AcadSci U S A. 2015; 112 (9): 2817-22. https://doi.org/10.1073/pnas.1414055112 PMID: 25730848
- 13. Lood C, Blanco LP, Purmalek MM, Carmona-Rivera C, De Ravin SS, Smith CK, et al. Neutrophil extracellular trapsenriched in oxidized mitochondrial DNA are interferogenic and contribute to lupus-like disease. Nat Med. 2016; 22(2):146-53. https://doi.org/10.1038/nm.4027 PMID: 26779811
- 14. Clark SR, Ma AC, Tavener SA, McDonald B, Goodarzi Z, Kelly MM, et al. Platelet TLR4 activates neutrophil extracellular traps to ensnare bacteria in septic blood. Nat Med. 2007; 13(4):463-9. Epub 2007. https://doi.org/10.1038/nm1565 PMID: 17384648
- 15. Naccache PH, Fernandes MJ. Challenges in the characterization of neutrophilextracellular traps: The truth is in the details. Eur J Immunol. 2016; 46(1):52-5. https://doi.org/10.1002/eji.201546022 PMID: 26635275

- 16. Kenny EF, Herzig A, Kru ger R, Muth A, Mondal S, Thompson PR, et al. Diverse stimuli engage different neutrophil extracellular trap pathways. Elife. 2017 Jun 2; 6. pii: e24437. https://doi.org/10.7554/eLife. 24437 PMID: 28574339
- 17. Urban CF, Ermert D, Schmid M, Abu-Abed U, Goosmann C, Nacken W, et al. Neutrophil extracellular traps contain calprotectin, acytosolic protein complex involved in host defense against Candida albicans . PLoS Pathog. 2009; 5(10): e1000639. https://doi.org/10.1371/journal.ppat.1000639 PMID: 19876394
- 18. Fadini GP, Menegazzo L, Rigato M, Scattolini V, Poncina N, Bruttocao A, et al. NETosis Delays Diabetic Wound Healing in Mice and Humans. Diabetes. 2016; 65(4):1061-71. https://doi.org/10.2337/db150863 PMID: 26740598
- 19. Metzler KD, Fuchs TA, Nauseef WM, Reumaux D, Roesler J, Schulze I, et al. Myeloperoxidase is required for neutrophil extracellular trap formation: implications for innate immunity. Blood2011; 117:953-9. https://doi.org/10.1182/blood-2010-06-290171 PMID: 20974672
- 20. Pieterse E, Rother N, Yanginlar C, Hilbrands LB, van der Vlag J. Neutrophils discriminate between lipopolysaccharides of different bacterial sources and selectively release Neutrophil Extracellular Traps. Front Immunol. 2016; 7: 484. https://doi.org/10.3389/fimmu.2016.00484 PMID: 27867387
- 21. Khan MA, Farahvash A, Douda DN, Licht JC, Grasemann H, Sweezey N, et al. JNK activation turns on LPS- and Gram-negative bacteria-induced NADPH oxidase-dependent suicidal NETosis. Sci Rep. 2017; 7(1):3409. https://doi.org/10.1038/s41598-017-03257-z PMID: 28611461
- 22. Radic M, Marion TN. Neutrophil extracellular chromatin traps connect innate immune response to autoimmunity. Semin Immunopathol 2013; 35:465-80. https://doi.org/10.1007/s00281-013-0376-6 PMID: 23595413
- 23. Barnado A, Crofford LJ, Oates JC. At the Bedside: Neutrophil extracellular traps (NETs) as targets for biomarkers and therapies in autoimmune diseases. J Leukoc Biol. 2016; 99:265-78. https://doi.org/10. 1189/jlb.5BT0615-234R PMID: 26658004
- 24. Knight JS, Carmona-Rivera C, Kaplan MJ. Proteins derived from neutrophil extracellular traps may serve as self-antigens and mediate organ damage in autoimmune diseases. Front Immunol2012; 3:380. https://doi.org/10.3389/fimmu.2012.00380 PMID: 23248629
- 25. Lande R, Ganguly D, Facchinetti V, Frasca L, Conrad C, Gregorio J, et al. Neutrophils activate plasmacytoid dendritic cells by releasing self-DNA-peptide complexes in systemic lupus erythematosus. Sci Transl Med 2011; 3:73ra19. https://doi.org/10.1126/scitranslmed.3001180 PMID: 21389263
- 26. Hakkim A, Fu rnrohr BG, Amann K, Laube B, Abed UA, Brinkmann V, et al. Impairment of neutrophil extracellular trap degradation is associated with lupus nephritis. Proc Natl Acad Sci U S A 2010; 107:9813-8. https://doi.org/10.1073/pnas.0909927107 PMID: 20439745
- 27. Villanueva E, Yalavarthi S, Berthier CC, Hodgin JB, Khandpur R, Lin AM, et al. Netting neutrophils induce endothelial damage, infiltrate tissues, and expose immune stimulatory molecules in systemic lupus erythematosus. J Immunol 2011; 187:538-52. https://doi.org/10.4049/jimmunol.1100450 PMID: 21613614
- 28. Kessenbrock K, Krumbholz M, Scho nermarck U, Back W, Gross WL, Werb Z, et al. Netting neutrophils in autoimmune small-vessel vasculitis. Nat Med. 2009; 15(6):623-5. https://doi.org/10.1038/nm.1959 PMID: 19448636
- 29. Csernok E, Moosig F.Current and emerging techniques for ANCA detection in vasculitis. Nat Rev Rheumatol. 2014; 10(8):494-501. https://doi.org/10.1038/nrrheum.2014.78 PMID: 24890776
- 30. Korkmaz B, Lesner A, Letast S, Mahdi YK, Jourdan ML, Dallet-Choisy S, et al. Neutrophil proteinase 3 and dipeptidyl peptidase I (cathepsin C) as pharmacological targets in granulomatosis with polyangiitis (Wegener granulomatosis). Semin Immunopathol. 2013; 35(4):411-21. https://doi.org/10.1007/ s00281-013-0362-z PMID: 23385856
- 31. Kain R, Tadema H, McKinney EF, Benharkou A, Brandes R, Peschel A, et al. High prevalence of autoantibodies to hLAMP-2 in anti-neutrophil cytoplasmic antibody-associated vasculitis. J Am Soc Nephrol 2012; 23:556-66. https://doi.org/10.1681/ASN.2011090920 PMID: 22323643
- 32. Dwivedi N, Radic M. Citrullination of autoantigens implicates NETosis in the induction of autoimmunity. Ann Rheum Dis 2014; 73: 483-91. https://doi.org/10.1136/annrheumdis-2013-203844 PMID: 24291655
- 33. Corsiero E, Pratesi F, Prediletto E, Bombardieri M, Migliorini P. NETosis as Source of Autoantigens in Rheumatoid Arthritis. Front Immunol 2016; 7:485. https://doi.org/10.3389/fimmu.2016.00485 PMID: 27895639
- 34. HamamHJ,KhanMA, Palaniyar N. Histone Acetylation Promotes Neutrophil Extracellular Trap Formation. Biomolecules. 2019; 9(1). pii: E32. https://doi.org/10.3390/biom9010032 PMID: 30669408

<!-- image -->

- 35. Pieterse E, Hofstra J, Berden J, Herrmann M, Dieker J, van der Vlag J. Acetylated histones contribute to the immune stimulatory potential of neutrophil extracellular traps in systemic lupus erythematosus. Clin Exp Immunol. 2015; 179(1):68-74. https://doi.org/10.1111/cei.12359 PMID: 24758196
- 36. Liu CL, Tangsombatvisit S, Rosenberg JM, Mandelbaum G, Gillespie EC, et al. Specific post-translational histone modifications of neutrophil extracellular traps as immunogens and potential targets of lupus autoantibodies. Arthritis Res Ther. 2012; 14(1):R25. https://doi.org/10.1186/ar3707 PMID: 22300536
- 37. Lande R, Ganguly D, Facchinetti V, Frasca L, Conrad C, Gregorio J, et al. Neutrophils activate plasmacytoid dendritic cells by releasingself-DNA-peptide complexes in systemic lupus erythematosus. Sci Transl Med. 2011; 3(73):73ra19. https://doi.org/10.1126/scitranslmed.3001180 PMID: 21389263
- 38. English D, Andersen BR. Single-step separation of red blood cells. Granulocytes and mononuclearleukocytes on discontinuous density gradients of Ficoll-Hypaque. J Immunol Methods. 1974; 5(3):249-52. PMID: 4427075.
- 39. Vizcaı 'no JA, Csordas A, del-Toro N, Dianes JA, Griss J, Lavidas I, et al. 2016 update of the PRIDE database and related tools. Nucleic Acids Res 2016; 44(D1): D447-56. https://doi.org/10.1093/nar/ gkv1145 PMID: 26527722