Annu. Rev. Immunol. 2014. 32:547-77

The Annual Review of Immunology is online at immunol.annualreviews.org

This article's doi:

10.1146/annurev-immunol-032713-120254

Copyright c © 2014 by Annual Reviews. All rights reserved

## Systems-Level Analysis of Innate Immunity

Daniel E. Zak, Vincent C. Tam, and Alan Aderem

Seattle BioMed, Seattle, Washington 98109; email: alan.aderem@seattlebiomed.org

## Keywords

systems biology, transcriptomics, proteomics, lipidomics, influenza, systems vaccinology

## Abstract

Systems-level analysis of biological processes strives to comprehensively and quantitatively evaluate the interactions between the relevant molecular components over time, thereby enabling development of models that can be employed to ultimately predict behavior. Rapid development in measurement technologies (omics), when combined with the accessible nature of the cellular constituents themselves, is allowing the field of innate immunity to take significant strides toward this lofty goal. In this review, we survey exciting results derived from systems biology analyses of the immune system, ranging from gene regulatory networks to influenza pathogenesis and systems vaccinology.

## INTRODUCTION

## Emergent Properties and Immunity

The functions of the immune system, such as effective host defense (under ideal conditions) or inflammatory disease (when the system is dysregulated), arise from molecular and cellular interactions between its constituents. Immune system function is therefore an emergent property of a complex dynamic system. The relationship between immune responses-which we may wish to enhance to prevent infection or squelch in autoimmune disease-and the particular manipulations we can make-such as administering drugs or vaccines-is therefore not simple or straightforward. Predicting how our molecular interventions will alter immune phenotypes or identifying the molecular interventions that will give rise to the phenotypes we seek to induce requires an understanding of the overall system. For example, it is not enough simply to know the specific kinase target of a drug. We must know the cell types in which that kinase is functional, the targets of that kinase in those cells, how those targets control cellular functions during an immune response, and how the different cellular subsets regulate each other and the immune system. We define the analysis that will collect and integrate these multiple levels of information as systems analysis and the encompassing field as systems biology. The goal of applying systems analysis to immunity is to develop a holistic and predictive understanding that can be harnessed to rationally guide the development of new vaccines and treatments.

## Technology Revolutionizes Biology

To perform systems analysis, information must be captured and integrated from as many hierarchical levels as possible. These levels include DNA and RNA sequences, RNA abundances, protein abundances and localization, protein-protein and protein-DNA interactions, concentrations of lipids and their localization, abundances of cellular metabolites, cell-cell interactions, tissues, organs, organisms, and phenotypes. Our ability to query biological systems has rapidly advanced, particularly with the development of omics technologies: Genomics (sequencing of entire genomes) is now complemented by transcriptomics (characterization and quantification of all mRNA species), proteomics (comprehensive characterization and quantification of proteins and protein states), metabolomics, and lipidomics.

On the surface, omics technologies appear to be no more than large-scale versions of existing targeted assays, with the added throughput potentially coming at the expense of measurement quality (accuracy and precision). This is not the case. On the one hand, maturation of these technologies has greatly reduced the quality gap between omics measurements and their targeted counterparts, especially for transcriptomics. In fact, an expression estimate for a single gene derived from a high-depth RNA-Seq analysis, which is based on the counting of individual transcript molecules, is likely to be as accurate a representation of the true mRNA abundance as expression estimates based on targeted quantitative real-time PCR (qRT-PCR). On the other hand, the unbiased and comprehensive nature of omics measurements allows them to capture cellular states in a manner that targeted assays cannot. Omics measurements of cellular states allow inferences about cellular networks that were not previously possible. Take, for example, the comparative analysis of transcript levels in lipopolysaccharide (LPS)-treated and unstimulated macrophages. Using qRT-PCR, a researcher may measure levels of IL-6 mRNA and see that they increase after stimulation. From this measurement, she may infer that levels of secreted IL-6 protein will also increase ( Figure 1 a ), but the analysis does not go further. The same researcher may also use RNA-Seq to measure the levels of all mRNAs (the transcriptome) before and after LPS

·

·

Figure 1

<!-- image -->

Omics measurements-more than high throughput. ( ) Conventional targeted measurement of mRNA a levels-for example, by qRT-PCR-produces a measurement that can be used to make inferences about the levels of the encoded protein. ( b ) Although genome-level assessment of mRNA levels-for example, by RNA-Seq transcriptomics-may be similarly employed to make inferences about encoded proteins, the unbiased and systematic nature of the transcriptome measurement also allows it to be interpreted as a holistic readout of all gene regulatory activities within the cell. Transcriptomes interpreted in this manner may be interrogated by network analysis to make inferences about the activities of transcription factors, RNA-binding proteins, and miRNAs that result in differential transcriptome patterns across varying conditions.

<!-- image -->

The iterative cycle of systems biology. Biology dictates what new technology and computational tools must be developed to answer specific questions. In turn, newly developed technologies and tools open new frontiers, revolutionizing biology and generating new fields of inquiry. (Figure adapted from Reference 139.)

treatment and would again observe that IL-6 mRNA increases. This time, however, she would observe that IL-6 is a member of one of several waves of coordinately regulated genes that are up- and downregulated after LPS treatment, each with different kinetics. This observation of coordinated regulatory patterns in the mRNA levels allows her to interpret the mRNA data in a new way-not simply as large-scale protein precursor measurements, but as measures of the sum total of transcriptional and posttranscriptional regulatory outputs of the cell ( Figure 1 b ). Transcript levels change in response to variations in transcription factors, RNA binding proteins, and microRNA (miRNA) activities. Therefore, transcriptome measurements may be used to make inferences in the reverse direction-that is, about changes in the regulatory networks upstream of transcription ( Figure 1 b ). Transcriptomes thus do not serve solely as a prelude to proteins; they also provide a window into cellular signaling and gene regulatory networks and, surprisingly, may be highly informative even if the mRNA levels do not reflect the protein levels.

The interpretation of transcriptomes as regulatory outputs forms the basis of many successful gene regulatory network inference approaches (for example, see 1) and demonstrates how improvements in technology can usher in a reconceptualization of biology ( Figure 2 ). The need to measure transcripts more efficiently and systematically resulted in the rapid development of new technologies. These technologies, in turn, enabled new approaches to network analysis. This iterative cycle-in which a biological question drives the development of a new technology, and the new technology, in turn, enables greater advances in biology-is one of the underpinnings of systems biology.

## Systems Biology Weds Experimentation and Computation

A central role for computation in systems analysis arises naturally for two reasons: ( a ) the sheer scale of omics measurements necessitates computational approaches for the management, quality control, standardization, and distribution of the data sets, and ( b ) quantitative mathematical models are required to mine and interpret the data sets and to make biological inferences. Both a and b constitute fields unto themselves, with an ever-expanding array of tools and approaches being developed to address both general and idiosyncratic issues arising for particular data sets and analyses. The mathematical modeling approaches employed in systems biology comprise a broad spectrum, ranging from fine-grained dynamic models of small pathways and networks in which the

·

·

Table 1 Representative public resources for systems analysis of innate immunity a

| A: Examples of network analysis resources                                               | A: Examples of network analysis resources                                               | A: Examples of network analysis resources                                               | A: Examples of network analysis resources                                                |
|-----------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|
| Software tool                                                                           | Reference                                                                               | Capabilities                                                                            | Databases                                                                                |
| BiologicalNetworks                                                                      | 141                                                                                     | Network construction, layout, and analysis                                              | KEGG, BIND, TRANSFAC Public                                                              |
| Osprey                                                                                  | 142                                                                                     | Network construction, layout, and analysis                                              | BioGRID                                                                                  |
| InnateDB                                                                                | 143                                                                                     | Network construction and analysis [layout using Cerebral (144)]                         | InnateDB                                                                                 |
| Cytoscape                                                                               | 145                                                                                     | Network layout and analysis                                                             | KEGG                                                                                     |
| VisANT                                                                                  | 146                                                                                     | Network layout and analysis                                                             | BIND, MIPS, BioGRID, MINT                                                                |
| B: Examples of pathway resources                                                        | B: Examples of pathway resources                                                        | B: Examples of pathway resources                                                        | B: Examples of pathway resources                                                         |
| Database                                                                                | Reference                                                                               | Link to database                                                                        | Software tool used for data access or data mining                                        |
| Reactome                                                                                | 147                                                                                     | http://reactome.org                                                                     | Sky Painter                                                                              |
| BioCarta                                                                                | 148                                                                                     | http://biocarta.com                                                                     | Pathway Explorer, Gene Set Enrichment Analysis, Pathway Miner                            |
| BioCyc                                                                                  | 149                                                                                     | http://biocyc.org                                                                       | Pathway Tools, Mouse Genome Informatics                                                  |
| KEGG Pathways                                                                           | 150                                                                                     | http://www.genome.jp/kegg                                                               | KEGG Pathway Mapping tool, Pathway Explorer, Gene Set Enrichment Analysis, Pathway Miner |
| PANTHER                                                                                 | 151                                                                                     | http://pantherdb.org                                                                    | PANTHER Gene Expression tools                                                            |
| GenMAPP/ WikiPathways                                                                   | 152                                                                                     | http://wikipathways.org                                                                 | PathVisio, Pathway Explorer, Pathway Miner                                               |
| Lipid MAPS                                                                              | 153                                                                                     | http://lipidmaps.org                                                                    | VANTED, Pathway Editor                                                                   |
| C: Examples of molecular interaction resources                                          | C: Examples of molecular interaction resources                                          | C: Examples of molecular interaction resources                                          | C: Examples of molecular interaction resources                                           |
| Database                                                                                | Reference                                                                               | Link to database                                                                        | Area(s) of focus                                                                         |
| Pathway Commons                                                                         | 154                                                                                     | http://pathwaycommons.org                                                               | Meta-database of protein interactions                                                    |
| IntAct                                                                                  | 155                                                                                     | http://www.ebi.ac.uk/intact                                                             | Protein-protein interactions                                                             |
| HPRD                                                                                    | 156                                                                                     | http://hprd.org                                                                         | Protein-protein interactions                                                             |
| MINT                                                                                    | 157                                                                                     | http://mint.bio.uniroma2.it                                                             | Protein-protein interactions                                                             |
| BioGRID                                                                                 | 158                                                                                     | http://thebiogrid.org                                                                   | Protein-protein and genetic interactions                                                 |
| InnateDB                                                                                | 143                                                                                     | http://innatedb.ca                                                                      | Innate immunity pathways and interactions                                                |
| D: Examples of omics data resources relevant to the systems analysis of innate immunity | D: Examples of omics data resources relevant to the systems analysis of innate immunity | D: Examples of omics data resources relevant to the systems analysis of innate immunity | D: Examples of omics data resources relevant to the systems analysis of innate immunity  |
| Database                                                                                | Reference                                                                               | Link to database                                                                        | Area(s) of focus                                                                         |
| GEO                                                                                     | 159                                                                                     | http://ncbi.nlm.nih.gov/geo                                                             | Omics data repository                                                                    |
| ArrayExpress                                                                            | 160                                                                                     | http://www.ebi.ac.uk/arrayexpress                                                       | Omics data repository                                                                    |
| ImmGen                                                                                  | 120                                                                                     | http://immgen.org                                                                       | Atlas of mouse immune cell transcriptomes                                                |
| Immport                                                                                 | n/a                                                                                     | http://immport.niaid.nih.gov                                                            | Immunological data warehouse                                                             |

key players and interactions are largely known in advance (reviewed in 2) to larger-scale approaches tailored to inferring networks and interactions from omics data sets (several examples are shown in Table 1A and are reviewed in References 3-5). The latter approach can be especially powerful when the data are integrated with known molecular interactions, which may be in the form of systematically curated pathways ( Table 1B ) or comprehensive protein-protein or protein-DNA interactions ( Table 1C ), or when the data are integrated with and cross-compared to existing

relevant data sets (examples of omics data resources are given in Table 1D ). The reason for this is obvious: Biological systems are extraordinarily complex, and no matter the scale of the omics data sets, network inference methods will always be data-limited. Any help we can give the inference algorithms by constraining the universe of possible networks to those that contain established interactions will ensure that the data are spent discovering something new, rather than rediscovering what is already known. The networks resulting from modeling analyses suggest specific hypotheses that can be tested experimentally by selectively perturbing, at the molecular level, the factors that are predicted to play central roles. Results of the experimental perturbations, whether they confirm the hypothesis or not, allow the model to be refined, thereby creating an iterative cycle between computation and analysis that is at the heart of systems biology.

## Systems Analysis of Innate Immunity

The number of cell types, their diverse cellular states, and the varied dynamic scales of responses are among the many factors that make holistic analysis of the innate immune system a daunting task. Fortunately, among biological systems, the immune system is nevertheless well suited for comprehensive analysis. Immune cells circulate in various functional states during an immune response and therefore may be readily isolated. This accessibility allows exhaustive profiling of the molecular properties of diverse cellular subsets over the course of actual immune responses in humans. Additionally, the function of the immune system overall (e.g., fighting infection) and the function of specific cell subsets [e.g., killing of virus-infected cells by cytotoxic T lymphocytes (CTLs)] are definable; it is therefore possible to anchor molecular measurements within a physiologically relevant functional context. Finally, many aspects of the human immune response may be modeled in animals and/or in vitro, thereby allowing network perturbations to be made at a range of scales, from genetic to pharmacological. As mentioned above, these perturbations are central to systems biology and will allow the development of predictive models that can be used to design rational interventions to modulate the immune response. Although there are certainly caveats to the above claims, immune cells are clearly more accessible than neurons deep within the prefrontal cortex, and the relationship between the function of a given CTL and the overall immune response is more straightforward than the relationship between the function of a given neuron and consciousness.

In this review, we describe a number of case studies in which systems analysis techniques have been successfully applied to reveal new components and properties of the innate immune response. We first describe efforts for deciphering regulatory networks controlled by Toll-like receptors (TLRs) in innate immune cells. We follow with a detailed survey of systems efforts aimed at understanding inflammatory mechanisms of influenza pathogenesis. We then describe how the tools of systems biology can be used to gain an understanding of the molecular and cellular interactions that govern vaccine responses. Due to space constraints, we unfortunately are unable to provide an overview of many excellent studies that apply systems approaches to achieve greater understanding of the adaptive immune system outside the context of vaccination. TheseincluderegulatorynetworkinferenceforTh17development(6),exhaustivecharacterization of the phenotypic diversity of virus-specific CD8 + T cells (7), and comprehensive regulatory epigenomic analysis of T cell development (8).

## SYSTEMS ANALYSIS OF INNATE IMMUNITY: CASE STUDIES

Therecognition and phagocytosis of pathogens and the presentation of pathogen-derived antigens by cells of the innate immune system are emergent properties that arise from the concerted action of signaling and gene regulatory networks. These interactions ideally result in effective host defense

·

·

but can lead to either impaired resistance or inflammatory disease when perturbed. Integrative systems biology approaches are perfectly suited for deciphering these networks, especially given that many of these functional responses can be recapitulated in vitro using cells expanded from the bone marrow [bone marrow-derived macrophages (BMDMs) and bone marrow-derived dendritic cells (BMDCs)].

The majority of systems analyses of innate immunity to date involve BMDM or BMDC responses to the activation of TLR4. TLR4 is unique among TLRs in that it activates both the MyD88andTRIFpathways, resulting in more complex inflammatory responses than other TLRs that signal exclusively through a single adaptor (9). Nevertheless, analysis of a single receptor, no matter how comprehensive, is insufficient to gain a holistic understanding of innate immunity. Macrophages and dendritic cells (DCs) detect pathogens by recognizing microbial components through distinct pattern-recognition receptor (PRR) families, which include the TLRs, nucleotide binding and oligomerization domain-like receptors (NLRs), C-type lectin receptors, and the retinoic acid-inducible gene I-like receptors (10), as well as nearly a dozen putative sensors of intracellular DNA (11). Most studies have used purified PRR agonists, but these do not occur in nature. Rather, during a natural infection, pathogens present a cocktail of molecules that activate diverse PRRs (12, 13). Cross talk between PRRs may enable macrophages and DCs to carry out multi-parameter analysis, which permits far greater accuracy in the determination of the threat. For example, simultaneous recognition of fungal zymosan particles by TLR2 and the C-type lectin Dectin-1 leads to synergistic inflammatory cytokine induction, reactive oxygen species production, and arachidonic acid metabolism (14).

Signaling cross talk also occurs between PRRs and the numerous phagocytic receptors expressed by innate immune cells. For example, priming macrophages with LPS (ligand for TLR4) results in synergistic arachidonic acid metabolite production upon phagocytosis of zymosan particles (15). Similarly, cross talk between PRRs and phagocytic receptors can be inferred from the observation that phagosomal TLR agonists enhance MHC class II-mediated presentation of antigens (16). Thus, functional macrophage and DC responses to pathogens result from complex interactions within and between PRR, phagocytic, and other pathways.

## Identification and Definition of Regulatory Circuits that Control Innate Immunity

The extraordinary advances in genomics, proteomics, lipidomics, and metabolomics have set the stage for the unbiased reconstruction of the genetic and molecular networks that underlie the innate immune response. A comprehensive understanding of these networks will not only lead to a much deeper understanding of the basic tenets of the immune system but also set the stage for rational vaccine design and the discovery of drugs and diagnostics.

Current approaches have led to systems analysis at a wide variety of scales, all presenting unique opportunities and challenges, and we provide examples from each scale below. Because each analysis involves different aspects of unbiased omics profiling of the innate immune response, we categorize them in terms of the scale of the perturbations that are made in order to better understand the system. Small-scale perturbations involve targeted genetic alterations of specific regulatory subnetworks. Medium-scale perturbations involve systematic knockdown of dozens to hundreds of network components. Genome-wide RNAi in vitro screens (17) and in vivo forward genetics screens (18, 19) constitute large-scale perturbations. Due to space constraints, we do not describe strategies in detail and instead present computational strategies for discovering regulatory networks at a large scale. Small-scale innate immune network discovery leads to a finegrained, high-resolution map with significant predictive power. The disadvantage of this approach

is that it gives limited insight into the network as a whole. Large-scale network analysis leads to a coarse-grained map with limited predictive power but with global insight into the network. Medium-scale analysis falls between these extremes, providing intermediate granularity and predictive power. Small-scale networks are highly tractable to biological validation, whereas it is extremely difficult to validate large-scale networks at the bench.

Small-scale perturbation analysis of innate immune networks. In the paragraphs below, we describe several examples of systems analysis of small-scale regulatory networks that have been analyzed by genetic approaches.

Insulating the innate immune response from fluctuating inputs. Transcriptional programs are propagated by sequential cascades of transcription factors (20, 21). We have shown that stimulation of macrophages with LPS induces the sequential transcription of waves of transcription factors and their target genes (22, 23). We used a combination of mathematical modeling and biological experiments to predict and confirm the existence of a number of novel transcriptional networks involved in TLR4 activation. The power of this approach lies in its ability to identify, in an unbiased manner, the complex transcriptional and signaling networks that lead to the emergent properties underlying the diverse macrophage phenotypes. One of these networks contains the transcription factors NFκ B (C-Rel), ATF3, and C/EBP δ . Further analysis of this system revealed that NFκ B and C/EBP δ function in concert to establish a coherent feed-forward type I regulatory circuit that amplifies inflammatory cytokine production, whereas ATF3 attenuates the circuit by epigenetic means (23). This type of regulation may protect biological systems from unwanted responses to fluctuating inputs (24), which could enable the innate immune system to precisely regulate the balance between effective host defense and the harmful sequelae of the response that leads to inflammatory disease.

Defining antiviral circuits. Using similar strategies to those described above, we have also begun to dissect regulatory circuits that control antiviral responses in macrophages (25). We applied this approach to a set of expression data from macrophages stimulated with ligands for TLR2, TLR3, andTLR4[PAM3,Poly(I:C),andLPS,respectively].Usingtranscriptionfactormotifscanning(1), weidentified enrichment of predicted binding sites for the transcription factor FOXO3 in a module of genes preferentially upregulated by both LPS and Poly(I:C) (25). In response to Poly(I:C), many interferon regulatory factor 7 (IRF7)-dependent genes in the type I interferon pathway are strongly upregulated in Foxo3 - -/ macrophages relative to wild-type macrophages, suggesting that FOXO3 is a negative regulator of this network. ChIP-Seq experiments revealed that the transcription factor IRF7 is a direct target of FOXO3 and that expression of IRF7 is highly elevated in unstimulated Foxo3 - -/ macrophages. Additionally, expression of FOXO3 itself is repressed by type I interferons, completing a negative feedback loop ( Figure 3 a ). The model predicts that FOXO3 suppresses the IRF7-dependent antiviral response to curb the collateral damage associated with host defense, and thus the dynamic interplay between FOXO3, IRF7, and type I interferon may achieve a balance between host defense and rampant inflammation. We confirmed this hypothesis in vivo ( Figure 3 b ). Whereas intranasal vesicular stomatitis virus (VSV) infection of wild-type mice resulted in a low-grade inflammatory response and intermediate viral load two days after infection, Foxo3 - -/ mice had significantly decreased viral loads accompanied by pronounced neutrophil influx, hemorrhage, and tissue damage. In contrast, viral replication was not controlled in Irf7 - -/ mice. Thus, FOXO3 serves to fine-tune the antiviral response, balancing robust host defense against pathological inflammation (25).

·

·

Figure 3

<!-- image -->

<!-- image -->

The FOXO3/IRF7 regulatory network fine-tunes the antiviral response. ( a ) Model of FOXO3 regulation of IRF7-dependent gene expression and implications for fine-tuning of the antiviral response. ( b ) Hematoxylin and eosin staining of lung tissue sections from wild-type (WT), Foxo3 - -/ , and Irf7 - -/ mice 0, 2, and 5 days after intranasal infection with vesicular stomatitis virus serotype Indiana 105 plaque-forming units (p.f.u.). Data are from one experiment representative of three independent experiments ( n = 6 mice per group). (Figure adapted from Reference 25.)

Rapid pathway contextualization of novel innate immune regulators. Large databases of transcriptional profiles in innate immune cells can be employed to rapidly contextualize novel regulators within established networks. For example, we identified SHARPIN, a molecule not previously studied in the context of the innate immune system, as a potential regulator of TLR signaling in macrophages. By using a mouse with a spontaneous frameshift in the Sharpin gene (26) and integrating transcriptional measurements on macrophages derived from these mice with our database of expression profiles from both wild-type and mutant macrophages stimulated with TLR agonists, we determined the location of Sharpin within the TLR-signaling network. The effects of SHARPIN deficiency on TLR2-induced transcriptional responses mirrored those of panr2 , a point mutation in NEMO generated by the Beutler laboratory (27), suggesting that SHARPIN and NEMOinteract ( Figure 4 a ). This interaction was confirmed by biochemical analysis and was found to be abrogated by the panr2 mutation. These studies determined that SHARPIN mediates a subset of NEMO-dependent, NFκ B-activated genes, including the critical Th1-skewing factor IL-12 (28) ( Figure 4 b ).

Fold impairment

<!-- image -->

Fold enhancement

<!-- image -->

## Figure 4

Contextualizing Sharpin within the NFκ B pathway by comparative analysis of mutant transcriptomes. ( a ) The heat map shows the effects of numerous mutations on 284 genes robustly induced by Pam3 (12 h) in macrophages. Blue indicates impaired gene induction compared to wild type, whereas pink indicates enhanced gene induction compared to wild type. The mutant with the most similar responses to Sharpin cpdm was Nemo panr2 (correlation coefficient = 0.82). ( b ) SHARPIN is an essential adaptor downstream of the branch point defined by the panr2 mutation in NEMO. The signaling responses most strongly impaired by SHARPIN deficiency and NEMO L153P ( panr2 ) are the phosphorylation of p105 and ERK, suggesting that p105 I κ B activity and Tpl2 sequestration are dominant regulators of Toll-like receptor 2 (TLR2)-induced proinflammatory cytokine expression ( left ). The greater deficiency in signaling and proinflammatory cytokine induction observed in panr2 compared with cpdm macrophages may result from SHARPIN-independent interactions between NEMO and the SHARPIN paralog RBCK1, which are also abrogated by NEMO L153P. TLR2-induced I κ α B degradation, phosphorylation of p38 and c-Jun terminal kinase ( JNK), and Nfkbia gene induction were unimpaired in cpdm macrophages and panr2 mutant macrophages, implying the existence of a branch of NEMO-dependent I κ B kinase (IKK) and mitogen-activated protein kinase (MAPK) activity that proceeds independently of SHARPIN and NEMO L153 ( right ). (Figure adapted from Reference 28.)

Medium-scale perturbation analysis of innate immune networks. In the following paragraphs, we describe several medium-throughput perturbation analyses of innate immune networks.

Construction of a TLR-regulated transcriptional network. Regev and colleagues (29) identified 125 candidate regulators predicted to act on 118 target genes within TLR-activated DCs. Each regulator was then systematically perturbed using short-hairpin RNAs (shRNAs). The effects from the perturbation on target gene responses to TLR activation were analyzed to construct a regulatory network. This work represented a major technological accomplishment not only because knocking down genes in primary innate immune cells is notoriously difficult but also because the researchers' use of the nCounter system (30) allowed for highly parallel high-sensitivity transcript

·

·

Figure 5

<!-- image -->

## Hierarchical organization of transcription factors

<!-- image -->

High-throughput chromatin immunoprecipitation coupled to next generation sequencing (ChIP-Seq) systematically maps proteinDNAinteractions. Systematically profiling protein-DNA interactions in the innate immune response facilitates the discovery of the hierarchical genome-wide organization of transcription factors. Amit and colleagues (31) discovered that in LPS-stimulated BMDCs, transcription factors can function as differentiation regulators, priming factors for transcriptional induction, and regulators of specific gene programs. (Reproduced with permission from Reference 31.)

measurements to be made with greatly reduced material and manipulation requirements. Besides confirming well-established regulators of TLR responses, these authors identified many putative signaling circuitries that warrant further investigation. For example, an intriguing prediction is that TLR4-induced CBX4, a SUMO E3 ligase, and DNMT3A, a DNA methyltransferase, form a coherent feed-forward loop to negatively regulate the induction of IFNβ . Similarly, the hypothesis that established regulators of the cell cycle and circadian rhythms (RBL1, JUN, RB, E2F5, E2F8, NMI, FUS, and TIMELESS) may be co-opted to regulate specific antiviral-associated gene groups is unexpected and compelling. This study underscores the power of systems analysis to identify novel interactions that would likely have been overlooked using conventional approaches.

Many of the interactions identified in this study (29) may be indirect, given that they were identified through knockdown analysis and not direct measurement of transcription factors binding to target gene promoters. Recently, the same team has begun to address this issue by systematically evaluating transcription factor binding in the same system (31) ( Figure 5 ). They used high-throughput ChIP-Seq to measure genome-wide localization of 25 transcription factors,

3 epigenetic regulators, and Pol II at 4 time points after LPS stimulation in BMDCs. Ultimately, coupling unbiased perturbation of transcription factors (29) with genome-wide location analysis of those same factors (31) will be a powerful approach for exhaustively illuminating the gene regulatory networks controlling the innate immune response.

Epigenetic regulation of innate immunity. The application of ChIP-Seq to thoroughly characterize changes in the histone modification state that occur over the course of the innate immune response enables a detailed mechanistic understanding of how epigenetics controls inflammation (32). Recently, Glass and colleagues (33) performed a detailed epigenomic analysis of TLR4-induced gene expression in macrophages. They observed that induction of most TLRresponsive genes proceeds from transcriptional enhancers that are established by macrophage lineage-defining transcription factors such as PU.1 and C/EBP α . They also gathered strong evidence for de novo priming of thousands of enhancers in response to TLR4 activation, with the induced enhancers accounting for 10% of overall gene induction. Further analysis revealed that transcription from the induced enhancers depended on the methylation state of histone H3K4, whichinturndependedonenhancertranscription.Bycombiningsystematicknockdownofhistone methyltransferases with H3K4 ChIP-Seq, they discovered an essential role for the mixed-lineage leukemia family-particularly MLL1, MLL2/4, and MLL3-in the process.

## Novel virus-sensing circuits discovered by systematic analysis of innate signaling networks.

Hacohen and colleagues (34) used transcriptomic data from TLR-stimulated BMDCs to implicate 280 genes as components of the TLR-signaling pathway. Seventeen candidates were perturbed with shRNAs, and CRKL, a tyrosine kinase adaptor, was shown to regulate TLR signaling. SILAC-based quantitative phosphoproteomic analysis demonstrated LPS-induced CRKL phosphorylation. Transcriptional profiling, perturbation analysis, and phosphoproteomics suggested that CRKL modulates the JNK-mediated antiviral signaling pathway. Additionally, network perturbation with shRNAs and small molecule inhibitors demonstrated that Polo-like kinases 2 and 4 were novel and critical activators of the antiviral response.

## Identification of novel antiviral functions by targeted overexpression studies. Type I interferon

is a cornerstone of the antiviral response. Activation of the interferon-signaling cascade leads to antiviral effects mediated by proteins encoded by the large family of interferon-stimulated genes (ISGs). Nearly 400 ISGs have been identified, but most of the effector mechanisms mediated by these genes are unknown (35). Using an overexpression system to conduct a large-scale antiviral screen, 389 human ISGs were expressed in various cell lines to determine their antiviral activities against a panel of important human and animal viruses (36). This approach led to the identification of broad-acting as well as specifically targeted antiviral effectors. Moreover, additive antiviral effects were experimentally confirmed by combinatorial expression of ISG pairs. Interestingly, many of the putative effectors were capable of translational inhibition (36).

A similar comprehensive overexpression study was conducted to investigate the role of tripartite interaction motif (TRIM) proteins in modulating the antiviral response (37). TRIM proteins have been implicated in many biological processes, including cell differentiation, apoptosis, transcriptional regulation and signaling pathways, and viral restriction (38, 39). Seventy-five human TRIMsandmanyoftheir splice variants were expressed in HEK293T cells, and a significant number of the family members were found to enhance innate immune responses by inducing IFNβ protein and interferon-stimulated response element (ISRE) and NFκ B promoter activities (37). A functional screen further showed that some of the TRIM proteins enhanced the production of

·

·

antiviral cytokines and restricted VSV replication in vitro. A complementary analysis employing shRNAs to deplete individual TRIMs concordantly diminished IFNβ induction.

Enhancing network analysis by incorporation of protein-protein interaction data. Production of type I interferon during the innate immune response involves a complex network of regulatory pathways consisting of a variety of paracrine and autocrine interactions. Using a comprehensive proteomic approach to assess the interactome of 58 components of the type I interferon network, 260 interacting proteins were identified in a framework of 401 protein-protein interactions (40). Toconfirmthe functional consequences of these interactions, gain-of-function and RNAi analyses were conducted to assess their effects on transcription and antiviral effects. These approaches led to the discovery of a novel ubiquitin E3 ligase that proved to be a positive regulator of antiviral responses (40).

Identification of novel regulators of the innate immune response using an integrative systems approach. Innate immune recognition of cytosolic DNA is crucial for the detection of DNA viruses and retroviruses (RNA viruses that replicate via a DNA intermediate). Precise control of the process is critical because aberrant DNA sensing can lead to autoimmune disease. Stimulator of interferon genes (STING), absent in melanoma-2 (AIM2)-like receptors, and several DExD/H box RNAhelicases have been identified as important components for sensing cytosolic DNA; however, the underlying mechanism by which they function is poorly understood. To extend the cytosolic DNA-sensing network, 809 candidates selected after genomic (transcriptome) and proteomic (protein-protein- and protein-DNA-interacting partners) analysis were perturbed using RNAi (41). The candidate list also included genes identified by domain-based analyses (phosphatases and deubiquitinases). Fifteen genes, when knocked down, decreased interferon signaling upon DNA transfection. ABCF1, a unique cytosolic and ER-localized member of the ATP-binding cassette transporters, was identified as a critical node in the DNA-sensing network. The putative network was validated using RNAi and small molecule inhibitors.

Large-scale analysis of innate immune networks. In this final section on systems analysis of networks in innate immunity, we present several examples of large-scale transcriptional and metabolic networks.

Genome-scale transcriptional and signaling networks. A number of groups have approached inference of transcription and signaling networks controlling TLR responses from a global scale, and three examples are presented here. First, Elkon et al. (42) combined computational analysis of expression profiles and cis -regulatory promoter sequences to dissect the TLR-induced transcriptional program. Their model demonstrated that NFκ B mainly regulates an early-induced and sustained response, whereas the ISRE element functions primarily in the induction of a delayed wave. The model further suggested that co-occurrence of the NFκ B and ISRE elements in promoters endows the targets with enhanced responsiveness. Second, Ramsey et al. (1) combined network analysis with systems-level network perturbations to comprehensively deconstruct the TLR transcriptional regulatory pathways, which involve the regulation of nearly 2,000 genes. Several associations in the resulting inferred network were validated using targeted ChIP-on-chip experiments. The network incorporates known regulators and gives insight into the transcriptional control of macrophage activation. Recent analysis has shown how inclusion of epigenetic data sets within the network inference methodology can lead to improved predictions of regulatory interactions (43). Third, Li et al. (44) used global data sets to reconstruct the human TLR-signaling network, which contains kinases, phosphatases, and other associated proteins that mediate the

signaling cascade along with a delineation of their associated chemical reactions. A computational framework based on the methods of large-scale convex analysis was developed and applied to this network to characterize input-output relationships. The analysis ranked potential inhibitory targets within the TLR pathway according to their specificity and potency.

Genome-scale metabolic modeling. Palsson and colleagues (45) used genome-scale modeling and multiple omics data sets (transcriptomics, proteomics, and metabolomics) to assess metabolic features critical to the activation of the macrophage-like cell line RAW 264.7. A number of metabolites were implicated in macrophage regulation, including glucose and arginine (activation) and tryptophan (suppression). A suppressive role for de novo nucleotide synthesis was also found. Bordbar et al. (46) also integrated a genome-scale macrophage metabolic model with a Mycobacterium tuberculosis model to simulate the metabolic changes that occur during infection. High-throughput data from infected macrophages were mapped onto the host-pathogen network and revealed three distinct pathological states.

## Systems Analysis of Innate Immune Pathways at the Single-Cell Level

The network analyses described above were carried out in populations of cells. However, individual cells, even when isolated from an apparently homogeneous population, exhibit significant differences in gene expression, protein concentration, and phenotypic output. New technologies, including single-cell genomics (47, 48), mass cytometry (a combination of flow cytometry and mass spectrometry, also known as CyTOF) (49, 50), and microfluidic devices (51), have enabled single-cell systems-level analysis which will be critical for comprehensive analyses that address cellintrinsic stochasticity and noise. An extensive consideration of this topic is provided in Reference 52.

Single-cell transcriptomics. Shalek et al. (53) uncovered surprising heterogeneity in transcriptional responses in individual LPS-stimulated DCs. They also observed a strong bimodality in expression levels and distinct splicing isoforms among many of the highly expressed genes. Although gene expression was highly variable, some functional modules, including a module of antiviral genes, were strongly coregulated (53).

Single-cell measurement of proteins using microfluidic devices. We have designed and constructed a microfluidic device capable of quantifying cytokines secreted from single cells. The device, which can be multiplexed, has been used to quantify a range of cytokines secreted by LPS-treated macrophages (51). Similar microfluidic devices have also been used to measure polyfunctionality in T cells (54) and to examine the relationship between cytokine release and cytolytic activity in HIV-specific T cells (55).

Single-cell protein measurement using mass cytometry. Mass cytometry can measure dozens of proteins and posttranslational protein modifications in single cells at a rate of 1,000 cells per second (49, 50). Nolan and colleagues (56) have applied mass cytometry to increase the throughput and dimensionality of the analysis of signaling networks in cytokine-stimulated peripheral blood mononuclear cells (PBMCs) at the single-cell level. Experiments conducted with various kinase inhibitors resulted in the quantification of 18,816 phosphorylation levels in 1,344 cell populations from 96 multiplexed samples.

·

·

## SYSTEMS ANALYSIS OF INFECTIOUS DISEASE: INFLUENZA AS A CASE STUDY

Given space constraints, we focus on influenza as an example of recent work that has applied systems analysis of the innate immune response within the context of microbial infection. Other examples include M. tuberculosis (57, 58), HIV/SIV (59-61), Francisella tularensis (62), Salmonella (63), and respiratory syncytial virus (64).

Theinfluenza virus is an enveloped, negative-sense, single-stranded RNA virus. The virus consists of eight genomic segments, encoding up to thirteen viral proteins (65). Seasonal or pandemic viruses can cause serious disease, often leading to pneumonia. A major reason why influenza virus is such a dangerous human pathogen is its ability to acquire mutations. Antigenic drift-small mutations in the segments encoding two surface proteins, hemagglutinin and neuraminidase-allows the virus to evade the immune system. Antigenic shift-the reassortment of segments between multiple virus strains infecting the same cell-can drastically change the pathogenicity as well as host specificity of the reassorted virus. Because of this genetic mutability, different strains of influenza virus can have varying levels of pathogenicity. For example, the notorious and highly pathogenic 1918 H1N1 pandemic strain, which causes devastating tissue damage that often results in death, is believed to induce an exacerbated immune response called a cytokine storm (66, 67). Seasonal influenza strains typically have low mortality rates; however, the recently emerging avian strains H5N1 and H7N9 appear to be highly pathogenic. Avian influenza strains usually lack the ability to transmit efficiently between humans. Recently, two independent investigations have discovered that as few as four mutations are sufficient to improve the transmissibility of an H5N1 virus in a ferret infection model (68, 69). Vaccines are effective in providing protection against influenza; however, their production is time-consuming and the prediction of the dominant seasonal strain is imperfect. The emergence of novel influenza strains further exposes the inability of vaccines to prevent epidemics and pandemics. Few antiviral drugs are effective against influenza, and drug-resistant strains are frequently observed. Additional therapeutics are urgently needed. Systems biology approaches are useful strategies to understand the mechanisms behind the pathogenicity of influenza virus, particularly the dysregulation of the host response. These network-based approaches can reveal novel therapeutic targets that can be modulated to prevent adverse clinical outcomes.

## Transcriptional and Proteomic Analysis of Influenza Infection

Transcriptional analyses of the host response to the influenza virus have been conducted both in vitro and in vivo. Comparative analyses of the host response to the highly pathogenic 1918 pandemic strain (70) and the H5N1 avian strain (71) yielded insights into the differences in transcriptional responses to these strains during lethal influenza infections. These whole-tissue transcriptomic analyses demonstrated that although the type I interferon transcriptional profiles of the 1918 and H5N1 viral strains were similar, genes related to the inflammatory response and cell death (including key components of the inflammasome, NLRP3, and IL-1 β ) were highly induced by the 1918 virus but were downregulated by the H5N1 strain (71). Similarly, we identified distinct transcriptional programs activated in the upper and lower airways by influenza strains of varying pathogenicity (72). Moreover, we determined that the induction of inflammationrelated genes during swine-origin pandemic H1N1 infection in vitro is significantly delayed in comparisontoinfections with its parental strains (P. Dash, C.J. Sanders, A.H. Diercks, P. Askovich, J.A. Rutigliano, et al., manuscript submitted).

In another study, modules of genes associated with lethal infection were identified by combining whole-tissue transcriptional profiling with experimental variation of the dosage and pathogenesis

of influenza strains (73) ( Figure 6 ). Analysis of sorted cell transcriptional responses revealed that lethal infection was driven by a feed-forward circuit of cytokine expression involving recruitment of neutrophils to the infected tissue. Together, the above studies (i.e., 70-73) provide insights into the differential transcriptional programs that drive the biological consequences of influenza infection.

Recent studies have also implicated posttranscriptional gene regulation in influenza pathogenesis. Profiling of miRNA expression during infection of mice with the 1918 influenza virus identified a distinct set of miRNAs that regulate expression of genes related to inflammatory responses or cell death (74), a result concordant with the mRNA transcriptional response. RNA-Seq analysis provided additional information on noncoding RNA profiles during SARS and influenza virus infection, better defining regulatory interactions between small RNAs and mRNAs (75). In similar work, our analysis of miRNA expression in influenza-infected DCs revealed miR-451 to be a modulator of proinflammatory cytokine secretion that negatively regulates the YWHAZ-ZFP36 axis (76).

Employingproteomics for the analysis of virus-induced innate immune responses at the protein level, in conjunction with the transcriptomic analyses described above, promises to more fully elucidate the molecular mechanisms of influenza pathogenesis. Mass spectrometry analysis of macaque tissues post-influenza infection revealed that increased expression of ISGs and RNA binding/sensing proteins is associated with infection by the highly pathogenic 1918 virus (77, 78). Proteomics has also been employed to analyze the responses of airway epithelial cells (79) and macrophages (80, 81) to influenza infection in vitro. Interestingly, analysis of influenza-infected human macrophage subcellular proteomes and secretomes demonstrated that cytoplasmic leakage of cathepsin B induces inflammasome activation and apoptosis (80). These cellular processes, together with the eventual secretion of danger-associated molecular patterns (DAMPs), can have significant immunological consequences that influence the overall host response.

The application of proteomics to interrogate the host-virus interactome has yielded many additional insights, including identification of intriguing interactions between viral proteins and proteins in the NFκ B, Wnt, mitogen-activated protein kinase (MAPK), RNA binding, and apoptosis pathways (82). To ensure identification of physiologically relevant host-virus interactions by proteomics, we have used a replication-competent, tagged influenza virus to probe the host-pathogen interactome during in vitro influenza infection (S.M. Kaiser, J. Noonan, R. Podyminogin, A.H. Diercks, P. Askovich &amp; A. Aderem, manuscript in preparation). We have focused on the interactome of the multifunctional nonstructural protein 1 (NS1) and have identified cellular proteins differentially interacting with NS1 proteins from multiple H1N1 and H5N1 viral strains with varying pathogenicity and host-range restriction. The relevance of these interactions to pathogenicity is currently being tested.

## The Role of Lipid Mediators in Influenza Pathogenesis

In addition to transcripts and proteins, bioactive lipid mediators play critical roles in the induction and resolution of inflammation associated with influenza pathogenesis. We (83) and others (84) have employed the emerging technology of lipidomics to probe this response. Given that lipidomics is possibly the least well characterized of the omics technologies, we provide a description of it prior to describing the results of lipidomic analyses of influenza pathogenesis.

Lipids and inflammation. Upon phospholipase activation, arachidonic acid is released from the plasma membrane and metabolized by many different families of enzymes to produce a large array of eicosanoids with diverse physiological functions (85, 86). There are three major arachidonic acid metabolic pathways: ( a ) the cyclooxygenase pathway, which produces prostaglandins and thromboxanes; ( ) the lipoxygenase pathway (LOX), which produces leukotrienes, numerous b hydroperoxy and hydroxylated fatty acids, hepoxilins, and lipoxins; and ( ) the cytochrome P450 c

·

·

Figure 6

<!-- image -->

<!-- image -->

Transcriptional analysis reveals a critical role for neutrophil recruitment in driving lethal influenza infection. Germain and colleagues (73) performed detailed comparative transcriptional analysis of lung tissue during influenza infection and identified a transcriptional module 'A-8' that was strongly associated with lethality. Analysis of the A-8 module implicated inflammatory pathways and neutrophils in the pathogenesis of lethal influenza. ( a ) Inflammatory network indicating signaling components elevated in module A-8 ( outline red ) and preferential constitutive ( green ) and/or inducible ( yellow ) expression in neutrophils. ( b ) Neutrophil samples exhibit highest expression of downstream genes from inflammatory signaling cascades. (Reproduced with permission from Reference 73.)

pathway, which produces epoxy and dihydroxy derivatives of arachidonic acid. Prostaglandins and leukotrienes have long been known to induce inflammation by modulating vasculature permeability and stimulating immune cell infiltration to the site of infection. Furthermore, many of the metabolic enzymes associated with the arachidonic acid pathway can act on related unsaturated fatty acid precursors, such as linoleic and linolenic acids, to produce potent bioactive lipid mediators.

Although the induction of inflammation is essential for the innate immune system to control microbial assaults, the failure to resolve inflammation can lead to chronic disease or severe tissue damage. Recently, docosahexaenoic acid- and eicosapentaenoic acid-derived lipid mediators such as resolvins, protectins, and maresins, as well as arachidonic acid-derived lipoxins, were discovered to promote the resolution of inflammation (87). These anti-inflammatory/proresolving signaling molecules can stop the further infiltration of immune cells, prompt nonphlogistic phagocytosis of apoptotic neutrophils, and stimulate the tissue to return to homeostasis (88).

Lipidomics. Lipid metabolic networks have many inherent complexities that confound traditional analytical approaches. These include multiple fatty acid precursors, large enzyme families, and vast numbers of lipid species. Furthermore, cross talk between distinct metabolic pathways exists because multiple enzymes can act on a single substrate, and conversely, multiple substrates can be modified by the same enzyme (85, 86, 89). This confounds the determination of causal relationships by genetic perturbations because depletion of an enzyme may shunt its substrate through another pathway (89, 90). Furthermore, many lipid mediators are susceptible to degradation and modifications. Lastly, some lipid mediators are produced by transcellular biosynthesis, a process in which multiple cell types contribute distinct enzymes and substrates to generate the final lipid mediator (88, 89). These complexities make a compelling case for the use of systems approaches.

Recent advances in mass spectrometry and liquid chromatography, including multiple reaction monitoring, have enabled the isolation and identification of many individual lipid species in parallel (91). In addition, multiplexed quantification can be achieved using the stable isotope dilution method. Using this approach, over 100 lipid metabolites can be simultaneously measured within a single analytical run.

Lipidomic analysis of influenza infection. The high throughput and resolving power of lipidomics allows for systems analysis of the lipidome during microbial infection. Two recent studies have explored the behavior of the lipidome during influenza infection. In the first study, Morita et al. (84) demonstrated that, early in infection, several proresolving lipid mediators, including 12-HETE, 15-HETE, 17-HDoHE, and protectin D1 (PD1), were less abundant in infected animals compared to mock-infected controls. Interestingly, these mediators inhibited influenza infection in vitro. Comparative lipidomic analysis between animals infected with the 2009 H1N1 pandemic strain, a highly pathogenic H5N1 strain, and its avirulent variant indicated that production of PD1 is suppressed only in H5N1 infection (84). Furthermore, exogenous PD1 lowered the mortality rate of animals during a lethal influenza infection. PD1 prevented viral RNA export from the nucleus, a crucial step in viral replication, by disrupting viral RNA binding to NXF1, an mRNA transporter. This observation is supported by two high-throughput studies: A genome-wideRNAiscreenshowedthatNXF1isessentialforviralreplication(92),andaproteomic study identified it as a cellular interacting partner of the influenza viral polymerase complex (93).

In the second study, we compared the lipidomic, transcriptomic, and cytokine profiles measured during the course of influenza infection with the high-pathogenicity PR8 strain to those measured during infection with the low-pathogenicity X31 strain (83). The lipidomic profile of X31 consisted of early proinflammatory responses followed by later anti-inflammatory responses.

Figure 7

<!-- image -->

Lipidomic analysis of influenza infection. ( a ) Lipoxygenase (LOX) metabolism pathway of arachidonic acid. Rectangular boxes represent the enzyme catalyzing the reaction. Circles represent the lipid mediators within the pathway. ( b ) Stacked bar graph representing the percentages of 5-, 15-, 12-, or 8-LOX-derived metabolites of all lipoxygenase-derived metabolites in mouse and human samples. Each vertical line represents data from a single sample ( n = 8-11 per time point or group). (Figure adapted from Reference 83.)

This sequence was dysregulated during PR8 infection, wherein the pro- and anti-inflammatory responses overlapped. This dysregulated lipidomic response was recapitulated in nasopharyngeal lavages from human clinical samples obtained during the 2009-2011 influenza seasons. By further dissecting lipoxygenase pathway metabolites-which include both proinflammatory and anti-inflammatory/proresolving lipid mediators ( Figure 7 a )-we determined that the

proinflammatory 5-LOX metabolites correlated with the pathogenic phase of infection, whereas anti-inflammatory 12/15-LOX metabolites were associated with the resolution phase. Once again, these data were recapitulated in humans ( Figure 7 b ), demonstrating the power of holistic analysis of model systems for understanding human disease processes.

## SYSTEMS VACCINOLOGY

Systems-level analyses of vaccination promise to yield insights that will vastly improve vaccine development (reviewed in 94-99). This new field of systems vaccinology weds holistic analysis of innate and adaptive immunity within an engineering framework to enable rational design of new vaccines that elicit tailored immune responses to protect against targeted pathogens. Such an approach is necessary for several reasons, including: ( a ) the pathogens causing diseases such as AIDS, malaria, and tuberculosis have proven too complex to be overcome by simpler classical methods [such as the empirically developed, antibody-inducing vaccines that have succeeded in preventing many other diseases (100)]; ( b ) the efficacy of any vaccination in a human population depends on complex interactions between genetic, molecular, and environmental factors; and ( c ) molecular responses to vaccines are complex, as they activate several innate immune pathways in parallel (101-103).

## Bridging Innate and Adaptive Immunity in the Context of Vaccines

A major objective of systems vaccinology is to discover relationships between the earliest measurable innate inflammatory responses to vaccination and the subsequent vaccine-induced adaptive immune responses (immunogenicity) and efficacy. It follows that a good starting point for systems vaccinology is the comprehensive analysis of the innate immune response to vaccination, and numerous technologies have been employed to make these measurements, including highthroughput serum analyte profiling, proteomics, and transcriptomics. These measurements are made early after vaccination (hours to days) and in relevant (i.e., lymph nodes) and/or accessible (i.e., peripheral blood) tissue compartments. In contrast, immunogenicity quantifies the magnitude and quality (i.e., breadth and skewing) of antigen-specific humoral (104) and T cell (105) responses and can be measured months to years after vaccination in both peripheral and effector (i.e., mucosal) tissue compartments. The quality of vaccine-induced adaptive immune responses can be further defined by employing multiparameter flow cytometry, transcriptomics, and other technologies to exhaustively characterize purified antigen-specific, vaccine-induced T cells (106) and B cells. As technologies for quantifying and characterizing antigen-specific adaptive immune responses continue to develop-including sequencing-based methods to define the repertoires of B and T cell receptors (107)-the measurable immunogenicity space becomes infinite, and a pressing need develops to identify which aspects of the adaptive immune response are relevant to vaccine-induced protection. In the absence of direct measurements of vaccine efficacy, analysis of patient subpopulations exhibiting distinct responses to natural infection [i.e., nonprogressors versus rapid progressors (108, 109)] may help to rank different aspects of pathogen-specific adaptive immune responses in terms of clinical importance for vaccination.

Withcomprehensive measures of the innate and adaptive immune responses and efficacy (when possible) of a given vaccine, it becomes possible to mine the data and generate hypotheses about molecular rules that causally relate these responses (110). This task is computationally challenging, given that it can involve evaluation of millions of combinations of innate immune response genes and the manner in which they impact hundreds of measurements of immunogenicity and/or efficacy across multiple time points. Appropriate adaptation of algorithms from the fields of machine learning and pattern recognition can address this problem. Promising approaches include

·

·

discriminant analysis via mixed integer programming (DAMIP) (111), an algorithm that was successfully applied in analyses of the yellow fever vaccine YF-17D (112) and seasonal influenza (113), and Elastic Net feature selection combined with logistic regression discrimination (114).

Ultimately, the rules linking innate to adaptive immunity in the context of vaccination can be harnessed to accelerate vaccine development in several ways. First, they will yield correlates or biomarkers of immunogenicity and protection. These biomarkers can be evaluated in the field as early measures for successful vaccination and to aid interpretation of clinical trials. Second, they will lead to hypotheses about the regulatory networks within cells that must be activated to induce the desired immune responses. Knowledge of these networks will guide the reengineering of vaccine regimens. Third, applying these approaches to candidate vaccines that induce adverse responses can similarly identify regulatory networks that may be suppressed to improve vaccine safety.

## Systems Vaccinology in Practice

Systems vaccinology analyses of the type described above can be carried out in a wide variety of experimental settings, ranging from analyses of clinical trial samples to animal models and in vitro studies, with each presenting unique opportunities and challenges (94). Whereas in vitro and murine systems enable detailed mechanistic analysis, vaccine efficacy generally cannot be assessed in these models, and mouse immune responses may not accurately reflect those of humans for a given pathogen or disease (115), although humanized mouse models continue to improve (116). In contrast, analysis of clinical trial samples directly probes the system of interest, but often the relevant vaccine responses can be measured only indirectly using surrogate tissues. For example, analyses of early blood transcriptome responses are often used as a surrogate for vaccine-induced innate immune activation (when lymph node profiling may more directly assess vaccine-relevant innate immunity), and immunogenicity is usually quantified in peripheral blood (when mucosa, liver, or other tissues may be appropriate for a given pathogen). Notwithstanding this limitation, surrogate innate response measurement by profiling whole blood or blood cell subset transcriptomes can be highly informative because it is a robust and convenient measure, meaning that data quality will often be high and the measurements reproducible. This approach also probes at least three relevant biological processes, all occurring in parallel, including ( a ) direct cell-intrinsic responses to the vaccine, ( b ) bystander responses to inflammatory mediators induced by the vaccine, and ( ) changes in the composition and activation states of circulating cells. c

Given that the blood transcriptome is an integrated measure of many distinct processes, computational strategies are required to interpret the data and generate mechanistic hypotheses. One approach is to employ a modular analysis framework (117) that deconvolutes complex transcriptional profiles into functionally interpretable patterns through the evaluation of combined expression responses of predefined disease, cell type, and stimulus-specific coexpressed gene groups. Another is to integrate the data with cell population measurements, which may be used to distinguish the blood transcriptome responses explained by the trafficking of specific populations from the responses that are likely to be cell intrinsic, allowing the generation of hypotheses about transcriptional responses within specific cellular subsets (118). Interpretation of blood and other mixed cell population transcriptomes is also facilitated by transcriptome compendia measured in isolated immune cell types. These compendia can be mined to identify genes that are robustly and preferentially expressed in specific immune cell lineages or subsets of lineages, and differential expression of these lineage-specific genes in whole blood may indicate trafficking of the associated cell type. Two of the most extensive immune cell transcriptome compendia are a collection of profiles of 38 purified human hematopoietic cell populations (119) and the ImmGen database (120), which includes profiles of 249 murine immune cell types.

## Case Studies

Analysis of blood samples from clinical trials in systems vaccinology studies has proven productive, yielding new hypotheses concerning mechanisms of action of numerous vaccines. The vaccine best studied in this manner is YF-17D, the gold standard vaccine for yellow fever (112, 121). Application of DAMIP identified innate response gene signatures that predict CD8 + T cell and neutralizing antibody responses with 90% and 100% accuracy, respectively (112).

We applied similar approaches to better understand innate immune responses induced by MRKAd5/HIV, the Step Study vaccine (122) ( Figure 8 ). Although this vaccine did not offer protection, it elicited high-magnitude CD8 + T cell responses to the HIV-1 inserts (123-125) and exerted selective pressure on infecting HIV-1 strains (126), but it unexpectedly appeared to enhance HIV acquisition in subgroups with baseline Ad5 seropositivity. MRKAd5/HIV robustly and rapidly triggered innate immune responses within 24 h postvaccination. Our analyses revealed that the innate immune responses of vaccinees with preexisting Ad5 neutralizing antibodies were strongly attenuated ( Figure 8 a b , ), suggesting that enhanced HIV acquisition in Ad5-seropositive subgroups in the Step Study may relate to the lack of an appropriate innate activation context. Unexpectedly, the innate immune response induced by MRKAd5/HIV greatly exceeded that induced by YF-17D ( Figure 8 c ). In spite of kinetic and signaling differences between MRKAd5/HIV and YF-17D, we were able to identify two transcripts [encoding cysteine-rich protein 3 (CRIP3) and neuropeptide B] with vaccine-induced expression responses that were consistently associated with impaired CD8 + Tcell responses ( Figure 8 d ), suggesting shared mechanisms linking innate immune stimulation and immunogenicity.

An analysis of healthy adults given influenza vaccines demonstrated how systems analysis of clinical trials leads to verifiable mechanistic insights (113). In subjects vaccinated with trivalent inactivated influenza vaccine (TIV), early molecular signatures correlated with-and could be used to accurately predict-antibody titers in two independent trials. In particular, expression of the kinase CaMKIV at day 3 was inversely correlated with antibody titers. This association was tested by vaccinating CaMKIV-deficient mice, which revealed enhanced TIV-induced antigen-specific antibody titers, functionally validating an immunogenicity signature and confirming a novel role for CaMKIV in the regulation of antibody responses.

In another study, systems analysis of the malaria vaccine RTS,S revealed that differential expression of genes in the immunoproteasome pathway may distinguish protected from nonprotected vaccinees (127). Other key systems vaccinology studies include analyses of the candidate HIV vaccine MVA-C (103), meta-analyses of antibody-inducing vaccines (128, 129), and analysis of adverse events induced by the smallpox vaccine (130).

## Systems Vaccinology Is an Iterative Process

Whereas biomarkers discovered through systems analysis of clinical trials will achieve practical utility as soon as they are validated in additional cohorts, the true power of systems vaccinology is to deliver mechanistic insights that can drive rational vaccine design. This is realized only when systems perturbation experiments are executed that directly test the mechanistic hypotheses generated from the biomarkers. Perturbations most relevant to validating molecular signatures include overexpression or knockdown (in vitro) or genetic ablation (murine in vivo) of the relevant genes, as demonstrated in analyses of the influenza vaccine (113). Such an approach is not feasible for clinical or even preclinical nonhuman primate trials, however. It is more practical in terms of eventual clinical application to combine vaccines with small molecule agonists and inhibitors that are specific for the networks implicated by the biomarkers. Such an approach has been used successfully in model systems (131, 132). Another approach is to include host modulatory components

·

·

Figure 8

<!-- image -->

c

<!-- image -->

<!-- image -->

Systems-level analysis of the Step Study HIV vaccine. ( a ) MRKAd5/HIV induces interferon response genes and represses lymphoid cell-associated genes. This effect is shown in a gene module (117) radar plot in which the axes indicate the average expression of specific functional gene modules ('M' followed by a number). Responses are attenuated in moderate Ad5 neutralizing antibody (nAb) titer ( green lines ) compared with low nAb titer ( black line ) volunteers. ( b ) As an example, induction of IP-10 (CXCL10) transcript by MRKAd5/HIV was attenuated in moderate Ad5 nAb titer ( dark green ) compared with low ( light green ) and zero ( gray ) titer volunteers. ( c ) MRKAd5/HIV induces transcriptional responses that involve more genes but are shorter-lived than YF-17D. ( d ) A subset of MRKAd5/HIV innate immune response genes (72 h postvaccination) are associated with HIV-specific CD8 + T cell responses (1 month postvaccination). For example, CRIP3 is inversely correlated with both MRKAd5/HIV- and YF-17D-induced CD8 + T cell responses. (Figure adapted from Reference 122.)

within the vaccines themselves. For example, the natural killer cell signaling adaptor EAT-2 (133) enhanced vaccine-induced CD8 + T cell responses when expressed as part of an adenoviral vaccine strategy (134). Consistent with that result, our systems analysis of the MRKAd5/HIV vaccine revealed a positive association between vaccine-induced expression of EAT-2 at early time points and the subsequent CD8 + T cell response (122). These two studies collectively suggest that it should be possible to employ systems vaccinology to identify critical nodes in the host response and then construct improved vaccines that specifically activate or suppress those nodes.

<!-- image -->

## Systems Vaccinology Opportunities in HIV, Malaria, and Tuberculosis

Vaccines for HIV, malaria, and tuberculosis are critically needed. Recent results from efficacy trial results for malaria suggest that an efficacious vaccine for this disease may soon be achieved (135, 136). Integrative systems analysis may help determine how to build upon these results to construct improved vaccines that are protective and yet may be deployed in resource-poor settings. Although a recent clinical trial of a candidate tuberculosis vaccine did not show protection (137), systems analysis of this trial may help fully elucidate novel avenues to pursue in future candidates. Finally, modest but positive results from recent clinical trials suggest that a vaccine for HIV may be achievable; the RV144 prime-boost Thai trial demonstrated low-level efficacy in terms of acquisition (138), and the MRKAd5/HIV Step Study vaccine exerted selective pressure on the breakthrough virus (126). Integrative analysis of these trials may identify additional leads to pursue in the next generation of candidate HIV vaccines.

## CONCLUSIONS

Enormous progress has been made in defining the molecular circuitry underlying the innate immune response through classical approaches. The recent work described above demonstrates how this process is greatly accelerated when systems-level analyses are applied, particularly for discovering new regulatory networks, understanding infectious disease pathogenesis, and identifying the principles that underlie successful vaccination. As the characterization of innate immune networks progresses, we anticipate that the field of innate immunity will gradually transition from a discovery science to an applied one in which our knowledge of molecular networks is harnessed to develop host-based therapeutics that curb infection while preventing inflammatory disease. This will enable the rational design of vaccines that will overcome the most challenging pathogens.

## DISCLOSURE STATEMENT

The authors are not aware of any affiliations, memberships, funding, or financial holdings that might be perceived as affecting the objectivity of this review.

## ACKNOWLEDGMENTS

We thank Kathleen Kennedy and Alan Diercks for helpful comments. This work was supported by grants and contracts from the National Institutes of Health (R01AI025032, R01AI032972, HHSN272200700038C,HHSN272200800058C,andU19AI100627).TheauthorsalsoacknowledgesupportfromFredHutchinsonCancerResearchCenteraspartoftheCollaborationforAIDS Vaccine Discovery (M.J. McElrath, Principal Investigator) with support from the Bill &amp; Melinda Gates Foundation.

## LITERATURE CITED

- 1. Ramsey SA, Klemm SL, Zak DE, Kennedy KA, Thorsson V, et al. 2008. Uncovering a macrophage transcriptional program by integrating evidence from motif scanning and expression dynamics. PLoS Comput. Biol. 4:e1000021
- 2. Germain RN, Meier-Schellersheim M, Nita-Lazar A, Fraser ID. 2011. Systems biology in immunology: a computational modeling perspective. Annu. Rev. Immunol. 29:527-85
- 3. Poultney CS, Greenfield A, Bonneau R. 2012. Integrated inference and analysis of regulatory networks from multi-level measurements. Methods Cell Biol. 110:19-56

·

·

- 4. Ghosh S, Matsuoka Y, Asai Y, Hsin KY, Kitano H. 2011. Software for systems biology: from tools to integrated platforms. Nat. Rev. Genet. 12:821-32
- 5. De Smet R, Marchal K. 2010. Advantages and limitations of current network inference methods. Nat. Rev. Microbiol. 8:717-29
- 6. Ciofani M, Madar A, Galan C, Sellars M, Mace K, et al. 2012. A validated regulatory network for Th17 cell specification. Cell 151:289-303
- 7. Newell EW, Sigal N, Bendall SC, Nolan GP, Davis MM. 2012. Cytometry by time-of-flight shows combinatorial cytokine expression and virus-specific cell niches within a continuum of CD8 + T cell phenotypes. Immunity 36:142-52
- 8. Zhang JA, Mortazavi A, Williams BA, Wold BJ, Rothenberg EV. 2012. Dynamic transformations of genome-wide epigenetic marking and transcriptional control establish T cell identity. Cell 149:467-82
- 9. O'Neill LA, Bowie AG. 2007. The family of five: TIR-domain-containing adaptors in Toll-like receptor signalling. Nat. Rev. Immunol. 7:353-64
- 10. Ishii KJ, Koyama S, Nakagawa A, Coban C, Akira S. 2008. Host innate immune receptors and beyond: making sense of microbial infections. Cell Host Microbe 3:352-63
- 11. Paludan SR, Bowie AG. 2013. Immune sensing of DNA. Immunity 38:870-80
- 12. Underhill DM, Ozinsky A. 2002. Phagocytosis of microbes: complexity in action. Annu. Rev. Immunol. 20:825-52
- 13. Trinchieri G, Sher A. 2007. Cooperation of Toll-like receptor signals in innate immune defence. Nat. Rev. Immunol. 7:179-90
- 14. GoodridgeHS,Underhill DM.2008.Fungal recognition by TLR2 and Dectin-1. Handb. Exp. Pharmacol. 183:87-109
- 15. Aderem AA, Cohen DS, Wright SD, Cohn ZA. 1986. Bacterial lipopolysaccharides prime macrophages for enhanced release of arachidonic acid metabolites. J. Exp. Med. 164:165-79
- 16. Blander JM, Medzhitov R. 2006. Toll-dependent selection of microbial antigens for presentation by dendritic cells. Nature 440:808-12
- 17. Mohr S, Bakal C, Perrimon N. 2010. Genomic screening with RNAi: results and challenges. Annu. Rev. Biochem. 79:37-64
- 18. Beutler B, Moresco EM. 2008. The forward genetic dissection of afferent innate immunity. Curr. Top. Microbiol. Immunol. 321:3-26
- 19. Cook MC, Vinuesa CG, Goodnow CC. 2006. ENU-mutagenesis: insight into immune function and pathology. Curr. Opin. Immunol. 18:627-33
- 20. Bolouri H, Davidson EH. 2003. Transcriptional regulatory cascades in development: initial rates, not steady state, determine network kinetics. Proc. Natl. Acad. Sci. USA 100:9371-76
- 21. Smith J, Theodoris C, Davidson EH. 2007. A gene regulatory network subcircuit drives a dynamic pattern of gene expression. Science 318:794-97
- 22. Gilchrist M, Thorsson V, Li B, Rust AG, Korb M, et al. 2006. Systems biology approaches identify ATF3 as a negative regulator of Toll-like receptor 4. Nature 441:173-78
- 23. Litvak V, Ramsey SA, Rust AG, Zak DE, Kennedy KA, et al. 2009. Function of C/EBP δ in a regulatory circuit that discriminates between transient and persistent TLR4-induced signals. Nat. Immunol. 10:43743
- 24. Alon U. 2007. Network motifs: theory and experimental approaches. Nat. Rev. Genet. 8:450-61
- 25. Litvak V, Ratushny AV, Lampano AE, Schmitz F, Huang AC, et al. 2012. A FOXO3-IRF7 gene regulatory circuit limits inflammatory sequelae of antiviral responses. Nature 490:421-25
- 26. Seymour RE, Hasham MG, Cox GA, Shultz LD, Hogenesch H, et al. 2007. Spontaneous mutations in the mouse Sharpin gene result in multiorgan inflammation, immune system dysregulation and dermatitis. Genes Immun. 8:416-21
- 27. Siggs OM, Berger M, Krebs P, Arnold CN, Eidenschenk C, et al. 2010. A mutation of Ikbkg causes immune deficiency without impairing degradation of I κ α B . Proc. Natl. Acad. Sci. USA 107:3046-51
- 28. Zak DE, Schmitz F, Gold ES, Diercks AH, Peschon JJ, et al. 2011. Systems analysis identifies an essential role for SHANK-associated RH domain-interacting protein (SHARPIN) in macrophage Toll-like receptor 2 (TLR2) responses. Proc. Natl. Acad. Sci. USA 108:11536-41

- 29. AmitI, Garber M, Chevrier N, Leite AP, Donner Y, et al. 2009. Unbiased reconstruction of a mammalian transcriptional network mediating pathogen responses. Science 326:257-63
- 30. Kulkarni MM. 2011. Digital multiplexed gene expression analysis using the NanoString nCounter system. Curr. Protoc. Mol. Biol. 94:25B.10
- 31. Garber M, Yosef N, Goren A, Raychowdhury R, Thielke A, et al. 2012. A high-throughput chromatin immunoprecipitation approach reveals principles of dynamic gene regulation in mammals. Mol. Cell 47:810-22
- 32. Stender JD, Glass CK. 2013. Epigenomic control of the innate immune response. Curr. Opin. Pharmacol. 13:582-87
- 33. Kaikkonen MU,SpannNJ,HeinzS,RomanoskiCE,AllisonKA,etal.2013.Remodelingoftheenhancer landscape during macrophage activation is coupled to enhancer transcription. Mol. Cell 51:310-25
- 34. Chevrier N, Mertins P, Artyomov MN, Shalek AK, Iannacone M, et al. 2011. Systematic discovery of TLR signaling components delineates viral-sensing circuits. Cell 147:853-67
- 35. de Veer MJ, Holko M, Frevel M, Walker E, Der S, et al. 2001. Functional classification of interferonstimulated genes identified using microarrays. J. Leukoc. Biol. 69:912-20
- 36. Schoggins JW, Wilson SJ, Panis M, Murphy MY, Jones CT, et al. 2011. A diverse range of gene products are effectors of the type I interferon antiviral response. Nature 472:481-85
- 37. Versteeg GA, Rajsbaum R, S' anchez-Aparicio MT, Maestre AM, Valdiviezo J, et al. 2013. The E3-ligase TRIM family of proteins regulates signaling pathways triggered by innate immune pattern-recognition receptors. Immunity 38:384-98
- 38. McNab FW, Rajsbaum R, Stoye JP, O'Garra A. 2011. Tripartite-motif proteins and innate immune regulation. Curr. Opin. Immunol. 23:46-56
- 39. Ozato K, Shin DM, Chang TH, Morse HC III. 2008. TRIM family proteins and their emerging roles in innate immunity. Nat. Rev. Immunol. 8:849-60
- 40. Li S, Wang L, Berman M, Kong YY, Dorf ME. 2011. Mapping a dynamic innate immunity protein interaction network regulating type I interferon production. Immunity 35:426-40
- 41. Lee MN, Roy M, Ong SE, Mertins P, Villani AC, et al. 2013. Identification of regulators of the innate immune response to cytosolic DNA and retroviral infection by an integrative approach. Nat. Immunol. 14:179-85
- 42. Elkon R, Linhart C, Halperin Y, Shiloh Y, Shamir R. 2007. Functional genomic delineation of TLRinduced transcriptional networks. BMC Genomics 8:394
- 43. Ramsey SA, Knijnenburg TA, Kennedy KA, Zak DE, Gilchrist M, et al. 2010. Genome-wide histone acetylation data improve prediction of mammalian transcription factor binding sites. Bioinformatics 26:2071-75
- 44. Li F, Thiele I, Jamshidi N, Palsson BO. 2009. Identification of potential pathway mediation targets in Toll-like receptor signaling. PLoS Comput. Biol. 5:e1000292
- 45. Bordbar A, Mo ML, Nakayasu ES, Schrimpe-Rutledge AC, Kim YM, et al. 2012. Model-driven multiomic data analysis elucidates metabolic immunomodulators of macrophage activation. Mol. Syst. Biol. 8:558
- 46. Bordbar A, Lewis NE, Schellenberger J, Palsson BO, Jamshidi N. 2010. Insight into human alveolar macrophage and M. tuberculosis interactions via metabolic reconstructions. Mol. Syst. Biol. 6:422
- 47. Kalisky T, Blainey P, Quake SR. 2011. Genomic analysis at the single-cell level. Annu. Rev. Genet. 45:431-45
- 48. Kalisky T, Quake SR. 2011. Single-cell genomics. Nat. Methods 8:311-14
- 49. Bandura DR, Baranov VI, Ornatsky OI, Antonov A, Kinach R, et al. 2009. Mass cytometry: technique for real time single cell multitarget immunoassay based on inductively coupled plasma time-of-flight mass spectrometry. Anal. Chem. 81:6813-22
- 50. Bendall SC, Nolan GP. 2012. From single cells to deep phenotypes in cancer. Nat. Biotechnol. 30:639-47
- 51. Diercks AH, Ozinsky A, Hansen CL, Spotts JM, Rodriguez DJ, Aderem A. 2009. A microfluidic device for multiplexed protein detection in nano-liter volumes. Anal. Biochem. 386:30-35
- 52. Gottschalk RA, Martins AJ, Sjoelund VH, Angermann BR, Lin B, Germain RN. 2012. Recent progress using systems biology approaches to better understand molecular mechanisms of immunity. Semin. Immunol. 25:201-8

·

·

- 53. Shalek AK, Satija R, Adiconis X, Gertner RS, Gaublomme JT, et al. 2013. Single-cell transcriptomics reveals bimodality in expression and splicing in immune cells. Nature 498:236-40
- 54. Ma C, Fan R, Ahmad H, Shi Q, Comin-Anduix B, et al. 2011. A clinical microchip for evaluation of single immune cells reveals high functional heterogeneity in phenotypically similar T cells. Nat. Med. 17:738-43
- 55. Varadarajan N, Julg B, Yamanaka YJ, Chen H, Ogunniyi AO, et al. 2011. A high-throughput singlecell analysis of human CD8 + T cell functions reveals discordance for cytokine secretion and cytolysis. J. Clin. Investig. 121:4322-31
- 56. Bodenmiller B, Zunder ER, Finck R, Chen TJ, Savig ES, et al. 2012. Multiplexed mass cytometry profiling of cellular states perturbed by small-molecule regulators. Nat. Biotechnol. 30:858-67
- 57. Berry MP, Graham CM, McNab FW, Xu Z, Bloch SA, et al. 2010. An interferon-inducible neutrophildriven blood transcriptional signature in human tuberculosis. Nature 466:973-77
- 58. Weiner J, Maertzdorf J, Kaufmann SH. 2013. The dual role of biomarkers for understanding basic principles and devising novel intervention strategies in tuberculosis. Ann. N.Y. Acad. Sci. 1283:22-29
- 59. Palermo RE, Fuller DH. 2013. 'Omics investigations of HIV and SIV pathogenesis and innate immunity. Curr. Top. Microbiol. Immunol. 363:87-116
- 60. Bosinger SE, Jacquelin B, Benecke A, Silvestri G, Muller-Trutwin M. 2012. Systems biology of natural simian immunodeficiency virus infections. Curr. Opin. HIV AIDS 7:71-78
- 61. Peretz Y, Cameron C, Sekaly RP. 2012. Dissecting the HIV-specific immune response: a systems biology approach. Curr. Opin. HIV AIDS 7:17-23
- 62. Walters KA, Olsufka R, Kuestner RE, Cho JH, Li H, et al. 2013. Francisella tularensis subsp. tularensis induces a unique pulmonary inflammatory response: role of bacterial gene expression in temporal regulation of host defense responses. PLoS ONE 8:e62412
- 63. Thompson LJ, Dunstan SJ, Dolecek C, Perkins T, House D, et al. 2009. Transcriptional response in the peripheral blood of patients infected with Salmonella enterica serovar Typhi. Proc. Natl. Acad. Sci. USA 106:22433-38
- 64. Ravi LI, Li L, Sutejo R, Chen H, Wong PS, et al. 2013. A systems-based approach to analyse the host response in murine lung macrophages challenged with respiratory syncytial virus. BMCGenomics 14:190
- 65. Jagger BW, Wise HM, Kash JC, Walters KA, Wills NM, et al. 2012. An overlapping protein-coding region in influenza A virus segment 3 modulates the host response. Science 337:199-204
- 66. Kash JC, Walters KA, Davis AS, Sandouk A, Schwartzman LM, et al. 2011. Lethal synergism of 2009 pandemic H1N1 influenza virus and Streptococcus pneumoniae coinfection is associated with loss of murine lung repair responses. mBio 2:e00172-11
- 67. Tisoncik JR, Korth MJ, Simmons CP, Farrar J, Martin TR, Katze MG. 2012. Into the eye of the cytokine storm. Microbiol. Mol. Biol. Rev. 76:16-32
- 68. Imai M, Watanabe T, Hatta M, Das SC, Ozawa M, et al. 2012. Experimental adaptation of an influenza H5 haemagglutinin (HA) confers respiratory droplet transmission to a reassortant H5 HA/H1N1 virus in ferrets. Nature 486:420-28
- 69. Herfst S, Schrauwen EJA, Linster M, Chutinimitkul S, de Wit E, et al. 2012. Airborne transmission of influenza A/H5N1 virus between ferrets. Science 336:1534-41
- 70. Kash JC, Tumpey TM, Proll SC, Carter V, Perwitasari O, et al. 2006. Genomic analysis of increased host immune and cell death responses induced by 1918 influenza virus. Nature 443:578-81
- 71. Cill'niz C, Shinya K, Peng X, Korth MJ, Proll SC, et al. 2009. Lethal influenza virus infection in o macaques is associated with early dysregulation of inflammatory related genes. PLoS Pathog. 5:e1000604
- 72. Askovich PS, Sanders CJ, Rosenberger CM, Diercks AH, Dash P, et al. 2013. Differential host response, rather than early viral replication efficiency, correlates with pathogenicity caused by influenza viruses. PLoS ONE 8:e74863
- 73. Brandes M, Klauschen F, Kuchen S, Germain RN. 2013. A systems analysis identifies a feedforward inflammatory circuit leading to lethal influenza infection. Cell 154:197-212
- 74. Li Y, Chan EY, Li J, Ni C, Peng X, et al. 2010. MicroRNA expression and virulence in pandemic influenza virus-infected mice. J. Virol. 84:3023-32

- 75. Peng X, Gralinski L, Ferris MT, Frieman MB, Thomas MJ, et al. 2011. Integrative deep sequencing of the mouse lung transcriptome reveals differential expression of diverse classes of small RNAs in response to respiratory virus infection. mBio 2:e00198-11
- 76. Rosenberger CM, Podyminogin RL, Navarro G, Zhao GW, Askovich PS, et al. 2012. miR-451 regulates dendritic cell cytokine responses to influenza infection. J. Immunol. 189:5965-75
- 77. Baas T, Baskin CR, Diamond DL, Garc' ıa-Sastre A, Bielefeldt-Ohmann H, et al. 2006. Integrated molecular signature of disease: analysis of influenza virus-infected macaques through functional genomics and proteomics. J. Virol. 80:10813-28
- 78. Brown JN, Palermo RE, Baskin CR, Gritsenko M, Sabourin PJ, et al. 2010. Macaque proteome response to highly pathogenic avian influenza and 1918 reassortant influenza virus infections. J. Virol. 84:12058-68
- 79. Kroeker AL, Ezzati P, Halayko AJ, Coombs KM. 2012. Response of primary human airway epithelial cells to influenza infection: a quantitative proteomic study. J. Proteome Res. 11:4132-46
- 80. Lietz' en N, Ohman T, Rintahaka J, Julkunen I, Aittokallio T, et al. 2011. Quantitative subcellular pro-teome and secretome profiling of influenza A virus-infected human primary macrophages. PLoS Pathog. 7:e1001340
- 81. Cheung CY, Chan EY, Krasnoselsky A, Purdy D, Navare AT, et al. 2012. H5N1 virus causes significant perturbations in host proteome very early in influenza virus-infected primary human monocyte-derived macrophages. J. Infect. Dis. 206:640-45
- 82. Shapira SD, Gat-Viks I, Shum BOV, Dricot A, de Grace MM, et al. 2009. A physical and regulatory map of host-influenza interactions reveals pathways in H1N1 infection. Cell 139:1255-67
- 83. Tam VC, Quehenberger O, Oshansky CM, Suen R, Armando AM, et al. 2013. Lipidomic profiling of influenza infection identifies mediators that induce and resolve inflammation. Cell 154:213-27
- 84. Morita M, Kuba K, Ichikawa A, Nakayama M, Katahira J, et al. 2013. The lipid mediator protectin D1 inhibits influenza virus replication and improves severe influenza. Cell 153:112-25
- 85. Lawrence T, Willoughby DA, Gilroy DW. 2002. Anti-inflammatory lipid mediators and insights into the resolution of inflammation. Nat. Rev. Immunol. 2:787-95
- 86. Quehenberger O, Dennis EA. 2011. The human plasma lipidome. N. Engl. J. Med. 365:1812-23
- 87. Serhan CN, Krishnamoorthy S, Recchiuti A, Chiang N. 2011. Novel anti-inflammatory-pro-resolving mediators and their receptors. Curr. Top. Med. Chem. 11:629-47
- 88. Serhan CN, Chiang N, Van Dyke TE. 2008. Resolving inflammation: dual anti-inflammatory and proresolution lipid mediators. Nat. Rev. Immunol. 8:349-61
- 89. Buczynski M, Dumlao D, Dennis E. 2009. An integrated omics analysis of eicosanoid biology. J. Lipid Res. 6:1015-38
- 90. Norris PC, Dennis EA. 2012. Omega-3 fatty acids cause dramatic changes in TLR4 and purinergic eicosanoid signaling. Proc. Natl. Acad. Sci. USA 109:8517-22
- 91. Yang R, Chiang N, Oh SF, Serhan CN. 2001. Metabolomics-Lipidomics of Eicosanoids and Docosanoids Generated by Phagocytes . Hoboken, NJ: Wiley
- 92. Karlas A, Machuy N, Shin Y, Pleissner K-P, Artarini A, et al. 2010. Genome-wide RNAi screen identifies human host factors crucial for influenza virus replication. Nature 463:818-22
- 93. Munier S, Rolland T, Diot C, Jacob Y, Naffakh N. 2013. Exploration of binary virus-host interactions using an infectious protein complementation assay. Mol. Cell. Proteomics 12:2845-55
- 94. Zak DE, Aderem A. 2012. Overcoming limitations in the systems vaccinology approach: a pathway for accelerated HIV vaccine development. Curr. Opin. HIV AIDS 7:58-63
- 95. Pulendran B, Li S, Nakaya HI. 2010. Systems vaccinology. Immunity 33(4):516-29
- 96. Andersen-Nissen E, Heit A, McElrath MJ. 2012. Profiling immunity to HIV vaccines with systems biology. Curr. Opin. HIV AIDS 7:32-37
- 97. Rappuoli R, Aderem A. 2011. A 2020 vision for vaccines against HIV, tuberculosis and malaria. Nature 473:463-69
- 98. Koff WC, Burton DR, Johnson PR, Walker BD, King CR, et al. 2013. Accelerating next-generation vaccine development for global disease prevention. Science 340(6136):1232910
- 99. Mooney M, McWeeney S, Sekaly RP. 2013. Systems immunogenetics of vaccines. Semin. Immunol. 25(2):124-29

- 100. Plotkin SA. 2008. Vaccines: correlates of vaccine-induced immunity. Clin. Infect. Dis. 47(3): 401-9
- 101. Querec T, Bennouna S, Alkan S, Laouar Y, Gorden K, et al. 2006. Yellow fever vaccine YF-17D activates multiple dendritic cell subsets via TLR2, 7, 8, and 9 to stimulate polyvalent immunity. J. Exp. Med. 203:413-24
- 102. Lindsay RW, Darrah PA, Quinn KM, Wille-Reece U, Mattei LM, et al. 2010. CD8 + T cell responses following replication-defective adenovirus serotype 5 immunization are dependent on CD11c + dendritic cells but show redundancy in their requirement of TLR and nucleotide-binding oligomerization domainlike receptor signaling. J. Immunol. 185:1513-21
- 103. Delaloye J, Roger T, Steiner-Tardivel QG, Le Roy D, Knaup Reymond M, et al. 2009. Innate immune sensing of modified vaccinia virus Ankara (MVA) is mediated by TLR2-TLR6, MDA-5 and the NALP3 inflammasome. PLoS Pathog. 5:e1000480
- 104. Baum LL. 2010. Role of humoral immunity in host defense against HIV. Curr. HIV/AIDS Rep. 7:11-18
- 105. Seder RA, Darrah PA, Roederer M. 2008. T-cell quality in memory and protection: implications for vaccine design. Nat. Rev. Immunol. 8:247-58
- 106. Flatz L, Roychoudhuri R, Honda M, Filali-Mouhim A, Goulet JP, et al. 2011. Single-cell gene-expression profiling reveals qualitatively distinct CD8 T cells elicited by different gene-based vaccines. Proc. Natl. Acad. Sci. USA 108:5724-29
- 107. Baum PD, Venturi V, Price DA. 2012. Wrestling with the repertoire: the promise and perils of next generation sequencing for antigen receptors. Eur. J. Immunol. 42:2834-39
- 108. Autran B, Descours B, Avettand-Fenoel V, Rouzioux C. 2011. Elite controllers as a model of functional cure. Curr. Opin. HIV AIDS 6:181-87
- 109. Walker BD. 2007. Elite control of HIV infection: implications for vaccines and treatment. Top. HIV Med. 15:134-36
- 110. Thomas PG, Doherty PC. 2009. Rules to 'prime' by. Nat. Immunol. 10:14-16
- 111. Brooks JPL, Lee EK. 2008. Analysis of the consistency of a mixed integer programming-based multicategory constrained discriminant model. Ann. Oper. Res. 174:147-68
- 112. Querec TD, Akondy RS, Lee EK, Cao W, Nakaya HI, et al. 2009. Systems biology approach predicts immunogenicity of the yellow fever vaccine in humans. Nat. Immunol. 10:116-25
- 113. NakayaHI,WrammertJ,LeeEK,RacioppiL,Marie-KunzeS,etal.2011.Systemsbiologyofvaccination for seasonal influenza in humans. Nat. Immunol. 12:786-95
- 114. Zou H, Hastie T. 2005. Regularization and variable selection via the elastic net. J. R. Stat. Soc. B 67:30120
- 115. Seok J, Warren HS, Cuenca AG, Mindrinos MN, Baker HV, et al. 2013. Genomic responses in mouse models poorly mimic human inflammatory diseases. Proc. Natl. Acad. Sci. USA 110(9):3507-12
- 116. Shultz LD, Brehm MA, Garcia-Martinez JV, Greiner DL. 2012. Humanized mice for immune system investigation: progress, promise and challenges. Nat. Rev. Immunol. 12:786-98
- 117. Chaussabel D, Quinn C, Shen J, Patel P, Glaser C, et al. 2008. A modular analysis framework for blood genomics studies: application to systemic lupus erythematosus. Immunity 29:150-64
- 118. Shen-Orr SS, Tibshirani R, Khatri P, Bodian DL, Staedtler F, et al. 2010. Cell type-specific gene expression differences in complex tissues. Nat. Methods 7:287-89
- 119. Novershtern N, Subramanian A, Lawton LN, Mak RH, Haining WN, et al. 2011. Densely interconnected transcriptional circuits control cell states in human hematopoiesis. Cell 144(2):296-309
- 120. Shay T, Kang J. 2013. Immunological Genome Project and systems immunology. Trends Immunol. 34(12):602-9
- 121. Gaucher D, Therrien R, Kettaf N, Angermann BR, Boucher G, et al. 2008. Yellow fever vaccine induces integrated multilineage and polyfunctional immune responses. J. Exp. Med. 205:3119-31
- 122. Zak DE, Andersen-Nissen E, Peterson ER, Sato A, Hamilton MK, et al. 2012. Merck Ad5/HIV induces broad innate immune activation that predicts CD8 + T-cell responses but is attenuated by preexisting Ad5 immunity. Proc. Natl. Acad. Sci. USA 109:E3503-12
- 123. Buchbinder SP, Mehrotra DV, Duerr A, Fitzgerald DW, Mogg R, et al. 2008. Efficacy assessment of a cell-mediated immunity HIV-1 vaccine (the Step Study): a double-blind, randomised, placebocontrolled, test-of-concept trial. Lancet 372:1881-93

- 124. Gray GE, Allen M, Moodie Z, Churchyard G, Bekker LG, et al. 2011. Safety and efficacy of the HVTN 503/Phambili study of a clade-B-based HIV-1 vaccine in South Africa: a double-blind, randomised, placebo-controlled test-of-concept phase 2b study. Lancet Infect. Dis. 11:507-15
- 125. McElrath MJ, De Rosa SC, Moodie Z, Dubey S, Kierstead L, et al. 2008. HIV-1 vaccine-induced immunity in the test-of-concept Step Study: a case-cohort analysis. Lancet 372:1894-905
- 126. Rolland M, Tovanabutra S, deCamp AC, Frahm N, Gilbert PB, et al. 2011. Genetic impact of vaccination on breakthrough HIV-1 sequences from the STEP trial. Nat. Med. 17:366-71
- 127. Vahey MT, Wang Z, Kester KE, Cummings J, Heppner DG Jr, et al. 2010. Expression of genes associated with immunoproteasome processing of major histocompatibility complex peptides is indicative of protection with adjuvanted RTS,S malaria vaccine. J. Infect. Dis. 201:580-89
- 128. Li S, Rouphael N, Duraisingham S, Romero-Steiner S, Presnell S, et al. 2014. Molecular signatures of antibody responses derived from a systems biology study of five human vaccines. Nat. Immunol. 15:195204
- 129. Obermoser G, Presnell S, Domico K, Xu H, Wang Y, et al. 2013. Systems scale interactive exploration reveals quantitative and qualitative differences in response to influenza and pneumococcal vaccines. Immunity 38(4):831-44
- 130. Reif DM, Motsinger-Reif AA, McKinney BA, Rock MT, Crowe JE Jr, Moore JH. 2009. Integrated analysis of genetic and proteomic data identifies biomarkers associated with adverse events following smallpox vaccination. Genes Immun. 10(2):112-19
- 131. Araki K, Turner AP, Shaffer VO, Gangappa S, Keller SA, et al. 2009. mTOR regulates memory CD8 T-cell differentiation. Nature 460:108-12
- 132. Tan X, Sande JL, Pufnock JS, Blattman JN, Greenberg PD. 2011. Retinoic acid as a vaccine adjuvant enhances CD8 + T cell response and mucosal protection from viral challenge. J. Virol. 85:8316-27
- 133. Wang N, Calpe S, Westcott J, Castro W, Ma C, et al. 2010. Cutting edge: The adapters EAT-2A and -2B are positive regulators of CD244- and CD84-dependent NK cell functions in the C57BL/6 mouse. J. Immunol. 185:5683-87
- 134. Aldhamen YA, Appledorn DM, Seregin SS, Liu CJ, Schuldt NJ, et al. 2011. Expression of the SLAM family of receptors adapter EAT-2 as a novel strategy for enhancing beneficial immune responses to vaccine antigens. J. Immunol. 186:722-32
- 135. Olotu A, Fegan G, Wambua J, Nyangweso G, Awuondo KO, et al. 2013. Four-year efficacy of RTS,S/AS01E and its interaction with malaria exposure. N. Engl. J. Med. 368(12): 1111-20
- 136. Seder RA, Chang LJ, Enama ME, Zephir KL, Sarwar UN, et al. 2013. Protection against malaria by intravenous immunization with a nonreplicating sporozoite vaccine. Science 341:1359-65
- 137. Tameris MD, Hatherill M, Landry BS, Scriba TJ, Snowden MA, et al. 2013. Safety and efficacy of MVA85A,anewtuberculosis vaccine, in infants previously vaccinated with BCG: a randomised, placebocontrolled phase 2b trial. Lancet 381(9871):1021-28
- 138. Rerks-Ngarm S, Pitisuttithum P, Nitayaphan S, Kaewkungwal J, Chiu J, et al. 2009. Vaccination with ALVAC and AIDSVAX to prevent HIV-1 infection in Thailand. N. Engl. J. Med. 361:2209-20
- 139. Zak DE, Aderem A. 2009. Systems biology of innate immunity. Immunol. Rev. 227:264-82
- 140. Ramsey SA, Gold ES, Aderem A. 2010. A systems biology approach to understanding atherosclerosis. EMBO Mol. Med. 2:79-89
- 141. Kozhenkov S, Dubinina Y, Sedova M, Gupta A, Ponomarenko J, Baitaluk M. 2010. BiologicalNetworks 2.0-an integrative view of genome biology data. BMC Bioinforma. 11:610
- 142. Breitkreutz BJ, Stark C, Tyers M. 2003. Osprey: a network visualization system. Genome Biol. 4:R22
- 143. Breuer K, Foroushani AK, Laird MR, Chen C, Sribnaia A, et al. 2013. InnateDB: systems biology of innate immunity and beyond-recent updates and continuing curation. Nucleic Acids Res. 41:D1228-33
- 144. Barsky A, Gardy JL, Hancock RE, Munzner T. 2007. Cerebral: a Cytoscape plugin for layout of and interaction with biological networks using subcellular localization annotation. Bioinformatics 23:1040-42
- 145. Smoot ME, Ono K, Ruscheinski J, Wang PL, Ideker T. 2011. Cytoscape 2.8: new features for data integration and network visualization. Bioinformatics 27:431-32
- 146. Hu Z, Chang YC, Wang Y, Huang CL, Liu Y, et al. 2013. VisANT 4.0: Integrative network platform to connect genes, drugs, diseases and therapies. Nucleic Acids Res. 41:W225-31

- 147. Croft D, O'Kelly G, Wu G, Haw R, Gillespie M, et al. 2011. Reactome: a database of reactions, pathways and biological processes. Nucleic Acids Res. 39:D691-97
- 148. Nishimura D. 2001. BioCarta. Biotech Softw. Internet Rep. 2:117-20
- 149. Latendresse M, Paley S, Karp PD. 2012. Browsing metabolic and regulatory networks with BioCyc. Methods Mol. Biol. 804:197-216
- 150. Kanehisa M. 2013. Molecular network analysis of diseases and drugs in KEGG. Methods Mol. Biol. 939:263-75
- 151. MiH,MuruganujanA,ThomasPD.2013.PANTHERin2013:modelingtheevolutionofgenefunction, and other gene attributes, in the context of phylogenetic trees. Nucleic Acids Res. 41:D377-86
- 152. Kelder T, van Iersel MP, Hanspers K, Kutmon M, Conklin BR, et al. 2012. WikiPathways: building research communities on biological pathways. Nucleic Acids Res. 40:D1301-7
- 153. Fahy E, Sud M, Cotter D, Subramaniam S. 2007. LIPID MAPS online tools for lipid research. Nucleic Acids Res. 35:W606-12
- 154. CeramiEG,GrossBE,DemirE,RodchenkovI,BaburO,etal.2011.PathwayCommons,awebresource for biological pathway data. Nucleic Acids Res. 39:D685-90
- 155. KerrienS,ArandaB,BreuzaL,BridgeA,Broackes-CarterF,etal.2012.TheIntActmolecularinteraction database in 2012. Nucleic Acids Res. 40:D841-46
- 156. Keshava Prasad TS, Goel R, Kandasamy K, Keerthikumar S, Kumar S, et al. 2009. Human Protein Reference Database-2009 update. Nucleic Acids Res. 37:D767-72
- 157. Licata L, Briganti L, Peluso D, Perfetto L, Iannuccelli M, et al. 2012. MINT, the molecular interaction database: 2012 update. Nucleic Acids Res. 40:D857-61
- 158. Chatr-Aryamontri A, Breitkreutz BJ, Heinicke S, Boucher L, Winter A, et al. 2013. The BioGRID interaction database: 2013 update. Nucleic Acids Res. 41:D816-23
- 159. Barrett T, Edgar R. 2006. Gene expression omnibus: microarray data storage, submission, retrieval, and analysis. Methods Enzymol. 411:352-69
- 160. Rustici G, Kolesnikov N, Brandizi M, Burdett T, Dylag M, et al. 2013. ArrayExpress update-trends in database growth and links to data analysis tools. Nucleic Acids Res. 41:D987-90